{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import polars as pl\n",
    "from IPython.display import display\n",
    "import json\n",
    "import ast"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "encode_df = pl.read_parquet('../encode_files.parquet')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF819LHK&quot;</td></tr><tr><td>&quot;ENCFF978MSE&quot;</td></tr><tr><td>&quot;ENCFF111MFR&quot;</td></tr><tr><td>&quot;ENCFF822KRV&quot;</td></tr><tr><td>&quot;ENCFF039ZBL&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;ENCFF514KLY&quot;</td></tr><tr><td>&quot;ENCFF332FTS&quot;</td></tr><tr><td>&quot;ENCFF594BFE&quot;</td></tr><tr><td>&quot;ENCFF112KGH&quot;</td></tr><tr><td>&quot;ENCFF673BQM&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008,)\n",
       "Series: 'title' [str]\n",
       "[\n",
       "\t\"ENCFF819LHK\"\n",
       "\t\"ENCFF978MSE\"\n",
       "\t\"ENCFF111MFR\"\n",
       "\t\"ENCFF822KRV\"\n",
       "\t\"ENCFF039ZBL\"\n",
       "\t…\n",
       "\t\"ENCFF514KLY\"\n",
       "\t\"ENCFF332FTS\"\n",
       "\t\"ENCFF594BFE\"\n",
       "\t\"ENCFF112KGH\"\n",
       "\t\"ENCFF673BQM\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (760_797,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>accession</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF606MHR&quot;</td></tr><tr><td>&quot;ENCFF545OLF&quot;</td></tr><tr><td>&quot;ENCFF020IMS&quot;</td></tr><tr><td>&quot;ENCFF822EAJ&quot;</td></tr><tr><td>&quot;ENCFF323ZIV&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;ENCFF615GWD&quot;</td></tr><tr><td>&quot;ENCFF633CMO&quot;</td></tr><tr><td>&quot;ENCFF997MRC&quot;</td></tr><tr><td>&quot;ENCFF635BOE&quot;</td></tr><tr><td>&quot;ENCFF157LRK&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (760_797,)\n",
       "Series: 'accession' [str]\n",
       "[\n",
       "\t\"ENCFF606MHR\"\n",
       "\t\"ENCFF545OLF\"\n",
       "\t\"ENCFF020IMS\"\n",
       "\t\"ENCFF822EAJ\"\n",
       "\t\"ENCFF323ZIV\"\n",
       "\t…\n",
       "\t\"ENCFF615GWD\"\n",
       "\t\"ENCFF633CMO\"\n",
       "\t\"ENCFF997MRC\"\n",
       "\t\"ENCFF635BOE\"\n",
       "\t\"ENCFF157LRK\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (24_338,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>dataset</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;/experiments/ENCSR271VMW/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR411AFQ/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR847RRR/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR807KQN/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR812VCN/&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;/experiments/ENCSR392YGP/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR475ZYS/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR558SEE/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR000EQV/&quot;</td></tr><tr><td>&quot;/experiments/ENCSR200KWM/&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (24_338,)\n",
       "Series: 'dataset' [str]\n",
       "[\n",
       "\t\"/experiments/ENCSR271VMW/\"\n",
       "\t\"/experiments/ENCSR411AFQ/\"\n",
       "\t\"/experiments/ENCSR847RRR/\"\n",
       "\t\"/experiments/ENCSR807KQN/\"\n",
       "\t\"/experiments/ENCSR812VCN/\"\n",
       "\t…\n",
       "\t\"/experiments/ENCSR392YGP/\"\n",
       "\t\"/experiments/ENCSR475ZYS/\"\n",
       "\t\"/experiments/ENCSR558SEE/\"\n",
       "\t\"/experiments/ENCSR000EQV/\"\n",
       "\t\"/experiments/ENCSR200KWM/\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (11,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>assembly</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;ce10&quot;</td></tr><tr><td>&quot;dm6&quot;</td></tr><tr><td>&quot;dm3&quot;</td></tr><tr><td>&quot;mm10-minimal&quot;</td></tr><tr><td>&quot;GRCh38-minimal&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;hg19&quot;</td></tr><tr><td>&quot;mm10&quot;</td></tr><tr><td>&quot;ce11&quot;</td></tr><tr><td>&quot;mm9&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (11,)\n",
       "Series: 'assembly' [str]\n",
       "[\n",
       "\t\"ce10\"\n",
       "\t\"dm6\"\n",
       "\t\"dm3\"\n",
       "\t\"mm10-minimal\"\n",
       "\t\"GRCh38-minimal\"\n",
       "\t…\n",
       "\t\"NA\"\n",
       "\t\"hg19\"\n",
       "\t\"mm10\"\n",
       "\t\"ce11\"\n",
       "\t\"mm9\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (264,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>technical_replicates</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[&#x27;1_99&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;1_2&#x27;, &#x27;2_2&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;1_16&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;2_3&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;1_1&#x27;, &#x27;1_2&#x27;, &#x27;2_1&#x27;, &#x27;2_2&#x27;, &#x27;…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[&#x27;1_1&#x27;, &#x27;1_2&#x27;, &#x27;1_3&#x27;, &#x27;1_4&#x27;, &#x27;…</td></tr><tr><td>&quot;[&#x27;5_1&#x27;, &#x27;5_2&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;2_3&#x27;, &#x27;2_4&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;4_4&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;9_1&#x27;]&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (264,)\n",
       "Series: 'technical_replicates' [str]\n",
       "[\n",
       "\t\"['1_99']\"\n",
       "\t\"['1_2', '2_2']\"\n",
       "\t\"['1_16']\"\n",
       "\t\"['2_3']\"\n",
       "\t\"['1_1', '1_2', '2_1', '2_2', '…\n",
       "\t…\n",
       "\t\"['1_1', '1_2', '1_3', '1_4', '…\n",
       "\t\"['5_1', '5_2']\"\n",
       "\t\"['2_3', '2_4']\"\n",
       "\t\"['4_4']\"\n",
       "\t\"['9_1']\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (43,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>biological_replicates</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[2, 3]&quot;</td></tr><tr><td>&quot;[2, 4]&quot;</td></tr><tr><td>&quot;[3, 4, 5]&quot;</td></tr><tr><td>&quot;[3]&quot;</td></tr><tr><td>&quot;[3, 5]&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[4, 5]&quot;</td></tr><tr><td>&quot;[12]&quot;</td></tr><tr><td>&quot;[7]&quot;</td></tr><tr><td>&quot;[1, 7]&quot;</td></tr><tr><td>&quot;[6, 7]&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (43,)\n",
       "Series: 'biological_replicates' [str]\n",
       "[\n",
       "\t\"[2, 3]\"\n",
       "\t\"[2, 4]\"\n",
       "\t\"[3, 4, 5]\"\n",
       "\t\"[3]\"\n",
       "\t\"[3, 5]\"\n",
       "\t…\n",
       "\t\"[4, 5]\"\n",
       "\t\"[12]\"\n",
       "\t\"[7]\"\n",
       "\t\"[1, 7]\"\n",
       "\t\"[6, 7]\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (27,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>file_format</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;tagAlign&quot;</td></tr><tr><td>&quot;wig&quot;</td></tr><tr><td>&quot;gff&quot;</td></tr><tr><td>&quot;bigInteract&quot;</td></tr><tr><td>&quot;fastq&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;bam&quot;</td></tr><tr><td>&quot;hdf5&quot;</td></tr><tr><td>&quot;sra&quot;</td></tr><tr><td>&quot;bigWig&quot;</td></tr><tr><td>&quot;vcf&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (27,)\n",
       "Series: 'file_format' [str]\n",
       "[\n",
       "\t\"tagAlign\"\n",
       "\t\"wig\"\n",
       "\t\"gff\"\n",
       "\t\"bigInteract\"\n",
       "\t\"fastq\"\n",
       "\t…\n",
       "\t\"bam\"\n",
       "\t\"hdf5\"\n",
       "\t\"sra\"\n",
       "\t\"bigWig\"\n",
       "\t\"vcf\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (61,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>file_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;bed bedLogR&quot;</td></tr><tr><td>&quot;wig&quot;</td></tr><tr><td>&quot;bigBed modPepMap&quot;</td></tr><tr><td>&quot;bed bed6+&quot;</td></tr><tr><td>&quot;sra&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;bigBed broadPeak&quot;</td></tr><tr><td>&quot;bigBed bed3+&quot;</td></tr><tr><td>&quot;CEL&quot;</td></tr><tr><td>&quot;bigBed narrowPeak&quot;</td></tr><tr><td>&quot;pairs&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (61,)\n",
       "Series: 'file_type' [str]\n",
       "[\n",
       "\t\"bed bedLogR\"\n",
       "\t\"wig\"\n",
       "\t\"bigBed modPepMap\"\n",
       "\t\"bed bed6+\"\n",
       "\t\"sra\"\n",
       "\t…\n",
       "\t\"bigBed broadPeak\"\n",
       "\t\"bigBed bed3+\"\n",
       "\t\"CEL\"\n",
       "\t\"bigBed narrowPeak\"\n",
       "\t\"pairs\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (21,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>file_format_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;tss_peak&quot;</td></tr><tr><td>&quot;bedLogR&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;peptideMapping&quot;</td></tr><tr><td>&quot;idr_peak&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;bed3+&quot;</td></tr><tr><td>&quot;bed9&quot;</td></tr><tr><td>&quot;modPepMap&quot;</td></tr><tr><td>&quot;bedMethyl&quot;</td></tr><tr><td>&quot;bedExonScore&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (21,)\n",
       "Series: 'file_format_type' [str]\n",
       "[\n",
       "\t\"tss_peak\"\n",
       "\t\"bedLogR\"\n",
       "\t\"NA\"\n",
       "\t\"peptideMapping\"\n",
       "\t\"idr_peak\"\n",
       "\t…\n",
       "\t\"bed3+\"\n",
       "\t\"bed9\"\n",
       "\t\"modPepMap\"\n",
       "\t\"bedMethyl\"\n",
       "\t\"bedExonScore\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (751_746,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>file_size</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;84220107&quot;</td></tr><tr><td>&quot;9550190&quot;</td></tr><tr><td>&quot;916834617&quot;</td></tr><tr><td>&quot;29583&quot;</td></tr><tr><td>&quot;1939003&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;2427731946&quot;</td></tr><tr><td>&quot;309956014&quot;</td></tr><tr><td>&quot;1102924&quot;</td></tr><tr><td>&quot;162299162&quot;</td></tr><tr><td>&quot;558186&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (751_746,)\n",
       "Series: 'file_size' [str]\n",
       "[\n",
       "\t\"84220107\"\n",
       "\t\"9550190\"\n",
       "\t\"916834617\"\n",
       "\t\"29583\"\n",
       "\t\"1939003\"\n",
       "\t…\n",
       "\t\"2427731946\"\n",
       "\t\"309956014\"\n",
       "\t\"1102924\"\n",
       "\t\"162299162\"\n",
       "\t\"558186\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (59,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>assay_term_name</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;polyA minus RNA-seq&quot;</td></tr><tr><td>&quot;comparative genomic hybridizat…</td></tr><tr><td>&quot;5&#x27; RLM RACE&quot;</td></tr><tr><td>&quot;transcription profiling by arr…</td></tr><tr><td>&quot;RRBS&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;iCLIP&quot;</td></tr><tr><td>&quot;MeDIP-seq&quot;</td></tr><tr><td>&quot;5C&quot;</td></tr><tr><td>&quot;ChIP-seq&quot;</td></tr><tr><td>&quot;CAGE&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (59,)\n",
       "Series: 'assay_term_name' [str]\n",
       "[\n",
       "\t\"polyA minus RNA-seq\"\n",
       "\t\"comparative genomic hybridizat…\n",
       "\t\"5' RLM RACE\"\n",
       "\t\"transcription profiling by arr…\n",
       "\t\"RRBS\"\n",
       "\t…\n",
       "\t\"iCLIP\"\n",
       "\t\"MeDIP-seq\"\n",
       "\t\"5C\"\n",
       "\t\"ChIP-seq\"\n",
       "\t\"CAGE\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (816,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>term_name</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;A172&quot;</td></tr><tr><td>&quot;GM18507&quot;</td></tr><tr><td>&quot;effector memory CD4-positive, …</td></tr><tr><td>&quot;GM19455&quot;</td></tr><tr><td>&quot;R1&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;induced T-regulatory cell&quot;</td></tr><tr><td>&quot;transverse colon&quot;</td></tr><tr><td>&quot;GM18858&quot;</td></tr><tr><td>&quot;bile duct&quot;</td></tr><tr><td>&quot;L1-S8R&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (816,)\n",
       "Series: 'term_name' [str]\n",
       "[\n",
       "\t\"A172\"\n",
       "\t\"GM18507\"\n",
       "\t\"effector memory CD4-positive, …\n",
       "\t\"GM19455\"\n",
       "\t\"R1\"\n",
       "\t…\n",
       "\t\"induced T-regulatory cell\"\n",
       "\t\"transverse colon\"\n",
       "\t\"GM18858\"\n",
       "\t\"bile duct\"\n",
       "\t\"L1-S8R\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (254,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>organ_slims</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[&#x27;vasculature&#x27;, &#x27;kidney&#x27;, &#x27;epi…</td></tr><tr><td>&quot;[&#x27;epithelium&#x27;, &#x27;esophagus&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;liver&#x27;, &#x27;exocrine gland&#x27;, &#x27;e…</td></tr><tr><td>&quot;[&#x27;bone element&#x27;, &#x27;skeleton&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;vasculature&#x27;, &#x27;skin of body&#x27;…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[&#x27;pancreas&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;connective tissue&#x27;, &#x27;penis&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;intestine&#x27;, &#x27;epithelium&#x27;, &#x27;c…</td></tr><tr><td>&quot;[&#x27;placenta&#x27;, &#x27;extraembryonic c…</td></tr><tr><td>&quot;[&#x27;exocrine gland&#x27;, &#x27;mammary gl…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (254,)\n",
       "Series: 'organ_slims' [str]\n",
       "[\n",
       "\t\"['vasculature', 'kidney', 'epi…\n",
       "\t\"['epithelium', 'esophagus']\"\n",
       "\t\"['liver', 'exocrine gland', 'e…\n",
       "\t\"['bone element', 'skeleton']\"\n",
       "\t\"['vasculature', 'skin of body'…\n",
       "\t…\n",
       "\t\"['pancreas']\"\n",
       "\t\"['connective tissue', 'penis']\"\n",
       "\t\"['intestine', 'epithelium', 'c…\n",
       "\t\"['placenta', 'extraembryonic c…\n",
       "\t\"['exocrine gland', 'mammary gl…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (4_674,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>simple_biosample_summary</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;female embryo (103 days)&quot;</td></tr><tr><td>&quot;genetically modified (insertio…</td></tr><tr><td>&quot;genetically modified (interfer…</td></tr><tr><td>&quot;genetically modified (insertio…</td></tr><tr><td>&quot;treated with 10 nM estradiol f…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;embryo (0-24 hours) strain CG2…</td></tr><tr><td>&quot;genetically modified (deletion…</td></tr><tr><td>&quot;strain YL631 (gmeb-2(tm6823)X)…</td></tr><tr><td>&quot;genetically modified (insertio…</td></tr><tr><td>&quot;strain Oregon-R expressing RNA…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (4_674,)\n",
       "Series: 'simple_biosample_summary' [str]\n",
       "[\n",
       "\t\"female embryo (103 days)\"\n",
       "\t\"genetically modified (insertio…\n",
       "\t\"genetically modified (interfer…\n",
       "\t\"genetically modified (insertio…\n",
       "\t\"treated with 10 nM estradiol f…\n",
       "\t…\n",
       "\t\"embryo (0-24 hours) strain CG2…\n",
       "\t\"genetically modified (deletion…\n",
       "\t\"strain YL631 (gmeb-2(tm6823)X)…\n",
       "\t\"genetically modified (insertio…\n",
       "\t\"strain Oregon-R expressing RNA…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (14_270,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>origin_batches</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[&#x27;/biosamples/ENCBS690VKC/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS740SGB/&#x27;, &#x27;…</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS855EIV/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS308TTU/&#x27;, &#x27;…</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS162TUX/&#x27;]&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS435CFB/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS869ANE/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS975PJY/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS652QEI/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/biosamples/ENCBS391ENC/&#x27;]&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (14_270,)\n",
       "Series: 'origin_batches' [str]\n",
       "[\n",
       "\t\"['/biosamples/ENCBS690VKC/']\"\n",
       "\t\"['/biosamples/ENCBS740SGB/', '…\n",
       "\t\"['/biosamples/ENCBS855EIV/']\"\n",
       "\t\"['/biosamples/ENCBS308TTU/', '…\n",
       "\t\"['/biosamples/ENCBS162TUX/']\"\n",
       "\t…\n",
       "\t\"['/biosamples/ENCBS435CFB/']\"\n",
       "\t\"['/biosamples/ENCBS869ANE/']\"\n",
       "\t\"['/biosamples/ENCBS975PJY/']\"\n",
       "\t\"['/biosamples/ENCBS652QEI/']\"\n",
       "\t\"['/biosamples/ENCBS391ENC/']\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (2_752,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>label</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;DEAF1&quot;</td></tr><tr><td>&quot;NUP98&quot;</td></tr><tr><td>&quot;KLF4&quot;</td></tr><tr><td>&quot;ELAVL4&quot;</td></tr><tr><td>&quot;CCNT2&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;al&quot;</td></tr><tr><td>&quot;MEF2A&quot;</td></tr><tr><td>&quot;B52&quot;</td></tr><tr><td>&quot;CLNS1A&quot;</td></tr><tr><td>&quot;ZNF292&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (2_752,)\n",
       "Series: 'label' [str]\n",
       "[\n",
       "\t\"DEAF1\"\n",
       "\t\"NUP98\"\n",
       "\t\"KLF4\"\n",
       "\t\"ELAVL4\"\n",
       "\t\"CCNT2\"\n",
       "\t…\n",
       "\t\"al\"\n",
       "\t\"MEF2A\"\n",
       "\t\"B52\"\n",
       "\t\"CLNS1A\"\n",
       "\t\"ZNF292\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>download_url</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr><tr><td>&quot;https://encodeproject.org/file…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008,)\n",
       "Series: 'download_url' [str]\n",
       "[\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "\t\"https://encodeproject.org/file…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (340_589,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>derived_from</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[&#x27;/files/ENCFF720AUK/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/GRCh38_no_alt_analysi…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF001ZZG/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF300USB/&#x27;, &#x27;/file…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF057OAB/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF843JYN/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF056BGC/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF054AUB/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF331YVE/&#x27;, &#x27;/file…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (340_589,)\n",
       "Series: 'derived_from' [str]\n",
       "[\n",
       "\t\"['/files/ENCFF720AUK/', '/file…\n",
       "\t\"['/files/ENCFF723SJO/', '/file…\n",
       "\t\"['/files/GRCh38_no_alt_analysi…\n",
       "\t\"['/files/ENCFF001ZZG/', '/file…\n",
       "\t\"['/files/ENCFF300USB/', '/file…\n",
       "\t…\n",
       "\t\"['/files/ENCFF057OAB/']\"\n",
       "\t\"['/files/ENCFF843JYN/']\"\n",
       "\t\"['/files/ENCFF056BGC/', '/file…\n",
       "\t\"['/files/ENCFF054AUB/']\"\n",
       "\t\"['/files/ENCFF331YVE/', '/file…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (14,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>genome_annotation</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;V3c&quot;</td></tr><tr><td>&quot;V10&quot;</td></tr><tr><td>&quot;V29&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;M2&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;V24&quot;</td></tr><tr><td>&quot;V22&quot;</td></tr><tr><td>&quot;M3&quot;</td></tr><tr><td>&quot;M21&quot;</td></tr><tr><td>&quot;M4&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (14,)\n",
       "Series: 'genome_annotation' [str]\n",
       "[\n",
       "\t\"V3c\"\n",
       "\t\"V10\"\n",
       "\t\"V29\"\n",
       "\t\"NA\"\n",
       "\t\"M2\"\n",
       "\t…\n",
       "\t\"V24\"\n",
       "\t\"V22\"\n",
       "\t\"M3\"\n",
       "\t\"M21\"\n",
       "\t\"M4\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (4,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>paired_end</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;2&quot;</td></tr><tr><td>&quot;1,2&quot;</td></tr><tr><td>&quot;1&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (4,)\n",
       "Series: 'paired_end' [str]\n",
       "[\n",
       "\t\"2\"\n",
       "\t\"1,2\"\n",
       "\t\"1\"\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (96_288,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>paired_with</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;/files/ENCFF158LND/&quot;</td></tr><tr><td>&quot;/files/ENCFF118SCW/&quot;</td></tr><tr><td>&quot;/files/ENCFF539ALA/&quot;</td></tr><tr><td>&quot;/files/ENCFF571HBG/&quot;</td></tr><tr><td>&quot;/files/ENCFF843CUY/&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;/files/ENCFF181QHH/&quot;</td></tr><tr><td>&quot;/files/ENCFF744NPL/&quot;</td></tr><tr><td>&quot;/files/ENCFF242LHV/&quot;</td></tr><tr><td>&quot;/files/ENCFF104LBN/&quot;</td></tr><tr><td>&quot;/files/ENCFF665FQK/&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (96_288,)\n",
       "Series: 'paired_with' [str]\n",
       "[\n",
       "\t\"/files/ENCFF158LND/\"\n",
       "\t\"/files/ENCFF118SCW/\"\n",
       "\t\"/files/ENCFF539ALA/\"\n",
       "\t\"/files/ENCFF571HBG/\"\n",
       "\t\"/files/ENCFF843CUY/\"\n",
       "\t…\n",
       "\t\"/files/ENCFF181QHH/\"\n",
       "\t\"/files/ENCFF744NPL/\"\n",
       "\t\"/files/ENCFF242LHV/\"\n",
       "\t\"/files/ENCFF104LBN/\"\n",
       "\t\"/files/ENCFF665FQK/\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (3,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>preferred_default</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;False&quot;</td></tr><tr><td>&quot;True&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (3,)\n",
       "Series: 'preferred_default' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "\t\"False\"\n",
       "\t\"True\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (3,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>run_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;single-ended&quot;</td></tr><tr><td>&quot;paired-ended&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (3,)\n",
       "Series: 'run_type' [str]\n",
       "[\n",
       "\t\"single-ended\"\n",
       "\t\"paired-ended\"\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (88,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>read_length</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;41&quot;</td></tr><tr><td>&quot;98&quot;</td></tr><tr><td>&quot;28&quot;</td></tr><tr><td>&quot;71&quot;</td></tr><tr><td>&quot;36&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;50&quot;</td></tr><tr><td>&quot;31&quot;</td></tr><tr><td>&quot;132&quot;</td></tr><tr><td>&quot;73&quot;</td></tr><tr><td>&quot;45&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (88,)\n",
       "Series: 'read_length' [str]\n",
       "[\n",
       "\t\"41\"\n",
       "\t\"98\"\n",
       "\t\"28\"\n",
       "\t\"71\"\n",
       "\t\"36\"\n",
       "\t…\n",
       "\t\"50\"\n",
       "\t\"31\"\n",
       "\t\"132\"\n",
       "\t\"73\"\n",
       "\t\"45\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (83,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>mapped_read_length</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;83&quot;</td></tr><tr><td>&quot;37&quot;</td></tr><tr><td>&quot;41&quot;</td></tr><tr><td>&quot;88&quot;</td></tr><tr><td>&quot;51&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;25&quot;</td></tr><tr><td>&quot;123&quot;</td></tr><tr><td>&quot;131&quot;</td></tr><tr><td>&quot;22&quot;</td></tr><tr><td>&quot;108&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (83,)\n",
       "Series: 'mapped_read_length' [str]\n",
       "[\n",
       "\t\"83\"\n",
       "\t\"37\"\n",
       "\t\"41\"\n",
       "\t\"88\"\n",
       "\t\"51\"\n",
       "\t…\n",
       "\t\"25\"\n",
       "\t\"123\"\n",
       "\t\"131\"\n",
       "\t\"22\"\n",
       "\t\"108\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (27,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>cropped_read_length</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;83&quot;</td></tr><tr><td>&quot;36&quot;</td></tr><tr><td>&quot;151&quot;</td></tr><tr><td>&quot;45&quot;</td></tr><tr><td>&quot;47&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;100&quot;</td></tr><tr><td>&quot;101&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;72&quot;</td></tr><tr><td>&quot;50&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (27,)\n",
       "Series: 'cropped_read_length' [str]\n",
       "[\n",
       "\t\"83\"\n",
       "\t\"36\"\n",
       "\t\"151\"\n",
       "\t\"45\"\n",
       "\t\"47\"\n",
       "\t…\n",
       "\t\"100\"\n",
       "\t\"101\"\n",
       "\t\"NA\"\n",
       "\t\"72\"\n",
       "\t\"50\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (2,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>cropped_read_length_tolerance</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;2&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (2,)\n",
       "Series: 'cropped_read_length_tolerance' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "\t\"2\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (3,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>mapped_run_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;paired-ended&quot;</td></tr><tr><td>&quot;single-ended&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (3,)\n",
       "Series: 'mapped_run_type' [str]\n",
       "[\n",
       "\t\"paired-ended\"\n",
       "\t\"single-ended\"\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (2,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>read_length_units</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;nt&quot;</td></tr><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (2,)\n",
       "Series: 'read_length_units' [str]\n",
       "[\n",
       "\t\"nt\"\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (7,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>output_category</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;signal&quot;</td></tr><tr><td>&quot;raw data&quot;</td></tr><tr><td>&quot;quantification&quot;</td></tr><tr><td>&quot;annotation&quot;</td></tr><tr><td>&quot;alignment&quot;</td></tr><tr><td>&quot;validation&quot;</td></tr><tr><td>&quot;reference&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (7,)\n",
       "Series: 'output_category' [str]\n",
       "[\n",
       "\t\"signal\"\n",
       "\t\"raw data\"\n",
       "\t\"quantification\"\n",
       "\t\"annotation\"\n",
       "\t\"alignment\"\n",
       "\t\"validation\"\n",
       "\t\"reference\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (121,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>output_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;microRNA quantifications&quot;</td></tr><tr><td>&quot;rejected reads&quot;</td></tr><tr><td>&quot;filtered modified peptide quan…</td></tr><tr><td>&quot;filtered regions&quot;</td></tr><tr><td>&quot;cell coordinates&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;read annotations&quot;</td></tr><tr><td>&quot;subreads&quot;</td></tr><tr><td>&quot;raw signal&quot;</td></tr><tr><td>&quot;fragments&quot;</td></tr><tr><td>&quot;unfiltered sparse gene count m…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (121,)\n",
       "Series: 'output_type' [str]\n",
       "[\n",
       "\t\"microRNA quantifications\"\n",
       "\t\"rejected reads\"\n",
       "\t\"filtered modified peptide quan…\n",
       "\t\"filtered regions\"\n",
       "\t\"cell coordinates\"\n",
       "\t…\n",
       "\t\"read annotations\"\n",
       "\t\"subreads\"\n",
       "\t\"raw signal\"\n",
       "\t\"fragments\"\n",
       "\t\"unfiltered sparse gene count m…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (2_115,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>index_of</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[&#x27;/files/ENCFF920VYN/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF538VZY/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF432INX/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF786IGX/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF577MZP/&#x27;]&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF220KWE/&#x27;]&quot;</td></tr><tr><td>&quot;[&#x27;/files/ENCFF240WET/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF118HST/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF736WUH/&#x27;, &#x27;/file…</td></tr><tr><td>&quot;[&#x27;/files/ENCFF875HMZ/&#x27;]&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (2_115,)\n",
       "Series: 'index_of' [str]\n",
       "[\n",
       "\t\"['/files/ENCFF920VYN/']\"\n",
       "\t\"['/files/ENCFF538VZY/']\"\n",
       "\t\"['/files/ENCFF432INX/']\"\n",
       "\t\"['/files/ENCFF786IGX/']\"\n",
       "\t\"['/files/ENCFF577MZP/']\"\n",
       "\t…\n",
       "\t\"['/files/ENCFF220KWE/']\"\n",
       "\t\"['/files/ENCFF240WET/', '/file…\n",
       "\t\"['/files/ENCFF118HST/', '/file…\n",
       "\t\"['/files/ENCFF736WUH/', '/file…\n",
       "\t\"['/files/ENCFF875HMZ/']\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (210_871,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>quality_metrics</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;[{&#x27;submitted_by&#x27;: &#x27;/users/9851…</td></tr><tr><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;ChIP-seq…</td></tr><tr><td>&quot;[{&#x27;submitted_by&#x27;: &#x27;/users/4c23…</td></tr><tr><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;ChIP-seq…</td></tr><tr><td>&quot;[{&#x27;submitted_by&#x27;: &#x27;/users/81a6…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;[{&#x27;submitted_by&#x27;: &#x27;/users/6800…</td></tr><tr><td>&quot;[{&#x27;Average input read length&#x27;:…</td></tr><tr><td>&quot;[{&#x27;Average input read length&#x27;:…</td></tr><tr><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;HiC&#x27;, &#x27;s…</td></tr><tr><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;HiC&#x27;, &#x27;s…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (210_871,)\n",
       "Series: 'quality_metrics' [str]\n",
       "[\n",
       "\t\"[{'submitted_by': '/users/9851…\n",
       "\t\"[{'assay_term_name': 'ChIP-seq…\n",
       "\t\"[{'submitted_by': '/users/4c23…\n",
       "\t\"[{'assay_term_name': 'ChIP-seq…\n",
       "\t\"[{'submitted_by': '/users/81a6…\n",
       "\t…\n",
       "\t\"[{'submitted_by': '/users/6800…\n",
       "\t\"[{'Average input read length':…\n",
       "\t\"[{'Average input read length':…\n",
       "\t\"[{'assay_term_name': 'HiC', 's…\n",
       "\t\"[{'assay_term_name': 'HiC', 's…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (72,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>lab_title</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;Anshul Kundaje, Stanford&quot;</td></tr><tr><td>&quot;Marjorie Brand, OHRI&quot;</td></tr><tr><td>&quot;Joseph Costello, UCSF&quot;</td></tr><tr><td>&quot;Ana Conesa, UF&quot;</td></tr><tr><td>&quot;Mats Ljungman, UMichigan&quot;</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;Yin Shen, UCSF&quot;</td></tr><tr><td>&quot;Will Greenleaf, Stanford&quot;</td></tr><tr><td>&quot;Job Dekker, UMass&quot;</td></tr><tr><td>&quot;Keji Zhao, NIH&quot;</td></tr><tr><td>&quot;Christine Disteche, UW&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (72,)\n",
       "Series: 'lab_title' [str]\n",
       "[\n",
       "\t\"Anshul Kundaje, Stanford\"\n",
       "\t\"Marjorie Brand, OHRI\"\n",
       "\t\"Joseph Costello, UCSF\"\n",
       "\t\"Ana Conesa, UF\"\n",
       "\t\"Mats Ljungman, UMichigan\"\n",
       "\t…\n",
       "\t\"Yin Shen, UCSF\"\n",
       "\t\"Will Greenleaf, Stanford\"\n",
       "\t\"Job Dekker, UMass\"\n",
       "\t\"Keji Zhao, NIH\"\n",
       "\t\"Christine Disteche, UW\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (6,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>project</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;modERN&quot;</td></tr><tr><td>&quot;ENCODE&quot;</td></tr><tr><td>&quot;community&quot;</td></tr><tr><td>&quot;modENCODE&quot;</td></tr><tr><td>&quot;Roadmap&quot;</td></tr><tr><td>&quot;GGR&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (6,)\n",
       "Series: 'project' [str]\n",
       "[\n",
       "\t\"modERN\"\n",
       "\t\"ENCODE\"\n",
       "\t\"community\"\n",
       "\t\"modENCODE\"\n",
       "\t\"Roadmap\"\n",
       "\t\"GGR\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (229_108,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>step_run</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2017-08-29T0…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2016-08-29T1…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2017-05-05T1…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2016-05-03T2…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (229_108,)\n",
       "Series: 'step_run' [str]\n",
       "[\n",
       "\t\"{'date_created': '2017-08-29T0…\n",
       "\t\"{'status': 'released', 'date_c…\n",
       "\t\"{'date_created': '2016-08-29T1…\n",
       "\t\"{'status': 'released', 'lab': …\n",
       "\t\"{'date_created': '2017-05-05T1…\n",
       "\t…\n",
       "\t\"{'status': 'released', 'date_c…\n",
       "\t\"{'status': 'released', 'lab': …\n",
       "\t\"{'date_created': '2016-05-03T2…\n",
       "\t\"{'status': 'released', 'date_c…\n",
       "\t\"{'status': 'released', 'date_c…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (731_431,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>date_created</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;2016-04-28T16:47:37.915026+00:…</td></tr><tr><td>&quot;2023-01-21T08:32:27.169795+00:…</td></tr><tr><td>&quot;2017-11-28T22:31:25.507032+00:…</td></tr><tr><td>&quot;2020-11-27T23:56:29.417409+00:…</td></tr><tr><td>&quot;2020-08-17T23:26:52.368217+00:…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;2020-12-02T03:48:49.772936+00:…</td></tr><tr><td>&quot;2021-04-08T22:58:08.945629+00:…</td></tr><tr><td>&quot;2022-02-11T15:24:48.397719+00:…</td></tr><tr><td>&quot;2017-11-17T15:24:03.080829+00:…</td></tr><tr><td>&quot;2015-03-16T16:31:32.467119+00:…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (731_431,)\n",
       "Series: 'date_created' [str]\n",
       "[\n",
       "\t\"2016-04-28T16:47:37.915026+00:…\n",
       "\t\"2023-01-21T08:32:27.169795+00:…\n",
       "\t\"2017-11-28T22:31:25.507032+00:…\n",
       "\t\"2020-11-27T23:56:29.417409+00:…\n",
       "\t\"2020-08-17T23:26:52.368217+00:…\n",
       "\t…\n",
       "\t\"2020-12-02T03:48:49.772936+00:…\n",
       "\t\"2021-04-08T22:58:08.945629+00:…\n",
       "\t\"2022-02-11T15:24:48.397719+00:…\n",
       "\t\"2017-11-17T15:24:03.080829+00:…\n",
       "\t\"2015-03-16T16:31:32.467119+00:…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (404,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>analysis_step_version</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;{&#x27;schema_version&#x27;: &#x27;4&#x27;, &#x27;statu…</td></tr><tr><td>&quot;{&#x27;submitted_by&#x27;: &#x27;/users/7e944…</td></tr><tr><td>&quot;{&#x27;aliases&#x27;: [&#x27;dnanexus:dnase-a…</td></tr><tr><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;aliase…</td></tr><tr><td>&quot;{&#x27;aliases&#x27;: [&#x27;dnanexus:rampage…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-10-13T1…</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2022-03-12T0…</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2022-05-27T1…</td></tr><tr><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-11-03T2…</td></tr><tr><td>&quot;{&#x27;aliases&#x27;: [], &#x27;analysis_step…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (404,)\n",
       "Series: 'analysis_step_version' [str]\n",
       "[\n",
       "\t\"{'schema_version': '4', 'statu…\n",
       "\t\"{'submitted_by': '/users/7e944…\n",
       "\t\"{'aliases': ['dnanexus:dnase-a…\n",
       "\t\"{'status': 'released', 'aliase…\n",
       "\t\"{'aliases': ['dnanexus:rampage…\n",
       "\t…\n",
       "\t\"{'date_created': '2020-10-13T1…\n",
       "\t\"{'date_created': '2022-03-12T0…\n",
       "\t\"{'date_created': '2022-05-27T1…\n",
       "\t\"{'date_created': '2020-11-03T2…\n",
       "\t\"{'aliases': [], 'analysis_step…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (3,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>restricted</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;True&quot;</td></tr><tr><td>&quot;False&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (3,)\n",
       "Series: 'restricted' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "\t\"True\"\n",
       "\t\"False\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (77,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>submitter_comment</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;Not all available reads were u…</td></tr><tr><td>&quot;Confirmed with submitter that …</td></tr><tr><td>&quot;File content md5sum conflicts …</td></tr><tr><td>&quot;NA&quot;</td></tr><tr><td>&quot;This file is a replacement sub…</td></tr><tr><td>&hellip;</td></tr><tr><td>&quot;Not all available reads were u…</td></tr><tr><td>&quot;Fastq file ENCFF000AUN contain…</td></tr><tr><td>&quot;Not all of the available reads…</td></tr><tr><td>&quot;DESeq result. SampleA: Knockdo…</td></tr><tr><td>&quot;The fastqs submitted are the r…</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (77,)\n",
       "Series: 'submitter_comment' [str]\n",
       "[\n",
       "\t\"Not all available reads were u…\n",
       "\t\"Confirmed with submitter that …\n",
       "\t\"File content md5sum conflicts …\n",
       "\t\"NA\"\n",
       "\t\"This file is a replacement sub…\n",
       "\t…\n",
       "\t\"Not all available reads were u…\n",
       "\t\"Fastq file ENCFF000AUN contain…\n",
       "\t\"Not all of the available reads…\n",
       "\t\"DESeq result. SampleA: Knockdo…\n",
       "\t\"The fastqs submitted are the r…\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (2,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>status</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;released&quot;</td></tr><tr><td>&quot;archived&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (2,)\n",
       "Series: 'status' [str]\n",
       "[\n",
       "\t\"released\"\n",
       "\t\"archived\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (1,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>annotation_type</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (1,)\n",
       "Series: 'annotation_type' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (1,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>annotation_subtype</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (1,)\n",
       "Series: 'annotation_subtype' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (1,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>biochemical_inputs</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (1,)\n",
       "Series: 'biochemical_inputs' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (1,)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>encyclopedia_version</th></tr><tr><td>str</td></tr></thead><tbody><tr><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (1,)\n",
       "Series: 'encyclopedia_version' [str]\n",
       "[\n",
       "\t\"NA\"\n",
       "]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for column in encode_df.iter_columns():\n",
    "    display(column.unique())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008, 45)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th><th>annotation_type</th><th>annotation_subtype</th><th>biochemical_inputs</th><th>encyclopedia_version</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>&quot;NA&quot;</td><td>&quot;3936091129&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;150&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>&quot;NA&quot;</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:07.433754+00:…</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>&quot;NA&quot;</td><td>&quot;4117922&quot;</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;True&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>&quot;NA&quot;</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2022-01-30T04:31:33.614985+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>&quot;NA&quot;</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>&quot;NA&quot;</td><td>&quot;263567745&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>&quot;NA&quot;</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>&quot;NA&quot;</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;NA&quot;</td><td>&quot;2022-02-11T14:58:17.561304+00:…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;NA&quot;</td><td>&quot;120998658&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>&quot;NA&quot;</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:14.572055+00:…</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>&quot;NA&quot;</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>&quot;NA&quot;</td><td>&quot;45801960&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>&quot;NA&quot;</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>&quot;NA&quot;</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;NA&quot;</td><td>&quot;2022-02-11T15:54:31.696914+00:…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>&quot;ENCFF266RHM&quot;</td><td>&quot;ENCFF266RHM&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>&quot;2466892&quot;</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;annotation&quot;</td><td>&quot;footprints&quot;</td><td>&quot;NA&quot;</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:17.058788+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF091WRX&quot;</td><td>&quot;ENCFF091WRX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>&quot;1980267&quot;</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF180EJG/&#x27;, &#x27;/file…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>&quot;NA&quot;</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:28.513391+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF563EQX&quot;</td><td>&quot;ENCFF563EQX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>&quot;3018176&quot;</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF091WRX/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>&quot;NA&quot;</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:37.977557+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF196LAE&quot;</td><td>&quot;ENCFF196LAE&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>&quot;218184446&quot;</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF731HGQ/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;quantification&quot;</td><td>&quot;FDR cut rate&quot;</td><td>&quot;NA&quot;</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:25.369881+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr><tr><td>&quot;ENCFF957MYA&quot;</td><td>&quot;ENCFF957MYA&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>&quot;1345926&quot;</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>&quot;NA&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>&quot;NA&quot;</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:33.939195+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;released&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td><td>&quot;NA&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008, 45)\n",
       "┌───────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬───────────┐\n",
       "│ title     ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ annotatio ┆ annotatio ┆ biochemic ┆ encyclope │\n",
       "│ ---       ┆ ---       ┆ ---       ┆ ---      ┆   ┆ n_type    ┆ n_subtype ┆ al_inputs ┆ dia_versi │\n",
       "│ str       ┆ str       ┆ str       ┆ str      ┆   ┆ ---       ┆ ---       ┆ ---       ┆ on        │\n",
       "│           ┆           ┆           ┆          ┆   ┆ str       ┆ str       ┆ str       ┆ ---       │\n",
       "│           ┆           ┆           ┆          ┆   ┆           ┆           ┆           ┆ str       │\n",
       "╞═══════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪═══════════╡\n",
       "│ ENCFF566N ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ GG        ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 369MDF/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF642U ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ FR        ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 250SCW/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF094M ┆ ENCFF094M ┆ /experime ┆ NA       ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ ID        ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF506T ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ JN        ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 369MDF/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF174I ┆ ENCFF174I ┆ /experime ┆ NA       ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ GS        ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ …         ┆ …         ┆ …         ┆ …        ┆ … ┆ …         ┆ …         ┆ …         ┆ …         │\n",
       "│ ENCFF266R ┆ ENCFF266R ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ HM        ┆ HM        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 820WLP/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF091W ┆ ENCFF091W ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ RX        ┆ RX        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 820WLP/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF563E ┆ ENCFF563E ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ QX        ┆ QX        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 820WLP/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF196L ┆ ENCFF196L ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ AE        ┆ AE        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 820WLP/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF957M ┆ ENCFF957M ┆ /experime ┆ GRCh38   ┆ … ┆ NA        ┆ NA        ┆ NA        ┆ NA        │\n",
       "│ YA        ┆ YA        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 820WLP/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "└───────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴───────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(encode_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (5, 45)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th><th>annotation_type</th><th>annotation_subtype</th><th>biochemical_inputs</th><th>encyclopedia_version</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>&quot;3936091129&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;150&quot;</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:07.433754+00:…</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>&quot;4117922&quot;</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>&quot;True&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2022-01-30T04:31:33.614985+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>&quot;263567745&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T14:58:17.561304+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>&quot;120998658&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:14.572055+00:…</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>&quot;45801960&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T15:54:31.696914+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (5, 45)\n",
       "┌───────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬───────────┐\n",
       "│ title     ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ annotatio ┆ annotatio ┆ biochemic ┆ encyclope │\n",
       "│ ---       ┆ ---       ┆ ---       ┆ ---      ┆   ┆ n_type    ┆ n_subtype ┆ al_inputs ┆ dia_versi │\n",
       "│ str       ┆ str       ┆ str       ┆ str      ┆   ┆ ---       ┆ ---       ┆ ---       ┆ on        │\n",
       "│           ┆           ┆           ┆          ┆   ┆ str       ┆ str       ┆ str       ┆ ---       │\n",
       "│           ┆           ┆           ┆          ┆   ┆           ┆           ┆           ┆ str       │\n",
       "╞═══════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪═══════════╡\n",
       "│ ENCFF566N ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ null      ┆ null      ┆ null      ┆ null      │\n",
       "│ GG        ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 369MDF/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF642U ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ null      ┆ null      ┆ null      ┆ null      │\n",
       "│ FR        ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 250SCW/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF094M ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ null      │\n",
       "│ ID        ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF506T ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ null      ┆ null      ┆ null      ┆ null      │\n",
       "│ JN        ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 369MDF/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│ ENCFF174I ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ null      │\n",
       "│ GS        ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "│           ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆           │\n",
       "└───────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴───────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Convert \"NA\" to None\n",
    "\n",
    "encode_df = pl.DataFrame([column.replace({\"NA\": None}) for column in encode_df.iter_columns()])\n",
    "display(encode_df.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (5, 41)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>&quot;3936091129&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;150&quot;</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:07.433754+00:…</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>&quot;4117922&quot;</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>&quot;True&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2022-01-30T04:31:33.614985+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>&quot;263567745&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T14:58:17.561304+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>&quot;120998658&quot;</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:14.572055+00:…</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>&quot;45801960&quot;</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;58&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T15:54:31.696914+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (5, 41)\n",
       "┌────────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
       "│ title      ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ analysis_ ┆ restricte ┆ submitter ┆ status   │\n",
       "│ ---        ┆ ---       ┆ ---       ┆ ---      ┆   ┆ step_vers ┆ d         ┆ _comment  ┆ ---      │\n",
       "│ str        ┆ str       ┆ str       ┆ str      ┆   ┆ ion       ┆ ---       ┆ ---       ┆ str      │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ---       ┆ str       ┆ str       ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ str       ┆           ┆           ┆          │\n",
       "╞════════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪══════════╡\n",
       "│ ENCFF566NG ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ {'softwar ┆ null      ┆ null      ┆ released │\n",
       "│ G          ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆ e_version ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ s':       ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ [{'submi… ┆           ┆           ┆          │\n",
       "│ ENCFF642UF ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ R          ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 250SCW/   ┆          ┆   ┆ '2021-12- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 14T2…     ┆           ┆           ┆          │\n",
       "│ ENCFF094MI ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ D          ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ ENCFF506TJ ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ {'analysi ┆ null      ┆ null      ┆ released │\n",
       "│ N          ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆ s_step':  ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ {'documen ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ts'…      ┆           ┆           ┆          │\n",
       "│ ENCFF174IG ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ S          ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "└────────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴──────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Drop last 4 columns (contain no information)\n",
    "\n",
    "encode_df = encode_df.drop(pl.col(\"annotation_type\", \"annotation_subtype\", \"biochemical_inputs\", \"encyclopedia_version\"))\n",
    "display(encode_df.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008, 41)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>f64</td><td>f64</td><td>f64</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>3.9361e9</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>150.0</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:07.433754+00:…</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>4.117922e6</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>&quot;True&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2022-01-30T04:31:33.614985+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>2.63567745e8</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T14:58:17.561304+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>1.20998658e8</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:14.572055+00:…</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>4.580196e7</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T15:54:31.696914+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>&quot;ENCFF266RHM&quot;</td><td>&quot;ENCFF266RHM&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.466892e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;footprints&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:17.058788+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF091WRX&quot;</td><td>&quot;ENCFF091WRX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.980267e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF180EJG/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:28.513391+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF563EQX&quot;</td><td>&quot;ENCFF563EQX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>3.018176e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF091WRX/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:37.977557+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF196LAE&quot;</td><td>&quot;ENCFF196LAE&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.18184446e8</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF731HGQ/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;FDR cut rate&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:25.369881+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF957MYA&quot;</td><td>&quot;ENCFF957MYA&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.345926e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:33.939195+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008, 41)\n",
       "┌────────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
       "│ title      ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ analysis_ ┆ restricte ┆ submitter ┆ status   │\n",
       "│ ---        ┆ ---       ┆ ---       ┆ ---      ┆   ┆ step_vers ┆ d         ┆ _comment  ┆ ---      │\n",
       "│ str        ┆ str       ┆ str       ┆ str      ┆   ┆ ion       ┆ ---       ┆ ---       ┆ str      │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ---       ┆ str       ┆ str       ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ str       ┆           ┆           ┆          │\n",
       "╞════════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪══════════╡\n",
       "│ ENCFF566NG ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ {'softwar ┆ null      ┆ null      ┆ released │\n",
       "│ G          ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆ e_version ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ s':       ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ [{'submi… ┆           ┆           ┆          │\n",
       "│ ENCFF642UF ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ R          ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 250SCW/   ┆          ┆   ┆ '2021-12- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 14T2…     ┆           ┆           ┆          │\n",
       "│ ENCFF094MI ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ D          ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ ENCFF506TJ ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ {'analysi ┆ null      ┆ null      ┆ released │\n",
       "│ N          ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆ s_step':  ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ {'documen ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ts'…      ┆           ┆           ┆          │\n",
       "│ ENCFF174IG ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ S          ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ …          ┆ …         ┆ …         ┆ …        ┆ … ┆ …         ┆ …         ┆ …         ┆ …        │\n",
       "│ ENCFF266RH ┆ ENCFF266R ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ M          ┆ HM        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF091WR ┆ ENCFF091W ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ RX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF563EQ ┆ ENCFF563E ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ QX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF196LA ┆ ENCFF196L ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ E          ┆ AE        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF957MY ┆ ENCFF957M ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ A          ┆ YA        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "└────────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴──────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Convert select columns to numeric\n",
    "\n",
    "\n",
    "encode_df = encode_df.with_columns(pl.col(\"file_size\", \"read_length\", \"mapped_read_length\", \"cropped_read_length\", \"cropped_read_length_tolerance\").cast(pl.Float64))\n",
    "display(encode_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008, 41)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>bool</td><td>str</td><td>f64</td><td>f64</td><td>f64</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>bool</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>3.9361e9</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>150.0</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:07.433754+00:…</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>4.117922e6</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>true</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2022-01-30T04:31:33.614985+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>2.63567745e8</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T14:58:17.561304+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>1.20998658e8</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>&quot;2021-12-31T08:51:14.572055+00:…</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>4.580196e7</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>&quot;2022-02-11T15:54:31.696914+00:…</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>&quot;ENCFF266RHM&quot;</td><td>&quot;ENCFF266RHM&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.466892e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;footprints&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:17.058788+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF091WRX&quot;</td><td>&quot;ENCFF091WRX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.980267e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF180EJG/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:28.513391+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF563EQX&quot;</td><td>&quot;ENCFF563EQX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>3.018176e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF091WRX/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:37.977557+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF196LAE&quot;</td><td>&quot;ENCFF196LAE&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.18184446e8</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF731HGQ/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;FDR cut rate&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:25.369881+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF957MYA&quot;</td><td>&quot;ENCFF957MYA&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.345926e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>&quot;2020-11-06T18:02:33.939195+00:…</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008, 41)\n",
       "┌────────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
       "│ title      ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ analysis_ ┆ restricte ┆ submitter ┆ status   │\n",
       "│ ---        ┆ ---       ┆ ---       ┆ ---      ┆   ┆ step_vers ┆ d         ┆ _comment  ┆ ---      │\n",
       "│ str        ┆ str       ┆ str       ┆ str      ┆   ┆ ion       ┆ ---       ┆ ---       ┆ str      │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ---       ┆ bool      ┆ str       ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ str       ┆           ┆           ┆          │\n",
       "╞════════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪══════════╡\n",
       "│ ENCFF566NG ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ {'softwar ┆ null      ┆ null      ┆ released │\n",
       "│ G          ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆ e_version ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ s':       ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ [{'submi… ┆           ┆           ┆          │\n",
       "│ ENCFF642UF ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ R          ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 250SCW/   ┆          ┆   ┆ '2021-12- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 14T2…     ┆           ┆           ┆          │\n",
       "│ ENCFF094MI ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ D          ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ ENCFF506TJ ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ {'analysi ┆ null      ┆ null      ┆ released │\n",
       "│ N          ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆ s_step':  ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ {'documen ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ts'…      ┆           ┆           ┆          │\n",
       "│ ENCFF174IG ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ S          ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ …          ┆ …         ┆ …         ┆ …        ┆ … ┆ …         ┆ …         ┆ …         ┆ …        │\n",
       "│ ENCFF266RH ┆ ENCFF266R ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ M          ┆ HM        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF091WR ┆ ENCFF091W ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ RX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF563EQ ┆ ENCFF563E ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ QX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF196LA ┆ ENCFF196L ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ E          ┆ AE        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF957MY ┆ ENCFF957M ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ A          ┆ YA        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "└────────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴──────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Convert select columns to boolean\n",
    "\n",
    "encode_df = encode_df.with_columns(\n",
    "    pl.col(\"preferred_default\", \"restricted\").map_elements(\n",
    "        lambda x: True if x.lower() in [\"true\"] else \n",
    "                  False if x.lower() in [\"false\"] else None, return_dtype=pl.Boolean)\n",
    ")\n",
    "display(encode_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008, 41)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>bool</td><td>str</td><td>f64</td><td>f64</td><td>f64</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>datetime[μs]</td><td>str</td><td>bool</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>3.9361e9</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS291NHT/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF598IDH/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>150.0</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>2021-12-31 08:51:07.433754</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>4.117922e6</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>&quot;[&#x27;brain&#x27;]&quot;</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF723SJO/&#x27;, &#x27;/file…</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>true</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2022-01-30 04:31:33.614985</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>2.63567745e8</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;female adult (39 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS517ZRS/&#x27;]&quot;</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>2022-02-11 14:58:17.561304</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>1.20998658e8</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>&quot;[&#x27;blood&#x27;, &#x27;bodily fluid&#x27;]&quot;</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>&quot;[&#x27;/biosamples/ENCBS798JQV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF124OPS/&#x27;, &#x27;/file…</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>2021-12-31 08:51:14.572055</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>&quot;[&#x27;1_2&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>4.580196e7</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>&quot;[]&quot;</td><td>&quot;male adult (36 years)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS181QGR/&#x27;]&quot;</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>2022-02-11 15:54:31.696914</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>&quot;ENCFF266RHM&quot;</td><td>&quot;ENCFF266RHM&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.466892e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;footprints&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:17.058788</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF091WRX&quot;</td><td>&quot;ENCFF091WRX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.980267e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF180EJG/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:28.513391</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF563EQX&quot;</td><td>&quot;ENCFF563EQX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>3.018176e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF091WRX/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:37.977557</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF196LAE&quot;</td><td>&quot;ENCFF196LAE&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;1_1&#x27;]&quot;</td><td>&quot;[1]&quot;</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.18184446e8</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS469CDX/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF731HGQ/&#x27;]&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;FDR cut rate&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:25.369881</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF957MYA&quot;</td><td>&quot;ENCFF957MYA&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>&quot;[&#x27;2_1&#x27;]&quot;</td><td>&quot;[2]&quot;</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.345926e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>&quot;[&#x27;connective tissue&#x27;, &#x27;extraem…</td><td>&quot;male embryo (15 weeks)&quot;</td><td>&quot;[&#x27;/biosamples/ENCBS167CXV/&#x27;]&quot;</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>&quot;[&#x27;/files/ENCFF381FQU/&#x27;, &#x27;/file…</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:33.939195</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008, 41)\n",
       "┌────────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
       "│ title      ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ analysis_ ┆ restricte ┆ submitter ┆ status   │\n",
       "│ ---        ┆ ---       ┆ ---       ┆ ---      ┆   ┆ step_vers ┆ d         ┆ _comment  ┆ ---      │\n",
       "│ str        ┆ str       ┆ str       ┆ str      ┆   ┆ ion       ┆ ---       ┆ ---       ┆ str      │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ---       ┆ bool      ┆ str       ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ str       ┆           ┆           ┆          │\n",
       "╞════════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪══════════╡\n",
       "│ ENCFF566NG ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ {'softwar ┆ null      ┆ null      ┆ released │\n",
       "│ G          ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆ e_version ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ s':       ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ [{'submi… ┆           ┆           ┆          │\n",
       "│ ENCFF642UF ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ R          ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 250SCW/   ┆          ┆   ┆ '2021-12- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 14T2…     ┆           ┆           ┆          │\n",
       "│ ENCFF094MI ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ D          ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ ENCFF506TJ ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ {'analysi ┆ null      ┆ null      ┆ released │\n",
       "│ N          ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆ s_step':  ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ {'documen ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ts'…      ┆           ┆           ┆          │\n",
       "│ ENCFF174IG ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ S          ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ …          ┆ …         ┆ …         ┆ …        ┆ … ┆ …         ┆ …         ┆ …         ┆ …        │\n",
       "│ ENCFF266RH ┆ ENCFF266R ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ M          ┆ HM        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF091WR ┆ ENCFF091W ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ RX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF563EQ ┆ ENCFF563E ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ QX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF196LA ┆ ENCFF196L ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ E          ┆ AE        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF957MY ┆ ENCFF957M ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ A          ┆ YA        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "└────────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴──────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Convert select columns to datetime\n",
    "\n",
    "encode_df = encode_df.with_columns(pl.col(\"date_created\").cast(pl.Datetime))\n",
    "display(encode_df)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[]\n",
      "<class 'list'>\n"
     ]
    }
   ],
   "source": [
    "# Test ast.literal_eval\n",
    "\n",
    "example = \"[]\"\n",
    "result = ast.literal_eval(example)\n",
    "print(result)\n",
    "print(type(result))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><style>\n",
       ".dataframe > thead > tr,\n",
       ".dataframe > tbody > tr {\n",
       "  text-align: right;\n",
       "  white-space: pre-wrap;\n",
       "}\n",
       "</style>\n",
       "<small>shape: (762_008, 41)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>title</th><th>accession</th><th>dataset</th><th>assembly</th><th>technical_replicates</th><th>biological_replicates</th><th>file_format</th><th>file_type</th><th>file_format_type</th><th>file_size</th><th>assay_term_name</th><th>term_name</th><th>organ_slims</th><th>simple_biosample_summary</th><th>origin_batches</th><th>label</th><th>download_url</th><th>derived_from</th><th>genome_annotation</th><th>paired_end</th><th>paired_with</th><th>preferred_default</th><th>run_type</th><th>read_length</th><th>mapped_read_length</th><th>cropped_read_length</th><th>cropped_read_length_tolerance</th><th>mapped_run_type</th><th>read_length_units</th><th>output_category</th><th>output_type</th><th>index_of</th><th>quality_metrics</th><th>lab_title</th><th>project</th><th>step_run</th><th>date_created</th><th>analysis_step_version</th><th>restricted</th><th>submitter_comment</th><th>status</th></tr><tr><td>str</td><td>str</td><td>str</td><td>str</td><td>list[str]</td><td>list[i64]</td><td>str</td><td>str</td><td>str</td><td>f64</td><td>str</td><td>str</td><td>list[str]</td><td>str</td><td>list[str]</td><td>str</td><td>str</td><td>list[str]</td><td>str</td><td>str</td><td>str</td><td>bool</td><td>str</td><td>f64</td><td>f64</td><td>f64</td><td>f64</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>str</td><td>datetime[μs]</td><td>str</td><td>bool</td><td>str</td><td>str</td></tr></thead><tbody><tr><td>&quot;ENCFF566NGG&quot;</td><td>&quot;ENCFF566NGG&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;1_1&quot;]</td><td>[1]</td><td>&quot;bam&quot;</td><td>&quot;bam&quot;</td><td>null</td><td>3.9361e9</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>[&quot;blood&quot;, &quot;bodily fluid&quot;]</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>[&quot;/biosamples/ENCBS291NHT/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF598IDH/&quot;, &quot;/files/ENCFF146IQN/&quot;, &quot;/files/ENCFF967MFO/&quot;]</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>150.0</td><td>null</td><td>null</td><td>&quot;paired-ended&quot;</td><td>&quot;nt&quot;</td><td>&quot;alignment&quot;</td><td>&quot;transcriptome alignments&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;RNA-seq&#x27;…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>2021-12-31 08:51:07.433754</td><td>&quot;{&#x27;software_versions&#x27;: [{&#x27;submi…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF642UFR&quot;</td><td>&quot;ENCFF642UFR&quot;</td><td>&quot;/experiments/ENCSR250SCW/&quot;</td><td>&quot;mm10&quot;</td><td>[&quot;2_1&quot;]</td><td>[2]</td><td>&quot;tar&quot;</td><td>&quot;tar&quot;</td><td>null</td><td>4.117922e6</td><td>&quot;single-cell RNA sequencing ass…</td><td>&quot;left cerebral cortex&quot;</td><td>[&quot;brain&quot;]</td><td>&quot;male adult (2 months) strain B…</td><td>null</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF723SJO/&quot;, &quot;/files/ENCFF192SYW/&quot;]</td><td>&quot;M21&quot;</td><td>null</td><td>null</td><td>true</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;unfiltered sparse gene count m…</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;single-c…</td><td>&quot;Barbara Wold, Caltech&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2022-01-30 04:31:33.614985</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2021-12-14T2…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF094MID&quot;</td><td>&quot;ENCFF094MID&quot;</td><td>&quot;/experiments/ENCSR414JJE/&quot;</td><td>null</td><td>[&quot;1_2&quot;]</td><td>[1]</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>2.63567745e8</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;naive B cell&quot;</td><td>[&quot;blood&quot;, &quot;bodily fluid&quot;]</td><td>&quot;female adult (39 years)&quot;</td><td>[&quot;/biosamples/ENCBS517ZRS/&quot;]</td><td>&quot;H3K27me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF481AOK/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>2022-02-11 14:58:17.561304</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF506TJN&quot;</td><td>&quot;ENCFF506TJN&quot;</td><td>&quot;/experiments/ENCSR369MDF/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;2_1&quot;]</td><td>[2]</td><td>&quot;bigWig&quot;</td><td>&quot;bigWig&quot;</td><td>null</td><td>1.20998658e8</td><td>&quot;RNA-seq&quot;</td><td>&quot;K562&quot;</td><td>[&quot;blood&quot;, &quot;bodily fluid&quot;]</td><td>&quot;treated with 5 μM JQ1 for 4 ho…</td><td>[&quot;/biosamples/ENCBS798JQV/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF124OPS/&quot;, &quot;/files/GRCh38_EBV.chrom.sizes/&quot;]</td><td>&quot;V29&quot;</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;signal&quot;</td><td>&quot;signal of all reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;lab&#x27;: …</td><td>2021-12-31 08:51:14.572055</td><td>&quot;{&#x27;analysis_step&#x27;: {&#x27;documents&#x27;…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF174IGS&quot;</td><td>&quot;ENCFF174IGS&quot;</td><td>&quot;/experiments/ENCSR040TXN/&quot;</td><td>null</td><td>[&quot;1_2&quot;]</td><td>[1]</td><td>&quot;fastq&quot;</td><td>&quot;fastq&quot;</td><td>null</td><td>4.580196e7</td><td>&quot;Mint-ChIP-seq&quot;</td><td>&quot;central memory CD8-positive, a…</td><td>[]</td><td>&quot;male adult (36 years)&quot;</td><td>[&quot;/biosamples/ENCBS181QGR/&quot;]</td><td>&quot;H3K4me3&quot;</td><td>&quot;https://encodeproject.org/file…</td><td>null</td><td>null</td><td>&quot;2&quot;</td><td>&quot;/files/ENCFF066EBL/&quot;</td><td>null</td><td>&quot;paired-ended&quot;</td><td>58.0</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;nt&quot;</td><td>&quot;raw data&quot;</td><td>&quot;reads&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;Bradley Bernstein, Broad&quot;</td><td>&quot;ENCODE&quot;</td><td>null</td><td>2022-02-11 15:54:31.696914</td><td>null</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td><td>&hellip;</td></tr><tr><td>&quot;ENCFF266RHM&quot;</td><td>&quot;ENCFF266RHM&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;2_1&quot;]</td><td>[2]</td><td>&quot;bed&quot;</td><td>&quot;bed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.466892e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>[&quot;connective tissue&quot;, &quot;extraembryonic component&quot;]</td><td>&quot;male embryo (15 weeks)&quot;</td><td>[&quot;/biosamples/ENCBS167CXV/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF381FQU/&quot;, &quot;/files/ENCFF968VMC/&quot;, … &quot;/files/ENCFF499QKB/&quot;]</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;footprints&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:17.058788</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF091WRX&quot;</td><td>&quot;ENCFF091WRX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;1_1&quot;]</td><td>[1]</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.980267e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>[&quot;connective tissue&quot;, &quot;extraembryonic component&quot;]</td><td>&quot;male embryo (15 weeks)&quot;</td><td>[&quot;/biosamples/ENCBS469CDX/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF180EJG/&quot;, &quot;/files/ENCFF833UTG/&quot;]</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:28.513391</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF563EQX&quot;</td><td>&quot;ENCFF563EQX&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;1_1&quot;]</td><td>[1]</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>3.018176e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>[&quot;connective tissue&quot;, &quot;extraembryonic component&quot;]</td><td>&quot;male embryo (15 weeks)&quot;</td><td>[&quot;/biosamples/ENCBS469CDX/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF091WRX/&quot;]</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:37.977557</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF196LAE&quot;</td><td>&quot;ENCFF196LAE&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;1_1&quot;]</td><td>[1]</td><td>&quot;bigBed&quot;</td><td>&quot;bigBed bed3+&quot;</td><td>&quot;bed3+&quot;</td><td>2.18184446e8</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>[&quot;connective tissue&quot;, &quot;extraembryonic component&quot;]</td><td>&quot;male embryo (15 weeks)&quot;</td><td>[&quot;/biosamples/ENCBS469CDX/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF731HGQ/&quot;]</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;quantification&quot;</td><td>&quot;FDR cut rate&quot;</td><td>null</td><td>&quot;[]&quot;</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:25.369881</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr><tr><td>&quot;ENCFF957MYA&quot;</td><td>&quot;ENCFF957MYA&quot;</td><td>&quot;/experiments/ENCSR820WLP/&quot;</td><td>&quot;GRCh38&quot;</td><td>[&quot;2_1&quot;]</td><td>[2]</td><td>&quot;bed&quot;</td><td>&quot;bed narrowPeak&quot;</td><td>&quot;narrowPeak&quot;</td><td>1.345926e6</td><td>&quot;DNase-seq&quot;</td><td>&quot;dedifferentiated amniotic flui…</td><td>[&quot;connective tissue&quot;, &quot;extraembryonic component&quot;]</td><td>&quot;male embryo (15 weeks)&quot;</td><td>[&quot;/biosamples/ENCBS167CXV/&quot;]</td><td>null</td><td>&quot;https://encodeproject.org/file…</td><td>[&quot;/files/ENCFF381FQU/&quot;, &quot;/files/ENCFF180EJG/&quot;]</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>null</td><td>&quot;annotation&quot;</td><td>&quot;peaks&quot;</td><td>null</td><td>&quot;[{&#x27;assay_term_name&#x27;: &#x27;DNase-se…</td><td>&quot;ENCODE Processing Pipeline&quot;</td><td>&quot;ENCODE&quot;</td><td>&quot;{&#x27;status&#x27;: &#x27;released&#x27;, &#x27;date_c…</td><td>2020-11-06 18:02:33.939195</td><td>&quot;{&#x27;date_created&#x27;: &#x27;2020-08-08T0…</td><td>null</td><td>null</td><td>&quot;released&quot;</td></tr></tbody></table></div>"
      ],
      "text/plain": [
       "shape: (762_008, 41)\n",
       "┌────────────┬───────────┬───────────┬──────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
       "│ title      ┆ accession ┆ dataset   ┆ assembly ┆ … ┆ analysis_ ┆ restricte ┆ submitter ┆ status   │\n",
       "│ ---        ┆ ---       ┆ ---       ┆ ---      ┆   ┆ step_vers ┆ d         ┆ _comment  ┆ ---      │\n",
       "│ str        ┆ str       ┆ str       ┆ str      ┆   ┆ ion       ┆ ---       ┆ ---       ┆ str      │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ---       ┆ bool      ┆ str       ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ str       ┆           ┆           ┆          │\n",
       "╞════════════╪═══════════╪═══════════╪══════════╪═══╪═══════════╪═══════════╪═══════════╪══════════╡\n",
       "│ ENCFF566NG ┆ ENCFF566N ┆ /experime ┆ GRCh38   ┆ … ┆ {'softwar ┆ null      ┆ null      ┆ released │\n",
       "│ G          ┆ GG        ┆ nts/ENCSR ┆          ┆   ┆ e_version ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ s':       ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ [{'submi… ┆           ┆           ┆          │\n",
       "│ ENCFF642UF ┆ ENCFF642U ┆ /experime ┆ mm10     ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ R          ┆ FR        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 250SCW/   ┆          ┆   ┆ '2021-12- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 14T2…     ┆           ┆           ┆          │\n",
       "│ ENCFF094MI ┆ ENCFF094M ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ D          ┆ ID        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 414JJE/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ ENCFF506TJ ┆ ENCFF506T ┆ /experime ┆ GRCh38   ┆ … ┆ {'analysi ┆ null      ┆ null      ┆ released │\n",
       "│ N          ┆ JN        ┆ nts/ENCSR ┆          ┆   ┆ s_step':  ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 369MDF/   ┆          ┆   ┆ {'documen ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ ts'…      ┆           ┆           ┆          │\n",
       "│ ENCFF174IG ┆ ENCFF174I ┆ /experime ┆ null     ┆ … ┆ null      ┆ null      ┆ null      ┆ released │\n",
       "│ S          ┆ GS        ┆ nts/ENCSR ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 040TXN/   ┆          ┆   ┆           ┆           ┆           ┆          │\n",
       "│ …          ┆ …         ┆ …         ┆ …        ┆ … ┆ …         ┆ …         ┆ …         ┆ …        │\n",
       "│ ENCFF266RH ┆ ENCFF266R ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ M          ┆ HM        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF091WR ┆ ENCFF091W ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ RX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF563EQ ┆ ENCFF563E ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ X          ┆ QX        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF196LA ┆ ENCFF196L ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ E          ┆ AE        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "│ ENCFF957MY ┆ ENCFF957M ┆ /experime ┆ GRCh38   ┆ … ┆ {'date_cr ┆ null      ┆ null      ┆ released │\n",
       "│ A          ┆ YA        ┆ nts/ENCSR ┆          ┆   ┆ eated':   ┆           ┆           ┆          │\n",
       "│            ┆           ┆ 820WLP/   ┆          ┆   ┆ '2020-08- ┆           ┆           ┆          │\n",
       "│            ┆           ┆           ┆          ┆   ┆ 08T0…     ┆           ┆           ┆          │\n",
       "└────────────┴───────────┴───────────┴──────────┴───┴───────────┴───────────┴───────────┴──────────┘"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Convert select columns to valid JSON\n",
    "\n",
    "encode_df = encode_df.with_columns(pl.col(\"technical_replicates\", \"biological_replicates\", \"organ_slims\", \"origin_batches\", \"derived_from\").map_elements(lambda x: json.loads(x.replace(\"'\", '\"'))))\n",
    "display(encode_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[\"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.402334+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF566NGG/'], 'attachment': {'download': 'rep1ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep1ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': '43baa9222d21578ad7d99663f603ec04'}, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, 'duplicates': 0, 'duplicates_qc_failed': 0, 'mapped': 84938342, 'mapped_pct': '97.2', 'mapped_qc_failed': 0, 'paired': 28108292, 'paired_properly': 25661984, 'paired_properly_pct': '91.3', 'paired_properly_qc_failed': 0, 'paired_qc_failed': 0, 'read1': 14054146, 'read1_qc_failed': 0, 'read2': 14054146, 'read2_qc_failed': 0, 'singletons': 0, 'singletons_pct': '0.0', 'singletons_qc_failed': 0, 'total': 87384650, 'total_qc_failed': 0, 'with_itself': 25661984, 'with_itself_qc_failed': 0, '@id': '/samtools-flagstats-quality-metrics/cfb189af-7ec8-4558-9093-6546925fca73/', '@type': ['SamtoolsFlagstatsQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cfb189af-7ec8-4558-9093-6546925fca73', 'assay_term_id': 'OBI:0001271'}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.233994+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF566NGG/'], 'attachment': {'download': 'rep1ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep1ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': 'daf55156d2fb4d54385726d7b8ba8722'}, '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.62%', '% of reads mapped to too many loci': '0.01%', '% of reads unmapped: other': '0.22%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.68%', 'Average input read length': 300, 'Average mapped length': 297.27, 'Deletion average length': 1.89, 'Deletion rate per base': '0.01%', 'Insertion average length': 1.85, 'Insertion rate per base': '0.01%', 'Mapping speed, Million of reads per hour': 136.07, 'Mismatch rate per base, %': '0.37%', 'Number of chimeric reads': 0, 'Number of input reads': 17688624, 'Number of reads mapped to multiple loci': 993429, 'Number of reads mapped to too many loci': 2472, 'Number of splices: AT/AC': 15439, 'Number of splices: Annotated (sjdb)': 16797458, 'Number of splices: GC/AG': 122724, 'Number of splices: GT/AG': 16858848, 'Number of splices: Non-canonical': 8429, 'Number of splices: Total': 17005440, 'Uniquely mapped reads %': '87.47%', 'Uniquely mapped reads number': 15472041, '@id': '/star-quality-metrics/82e662dc-5920-4d77-a715-584c30445b3d/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '82e662dc-5920-4d77-a715-584c30445b3d', 'assay_term_id': 'OBI:0001271', 'read_depth': 16465470}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.525219+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF566NGG/'], 'attachment': {'download': 'rep1ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep1ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': '33dcc693e9751d5c86a7420efdcefe98'}, 'Mt_rRNA': 288662, 'antisense': 99712, 'miRNA': 2, 'processed_transcript': 110684, 'protein_coding': 24770480, 'rRNA': 0, 'ribozyme': 4, 'sRNA': 0, 'scaRNA': 4, 'sense_intronic': 3878, 'sense_overlapping': 3362, 'snRNA': 12, 'snoRNA': 30, 'spikein': 0, '@id': '/gene-type-quantification-quality-metrics/922c6019-ae55-4c80-9d04-8d461a075a0d/', '@type': ['GeneTypeQuantificationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '922c6019-ae55-4c80-9d04-8d461a075a0d', 'assay_term_id': 'OBI:0001271'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:32:21.344130+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF642UFR/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c8e2ace5b7a831cf25e8589d335e17ed', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '08203adc3a11365cbcfd4234979e0e0e', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '149b3766182acc30a927520c205ab3cc', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/7b3da4a4-9e39-4c04-9c54-01e8fa2aa708/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7b3da4a4-9e39-4c04-9c54-01e8fa2aa708', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:27:06.618010+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF982HYB/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'e70204b19eba6e8bda1fd7b88dda6108', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '60370cd525d702d727ac5d836714d131', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '017e8b5c24f8434fd3d9bad5d4b4a757', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/31dece2e-d3b4-4970-8fa7-39826291668c/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '31dece2e-d3b4-4970-8fa7-39826291668c', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:07.243385+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', 'quality_metric_of': ['/files/ENCFF738LUH/'], 'unpaired_reads': 0, 'paired_reads': 37753381, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 4217430, 'paired_optical_duplicate_reads': 9620, 'pct_duplicate_reads': 11.171000000000001, 'total_fragments': 37752416, 'distinct_fragments': 33881158, 'positions_with_one_read': 2737007, 'NRF': 0.897457, 'PBC1': 0.905272, 'PBC2': 11.206279, '@id': '/chip-library-quality-metrics/0eb15c4f-8288-4c93-9ac4-55e06588777c/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0eb15c4f-8288-4c93-9ac4-55e06588777c', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:07.007676+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', 'quality_metric_of': ['/files/ENCFF738LUH/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 125, 'corr_estimated_fragment_len': 0.0991376292782874, 'phantom_peak': 40, 'corr_phantom_peak': 0.1035618, 'argmin_corr': 1500, 'min_corr': 0.09175083, 'NSC': 1.080509, 'cross_correlation_plot': {'download': 'ENCFF343QPX.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF343QPX.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'd52af395f22a62020eb15869978f9676'}, 'auc': 0.052701485205041594, 'syn_auc': 0.4945829094891469, 'x_intercept': 0.6027390960982876, 'syn_x_intercept': 1.4237620643249638e-292, 'elbow_pt': 0.8456489358511691, 'syn_elbow_pt': 0.5022017456071198, 'jsd': 0.29226907447182027, 'syn_jsd': 0.5753044538088874, 'pct_genome_enrich': 14.184719042715903, 'diff_enrich': 70.97984295386662, 'ch_div': 0.6404704821106504, 'jsd_plot': {'download': 'ENCFF343QPX.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF343QPX.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '3c367382b7542c218947738039716990', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF343QPX.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF343QPX.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '9a5f69e55eccb055e80b113c9ea1feaf', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/02dc986f-0fb4-4741-9d76-048035805205/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '02dc986f-0fb4-4741-9d76-048035805205', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:06.869927+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', 'quality_metric_of': ['/files/ENCFF738LUH/'], 'processing_stage': 'filtered', 'total_reads': 67071902, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 67071902, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 67071902, 'paired_reads_qc_failed': 0, 'read1': 33535951, 'read1_qc_failed': 0, 'read2': 33535951, 'read2_qc_failed': 0, 'properly_paired_reads': 67071902, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 67071902, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/eba3a412-63ca-4084-8146-7a020abac940/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'eba3a412-63ca-4084-8146-7a020abac940', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 33535951}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.818932+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF124OPS/'], 'attachment': {'download': 'rep2ENCSR369MDF_genome.bam.json', 'href': '@@download/attachment/rep2ENCSR369MDF_genome.bam.json', 'type': 'application/json', 'md5sum': 'd0a4b80fd0005c6f09119e95876ab4e9'}, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, 'duplicates': 0, 'duplicates_qc_failed': 0, 'mapped': 41420550, 'mapped_pct': '93.97', 'mapped_qc_failed': 0, 'paired': 39469572, 'paired_properly': 36809978, 'paired_properly_pct': '93.26', 'paired_properly_qc_failed': 0, 'paired_qc_failed': 0, 'read1': 19734786, 'read1_qc_failed': 0, 'read2': 19734786, 'read2_qc_failed': 0, 'singletons': 0, 'singletons_pct': '0.0', 'singletons_qc_failed': 0, 'total': 44080144, 'total_qc_failed': 0, 'with_itself': 36809978, 'with_itself_qc_failed': 0, '@id': '/samtools-flagstats-quality-metrics/91413ba5-5efa-446d-a426-28f69fb23d8a/', '@type': ['SamtoolsFlagstatsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '91413ba5-5efa-446d-a426-28f69fb23d8a', 'assay_term_id': 'OBI:0001271'}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.679763+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF124OPS/'], 'attachment': {'download': 'rep2ENCSR369MDF_genome.bam.json', 'href': '@@download/attachment/rep2ENCSR369MDF_genome.bam.json', 'type': 'application/json', 'md5sum': '0b0424e807c6e08ab686fc789224d0c7'}, '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.77%', '% of reads mapped to too many loci': '0.01%', '% of reads unmapped: other': '0.24%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.49%', 'Average input read length': 300, 'Average mapped length': 297.33, 'Deletion average length': 1.85, 'Deletion rate per base': '0.01%', 'Insertion average length': 1.84, 'Insertion rate per base': '0.01%', 'Mapping speed, Million of reads per hour': 141.52, 'Mismatch rate per base, %': '0.37%', 'Number of chimeric reads': 0, 'Number of input reads': 19734786, 'Number of reads mapped to multiple loci': 1137717, 'Number of reads mapped to too many loci': 2440, 'Number of splices: AT/AC': 17843, 'Number of splices: Annotated (sjdb)': 19400675, 'Number of splices: GC/AG': 138522, 'Number of splices: GT/AG': 19465736, 'Number of splices: Non-canonical': 9987, 'Number of splices: Total': 19632088, 'Uniquely mapped reads %': '87.50%', 'Uniquely mapped reads number': 17267272, '@id': '/star-quality-metrics/21777661-4961-4432-8967-cf7cfad2cb72/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '21777661-4961-4432-8967-cf7cfad2cb72', 'assay_term_id': 'OBI:0001271', 'read_depth': 18404989}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:06.792435+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', 'quality_metric_of': ['/files/ENCFF932BAE/'], 'processing_stage': 'unfiltered', 'total_reads': 82707110, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 81792451, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.9, 'paired_reads': 82707110, 'paired_reads_qc_failed': 0, 'read1': 41353555, 'read1_qc_failed': 0, 'read2': 41353555, 'read2_qc_failed': 0, 'properly_paired_reads': 80162702, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.89999999999999, 'with_itself': 81166710, 'with_itself_qc_failed': 0, 'singletons': 625741, 'singletons_qc_failed': 0, 'pct_singletons': 0.8, 'diff_chroms': 216799, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/70dfef4d-423d-483a-98af-4c7c154cbb71/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '70dfef4d-423d-483a-98af-4c7c154cbb71', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:50.651230+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b/', 'quality_metric_of': ['/files/ENCFF917WZE/'], 'frip': 0.01081428407129738, '@id': '/chip-peak-enrichment-quality-metrics/1bf2c3df-f691-4ed1-a0c6-3daade6041db/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1bf2c3df-f691-4ed1-a0c6-3daade6041db', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:50.439273+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b/', 'quality_metric_of': ['/files/ENCFF917WZE/'], 'reproducible_peaks': 983, 'idr_cutoff': 0.01, 'idr_dispersion_plot': {'download': 'pooled-pr1_vs_pooled-pr2.idr0.01.unthresholded-peaks.txt.png', 'href': '@@download/idr_dispersion_plot/pooled-pr1_vs_pooled-pr2.idr0.01.unthresholded-peaks.txt.png', 'type': 'image/png', 'md5sum': '2d987394d88144b470af7f3c2a924d00', 'width': 1200, 'height': 1200}, 'idr_parameters': {'download': 'pooled-pr1_vs_pooled-pr2.idr0.01.log.txt', 'href': '@@download/idr_parameters/pooled-pr1_vs_pooled-pr2.idr0.01.log.txt', 'type': 'text/plain', 'md5sum': 'e7befad95d58e3b37d7297a2a27cd940'}, '@id': '/chip-replication-quality-metrics/d46a69ea-3699-4f13-b953-a453f6a46877/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd46a69ea-3699-4f13-b953-a453f6a46877', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:07.320236+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/1e93b456-4047-4a11-9c1b-5407955b6328/', 'quality_metric_of': ['/files/ENCFF253CPS/'], 'reproducible_peaks': 307988, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/bf829c00-d694-44d9-9df9-7119474859f3/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bf829c00-d694-44d9-9df9-7119474859f3', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:05:07.424876+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/1e93b456-4047-4a11-9c1b-5407955b6328/', 'quality_metric_of': ['/files/ENCFF253CPS/'], 'frip': 0.7300927145319361, 'min_size': 125, '25_pct': 243, '50_pct': 378, '75_pct': 648, 'max_size': 19756, 'mean': 577.4617550034417, '@id': '/chip-peak-enrichment-quality-metrics/9617c610-e96e-40c9-a10b-81274a577d18/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9617c610-e96e-40c9-a10b-81274a577d18', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.624226+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', 'quality_metric_of': ['/files/ENCFF765RCK/'], 'reproducible_peaks': 452, 'idr_cutoff': 0.01, 'idr_dispersion_plot': {'download': 'rep1-pr1_vs_rep1-pr2.idr0.01.unthresholded-peaks.txt.png', 'href': '@@download/idr_dispersion_plot/rep1-pr1_vs_rep1-pr2.idr0.01.unthresholded-peaks.txt.png', 'type': 'image/png', 'md5sum': '3ca7d2bbf4b1f61f50fa813a9dbfc749', 'width': 1200, 'height': 1200}, 'idr_parameters': {'download': 'rep1-pr1_vs_rep1-pr2.idr0.01.log.txt', 'href': '@@download/idr_parameters/rep1-pr1_vs_rep1-pr2.idr0.01.log.txt', 'type': 'text/plain', 'md5sum': '4a12aec7074e7008877feb466058a216'}, '@id': '/chip-replication-quality-metrics/b6e1f170-4577-473d-96c6-7a650aaebc23/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b6e1f170-4577-473d-96c6-7a650aaebc23', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.826464+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', 'quality_metric_of': ['/files/ENCFF765RCK/'], 'frip': 0.0064059911428217394, '@id': '/chip-peak-enrichment-quality-metrics/ca4937fd-a38c-4b51-a788-58cdced0bea3/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ca4937fd-a38c-4b51-a788-58cdced0bea3', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:32:21.721658+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF192SYW/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.07%', '% of reads mapped to too many loci': '0.36%', '% of reads unmapped: other': '1.30%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '10.19%', 'Average input read length': 101.0, 'Average mapped length': 98.44, 'Deletion average length': 2.21, 'Deletion rate per base': '0.07', 'Insertion average length': 1.95, 'Insertion rate per base': '0.06', 'Mapping speed, Million of reads per hour': 235.54, 'Mismatch rate per base, %': '0.66%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2813450.0, 'Number of reads mapped to multiple loci': 142510.0, 'Number of reads mapped to too many loci': 10173.0, 'Number of splices: AT/AC': 114.0, 'Number of splices: Annotated (sjdb)': 76122.0, 'Number of splices: GC/AG': 834.0, 'Number of splices: GT/AG': 77393.0, 'Number of splices: Non-canonical': 39.0, 'Number of splices: Total': 78380.0, 'Uniquely mapped reads %': '83.08', 'Uniquely mapped reads number': 2337325.0, '@id': '/star-quality-metrics/d59e8a37-4c06-4d84-aa92-1089486194fe/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd59e8a37-4c06-4d84-aa92-1089486194fe', 'read_depth': 2479835.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:32:21.766312+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF192SYW/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 108, 'fraction_of_unique_reads_in_cells': 0.610011, 'mean_genefull_ex50pas_per_cell': 1388, 'mean_reads_per_cell': 11554, 'mean_UMI_per_cell': 3953, 'median_genefull_ex50pas_per_cell': 903, 'median_reads_per_cell': 4495, 'median_UMI_per_cell': 1529, 'number_of_reads': 2813450, 'q30_bases_in_CB_UMI': 0.915077, 'q30_bases_in_rna_read': 0.934384, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.727075, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.746467, 'reads_mapped_to_genome_unique': 0.830768, 'reads_mapped_to_genome_unique_and_multiple': 0.881421, 'reads_with_valid_barcodes': 0.981937, 'sequencing_saturation': 0.517239, 'total_genefull_ex50pas_detected': 15382, 'umis_in_cells': 426927, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1247832, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '487823dfd9040a88ab4e02e4614b947a', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/a1194fc8-1ae8-459f-8093-58f13b4f325e/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a1194fc8-1ae8-459f-8093-58f13b4f325e', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:27:06.434011+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF936GHR/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '8.01%', '% of reads mapped to too many loci': '0.42%', '% of reads unmapped: other': '1.29%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '14.07%', 'Average input read length': 99.0, 'Average mapped length': 94.87, 'Deletion average length': 2.27, 'Deletion rate per base': '0.08', 'Insertion average length': 1.99, 'Insertion rate per base': '0.06', 'Mapping speed, Million of reads per hour': 222.69, 'Mismatch rate per base, %': '0.70%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1422769.0, 'Number of reads mapped to multiple loci': 113987.0, 'Number of reads mapped to too many loci': 5924.0, 'Number of splices: AT/AC': 53.0, 'Number of splices: Annotated (sjdb)': 46229.0, 'Number of splices: GC/AG': 471.0, 'Number of splices: GT/AG': 46777.0, 'Number of splices: Non-canonical': 45.0, 'Number of splices: Total': 47346.0, 'Uniquely mapped reads %': '76.21', 'Uniquely mapped reads number': 1084301.0, '@id': '/star-quality-metrics/b379766d-6c3b-48de-ac62-0303a4cebf6d/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b379766d-6c3b-48de-ac62-0303a4cebf6d', 'read_depth': 1198288.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T04:27:06.470009+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF936GHR/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 208, 'fraction_of_unique_reads_in_cells': 0.560442, 'mean_genefull_ex50pas_per_cell': 478, 'mean_reads_per_cell': 2585, 'mean_UMI_per_cell': 884, 'median_genefull_ex50pas_per_cell': 330, 'median_reads_per_cell': 1547, 'median_UMI_per_cell': 572, 'number_of_reads': 1422769, 'q30_bases_in_CB_UMI': 0.911004, 'q30_bases_in_rna_read': 0.930745, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.674514, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.696214, 'reads_mapped_to_genome_unique': 0.762106, 'reads_mapped_to_genome_unique_and_multiple': 0.842222, 'reads_with_valid_barcodes': 0.979776, 'sequencing_saturation': 0.514912, 'total_genefull_ex50pas_detected': 12462, 'umis_in_cells': 183975, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 537843, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '93ade9b934917b0b6f1119b1275c43c6', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/84671f21-bfb1-428a-8532-ef199a5d84e5/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '84671f21-bfb1-428a-8532-ef199a5d84e5', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep2_novaseq_hic_chr_interactions_quality_metric'], 'date_created': '2022-02-03T05:39:43.492208+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', 'quality_metric_of': ['/files/ENCFF936BJV/'], 'intra_inter_pet_ratio': 4.6, 'intra_inter_clust_pet2_ratio': 10.05, 'intra_inter_clust_pet5_ratio': 82.71, 'intra_inter_clust_pet10_ratio': 1457.24, '@id': '/chia-pet-chr-interactions-quality-metrics/bc7b01c0-ae88-4ee8-9891-54b0601267f5/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bc7b01c0-ae88-4ee8-9891-54b0601267f5'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:50.790481+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/36e582a0-9b8e-4a3d-a109-799d59a5d72a/', 'quality_metric_of': ['/files/ENCFF295BTK/'], 'reproducible_peaks': 1009, 'idr_cutoff': 0.01, 'rescue_ratio': 1.0264496439471007, 'self_consistency_ratio': 1.011061946902655, 'reproducibility': 'pass', 'idr_dispersion_plot': {'download': 'rep1_vs_rep2.idr0.01.unthresholded-peaks.txt.png', 'href': '@@download/idr_dispersion_plot/rep1_vs_rep2.idr0.01.unthresholded-peaks.txt.png', 'type': 'image/png', 'md5sum': 'e67d0cebed285b118583cf81aea893b7', 'width': 1200, 'height': 1200}, 'idr_parameters': {'download': 'rep1_vs_rep2.idr0.01.log.txt', 'href': '@@download/idr_parameters/rep1_vs_rep2.idr0.01.log.txt', 'type': 'text/plain', 'md5sum': '5e6e5c6a30a16408d02e0aef21a0b5e1'}, '@id': '/chip-replication-quality-metrics/6cfbb778-c1ac-4493-8fa9-7db25e746f85/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6cfbb778-c1ac-4493-8fa9-7db25e746f85', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:51.070939+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/36e582a0-9b8e-4a3d-a109-799d59a5d72a/', 'quality_metric_of': ['/files/ENCFF295BTK/'], 'frip': 0.010963674765114527, 'min_size': 65, '25_pct': 260, '50_pct': 260, '75_pct': 260, 'max_size': 469, 'mean': 254.25966303270565, '@id': '/chip-peak-enrichment-quality-metrics/494774f3-728b-437a-ab27-51bd4072e06c/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '494774f3-728b-437a-ab27-51bd4072e06c', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep2_novaseq_peak_enrichment_quality_metric'], 'date_created': '2022-02-03T05:37:24.569746+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', 'quality_metric_of': ['/files/ENCFF500RYH/'], 'binding_peaks': 116457, '@id': '/chia-pet-peak-enrichment-quality-metrics/86632249-8b82-423c-aecc-f1dbac926eeb/', '@type': ['ChiaPetPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '86632249-8b82-423c-aecc-f1dbac926eeb'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep2_novaseq_loop_chr_interactions_quality_metric'], 'date_created': '2022-02-03T05:39:44.228099+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', 'quality_metric_of': ['/files/ENCFF801VZT/'], 'intra_inter_pet_ratio': 4.6, 'intra_inter_clust_pet2_ratio': 10.05, 'intra_inter_clust_pet5_ratio': 82.71, 'intra_inter_clust_pet10_ratio': 1457.24, '@id': '/chia-pet-chr-interactions-quality-metrics/aa283a72-ca84-44b8-9366-9f7f5c17afce/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'aa283a72-ca84-44b8-9366-9f7f5c17afce'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-08-12T16:25:48.248393+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/285487eb-3941-40ea-b52c-38e0087a09b7/', 'quality_metric_of': ['/files/ENCFF028PJB/'], 'processing_stage': 'filtered', 'total_reads': 18901841, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 18901841, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 0, 'paired_reads_qc_failed': 0, 'read1': 0, 'read1_qc_failed': 0, 'read2': 0, 'read2_qc_failed': 0, 'properly_paired_reads': 0, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 0.0, 'with_itself': 0, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/3540a84e-b64c-40f9-bddd-66568fde7e17/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3540a84e-b64c-40f9-bddd-66568fde7e17', 'assay_term_id': 'OBI:0000716', 'usable_fragments': 18901841}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-08-12T16:25:48.344457+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/285487eb-3941-40ea-b52c-38e0087a09b7/', 'quality_metric_of': ['/files/ENCFF028PJB/'], 'unpaired_reads': 21879189, 'paired_reads': 0, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 2977348, 'paired_duplicate_reads': 0, 'paired_optical_duplicate_reads': 0, 'pct_duplicate_reads': 13.6081, 'total_fragments': 21437651, 'distinct_fragments': 18884442, 'positions_with_one_read': 2021162, 'NRF': 0.880901, 'PBC1': 0.879669, 'PBC2': 8.219059, '@id': '/chip-library-quality-metrics/a5091c2f-8959-4ae1-b55f-595f60816703/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a5091c2f-8959-4ae1-b55f-595f60816703', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-08-12T16:25:48.143134+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/285487eb-3941-40ea-b52c-38e0087a09b7/', 'quality_metric_of': ['/files/ENCFF539QXE/'], 'processing_stage': 'unfiltered', 'total_reads': 36863860, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 29536323, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 80.10000000000001, 'paired_reads': 0, 'paired_reads_qc_failed': 0, 'read1': 0, 'read1_qc_failed': 0, 'read2': 0, 'read2_qc_failed': 0, 'properly_paired_reads': 0, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 0.0, 'with_itself': 0, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/ec9e4c3b-1da5-4692-b184-8de451c651b7/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ec9e4c3b-1da5-4692-b184-8de451c651b7', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:48.650029+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF331YPK/'], 'subsampled_reads': 6350422, 'estimated_fragment_len': 160, 'corr_estimated_fragment_len': 0.655615586800617, 'phantom_peak': 50, 'corr_phantom_peak': 0.6553722, 'argmin_corr': 1500, 'min_corr': 0.6502919, 'NSC': 1.008187, 'RSC': 1.047907, 'cross_correlation_plot': {'download': 'MXL2IP1_336_241_S10_L001_R1_001.trim_50bp.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/MXL2IP1_336_241_S10_L001_R1_001.trim_50bp.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'da58a6b20c6a8722621477126b8e23be'}, 'auc': 0.4099169117710638, 'syn_auc': 0.49811097966665074, 'x_intercept': 0.019585970737781466, 'syn_x_intercept': 0.0, 'elbow_pt': 0.512725393315826, 'syn_elbow_pt': 0.5025240959499493, 'jsd': 0.048111350702257034, 'syn_jsd': 0.13660676432887348, 'pct_genome_enrich': 44.24932751175358, 'diff_enrich': 9.028530281825574, 'ch_div': 0.07678186188705499, 'jsd_plot': {'download': 'MXL2IP1_336_241_S10_L001_R1_001.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/MXL2IP1_336_241_S10_L001_R1_001.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '28abe59aece7d32fc2175dc6b8c1946b', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'MXL2IP1_336_241_S10_L001_R1_001.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/MXL2IP1_336_241_S10_L001_R1_001.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '16290c4bd111cdb546253a0c2b09d327', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/76bab10d-d452-4ccf-97f5-b2e61dbf2d61/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '76bab10d-d452-4ccf-97f5-b2e61dbf2d61', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:48.969286+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF331YPK/'], 'unpaired_reads': 0, 'paired_reads': 6777410, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 1015304, 'paired_optical_duplicate_reads': 55852, 'pct_duplicate_reads': 14.980699999999999, 'total_fragments': 6737746, 'distinct_fragments': 5731667, 'positions_with_one_read': 772576, 'NRF': 0.85068, 'PBC1': 0.84614, 'PBC2': 6.277434, '@id': '/chip-library-quality-metrics/f8d91577-c2c8-41f4-9a19-92d69524a95e/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f8d91577-c2c8-41f4-9a19-92d69524a95e', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:48.531887+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF331YPK/'], 'processing_stage': 'filtered', 'total_reads': 11524212, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 11524212, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 11524212, 'paired_reads_qc_failed': 0, 'read1': 5762106, 'read1_qc_failed': 0, 'read2': 5762106, 'read2_qc_failed': 0, 'properly_paired_reads': 11524212, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 11524212, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/c073dc9f-dd9f-402e-9682-60c427517b0c/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c073dc9f-dd9f-402e-9682-60c427517b0c', 'assay_term_id': 'OBI:0000716', 'usable_fragments': 5762106}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:f4fdfed8-aa62-4e59-8c94-b51beb197f4d_replacement'], 'date_created': '2022-06-30T18:48:58.517823+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/15607b8b-0c8b-4a62-bd7f-cb14add2813b/', 'quality_metric_of': ['/files/ENCFF660JWA/'], 'attachment': {'download': 'f4fdfed8-aa62-4e59-8c94-b51beb197f4d.txt', 'href': '@@download/attachment/f4fdfed8-aa62-4e59-8c94-b51beb197f4d.txt', 'type': 'text/plain', 'md5sum': '502d44f5688372f7cf0283a2357fec89'}, 'run_type': 'single-ended', 'sequenced_reads': 3226136971, 'no_chimera_found': 772387007, '0_alignments': 6340553, '1_alignment': 766046454, '1_alignment_unique': 647313858, '1_alignment_duplicates': 118732596, '2_alignments': 2270768487, '2_alignment_duplicates': 332166120, '2_alignment_unique': 1938602367, '3_or_more_alignments': 182981477, 'pct_no_chimera_found': 23.94, 'pct_0_alignments': 0.2, 'pct_1_alignment': 23.75, 'pct_1_alignment_unique': 84.5, 'pct_1_alignment_duplicates': 15.5, 'pct_sequenced_1_alignment_unique': 20.06, 'pct_sequenced_1_alignment_duplicates': 3.68, 'pct_2_alignments': 70.39, 'pct_2_alignment_unique': 85.37, 'pct_2_alignment_duplicates': 14.63, 'pct_sequenced_2_alignment_unique': 60.09, 'pct_sequenced_2_alignment_duplicates': 10.3, 'pct_3_or_more_alignments': 5.67, 'pct_unique_total_duplicates': 14.85, 'pct_unique_total_unique': 85.15, 'total_unique': 2585916225, 'pct_sequenced_total_unique': 80.16, 'total_duplicates': 450898716, 'pct_sequenced_total_duplicates': 13.98, 'library_complexity_estimate_1_alignment': 2208659399, 'library_complexity_estimate_2_alignments': 6984789489, 'library_complexity_estimate_1_and_2_alignments': 9193448888, 'below_mapq_threshold': 590893447, 'pct_sequenced_below_mapq_threshold': 18.32, 'pct_unique_below_mapq_threshold': 30.48, 'hic_contacts': 1347708920, 'pct_sequenced_hic_contacts': 41.77, 'pct_unique_hic_contacts': 69.52, 'lior_convergence': 1874, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 889926362, 'pct_sequenced_inter_chromosomal': 27.58, 'pct_unique_inter_chromosomal': 45.91, 'intra_chromosomal': 457782558, 'pct_sequenced_intra_chromosomal': 14.19, 'pct_unique_intra_chromosomal': 23.61, 'short_range_less_than_500bp': 21399325, 'pct_sequenced_short_range_less_than_500bp': 0.66, 'pct_unique_short_range_less_than_500bp': 1.1, 'short_range_500bp_to_5kb': 35820659, 'pct_sequenced_short_range_500bp_to_5kb': 1.11, 'pct_unique_short_range_500bp_to_5kb': 1.85, 'short_range_5kb_to_20kb': 17614065, 'pct_sequenced_short_range_5kb_to_20kb': 0.55, 'pct_unique_short_range_5kb_to_20kb': 0.91, 'long_range_greater_than_20kb': 382948509, 'pct_sequenced_long_range_greater_than_20kb': 11.87, 'pct_unique_long_range_greater_than_20kb': 19.75, '@id': '/hic-quality-metrics/fcb44a98-495b-4f22-83fa-8446c6a87e27/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'fcb44a98-495b-4f22-83fa-8446c6a87e27', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:48.461366+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF374AUG/'], 'processing_stage': 'unfiltered', 'total_reads': 15190250, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 14916534, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.2, 'paired_reads': 15190250, 'paired_reads_qc_failed': 0, 'read1': 7595125, 'read1_qc_failed': 0, 'read2': 7595125, 'read2_qc_failed': 0, 'properly_paired_reads': 14854252, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.8, 'with_itself': 14906710, 'with_itself_qc_failed': 0, 'singletons': 9824, 'singletons_qc_failed': 0, 'pct_singletons': 0.1, 'diff_chroms': 4162, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/871517ea-2c32-45aa-b70c-b1870e50cd26/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '871517ea-2c32-45aa-b70c-b1870e50cd26', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep2_novaseq_alignment_quality_metric'], 'date_created': '2022-02-03T05:35:14.136848+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', 'quality_metric_of': ['/files/ENCFF429LRL/'], 'total_rp': 439238920, 'rp_bl': 411662669, 'frp_bl': 0.94, 'pet': 262789004, 'um_pet': 209479719, 'nr_pet': 61709758, 'pet_red': 0.71, '@id': '/chia-pet-alignment-quality-metrics/3211a5e7-5eda-45c2-9803-3d40be13d24b/', '@type': ['ChiaPetAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3211a5e7-5eda-45c2-9803-3d40be13d24b'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:48.393132+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF082FDT/'], 'processing_stage': 'unfiltered', 'total_reads': 15348742, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 14854866, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 96.8, 'paired_reads': 15348742, 'paired_reads_qc_failed': 0, 'read1': 7674371, 'read1_qc_failed': 0, 'read2': 7674371, 'read2_qc_failed': 0, 'properly_paired_reads': 14796244, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.39999999999999, 'with_itself': 14844052, 'with_itself_qc_failed': 0, 'singletons': 10814, 'singletons_qc_failed': 0, 'pct_singletons': 0.1, 'diff_chroms': 3884, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/d4273d27-0c6d-4852-8733-52255a189344/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd4273d27-0c6d-4852-8733-52255a189344', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:27.415839+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/e2ea0042-83a3-4215-8c54-b13e67f25cf6/', 'quality_metric_of': ['/files/ENCFF887EQC/'], 'frip': 0.6198621265855868, 'min_size': 125, '25_pct': 176, '50_pct': 266, '75_pct': 473, 'max_size': 23847, 'mean': 492.10490511418465, '@id': '/chip-peak-enrichment-quality-metrics/3b8eeac7-8df6-474f-8a04-5149d594b35a/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3b8eeac7-8df6-474f-8a04-5149d594b35a', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:27.331600+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/e2ea0042-83a3-4215-8c54-b13e67f25cf6/', 'quality_metric_of': ['/files/ENCFF887EQC/'], 'reproducible_peaks': 186540, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/64e0038e-5c79-44c1-a50a-74d72264ec50/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '64e0038e-5c79-44c1-a50a-74d72264ec50', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.038276+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF473OOA/'], 'processing_stage': 'filtered', 'total_reads': 11569596, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 11569596, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 11569596, 'paired_reads_qc_failed': 0, 'read1': 5784798, 'read1_qc_failed': 0, 'read2': 5784798, 'read2_qc_failed': 0, 'properly_paired_reads': 11569596, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 11569596, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/1a75ea72-8967-4567-abd0-ab9e722ff482/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1a75ea72-8967-4567-abd0-ab9e722ff482', 'assay_term_id': 'OBI:0000716', 'usable_fragments': 5784798}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.513291+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF473OOA/'], 'unpaired_reads': 0, 'paired_reads': 6803386, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 1018588, 'paired_optical_duplicate_reads': 55727, 'pct_duplicate_reads': 14.971799999999998, 'total_fragments': 6757842, 'distinct_fragments': 5750946, 'positions_with_one_read': 777135, 'NRF': 0.851003, 'PBC1': 0.846139, 'PBC2': 6.261589, '@id': '/chip-library-quality-metrics/7fb8969d-2307-40e6-bd0c-edad5b365c54/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7fb8969d-2307-40e6-bd0c-edad5b365c54', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.140395+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', 'quality_metric_of': ['/files/ENCFF473OOA/'], 'subsampled_reads': 6379097, 'estimated_fragment_len': 160, 'corr_estimated_fragment_len': 0.657039061948679, 'phantom_peak': 50, 'corr_phantom_peak': 0.656548, 'argmin_corr': 1500, 'min_corr': 0.6514796, 'NSC': 1.008534, 'RSC': 1.096878, 'cross_correlation_plot': {'download': 'MXL2IP2_324_253_S11_L001_R1_001.trim_50bp.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/MXL2IP2_324_253_S11_L001_R1_001.trim_50bp.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '555d74c35bde2168299f7ecdd4d82813'}, 'auc': 0.4093516856181763, 'syn_auc': 0.49811635803444926, 'x_intercept': 0.01965980475890199, 'syn_x_intercept': 0.0, 'elbow_pt': 0.5156847407028999, 'syn_elbow_pt': 0.49890585339861915, 'jsd': 0.04658524126380156, 'syn_jsd': 0.1382923076193169, 'pct_genome_enrich': 44.39779375962836, 'diff_enrich': 9.060133429530188, 'ch_div': 0.07706239786671744, 'jsd_plot': {'download': 'MXL2IP1_336_241_S10_L001_R1_001.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/MXL2IP1_336_241_S10_L001_R1_001.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '28abe59aece7d32fc2175dc6b8c1946b', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'MXL2IP2_324_253_S11_L001_R1_001.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/MXL2IP2_324_253_S11_L001_R1_001.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '9316bdd323b55a0130d78d77457185ce', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/ca2a157f-c956-4ba4-afc6-68947d47744a/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ca2a157f-c956-4ba4-afc6-68947d47744a', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:49.980612+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', 'quality_metric_of': ['/files/ENCFF784QXW/'], 'reproducible_peaks': 457, 'idr_cutoff': 0.01, 'idr_dispersion_plot': {'download': 'rep2-pr1_vs_rep2-pr2.idr0.01.unthresholded-peaks.txt.png', 'href': '@@download/idr_dispersion_plot/rep2-pr1_vs_rep2-pr2.idr0.01.unthresholded-peaks.txt.png', 'type': 'image/png', 'md5sum': '05cf5a083d973c197135453957ca636e', 'width': 1200, 'height': 1200}, 'idr_parameters': {'download': 'rep2-pr1_vs_rep2-pr2.idr0.01.log.txt', 'href': '@@download/idr_parameters/rep2-pr1_vs_rep2-pr2.idr0.01.log.txt', 'type': 'text/plain', 'md5sum': 'e09b799103923ea02df55f5a7d0bfb07'}, '@id': '/chip-replication-quality-metrics/5fafc88a-8540-414f-a907-8943226e87e7/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5fafc88a-8540-414f-a907-8943226e87e7', 'assay_term_id': 'OBI:0000716'}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-06-02T13:42:50.313624+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', 'quality_metric_of': ['/files/ENCFF784QXW/'], 'frip': 0.006608182342754233, '@id': '/chip-peak-enrichment-quality-metrics/dca311d0-d675-468b-a9e1-9dab8040f172/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'dca311d0-d675-468b-a9e1-9dab8040f172', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:26.829325+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', 'quality_metric_of': ['/files/ENCFF838HLU/'], 'processing_stage': 'unfiltered', 'total_reads': 91633584, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 88455223, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 96.5, 'paired_reads': 91633584, 'paired_reads_qc_failed': 0, 'read1': 45816792, 'read1_qc_failed': 0, 'read2': 45816792, 'read2_qc_failed': 0, 'properly_paired_reads': 84917058, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 92.7, 'with_itself': 85472670, 'with_itself_qc_failed': 0, 'singletons': 2982553, 'singletons_qc_failed': 0, 'pct_singletons': 3.3000000000000003, 'diff_chroms': 177361, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/254568af-479c-48e0-adf5-d5edd921d1cb/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '254568af-479c-48e0-adf5-d5edd921d1cb', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep1_novaseq_hic_chr_interactions_quality_metric'], 'date_created': '2022-02-03T05:39:43.150581+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', 'quality_metric_of': ['/files/ENCFF693HYY/'], 'intra_inter_pet_ratio': 3.53, 'intra_inter_clust_pet2_ratio': 5.68, 'intra_inter_clust_pet5_ratio': 23.33, 'intra_inter_clust_pet10_ratio': 201.47, '@id': '/chia-pet-chr-interactions-quality-metrics/0a568610-52da-442d-8ac7-3ebe96365493/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0a568610-52da-442d-8ac7-3ebe96365493'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:0d2e8487-2ee5-46a1-a742-f2d3ef0addc6_replacement'], 'date_created': '2022-06-30T18:48:06.142690+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/29f56206-d24c-4f6c-b0d0-d30fa6def06f/', 'quality_metric_of': ['/files/ENCFF932ATT/'], 'attachment': {'download': '0d2e8487-2ee5-46a1-a742-f2d3ef0addc6.txt', 'href': '@@download/attachment/0d2e8487-2ee5-46a1-a742-f2d3ef0addc6.txt', 'type': 'text/plain', 'md5sum': '167a693bdaa44a73f09d97eeaed1b9b5'}, 'run_type': 'single-ended', 'sequenced_reads': 2486221790, 'no_chimera_found': 931679400, '0_alignments': 24901486, '1_alignment': 906777914, '1_alignment_unique': 812319119, '1_alignment_duplicates': 94458795, '2_alignments': 1442576704, '2_alignment_duplicates': 72863045, '2_alignment_unique': 1369713659, '3_or_more_alignments': 111965686, 'pct_no_chimera_found': 37.47, 'pct_0_alignments': 1.0, 'pct_1_alignment': 36.47, 'pct_1_alignment_unique': 89.58, 'pct_1_alignment_duplicates': 10.42, 'pct_sequenced_1_alignment_unique': 32.67, 'pct_sequenced_1_alignment_duplicates': 3.8, 'pct_2_alignments': 58.02, 'pct_2_alignment_unique': 94.95, 'pct_2_alignment_duplicates': 5.05, 'pct_sequenced_2_alignment_unique': 55.09, 'pct_sequenced_2_alignment_duplicates': 2.93, 'pct_3_or_more_alignments': 4.5, 'pct_unique_total_duplicates': 7.12, 'pct_unique_total_unique': 92.88, 'total_unique': 2182032778, 'pct_sequenced_total_unique': 87.77, 'total_duplicates': 167321840, 'pct_sequenced_total_duplicates': 6.73, 'library_complexity_estimate_1_alignment': 4044586199, 'library_complexity_estimate_2_alignments': 13795385037, 'library_complexity_estimate_1_and_2_alignments': 17839971236, 'below_mapq_threshold': 247310385, 'pct_sequenced_below_mapq_threshold': 9.95, 'pct_unique_below_mapq_threshold': 18.06, 'hic_contacts': 1122403274, 'pct_sequenced_hic_contacts': 45.14, 'pct_unique_hic_contacts': 81.94, 'lior_convergence': 285, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 552509089, 'pct_sequenced_inter_chromosomal': 22.22, 'pct_unique_inter_chromosomal': 40.34, 'intra_chromosomal': 569894185, 'pct_sequenced_intra_chromosomal': 22.92, 'pct_unique_intra_chromosomal': 41.61, 'short_range_less_than_500bp': 29363999, 'pct_sequenced_short_range_less_than_500bp': 1.18, 'pct_unique_short_range_less_than_500bp': 2.14, 'short_range_500bp_to_5kb': 91891271, 'pct_sequenced_short_range_500bp_to_5kb': 3.7, 'pct_unique_short_range_500bp_to_5kb': 6.71, 'short_range_5kb_to_20kb': 42102677, 'pct_sequenced_short_range_5kb_to_20kb': 1.69, 'pct_unique_short_range_5kb_to_20kb': 3.07, 'long_range_greater_than_20kb': 406536238, 'pct_sequenced_long_range_greater_than_20kb': 16.35, 'pct_unique_long_range_greater_than_20kb': 29.68, '@id': '/hic-quality-metrics/0195f518-bd05-4bde-8639-e64a135f6be8/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0195f518-bd05-4bde-8639-e64a135f6be8', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:27.045454+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', 'quality_metric_of': ['/files/ENCFF721YSY/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 125, 'corr_estimated_fragment_len': 0.094779861405117, 'phantom_peak': 40, 'corr_phantom_peak': 0.09683634, 'argmin_corr': 1500, 'min_corr': 0.05737649, 'NSC': 1.651894, 'cross_correlation_plot': {'download': 'ENCFF228WSB.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF228WSB.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'a1d0faea6caf99b66eabe88905a67231'}, 'auc': 0.058366869014596985, 'syn_auc': 0.4937159381865422, 'x_intercept': 0.5336576237231816, 'syn_x_intercept': 5.219840498317384e-217, 'elbow_pt': 0.8337521917591854, 'syn_elbow_pt': 0.5076949212985095, 'jsd': 0.4056370085639425, 'syn_jsd': 0.59634651963509, 'pct_genome_enrich': 13.44525550363977, 'diff_enrich': 64.85325564302943, 'ch_div': 0.5743567254545193, 'jsd_plot': {'download': 'ENCFF228WSB.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF228WSB.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '6ce9a2cb1690402514e6ed9ec5038178', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF228WSB.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF228WSB.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '0d68a2ed3b111b1fc0dc740535ae15e4', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/1ed42b00-a622-4293-a443-c01524da659b/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1ed42b00-a622-4293-a443-c01524da659b', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:27.244848+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', 'quality_metric_of': ['/files/ENCFF721YSY/'], 'unpaired_reads': 0, 'paired_reads': 40209210, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 14344180, 'paired_optical_duplicate_reads': 15315, 'pct_duplicate_reads': 35.673899999999996, 'total_fragments': 40181896, 'distinct_fragments': 30043495, 'positions_with_one_read': 3440646, 'NRF': 0.747687, 'PBC1': 0.823871, 'PBC2': 7.193987, '@id': '/chip-library-quality-metrics/c0286323-8b19-485f-add8-33d9595cead6/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c0286323-8b19-485f-add8-33d9595cead6', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:29:26.918487+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', 'quality_metric_of': ['/files/ENCFF721YSY/'], 'processing_stage': 'filtered', 'total_reads': 51730060, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 51730060, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 51730060, 'paired_reads_qc_failed': 0, 'read1': 25865030, 'read1_qc_failed': 0, 'read2': 25865030, 'read2_qc_failed': 0, 'properly_paired_reads': 51730060, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 51730060, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/7c401aa3-35ff-438d-8fd1-efc7ac611f05/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7c401aa3-35ff-438d-8fd1-efc7ac611f05', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 25865030}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep1_novaseq_peak_enrichment_quality_metric'], 'date_created': '2022-02-03T05:37:24.222564+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', 'quality_metric_of': ['/files/ENCFF502VLW/'], 'binding_peaks': 106620, '@id': '/chia-pet-peak-enrichment-quality-metrics/79501541-d7f2-4e0a-be6c-516c199ffdcb/', '@type': ['ChiaPetPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '79501541-d7f2-4e0a-be6c-516c199ffdcb'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:c6ecf776-6369-423e-8aa9-feaccbab42a4_replacement'], 'date_created': '2022-06-30T18:48:10.923932+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/4712d703-abf1-4e9d-b339-137fc09bbc45/', 'quality_metric_of': ['/files/ENCFF667OOG/'], 'attachment': {'download': 'c6ecf776-6369-423e-8aa9-feaccbab42a4.txt', 'href': '@@download/attachment/c6ecf776-6369-423e-8aa9-feaccbab42a4.txt', 'type': 'text/plain', 'md5sum': '167a693bdaa44a73f09d97eeaed1b9b5'}, 'run_type': 'single-ended', 'sequenced_reads': 2486221790, 'no_chimera_found': 931679400, '0_alignments': 24901486, '1_alignment': 906777914, '1_alignment_unique': 812319119, '1_alignment_duplicates': 94458795, '2_alignments': 1442576704, '2_alignment_duplicates': 72863045, '2_alignment_unique': 1369713659, '3_or_more_alignments': 111965686, 'pct_no_chimera_found': 37.47, 'pct_0_alignments': 1.0, 'pct_1_alignment': 36.47, 'pct_1_alignment_unique': 89.58, 'pct_1_alignment_duplicates': 10.42, 'pct_sequenced_1_alignment_unique': 32.67, 'pct_sequenced_1_alignment_duplicates': 3.8, 'pct_2_alignments': 58.02, 'pct_2_alignment_unique': 94.95, 'pct_2_alignment_duplicates': 5.05, 'pct_sequenced_2_alignment_unique': 55.09, 'pct_sequenced_2_alignment_duplicates': 2.93, 'pct_3_or_more_alignments': 4.5, 'pct_unique_total_duplicates': 7.12, 'pct_unique_total_unique': 92.88, 'total_unique': 2182032778, 'pct_sequenced_total_unique': 87.77, 'total_duplicates': 167321840, 'pct_sequenced_total_duplicates': 6.73, 'library_complexity_estimate_1_alignment': 4044586199, 'library_complexity_estimate_2_alignments': 13795385037, 'library_complexity_estimate_1_and_2_alignments': 17839971236, 'below_mapq_threshold': 247310385, 'pct_sequenced_below_mapq_threshold': 9.95, 'pct_unique_below_mapq_threshold': 18.06, 'hic_contacts': 1122403274, 'pct_sequenced_hic_contacts': 45.14, 'pct_unique_hic_contacts': 81.94, 'lior_convergence': 285, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 552509089, 'pct_sequenced_inter_chromosomal': 22.22, 'pct_unique_inter_chromosomal': 40.34, 'intra_chromosomal': 569894185, 'pct_sequenced_intra_chromosomal': 22.92, 'pct_unique_intra_chromosomal': 41.61, 'short_range_less_than_500bp': 29363999, 'pct_sequenced_short_range_less_than_500bp': 1.18, 'pct_unique_short_range_less_than_500bp': 2.14, 'short_range_500bp_to_5kb': 91891271, 'pct_sequenced_short_range_500bp_to_5kb': 3.7, 'pct_unique_short_range_500bp_to_5kb': 6.71, 'short_range_5kb_to_20kb': 42102677, 'pct_sequenced_short_range_5kb_to_20kb': 1.69, 'pct_unique_short_range_5kb_to_20kb': 3.07, 'long_range_greater_than_20kb': 406536238, 'pct_sequenced_long_range_greater_than_20kb': 16.35, 'pct_unique_long_range_greater_than_20kb': 29.68, '@id': '/hic-quality-metrics/1be3954d-91de-4001-a0f1-968b0e8d8c3a/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1be3954d-91de-4001-a0f1-968b0e8d8c3a', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep1_novaseq_loop_chr_interactions_quality_metric'], 'date_created': '2022-02-03T05:39:43.867641+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', 'quality_metric_of': ['/files/ENCFF316EBJ/'], 'intra_inter_pet_ratio': 3.53, 'intra_inter_clust_pet2_ratio': 5.68, 'intra_inter_clust_pet5_ratio': 23.33, 'intra_inter_clust_pet10_ratio': 201.47, '@id': '/chia-pet-chr-interactions-quality-metrics/1c9b6c07-5514-4b1c-8a31-e54b015638af/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1c9b6c07-5514-4b1c-8a31-e54b015638af'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:43.622982+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF820HVB/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '8.85%', '% of reads mapped to too many loci': '0.23%', '% of reads unmapped: other': '1.05%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '15.54%', 'Average input read length': 96.0, 'Average mapped length': 93.23, 'Deletion average length': 2.27, 'Deletion rate per base': '0.08', 'Insertion average length': 1.7, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 152.61, 'Mismatch rate per base, %': '0.63%', 'Number of chimeric reads': 0.0, 'Number of input reads': 466301.0, 'Number of reads mapped to multiple loci': 41288.0, 'Number of reads mapped to too many loci': 1080.0, 'Number of splices: AT/AC': 10.0, 'Number of splices: Annotated (sjdb)': 19601.0, 'Number of splices: GC/AG': 219.0, 'Number of splices: GT/AG': 19816.0, 'Number of splices: Non-canonical': 23.0, 'Number of splices: Total': 20068.0, 'Uniquely mapped reads %': '74.32', 'Uniquely mapped reads number': 346554.0, '@id': '/star-quality-metrics/2650a86e-b179-4033-96a1-9700bd30b02e/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2650a86e-b179-4033-96a1-9700bd30b02e', 'read_depth': 387842.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:43.659761+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF820HVB/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 436, 'fraction_of_unique_reads_in_cells': 0.700104, 'mean_genefull_ex50pas_per_cell': 95, 'mean_reads_per_cell': 459, 'mean_UMI_per_cell': 178, 'median_genefull_ex50pas_per_cell': 46, 'median_reads_per_cell': 100, 'median_UMI_per_cell': 65, 'number_of_reads': 466301, 'q30_bases_in_CB_UMI': 0.940626, 'q30_bases_in_rna_read': 0.936508, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.614344, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.637507, 'reads_mapped_to_genome_unique': 0.743198, 'reads_mapped_to_genome_unique_and_multiple': 0.831742, 'reads_with_valid_barcodes': 0.961336, 'sequencing_saturation': 0.537021, 'total_genefull_ex50pas_detected': 8685, 'umis_in_cells': 77938, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 200558, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'c039bdbb15f1f2498fc2590dbbec53bb', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/f8d40a8a-7fd5-4246-97d2-58fb265def2b/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f8d40a8a-7fd5-4246-97d2-58fb265def2b', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:44.399088+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF649ODJ/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'd07c9da67208782ab57cb4bd12b45cb1', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '108a955ad1bb388ca1ba8daae69ae3fc', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '28aceee832e98553728c5c2299a422bf', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/e93475f6-c36f-44d9-bdcd-605f19e65988/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e93475f6-c36f-44d9-bdcd-605f19e65988', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:c4c159c4-0131-476d-9e19-3bdcadb7c93f_replacement'], 'date_created': '2022-06-30T18:48:09.234780+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/29f56206-d24c-4f6c-b0d0-d30fa6def06f/', 'quality_metric_of': ['/files/ENCFF459FRB/'], 'attachment': {'download': 'c4c159c4-0131-476d-9e19-3bdcadb7c93f.txt', 'href': '@@download/attachment/c4c159c4-0131-476d-9e19-3bdcadb7c93f.txt', 'type': 'text/plain', 'md5sum': 'd285b9bac6a0b294303f57d9fd030566'}, 'run_type': 'single-ended', 'sequenced_reads': 2486221790, 'no_chimera_found': 931679400, '0_alignments': 24901486, '1_alignment': 906777914, '1_alignment_unique': 812319119, '1_alignment_duplicates': 94458795, '2_alignments': 1442576704, '2_alignment_duplicates': 72863045, '2_alignment_unique': 1369713659, '3_or_more_alignments': 111965686, 'pct_no_chimera_found': 37.47, 'pct_0_alignments': 1.0, 'pct_1_alignment': 36.47, 'pct_1_alignment_unique': 89.58, 'pct_1_alignment_duplicates': 10.42, 'pct_sequenced_1_alignment_unique': 32.67, 'pct_sequenced_1_alignment_duplicates': 3.8, 'pct_2_alignments': 58.02, 'pct_2_alignment_unique': 94.95, 'pct_2_alignment_duplicates': 5.05, 'pct_sequenced_2_alignment_unique': 55.09, 'pct_sequenced_2_alignment_duplicates': 2.93, 'pct_3_or_more_alignments': 4.5, 'pct_unique_total_duplicates': 7.12, 'pct_unique_total_unique': 92.88, 'total_unique': 2182032778, 'pct_sequenced_total_unique': 87.77, 'total_duplicates': 167321840, 'pct_sequenced_total_duplicates': 6.73, 'library_complexity_estimate_1_alignment': 4044586199, 'library_complexity_estimate_2_alignments': 13795385037, 'library_complexity_estimate_1_and_2_alignments': 17839971236, 'below_mapq_threshold': 368537925, 'pct_sequenced_below_mapq_threshold': 14.82, 'pct_unique_below_mapq_threshold': 26.91, 'hic_contacts': 1001175734, 'pct_sequenced_hic_contacts': 40.27, 'pct_unique_hic_contacts': 73.09, 'lior_convergence': 285, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 481833089, 'pct_sequenced_inter_chromosomal': 19.38, 'pct_unique_inter_chromosomal': 35.18, 'intra_chromosomal': 519342645, 'pct_sequenced_intra_chromosomal': 20.89, 'pct_unique_intra_chromosomal': 37.92, 'short_range_less_than_500bp': 27093650, 'pct_sequenced_short_range_less_than_500bp': 1.09, 'pct_unique_short_range_less_than_500bp': 1.98, 'short_range_500bp_to_5kb': 84415592, 'pct_sequenced_short_range_500bp_to_5kb': 3.4, 'pct_unique_short_range_500bp_to_5kb': 6.16, 'short_range_5kb_to_20kb': 38381384, 'pct_sequenced_short_range_5kb_to_20kb': 1.54, 'pct_unique_short_range_5kb_to_20kb': 2.8, 'long_range_greater_than_20kb': 369452019, 'pct_sequenced_long_range_greater_than_20kb': 14.86, 'pct_unique_long_range_greater_than_20kb': 26.97, '@id': '/hic-quality-metrics/da19254d-9aff-4383-aa31-4b40b5231ba3/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'da19254d-9aff-4383-aa31-4b40b5231ba3', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-CDK7-Treated_CTCF_rep1_novaseq_alignment_quality_metric'], 'date_created': '2022-02-03T05:35:13.788230+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', 'quality_metric_of': ['/files/ENCFF032BUO/'], 'total_rp': 486545274, 'rp_bl': 451944772, 'frp_bl': 0.93, 'pet': 297554989, 'um_pet': 240649547, 'nr_pet': 47406253, 'pet_red': 0.8, '@id': '/chia-pet-alignment-quality-metrics/07073313-0c0c-4169-a578-f326ed7e3f97/', '@type': ['ChiaPetAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '07073313-0c0c-4169-a578-f326ed7e3f97'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:11:56.552914+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF300DWP/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '0b6233c6fbf7f703e6588a1bc9d78dd8', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '795466198a0ad7e00c3fef12d1b9e2b9', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '296d6b81291eafcb3e7ebd7849fc893d', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/580698c4-32e3-451d-b762-cb639b82124e/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '580698c4-32e3-451d-b762-cb639b82124e', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:11:57.097636+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF475VQD/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 366, 'fraction_of_unique_reads_in_cells': 0.719582, 'mean_genefull_ex50pas_per_cell': 648, 'mean_reads_per_cell': 4467, 'mean_UMI_per_cell': 1591, 'median_genefull_ex50pas_per_cell': 425, 'median_reads_per_cell': 1250, 'median_UMI_per_cell': 715, 'number_of_reads': 3170996, 'q30_bases_in_CB_UMI': 0.936692, 'q30_bases_in_rna_read': 0.938806, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.716615, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.738273, 'reads_mapped_to_genome_unique': 0.83313, 'reads_mapped_to_genome_unique_and_multiple': 0.901951, 'reads_with_valid_barcodes': 0.96921, 'sequencing_saturation': 0.547881, 'total_genefull_ex50pas_detected': 15749, 'umis_in_cells': 582644, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1635167, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'fdd3cd4a715d4f14a5fa523c4dd636ad', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/8d9cf59a-78a4-484b-bd96-69201c4f1450/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8d9cf59a-78a4-484b-bd96-69201c4f1450', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:11:57.582707+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF475VQD/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '6.88%', '% of reads mapped to too many loci': '0.25%', '% of reads unmapped: other': '0.95%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '8.61%', 'Average input read length': 100.0, 'Average mapped length': 98.55, 'Deletion average length': 2.26, 'Deletion rate per base': '0.06', 'Insertion average length': 1.94, 'Insertion rate per base': '0.06', 'Mapping speed, Million of reads per hour': 300.41, 'Mismatch rate per base, %': '0.51%', 'Number of chimeric reads': 0.0, 'Number of input reads': 3170996.0, 'Number of reads mapped to multiple loci': 218232.0, 'Number of reads mapped to too many loci': 7858.0, 'Number of splices: AT/AC': 138.0, 'Number of splices: Annotated (sjdb)': 181077.0, 'Number of splices: GC/AG': 1908.0, 'Number of splices: GT/AG': 181883.0, 'Number of splices: Non-canonical': 110.0, 'Number of splices: Total': 184039.0, 'Uniquely mapped reads %': '83.31', 'Uniquely mapped reads number': 2641851.0, '@id': '/star-quality-metrics/330aa6f5-684e-49da-a02d-747a87a8b0f9/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '330aa6f5-684e-49da-a02d-747a87a8b0f9', 'read_depth': 2860083.0}]\",\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-09-26T20:45:59.461402+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/86cfd783-e120-4e9d-9004-9c5205b7de25/', 'quality_metric_of': ['/files/ENCFF687NPG/'], 'processing_stage': 'filtered', 'total_reads': 6142512, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 6142512, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 6142512, 'paired_reads_qc_failed': 0, 'read1': 3071256, 'read1_qc_failed': 0, 'read2': 3071256, 'read2_qc_failed': 0, 'properly_paired_reads': 6142512, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 6142512, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/6aa8769e-2357-40e2-8c00-d7fa81968913/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6aa8769e-2357-40e2-8c00-d7fa81968913', 'assay_term_id': 'OBI:0000716', 'usable_fragments': 3071256}, {'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-09-26T20:45:59.527293+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/86cfd783-e120-4e9d-9004-9c5205b7de25/', 'quality_metric_of': ['/files/ENCFF687NPG/'], 'unpaired_reads': 0, 'paired_reads': 3268066, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 196810, 'paired_optical_duplicate_reads': 31473, 'pct_duplicate_reads': 6.0222, 'total_fragments': 3220099, 'distinct_fragments': 3031599, 'positions_with_one_read': 162010, 'NRF': 0.941461, 'PBC1': 0.94403, 'PBC2': 17.665082, '@id': '/chip-library-quality-metrics/cad1fe66-8ec3-4396-9029-232fb6247ca4/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cad1fe66-8ec3-4396-9029-232fb6247ca4', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'ChIP-seq', 'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'aliases': [], 'date_created': '2022-09-26T20:45:59.387521+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'schema_version': '1', 'step_run': '/analysis-step-runs/86cfd783-e120-4e9d-9004-9c5205b7de25/', 'quality_metric_of': ['/files/ENCFF371SSY/'], 'processing_stage': 'unfiltered', 'total_reads': 8236014, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 8175373, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 99.3, 'paired_reads': 8236014, 'paired_reads_qc_failed': 0, 'read1': 4118007, 'read1_qc_failed': 0, 'read2': 4118007, 'read2_qc_failed': 0, 'properly_paired_reads': 7951882, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.6, 'with_itself': 8139796, 'with_itself_qc_failed': 0, 'singletons': 35577, 'singletons_qc_failed': 0, 'pct_singletons': 0.4, 'diff_chroms': 114272, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/34754d39-712c-4b9e-9e47-73b0957a851e/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '34754d39-712c-4b9e-9e47-73b0957a851e', 'assay_term_id': 'OBI:0000716'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:0070350b-62e4-4cb6-bf9f-557b0e29a7ff_replacement'], 'date_created': '2022-06-30T18:48:50.603395+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/15607b8b-0c8b-4a62-bd7f-cb14add2813b/', 'quality_metric_of': ['/files/ENCFF309MSH/'], 'attachment': {'download': '0070350b-62e4-4cb6-bf9f-557b0e29a7ff.txt', 'href': '@@download/attachment/0070350b-62e4-4cb6-bf9f-557b0e29a7ff.txt', 'type': 'text/plain', 'md5sum': '6473e425210787abbc9123aece887168'}, 'run_type': 'single-ended', 'sequenced_reads': 3226136971, 'no_chimera_found': 772387007, '0_alignments': 6340553, '1_alignment': 766046454, '1_alignment_unique': 647313858, '1_alignment_duplicates': 118732596, '2_alignments': 2270768487, '2_alignment_duplicates': 332166120, '2_alignment_unique': 1938602367, '3_or_more_alignments': 182981477, 'pct_no_chimera_found': 23.94, 'pct_0_alignments': 0.2, 'pct_1_alignment': 23.75, 'pct_1_alignment_unique': 84.5, 'pct_1_alignment_duplicates': 15.5, 'pct_sequenced_1_alignment_unique': 20.06, 'pct_sequenced_1_alignment_duplicates': 3.68, 'pct_2_alignments': 70.39, 'pct_2_alignment_unique': 85.37, 'pct_2_alignment_duplicates': 14.63, 'pct_sequenced_2_alignment_unique': 60.09, 'pct_sequenced_2_alignment_duplicates': 10.3, 'pct_3_or_more_alignments': 5.67, 'pct_unique_total_duplicates': 14.85, 'pct_unique_total_unique': 85.15, 'total_unique': 2585916225, 'pct_sequenced_total_unique': 80.16, 'total_duplicates': 450898716, 'pct_sequenced_total_duplicates': 13.98, 'library_complexity_estimate_1_alignment': 2208659399, 'library_complexity_estimate_2_alignments': 6984789489, 'library_complexity_estimate_1_and_2_alignments': 9193448888, 'below_mapq_threshold': 421342592, 'pct_sequenced_below_mapq_threshold': 13.06, 'pct_unique_below_mapq_threshold': 21.73, 'hic_contacts': 1517259775, 'pct_sequenced_hic_contacts': 47.03, 'pct_unique_hic_contacts': 78.27, 'lior_convergence': 1874, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 1014053131, 'pct_sequenced_inter_chromosomal': 31.43, 'pct_unique_inter_chromosomal': 52.31, 'intra_chromosomal': 503206644, 'pct_sequenced_intra_chromosomal': 15.6, 'pct_unique_intra_chromosomal': 25.96, 'short_range_less_than_500bp': 23040300, 'pct_sequenced_short_range_less_than_500bp': 0.71, 'pct_unique_short_range_less_than_500bp': 1.19, 'short_range_500bp_to_5kb': 38758575, 'pct_sequenced_short_range_500bp_to_5kb': 1.2, 'pct_unique_short_range_500bp_to_5kb': 2.0, 'short_range_5kb_to_20kb': 19232989, 'pct_sequenced_short_range_5kb_to_20kb': 0.6, 'pct_unique_short_range_5kb_to_20kb': 0.99, 'long_range_greater_than_20kb': 422174780, 'pct_sequenced_long_range_greater_than_20kb': 13.09, 'pct_unique_long_range_greater_than_20kb': 21.78, '@id': '/hic-quality-metrics/985b4c0c-6a4c-4a44-83d5-cca7dff01b1e/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': '985b4c0c-6a4c-4a44-83d5-cca7dff01b1e', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.856027+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/95674aac-9277-4b18-84d7-3b5aa227512c/', 'quality_metric_of': ['/files/ENCFF724EWA/'], 'frip': 0.508494279565402, 'min_size': 360, '25_pct': 586, '50_pct': 815, '75_pct': 1458, 'max_size': 93391, 'mean': 1516.6150249686648, '@id': '/chip-peak-enrichment-quality-metrics/de67e941-4db8-451c-b163-cba7a7ebdfb3/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'de67e941-4db8-451c-b163-cba7a7ebdfb3', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.779719+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/95674aac-9277-4b18-84d7-3b5aa227512c/', 'quality_metric_of': ['/files/ENCFF724EWA/'], 'reproducible_peaks': 150789, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/c54c0f46-5e22-464b-8d43-cfee2ff144f2/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c54c0f46-5e22-464b-8d43-cfee2ff144f2', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:41.284696+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/8dc44c46-1971-446d-8939-035443e82eab/', 'quality_metric_of': ['/files/ENCFF814ORT/'], 'frip': 0.47902198394423523, 'min_size': 530, '25_pct': 1054, '50_pct': 1579, '75_pct': 2650, 'max_size': 46583, 'mean': 2259.1449366760544, '@id': '/chip-peak-enrichment-quality-metrics/261cb04a-e1b7-4f4b-8775-e24b6bca045e/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '261cb04a-e1b7-4f4b-8775-e24b6bca045e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:41.194707+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/8dc44c46-1971-446d-8939-035443e82eab/', 'quality_metric_of': ['/files/ENCFF814ORT/'], 'reproducible_peaks': 115754, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/aca4e209-a7d3-4ee8-ab23-f532cee6334b/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'aca4e209-a7d3-4ee8-ab23-f532cee6334b', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.692638+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', 'quality_metric_of': ['/files/ENCFF959ASO/'], 'unpaired_reads': 0, 'paired_reads': 64076530, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 15840597, 'paired_optical_duplicate_reads': 23607, 'pct_duplicate_reads': 24.7214, 'total_fragments': 63952117, 'distinct_fragments': 52321290, 'positions_with_one_read': 5195421, 'NRF': 0.818132, 'PBC1': 0.859253, 'PBC2': 8.653245, '@id': '/chip-library-quality-metrics/efa6f9cc-8afc-401b-bbca-da1568a0e671/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'efa6f9cc-8afc-401b-bbca-da1568a0e671', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.359873+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', 'quality_metric_of': ['/files/ENCFF959ASO/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 360, 'corr_estimated_fragment_len': 0.0971697329065173, 'phantom_peak': 45, 'corr_phantom_peak': 0.09898907, 'argmin_corr': 1500, 'min_corr': 0.09567848, 'NSC': 1.015586, 'cross_correlation_plot': {'download': 'ENCFF337LOK.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF337LOK.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '68e2a368f7ca5b1835b46f6aec9fbde1'}, 'auc': 0.11381899255157467, 'syn_auc': 0.4954430238486618, 'x_intercept': 0.3178652112197365, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7659209500389852, 'syn_elbow_pt': 0.49616156182196114, 'jsd': 0.20459738676144626, 'syn_jsd': 0.5131311453114648, 'pct_genome_enrich': 18.95159839261081, 'diff_enrich': 48.65473011047955, 'ch_div': 0.42408889095149616, 'jsd_plot': {'download': 'ENCFF337LOK.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF337LOK.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '13fdd879e614eb9906a679963ad62a5d', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF337LOK.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF337LOK.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'dd89e700dea488ef3528a4991a5a1e18', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/b0142f6b-d091-45bf-864e-14481eef959e/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b0142f6b-d091-45bf-864e-14481eef959e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.199922+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', 'quality_metric_of': ['/files/ENCFF959ASO/'], 'processing_stage': 'filtered', 'total_reads': 96471866, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 96471866, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 96471866, 'paired_reads_qc_failed': 0, 'read1': 48235933, 'read1_qc_failed': 0, 'read2': 48235933, 'read2_qc_failed': 0, 'properly_paired_reads': 96471866, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 96471866, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/675cde88-c61b-49a5-a69d-579acc458d50/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '675cde88-c61b-49a5-a69d-579acc458d50', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 48235933}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:38:04.122256+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', 'quality_metric_of': ['/files/ENCFF646CRK/'], 'processing_stage': 'unfiltered', 'total_reads': 147507950, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 144955369, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.3, 'paired_reads': 147507950, 'paired_reads_qc_failed': 0, 'read1': 73753975, 'read1_qc_failed': 0, 'read2': 73753975, 'read2_qc_failed': 0, 'properly_paired_reads': 141437766, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 95.89999999999999, 'with_itself': 142955824, 'with_itself_qc_failed': 0, 'singletons': 1999545, 'singletons_qc_failed': 0, 'pct_singletons': 1.4000000000000001, 'diff_chroms': 646132, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/97743566-74c5-4a8f-8ecc-d53f4d7649c3/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '97743566-74c5-4a8f-8ecc-d53f4d7649c3', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:40.682136+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', 'quality_metric_of': ['/files/ENCFF896UDP/'], 'processing_stage': 'filtered', 'total_reads': 113288406, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 113288406, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 113288406, 'paired_reads_qc_failed': 0, 'read1': 56644203, 'read1_qc_failed': 0, 'read2': 56644203, 'read2_qc_failed': 0, 'properly_paired_reads': 113288406, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 113288406, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/4189f253-509b-4902-bd38-cac65d97e91f/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '4189f253-509b-4902-bd38-cac65d97e91f', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 56644203}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:41.102523+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', 'quality_metric_of': ['/files/ENCFF896UDP/'], 'unpaired_reads': 0, 'paired_reads': 64204058, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 7559855, 'paired_optical_duplicate_reads': 19661, 'pct_duplicate_reads': 11.774700000000001, 'total_fragments': 64164742, 'distinct_fragments': 57237859, 'positions_with_one_read': 4574602, 'NRF': 0.892045, 'PBC1': 0.903527, 'PBC2': 11.305013, '@id': '/chip-library-quality-metrics/36200b31-f873-4730-9ba6-8e7b50798975/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '36200b31-f873-4730-9ba6-8e7b50798975', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:40.807347+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', 'quality_metric_of': ['/files/ENCFF896UDP/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 530, 'corr_estimated_fragment_len': 0.0937348953085592, 'phantom_peak': 40, 'corr_phantom_peak': 0.09523691, 'argmin_corr': 1500, 'min_corr': 0.09296654, 'NSC': 1.008265, 'cross_correlation_plot': {'download': 'ENCFF339VNO.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF339VNO.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '64c0f1868cb8ba63ce78eb5fd1f637cf'}, 'auc': 0.11592741182875763, 'syn_auc': 0.4957580905859937, 'x_intercept': 0.27080954961043013, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7677335769852869, 'syn_elbow_pt': 0.49870186417259377, 'jsd': 0.1742982279128715, 'syn_jsd': 0.5277092444158432, 'pct_genome_enrich': 17.452170974894386, 'diff_enrich': 39.43850361947599, 'ch_div': 0.3432685090872067, 'jsd_plot': {'download': 'ENCFF339VNO.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF339VNO.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '7a50c61c60556e2332fdf55124844683', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF339VNO.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF339VNO.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'd8980c3d788717d6f98b94b0f8a7b8f9', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/2632967e-235d-462d-a47b-1f60f607df81/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2632967e-235d-462d-a47b-1f60f607df81', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:00:40.592956+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', 'quality_metric_of': ['/files/ENCFF512SDY/'], 'processing_stage': 'unfiltered', 'total_reads': 146278932, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 144447383, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.7, 'paired_reads': 146278932, 'paired_reads_qc_failed': 0, 'read1': 73139466, 'read1_qc_failed': 0, 'read2': 73139466, 'read2_qc_failed': 0, 'properly_paired_reads': 142230874, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.2, 'with_itself': 143113938, 'with_itself_qc_failed': 0, 'singletons': 1333445, 'singletons_qc_failed': 0, 'pct_singletons': 0.8999999999999999, 'diff_chroms': 268936, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/81681a0c-f728-473a-bfb5-cf52f949f79f/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '81681a0c-f728-473a-bfb5-cf52f949f79f', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:05.118549+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF141ARX/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '30a1f971e1cf23b0166d5f9695c38340', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'aaa17e9b2cdc98720b98e1fabc7f3d8b', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '271ffb8912828cb43162f0c1240b37c8', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/46d9574e-c020-4103-9c94-a2af6df43433/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '46d9574e-c020-4103-9c94-a2af6df43433', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:44.629993+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF620QPN/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '0307d1c514e12b86160caf4c40e8e590', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '6fc4f4adb1a776ca63bf3eb12ea4472c', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b426fcee3e2b4d2f857f69e08bf0d3ff', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/8d9fcd0a-ee27-4cf5-a9f7-b9ba1858954b/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8d9fcd0a-ee27-4cf5-a9f7-b9ba1858954b', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:02.269946+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF327FHP/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b920c28981feaec012e550ec565b46f9', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'e86da450a031ca9cd02568ad0620079b', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'caef5a5fea5ecd607c459775ea4d6446', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/fd54165c-c2f0-47b1-9c48-a25b8fa99046/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'fd54165c-c2f0-47b1-9c48-a25b8fa99046', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:43.827422+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF871YLC/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 304, 'fraction_of_unique_reads_in_cells': 0.69375, 'mean_genefull_ex50pas_per_cell': 968, 'mean_reads_per_cell': 7304, 'mean_UMI_per_cell': 2538, 'median_genefull_ex50pas_per_cell': 593, 'median_reads_per_cell': 2095, 'median_UMI_per_cell': 999, 'number_of_reads': 4452792, 'q30_bases_in_CB_UMI': 0.939592, 'q30_bases_in_rna_read': 0.940257, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.71882, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.758905, 'reads_mapped_to_genome_unique': 0.859704, 'reads_mapped_to_genome_unique_and_multiple': 0.912598, 'reads_with_valid_barcodes': 0.968345, 'sequencing_saturation': 0.538853, 'total_genefull_ex50pas_detected': 16880, 'umis_in_cells': 771590, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 2220523, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'f8147d8a714a9cdcccce711a893482c7', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/cf75507e-6ded-4055-a607-28e90801075d/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cf75507e-6ded-4055-a607-28e90801075d', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:13:43.886708+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF871YLC/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.29%', '% of reads mapped to too many loci': '0.25%', '% of reads unmapped: other': '0.91%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '7.58%', 'Average input read length': 100.0, 'Average mapped length': 98.62, 'Deletion average length': 2.26, 'Deletion rate per base': '0.06', 'Insertion average length': 1.88, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 364.32, 'Mismatch rate per base, %': '0.51%', 'Number of chimeric reads': 0.0, 'Number of input reads': 4452792.0, 'Number of reads mapped to multiple loci': 235528.0, 'Number of reads mapped to too many loci': 11057.0, 'Number of splices: AT/AC': 146.0, 'Number of splices: Annotated (sjdb)': 217225.0, 'Number of splices: GC/AG': 2275.0, 'Number of splices: GT/AG': 218678.0, 'Number of splices: Non-canonical': 177.0, 'Number of splices: Total': 221276.0, 'Uniquely mapped reads %': '85.97', 'Uniquely mapped reads number': 3828081.0, '@id': '/star-quality-metrics/89e807bf-628c-4128-b21a-03e8a39900d5/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '89e807bf-628c-4128-b21a-03e8a39900d5', 'read_depth': 4063609.0}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:08:17.251580+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF397POJ/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'afc1a1286e287975304c5f6b6eeb014b', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b44103202490fce936e4d6ade464dbbb', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '6fc1c747567f1f587def36004aa4b72b', 'width': 1509, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/6d4c95ea-65f8-4324-bc67-32b6316c6497/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6d4c95ea-65f8-4324-bc67-32b6316c6497', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:08:17.109990+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF282MSX/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 201, 'fraction_of_unique_reads_in_cells': 0.66263, 'mean_genefull_ex50pas_per_cell': 1096, 'mean_reads_per_cell': 8199, 'mean_UMI_per_cell': 2668, 'median_genefull_ex50pas_per_cell': 931, 'median_reads_per_cell': 4353, 'median_UMI_per_cell': 1554, 'number_of_reads': 3499107, 'q30_bases_in_CB_UMI': 0.941345, 'q30_bases_in_rna_read': 0.940232, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.710791, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.732714, 'reads_mapped_to_genome_unique': 0.831654, 'reads_mapped_to_genome_unique_and_multiple': 0.903089, 'reads_with_valid_barcodes': 0.964439, 'sequencing_saturation': 0.543902, 'total_genefull_ex50pas_detected': 15953, 'umis_in_cells': 536430, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1648050, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '50db02b0cba2aae20705c94ca8e7cdbf', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/30910edc-6f80-43e9-8ebb-3b05fa322f9c/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '30910edc-6f80-43e9-8ebb-3b05fa322f9c', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2022-01-30T06:08:17.446670+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF282MSX/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '7.14%', '% of reads mapped to too many loci': '0.23%', '% of reads unmapped: other': '0.89%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '8.57%', 'Average input read length': 99.0, 'Average mapped length': 96.79, 'Deletion average length': 2.25, 'Deletion rate per base': '0.07', 'Insertion average length': 1.92, 'Insertion rate per base': '0.06', 'Mapping speed, Million of reads per hour': 322.99, 'Mismatch rate per base, %': '0.56%', 'Number of chimeric reads': 0.0, 'Number of input reads': 3499107.0, 'Number of reads mapped to multiple loci': 249957.0, 'Number of reads mapped to too many loci': 8195.0, 'Number of splices: AT/AC': 112.0, 'Number of splices: Annotated (sjdb)': 135972.0, 'Number of splices: GC/AG': 1673.0, 'Number of splices: GT/AG': 137120.0, 'Number of splices: Non-canonical': 111.0, 'Number of splices: Total': 139016.0, 'Uniquely mapped reads %': '83.17', 'Uniquely mapped reads number': 2910048.0, '@id': '/star-quality-metrics/59ee826f-2c62-40bb-b82b-da9ab1b988eb/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '59ee826f-2c62-40bb-b82b-da9ab1b988eb', 'read_depth': 3160005.0}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:11.677228+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF823OIB/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.96%', '% of reads mapped to too many loci': '0.32%', '% of reads unmapped: other': '2.56%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '19.04%', 'Average input read length': 92.0, 'Average mapped length': 92.24, 'Deletion average length': 2.02, 'Deletion rate per base': '0.06', 'Insertion average length': 1.9, 'Insertion rate per base': '0.08', 'Mapping speed, Million of reads per hour': 193.41, 'Mismatch rate per base, %': '0.60%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1235656.0, 'Number of reads mapped to multiple loci': 160140.0, 'Number of reads mapped to too many loci': 4001.0, 'Number of splices: AT/AC': 20.0, 'Number of splices: Annotated (sjdb)': 63228.0, 'Number of splices: GC/AG': 681.0, 'Number of splices: GT/AG': 63513.0, 'Number of splices: Non-canonical': 60.0, 'Number of splices: Total': 64274.0, 'Uniquely mapped reads %': '65.12', 'Uniquely mapped reads number': 804603.0, '@id': '/star-quality-metrics/9ce3ed76-4f77-45b8-aac5-e6112527bcb1/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9ce3ed76-4f77-45b8-aac5-e6112527bcb1', 'read_depth': 964743.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:11.685947+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF823OIB/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1662, 'fraction_of_unique_reads_in_cells': 0.479745, 'mean_genefull_ex50pas_per_cell': 50, 'mean_reads_per_cell': 189, 'mean_UMI_per_cell': 105, 'median_genefull_ex50pas_per_cell': 35, 'median_reads_per_cell': 114, 'median_UMI_per_cell': 66, 'number_of_reads': 1235656, 'q30_bases_in_CB_UMI': 0.902932, 'q30_bases_in_rna_read': 0.937155, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.531145, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.563623, 'reads_mapped_to_genome_unique': 0.651155, 'reads_mapped_to_genome_unique_and_multiple': 0.780754, 'reads_with_valid_barcodes': 0.975483, 'sequencing_saturation': 0.398304, 'total_genefull_ex50pas_detected': 10568, 'umis_in_cells': 175807, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 314863, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'db3d355e28dd0755d1643524f8c7b13c', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/a134e86a-3235-4aca-8415-94a190863e22/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a134e86a-3235-4aca-8415-94a190863e22', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:11.601123+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF804CZU/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '14.01%', '% of reads mapped to too many loci': '0.28%', '% of reads unmapped: other': '2.20%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.81%', 'Average input read length': 94.0, 'Average mapped length': 93.51, 'Deletion average length': 2.0, 'Deletion rate per base': '0.06', 'Insertion average length': 1.87, 'Insertion rate per base': '0.08', 'Mapping speed, Million of reads per hour': 260.39, 'Mismatch rate per base, %': '0.58%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1157274.0, 'Number of reads mapped to multiple loci': 162124.0, 'Number of reads mapped to too many loci': 3295.0, 'Number of splices: AT/AC': 30.0, 'Number of splices: Annotated (sjdb)': 60215.0, 'Number of splices: GC/AG': 624.0, 'Number of splices: GT/AG': 60613.0, 'Number of splices: Non-canonical': 29.0, 'Number of splices: Total': 61296.0, 'Uniquely mapped reads %': '66.70', 'Uniquely mapped reads number': 771934.0, '@id': '/star-quality-metrics/58bea376-0a62-474a-aacc-3ad3cc9bbe60/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '58bea376-0a62-474a-aacc-3ad3cc9bbe60', 'read_depth': 934058.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:28:11.628866+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF804CZU/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1195, 'fraction_of_unique_reads_in_cells': 0.448426, 'mean_genefull_ex50pas_per_cell': 57, 'mean_reads_per_cell': 239, 'mean_UMI_per_cell': 136, 'median_genefull_ex50pas_per_cell': 39, 'median_reads_per_cell': 147, 'median_UMI_per_cell': 87, 'number_of_reads': 1157274, 'q30_bases_in_CB_UMI': 0.904726, 'q30_bases_in_rna_read': 0.939015, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.55052, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.583858, 'reads_mapped_to_genome_unique': 0.667028, 'reads_mapped_to_genome_unique_and_multiple': 0.807119, 'reads_with_valid_barcodes': 0.974283, 'sequencing_saturation': 0.383088, 'total_genefull_ex50pas_detected': 9772, 'umis_in_cells': 163670, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 285693, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'cb8d8d8270b5e0c43d7890bfae5384b1', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/2e060c03-f3b7-4d51-9268-c3ca193dd7d5/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2e060c03-f3b7-4d51-9268-c3ca193dd7d5', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:34.248024+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF269YJD/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '2ad24652caf31d9df44c0a8771cd6e4d', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '68e2081d685d7c2e943626243f9e4603', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '06e46091e88cb9e6f5b2cd312f68c3f5', 'width': 1490, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/e8898a57-b0aa-4a73-8fee-4a3062e09bca/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e8898a57-b0aa-4a73-8fee-4a3062e09bca', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:34.618388+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF005YVB/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '3471ceb8548dcc425fe8c703ca336076', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '14c3f6f2a63fed342ee21544f8a9a935', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '81b68e576991f2b9702024c09eb0fee5', 'width': 1490, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/cf1a6691-6216-4e90-86a9-a21992bd788b/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cf1a6691-6216-4e90-86a9-a21992bd788b', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.732565+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', 'quality_metric_of': ['/files/ENCFF687PZQ/'], 'unpaired_reads': 0, 'paired_reads': 77865553, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 5580141, 'paired_optical_duplicate_reads': 26618, 'pct_duplicate_reads': 7.1664, 'total_fragments': 77813743, 'distinct_fragments': 72966164, 'positions_with_one_read': 2973877, 'NRF': 0.937703, 'PBC1': 0.952876, 'PBC2': 23.379488, '@id': '/chip-library-quality-metrics/3ec3bec5-0ded-4a4f-aa0c-f14e6495b8d1/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3ec3bec5-0ded-4a4f-aa0c-f14e6495b8d1', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.473760+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', 'quality_metric_of': ['/files/ENCFF687PZQ/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 525, 'corr_estimated_fragment_len': 0.0989149329882129, 'phantom_peak': 45, 'corr_phantom_peak': 0.09995238, 'argmin_corr': 1500, 'min_corr': 0.09855626, 'NSC': 1.003639, 'cross_correlation_plot': {'download': 'ENCFF689MZS.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF689MZS.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'f1715bdaf99e804a08727038b0afca6e'}, 'auc': 0.17471250930541374, 'syn_auc': 0.496170861933593, 'x_intercept': 0.14384049133722254, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7128508194951478, 'syn_elbow_pt': 0.5018281405730713, 'jsd': 0.09527266751613947, 'syn_jsd': 0.45178775574052993, 'pct_genome_enrich': 22.384073124797574, 'diff_enrich': 21.330768521723975, 'ch_div': 0.18535381985051538, 'jsd_plot': {'download': 'ENCFF689MZS.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF689MZS.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': 'b80422d53e17a0ec9c5f30e646e3773d', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF689MZS.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF689MZS.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '4ee7941a81e9ed0e095a12e6c55d0e07', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/1ab4d870-a246-4136-b39b-42467e1c0b6f/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1ab4d870-a246-4136-b39b-42467e1c0b6f', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.328302+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', 'quality_metric_of': ['/files/ENCFF687PZQ/'], 'processing_stage': 'filtered', 'total_reads': 144570824, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 144570824, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 144570824, 'paired_reads_qc_failed': 0, 'read1': 72285412, 'read1_qc_failed': 0, 'read2': 72285412, 'read2_qc_failed': 0, 'properly_paired_reads': 144570824, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 144570824, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/52ae6dfa-0883-4f1f-b524-6ef35d6f32bd/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '52ae6dfa-0883-4f1f-b524-6ef35d6f32bd', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 72285412}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.892145+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/da82ff3b-4f7e-434b-81b5-01f1e69e6186/', 'quality_metric_of': ['/files/ENCFF666BMR/'], 'frip': 0.2580874893540069, 'min_size': 525, '25_pct': 955, '50_pct': 1169, '75_pct': 1680, 'max_size': 27142, 'mean': 1424.1809826925657, '@id': '/chip-peak-enrichment-quality-metrics/2d2510be-f227-4025-a0fb-9454797787b3/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2d2510be-f227-4025-a0fb-9454797787b3', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.814585+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/da82ff3b-4f7e-434b-81b5-01f1e69e6186/', 'quality_metric_of': ['/files/ENCFF666BMR/'], 'reproducible_peaks': 134162, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/4ec777eb-09bb-40f2-8216-def37c2b7d8b/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '4ec777eb-09bb-40f2-8216-def37c2b7d8b', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:54.811454+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF654YOP/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.71%', '% of reads mapped to too many loci': '0.21%', '% of reads unmapped: other': '1.88%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.51%', 'Average input read length': 95.0, 'Average mapped length': 93.97, 'Deletion average length': 2.05, 'Deletion rate per base': '0.07', 'Insertion average length': 2.16, 'Insertion rate per base': '0.09', 'Mapping speed, Million of reads per hour': 166.06, 'Mismatch rate per base, %': '0.63%', 'Number of chimeric reads': 0.0, 'Number of input reads': 784157.0, 'Number of reads mapped to multiple loci': 99699.0, 'Number of reads mapped to too many loci': 1668.0, 'Number of splices: AT/AC': 18.0, 'Number of splices: Annotated (sjdb)': 45361.0, 'Number of splices: GC/AG': 455.0, 'Number of splices: GT/AG': 45512.0, 'Number of splices: Non-canonical': 32.0, 'Number of splices: Total': 46017.0, 'Uniquely mapped reads %': '68.69', 'Uniquely mapped reads number': 538602.0, '@id': '/star-quality-metrics/60b2133c-aeb3-45e6-a6ea-9e2c761f1017/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '60b2133c-aeb3-45e6-a6ea-9e2c761f1017', 'read_depth': 638301.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:58.113796+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF654YOP/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1083, 'fraction_of_unique_reads_in_cells': 0.463879, 'mean_genefull_ex50pas_per_cell': 49, 'mean_reads_per_cell': 183, 'mean_UMI_per_cell': 105, 'median_genefull_ex50pas_per_cell': 36, 'median_reads_per_cell': 124, 'median_UMI_per_cell': 72, 'number_of_reads': 784157, 'q30_bases_in_CB_UMI': 0.935455, 'q30_bases_in_rna_read': 0.940525, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.547658, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.576873, 'reads_mapped_to_genome_unique': 0.686855, 'reads_mapped_to_genome_unique_and_multiple': 0.813996, 'reads_with_valid_barcodes': 0.973869, 'sequencing_saturation': 0.386823, 'total_genefull_ex50pas_detected': 8504, 'umis_in_cells': 114619, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 199213, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '95911b82f9b9af2df0ad6e41cad721af', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/d97745ce-ceea-41f2-a9b5-84e87c8f05bd/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd97745ce-ceea-41f2-a9b5-84e87c8f05bd', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:55.604257+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', 'quality_metric_of': ['/files/ENCFF391FXR/'], 'processing_stage': 'unfiltered', 'total_reads': 87131082, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 86133985, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.9, 'paired_reads': 87131082, 'paired_reads_qc_failed': 0, 'read1': 43565541, 'read1_qc_failed': 0, 'read2': 43565541, 'read2_qc_failed': 0, 'properly_paired_reads': 84982592, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.5, 'with_itself': 85423252, 'with_itself_qc_failed': 0, 'singletons': 710733, 'singletons_qc_failed': 0, 'pct_singletons': 0.8, 'diff_chroms': 138230, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/ebc27efa-7500-46d9-a580-6ac958bfb10a/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ebc27efa-7500-46d9-a580-6ac958bfb10a', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:56.254664+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/e46c8ef3-665a-4d9e-8a14-1238f10c2ae8/', 'quality_metric_of': ['/files/ENCFF784XLI/'], 'frip': 0.24061056862893923, 'min_size': 250, '25_pct': 399, '50_pct': 518, '75_pct': 749, 'max_size': 10534, 'mean': 651.9463008072356, '@id': '/chip-peak-enrichment-quality-metrics/f5ac3e26-1f97-4ee9-ac53-061e33a7aec8/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f5ac3e26-1f97-4ee9-ac53-061e33a7aec8', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:56.179561+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/e46c8ef3-665a-4d9e-8a14-1238f10c2ae8/', 'quality_metric_of': ['/files/ENCFF784XLI/'], 'reproducible_peaks': 183094, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/622ce0ef-cb92-49a8-8ad2-ab92b5184ee1/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '622ce0ef-cb92-49a8-8ad2-ab92b5184ee1', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:47.527651+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF628XLU/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.26%', '% of reads mapped to too many loci': '0.23%', '% of reads unmapped: other': '2.22%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '17.77%', 'Average input read length': 94.0, 'Average mapped length': 92.85, 'Deletion average length': 2.07, 'Deletion rate per base': '0.07', 'Insertion average length': 2.16, 'Insertion rate per base': '0.09', 'Mapping speed, Million of reads per hour': 170.99, 'Mismatch rate per base, %': '0.64%', 'Number of chimeric reads': 0.0, 'Number of input reads': 807472.0, 'Number of reads mapped to multiple loci': 98956.0, 'Number of reads mapped to too many loci': 1818.0, 'Number of splices: AT/AC': 6.0, 'Number of splices: Annotated (sjdb)': 44348.0, 'Number of splices: GC/AG': 502.0, 'Number of splices: GT/AG': 44504.0, 'Number of splices: Non-canonical': 35.0, 'Number of splices: Total': 45047.0, 'Uniquely mapped reads %': '67.53', 'Uniquely mapped reads number': 545256.0, '@id': '/star-quality-metrics/9d88fbd9-947b-4a59-84a7-96559b2572b0/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9d88fbd9-947b-4a59-84a7-96559b2572b0', 'read_depth': 644212.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T18:17:55.244568+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF628XLU/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1062, 'fraction_of_unique_reads_in_cells': 0.440015, 'mean_genefull_ex50pas_per_cell': 47, 'mean_reads_per_cell': 180, 'mean_UMI_per_cell': 99, 'median_genefull_ex50pas_per_cell': 35, 'median_reads_per_cell': 120, 'median_UMI_per_cell': 67, 'number_of_reads': 807472, 'q30_bases_in_CB_UMI': 0.933571, 'q30_bases_in_rna_read': 0.939282, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.538173, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.568023, 'reads_mapped_to_genome_unique': 0.675263, 'reads_mapped_to_genome_unique_and_multiple': 0.797813, 'reads_with_valid_barcodes': 0.97502, 'sequencing_saturation': 0.402175, 'total_genefull_ex50pas_detected': 8131, 'umis_in_cells': 106053, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 191213, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '2e519010809315c8986e1bbcded46033', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/2e37fa93-af46-4abb-8a14-6790dbd1698e/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2e37fa93-af46-4abb-8a14-6790dbd1698e', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:33:04.246443+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', 'quality_metric_of': ['/files/ENCFF521PTH/'], 'processing_stage': 'unfiltered', 'total_reads': 202152774, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 198217752, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.1, 'paired_reads': 202152774, 'paired_reads_qc_failed': 0, 'read1': 101076387, 'read1_qc_failed': 0, 'read2': 101076387, 'read2_qc_failed': 0, 'properly_paired_reads': 194027654, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.0, 'with_itself': 196079620, 'with_itself_qc_failed': 0, 'singletons': 2138132, 'singletons_qc_failed': 0, 'pct_singletons': 1.0999999999999999, 'diff_chroms': 561933, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/75a8752b-cbc4-4de0-8ea1-5f36c336221b/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '75a8752b-cbc4-4de0-8ea1-5f36c336221b', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'HiC', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': ['encode:304e55ce-6e8c-4eed-bf91-733a919ed1ea_replacement'], 'date_created': '2022-06-30T18:49:16.396189+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/0b515aa6-3db5-451b-870f-11b2388a464d/', 'quality_metric_of': ['/files/ENCFF740FMH/'], 'attachment': {'download': '304e55ce-6e8c-4eed-bf91-733a919ed1ea.txt', 'href': '@@download/attachment/304e55ce-6e8c-4eed-bf91-733a919ed1ea.txt', 'type': 'text/plain', 'md5sum': '6473e425210787abbc9123aece887168'}, 'run_type': 'single-ended', 'sequenced_reads': 3226136971, 'no_chimera_found': 772387007, '0_alignments': 6340553, '1_alignment': 766046454, '1_alignment_unique': 647313858, '1_alignment_duplicates': 118732596, '2_alignments': 2270768487, '2_alignment_duplicates': 332166120, '2_alignment_unique': 1938602367, '3_or_more_alignments': 182981477, 'pct_no_chimera_found': 23.94, 'pct_0_alignments': 0.2, 'pct_1_alignment': 23.75, 'pct_1_alignment_unique': 84.5, 'pct_1_alignment_duplicates': 15.5, 'pct_sequenced_1_alignment_unique': 20.06, 'pct_sequenced_1_alignment_duplicates': 3.68, 'pct_2_alignments': 70.39, 'pct_2_alignment_unique': 85.37, 'pct_2_alignment_duplicates': 14.63, 'pct_sequenced_2_alignment_unique': 60.09, 'pct_sequenced_2_alignment_duplicates': 10.3, 'pct_3_or_more_alignments': 5.67, 'pct_unique_total_duplicates': 14.85, 'pct_unique_total_unique': 85.15, 'total_unique': 2585916225, 'pct_sequenced_total_unique': 80.16, 'total_duplicates': 450898716, 'pct_sequenced_total_duplicates': 13.98, 'library_complexity_estimate_1_alignment': 2208659399, 'library_complexity_estimate_2_alignments': 6984789489, 'library_complexity_estimate_1_and_2_alignments': 9193448888, 'below_mapq_threshold': 421342592, 'pct_sequenced_below_mapq_threshold': 13.06, 'pct_unique_below_mapq_threshold': 21.73, 'hic_contacts': 1517259775, 'pct_sequenced_hic_contacts': 47.03, 'pct_unique_hic_contacts': 78.27, 'lior_convergence': 1874, 'pct_left_pair_type': 25.0, 'pct_right_pair_type': 25.0, 'pct_inner_pair_type': 25.0, 'pct_outer_pair_type': 25.0, 'inter_chromosomal': 1014053131, 'pct_sequenced_inter_chromosomal': 31.43, 'pct_unique_inter_chromosomal': 52.31, 'intra_chromosomal': 503206644, 'pct_sequenced_intra_chromosomal': 15.6, 'pct_unique_intra_chromosomal': 25.96, 'short_range_less_than_500bp': 23040300, 'pct_sequenced_short_range_less_than_500bp': 0.71, 'pct_unique_short_range_less_than_500bp': 1.19, 'short_range_500bp_to_5kb': 38758575, 'pct_sequenced_short_range_500bp_to_5kb': 1.2, 'pct_unique_short_range_500bp_to_5kb': 2.0, 'short_range_5kb_to_20kb': 19232989, 'pct_sequenced_short_range_5kb_to_20kb': 0.6, 'pct_unique_short_range_5kb_to_20kb': 0.99, 'long_range_greater_than_20kb': 422174780, 'pct_sequenced_long_range_greater_than_20kb': 13.09, 'pct_unique_long_range_greater_than_20kb': 21.78, '@id': '/hic-quality-metrics/8b5034cc-7f2d-48ec-8c45-abaa6f9a135f/', '@type': ['HicQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8b5034cc-7f2d-48ec-8c45-abaa6f9a135f', 'assay_term_id': 'OBI:0002042'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:55.814888+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', 'quality_metric_of': ['/files/ENCFF959WVG/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 250, 'corr_estimated_fragment_len': 0.11778843815529, 'phantom_peak': 45, 'corr_phantom_peak': 0.1189394, 'argmin_corr': 1500, 'min_corr': 0.1164621, 'NSC': 1.011389, 'cross_correlation_plot': {'download': 'ENCFF248KDR.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF248KDR.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '468a66732ef1dfcda5038b45a1462e39'}, 'auc': 0.174111310479791, 'syn_auc': 0.4948229667188256, 'x_intercept': 0.25258460572641867, 'syn_x_intercept': 0.0, 'elbow_pt': 0.6740127915741153, 'syn_elbow_pt': 0.4951000142785361, 'jsd': 0.13799938340407628, 'syn_jsd': 0.4010867517307077, 'pct_genome_enrich': 29.55157515499637, 'diff_enrich': 31.309820457876146, 'ch_div': 0.2783939237258225, 'jsd_plot': {'download': 'ENCFF248KDR.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF248KDR.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '2842332baa02f99f3a33ce55f93d2764', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF248KDR.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF248KDR.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '3588d1bad60012e7b7030b4529c829bd', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/76f99685-8328-4cda-911f-8826d7e4426f/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '76f99685-8328-4cda-911f-8826d7e4426f', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:56.094825+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', 'quality_metric_of': ['/files/ENCFF959WVG/'], 'unpaired_reads': 0, 'paired_reads': 39394593, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 2372383, 'paired_optical_duplicate_reads': 111092, 'pct_duplicate_reads': 6.0221, 'total_fragments': 39391499, 'distinct_fragments': 37102192, 'positions_with_one_read': 1940108, 'NRF': 0.941883, 'PBC1': 0.94343, 'PBC2': 18.041944, '@id': '/chip-library-quality-metrics/97a5131d-bf09-4eda-968b-a8e336ed529e/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '97a5131d-bf09-4eda-968b-a8e336ed529e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:11:55.692632+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', 'quality_metric_of': ['/files/ENCFF959WVG/'], 'processing_stage': 'filtered', 'total_reads': 74044420, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 74044420, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 74044420, 'paired_reads_qc_failed': 0, 'read1': 37022210, 'read1_qc_failed': 0, 'read2': 37022210, 'read2_qc_failed': 0, 'properly_paired_reads': 74044420, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 74044420, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/2b0ed5a0-6188-401f-aac3-b4edf1a9b7fd/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2b0ed5a0-6188-401f-aac3-b4edf1a9b7fd', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 37022210}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:51.845661+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', 'quality_metric_of': ['/files/ENCFF236NLD/'], 'processing_stage': 'filtered', 'total_reads': 128105302, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 128105302, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 128105302, 'paired_reads_qc_failed': 0, 'read1': 64052651, 'read1_qc_failed': 0, 'read2': 64052651, 'read2_qc_failed': 0, 'properly_paired_reads': 128105302, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 128105302, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/2b02fc27-8fee-4434-bf2d-d9024f025f80/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2b02fc27-8fee-4434-bf2d-d9024f025f80', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 64052651}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:52.298263+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', 'quality_metric_of': ['/files/ENCFF236NLD/'], 'unpaired_reads': 0, 'paired_reads': 76157868, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 12105217, 'paired_optical_duplicate_reads': 117743, 'pct_duplicate_reads': 15.8949, 'total_fragments': 76095975, 'distinct_fragments': 64945680, 'positions_with_one_read': 6324619, 'NRF': 0.853471, 'PBC1': 0.874428, 'PBC2': 8.979252, '@id': '/chip-library-quality-metrics/c3ee243d-ec33-4da9-888f-7312ab37c8cc/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c3ee243d-ec33-4da9-888f-7312ab37c8cc', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:51.986134+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', 'quality_metric_of': ['/files/ENCFF236NLD/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 355, 'corr_estimated_fragment_len': 0.102076412983727, 'phantom_peak': 40, 'corr_phantom_peak': 0.1027212, 'argmin_corr': 1500, 'min_corr': 0.100264, 'NSC': 1.018076, 'cross_correlation_plot': {'download': 'ENCFF778GMU.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF778GMU.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '432798d86270e24da6e01a0b3d8317e3'}, 'auc': 0.15978552080412384, 'syn_auc': 0.4960435874417548, 'x_intercept': 0.17392521271831618, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7296758044910754, 'syn_elbow_pt': 0.5030326857946561, 'jsd': 0.13790151441496581, 'syn_jsd': 0.4693453308090954, 'pct_genome_enrich': 16.71977960463182, 'diff_enrich': 25.966601332387196, 'ch_div': 0.22200832832685286, 'jsd_plot': {'download': 'ENCFF778GMU.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF778GMU.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '5d1f5c83cdc9c29b4e5847df954a2db9', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF778GMU.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF778GMU.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'e5b790499b7ab925f1cd74298ab1c69c', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/8d94fe60-afc1-406e-8590-e9f8c1229c50/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8d94fe60-afc1-406e-8590-e9f8c1229c50', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:51.761054+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', 'quality_metric_of': ['/files/ENCFF806PXN/'], 'processing_stage': 'unfiltered', 'total_reads': 170826046, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 168683443, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.7, 'paired_reads': 170826046, 'paired_reads_qc_failed': 0, 'read1': 85413023, 'read1_qc_failed': 0, 'read2': 85413023, 'read2_qc_failed': 0, 'properly_paired_reads': 165774826, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.0, 'with_itself': 167528920, 'with_itself_qc_failed': 0, 'singletons': 1154523, 'singletons_qc_failed': 0, 'pct_singletons': 0.7000000000000001, 'diff_chroms': 384208, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/dbeb5ca3-1bd8-4bf1-afa9-a205cd7f57f2/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'dbeb5ca3-1bd8-4bf1-afa9-a205cd7f57f2', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:52.453573+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/519ff374-a2f7-4232-be9f-a8a404783d0d/', 'quality_metric_of': ['/files/ENCFF714FWL/'], 'frip': 0.31383840771867505, 'min_size': 355, '25_pct': 594, '50_pct': 771, '75_pct': 1222, 'max_size': 19779, 'mean': 1059.307545286585, '@id': '/chip-peak-enrichment-quality-metrics/5e62c8b3-169b-44e6-95d4-43f44ed3f372/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5e62c8b3-169b-44e6-95d4-43f44ed3f372', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:05:52.376581+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/519ff374-a2f7-4232-be9f-a8a404783d0d/', 'quality_metric_of': ['/files/ENCFF714FWL/'], 'reproducible_peaks': 141267, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/5e3f876b-15f6-4430-b61e-4bdec95639bb/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5e3f876b-15f6-4430-b61e-4bdec95639bb', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.499920+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '7', 'step_run': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', 'quality_metric_of': ['/files/ENCFF176COJ/', '/files/ENCFF753ZLV/'], 'attachment': {'download': 'rep1ENCSR369MDF_anno_rsem-rep2ENCSR369MDF_anno_rsem_mad_plot.png', 'href': '@@download/attachment/rep1ENCSR369MDF_anno_rsem-rep2ENCSR369MDF_anno_rsem_mad_plot.png', 'type': 'image/png', 'md5sum': 'e6ea414f42b34b9ca5e7fbca9e8ac62b', 'width': 504, 'height': 504}, 'SD of log ratios': 0.513, 'Pearson correlation': 0.9730435, 'Spearman correlation': 0.974847, 'MAD of log ratios': 0.327, '@id': '/mad-quality-metrics/b0933b64-2c58-48da-93b0-6f85bc6bd1de/', '@type': ['MadQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b0933b64-2c58-48da-93b0-6f85bc6bd1de', 'assay_term_id': 'OBI:0001271'}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.396954+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', 'quality_metric_of': ['/files/ENCFF176COJ/'], 'number_of_genes_detected': 13433, '@id': '/gene-quantification-quality-metrics/cf4093df-f717-4828-9f9c-656b9608bd38/', '@type': ['GeneQuantificationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cf4093df-f717-4828-9f9c-656b9608bd38', 'assay_term_id': 'OBI:0001271'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:25.786547+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', 'quality_metric_of': ['/files/ENCFF874QSA/'], 'processing_stage': 'filtered', 'total_reads': 41331774, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 41331774, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 41331774, 'paired_reads_qc_failed': 0, 'read1': 20665887, 'read1_qc_failed': 0, 'read2': 20665887, 'read2_qc_failed': 0, 'properly_paired_reads': 41331774, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 41331774, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/8047b0a3-27f4-47eb-95cd-3c325fec8e91/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8047b0a3-27f4-47eb-95cd-3c325fec8e91', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 20665887}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:25.913253+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', 'quality_metric_of': ['/files/ENCFF874QSA/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 300, 'corr_estimated_fragment_len': 0.0858965738116216, 'phantom_peak': 40, 'corr_phantom_peak': 0.09070849, 'argmin_corr': 1500, 'min_corr': 0.04120798, 'NSC': 2.084464, 'cross_correlation_plot': {'download': 'ENCFF045XPJ.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF045XPJ.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'a173f509169e65721408a14f4bd55ebf'}, 'auc': 0.0132617315269923, 'syn_auc': 0.49298404696653936, 'x_intercept': 0.8101382985954236, 'syn_x_intercept': 8.571510465306483e-174, 'elbow_pt': 0.934227203121814, 'syn_elbow_pt': 0.5031779771085076, 'jsd': 0.46717702037743886, 'syn_jsd': 0.7082656713565707, 'pct_genome_enrich': 5.577323668291279, 'diff_enrich': 85.77483440504294, 'ch_div': 0.7943313521814694, 'jsd_plot': {'download': 'ENCFF045XPJ.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF045XPJ.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '0f2596acbc75191f9e955f1eaa5321cc', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF045XPJ.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF045XPJ.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'ad4486ad8ca15bab7d1c5667a28b8d68', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/344570a6-1023-46e4-a1b2-94aa66e7223e/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '344570a6-1023-46e4-a1b2-94aa66e7223e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:26.294526+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', 'quality_metric_of': ['/files/ENCFF874QSA/'], 'unpaired_reads': 0, 'paired_reads': 27761792, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 7095905, 'paired_optical_duplicate_reads': 7021, 'pct_duplicate_reads': 25.56, 'total_fragments': 27731861, 'distinct_fragments': 21139068, 'positions_with_one_read': 2438791, 'NRF': 0.762266, 'PBC1': 0.829836, 'PBC2': 7.192894, '@id': '/chip-library-quality-metrics/7ac89a69-d0f8-4161-8f7b-058bd369eeb4/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7ac89a69-d0f8-4161-8f7b-058bd369eeb4', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:26.373607+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/31fe7a00-7e41-47e6-bbfd-06ec91848ebf/', 'quality_metric_of': ['/files/ENCFF642AOJ/'], 'reproducible_peaks': 55517, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/90db9836-d9cf-4dd6-9be1-6cd182a636ee/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '90db9836-d9cf-4dd6-9be1-6cd182a636ee', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:26.446921+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/31fe7a00-7e41-47e6-bbfd-06ec91848ebf/', 'quality_metric_of': ['/files/ENCFF642AOJ/'], 'frip': 0.820362682714756, 'min_size': 300, '25_pct': 558, '50_pct': 958, '75_pct': 2040, 'max_size': 31452, 'mean': 1537.7503107156367, '@id': '/chip-peak-enrichment-quality-metrics/e482de65-451c-459a-9591-fdfbce71f011/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e482de65-451c-459a-9591-fdfbce71f011', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.963276+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF577OPE/'], 'attachment': {'download': 'rep2ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep2ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': '0b0424e807c6e08ab686fc789224d0c7'}, '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.77%', '% of reads mapped to too many loci': '0.01%', '% of reads unmapped: other': '0.24%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.49%', 'Average input read length': 300, 'Average mapped length': 297.33, 'Deletion average length': 1.85, 'Deletion rate per base': '0.01%', 'Insertion average length': 1.84, 'Insertion rate per base': '0.01%', 'Mapping speed, Million of reads per hour': 141.52, 'Mismatch rate per base, %': '0.37%', 'Number of chimeric reads': 0, 'Number of input reads': 19734786, 'Number of reads mapped to multiple loci': 1137717, 'Number of reads mapped to too many loci': 2440, 'Number of splices: AT/AC': 17843, 'Number of splices: Annotated (sjdb)': 19400675, 'Number of splices: GC/AG': 138522, 'Number of splices: GT/AG': 19465736, 'Number of splices: Non-canonical': 9987, 'Number of splices: Total': 19632088, 'Uniquely mapped reads %': '87.50%', 'Uniquely mapped reads number': 17267272, '@id': '/star-quality-metrics/e3f027a6-c3b8-4d59-963d-3ff5c29a3c2a/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e3f027a6-c3b8-4d59-963d-3ff5c29a3c2a', 'assay_term_id': 'OBI:0001271', 'read_depth': 18404989}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.111970+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF577OPE/'], 'attachment': {'download': 'rep2ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep2ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': '5b994a40094cf84fafbe48b5168dc8b3'}, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, 'duplicates': 0, 'duplicates_qc_failed': 0, 'mapped': 97657480, 'mapped_pct': '97.35', 'mapped_qc_failed': 0, 'paired': 31992460, 'paired_properly': 29332866, 'paired_properly_pct': '91.69', 'paired_properly_qc_failed': 0, 'paired_qc_failed': 0, 'read1': 15996230, 'read1_qc_failed': 0, 'read2': 15996230, 'read2_qc_failed': 0, 'singletons': 0, 'singletons_pct': '0.0', 'singletons_qc_failed': 0, 'total': 100317074, 'total_qc_failed': 0, 'with_itself': 29332866, 'with_itself_qc_failed': 0, '@id': '/samtools-flagstats-quality-metrics/05755f01-1476-45e3-88ad-ef224353c4c2/', '@type': ['SamtoolsFlagstatsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '05755f01-1476-45e3-88ad-ef224353c4c2', 'assay_term_id': 'OBI:0001271'}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.252221+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF577OPE/'], 'attachment': {'download': 'rep2ENCSR369MDF_anno.bam.json', 'href': '@@download/attachment/rep2ENCSR369MDF_anno.bam.json', 'type': 'application/json', 'md5sum': '42e9225d91ad6ff1849105481e377ae5'}, 'Mt_rRNA': 376630, 'antisense': 109764, 'miRNA': 0, 'processed_transcript': 123876, 'protein_coding': 28298384, 'rRNA': 0, 'ribozyme': 8, 'sRNA': 0, 'scaRNA': 10, 'sense_intronic': 3670, 'sense_overlapping': 3640, 'snRNA': 8, 'snoRNA': 14, 'spikein': 0, '@id': '/gene-type-quantification-quality-metrics/a5baa835-fedc-4f61-8795-a55c7e032cc4/', '@type': ['GeneTypeQuantificationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a5baa835-fedc-4f61-8795-a55c7e032cc4', 'assay_term_id': 'OBI:0001271'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.499920+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '7', 'step_run': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', 'quality_metric_of': ['/files/ENCFF176COJ/', '/files/ENCFF753ZLV/'], 'attachment': {'download': 'rep1ENCSR369MDF_anno_rsem-rep2ENCSR369MDF_anno_rsem_mad_plot.png', 'href': '@@download/attachment/rep1ENCSR369MDF_anno_rsem-rep2ENCSR369MDF_anno_rsem_mad_plot.png', 'type': 'image/png', 'md5sum': 'e6ea414f42b34b9ca5e7fbca9e8ac62b', 'width': 504, 'height': 504}, 'SD of log ratios': 0.513, 'Pearson correlation': 0.9730435, 'Spearman correlation': 0.974847, 'MAD of log ratios': 0.327, '@id': '/mad-quality-metrics/b0933b64-2c58-48da-93b0-6f85bc6bd1de/', '@type': ['MadQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b0933b64-2c58-48da-93b0-6f85bc6bd1de', 'assay_term_id': 'OBI:0001271'}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:23.646890+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', 'quality_metric_of': ['/files/ENCFF753ZLV/'], 'number_of_genes_detected': 13283, '@id': '/gene-quantification-quality-metrics/72309dde-748c-4a6a-a7b4-de80ae23a254/', '@type': ['GeneQuantificationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '72309dde-748c-4a6a-a7b4-de80ae23a254', 'assay_term_id': 'OBI:0001271'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:36:25.706331+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', 'quality_metric_of': ['/files/ENCFF868PAD/'], 'processing_stage': 'unfiltered', 'total_reads': 60150068, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 59219371, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.5, 'paired_reads': 60150068, 'paired_reads_qc_failed': 0, 'read1': 30075034, 'read1_qc_failed': 0, 'read2': 30075034, 'read2_qc_failed': 0, 'properly_paired_reads': 58475022, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.2, 'with_itself': 58662154, 'with_itself_qc_failed': 0, 'singletons': 557217, 'singletons_qc_failed': 0, 'pct_singletons': 0.8999999999999999, 'diff_chroms': 52905, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/38c7c878-44cb-463c-b150-83550a77bd10/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '38c7c878-44cb-463c-b150-83550a77bd10', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:21.956065+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF458TDU/'], 'attachment': {'download': 'rep1ENCSR369MDF_genome.bam.json', 'href': '@@download/attachment/rep1ENCSR369MDF_genome.bam.json', 'type': 'application/json', 'md5sum': 'daf55156d2fb4d54385726d7b8ba8722'}, '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.62%', '% of reads mapped to too many loci': '0.01%', '% of reads unmapped: other': '0.22%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.68%', 'Average input read length': 300, 'Average mapped length': 297.27, 'Deletion average length': 1.89, 'Deletion rate per base': '0.01%', 'Insertion average length': 1.85, 'Insertion rate per base': '0.01%', 'Mapping speed, Million of reads per hour': 136.07, 'Mismatch rate per base, %': '0.37%', 'Number of chimeric reads': 0, 'Number of input reads': 17688624, 'Number of reads mapped to multiple loci': 993429, 'Number of reads mapped to too many loci': 2472, 'Number of splices: AT/AC': 15439, 'Number of splices: Annotated (sjdb)': 16797458, 'Number of splices: GC/AG': 122724, 'Number of splices: GT/AG': 16858848, 'Number of splices: Non-canonical': 8429, 'Number of splices: Total': 17005440, 'Uniquely mapped reads %': '87.47%', 'Uniquely mapped reads number': 15472041, '@id': '/star-quality-metrics/77ef7488-05a8-425d-9faa-a141ba8bde8a/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '77ef7488-05a8-425d-9faa-a141ba8bde8a', 'assay_term_id': 'OBI:0001271', 'read_depth': 16465470}, {'assay_term_name': 'RNA-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2021-12-31T08:51:22.089757+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '8', 'step_run': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', 'quality_metric_of': ['/files/ENCFF458TDU/'], 'attachment': {'download': 'rep1ENCSR369MDF_genome.bam.json', 'href': '@@download/attachment/rep1ENCSR369MDF_genome.bam.json', 'type': 'application/json', 'md5sum': '4d17ba74388c10710c70b02c03128ca8'}, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, 'duplicates': 0, 'duplicates_qc_failed': 0, 'mapped': 36910536, 'mapped_pct': '93.78', 'mapped_qc_failed': 0, 'paired': 35377248, 'paired_properly': 32930940, 'paired_properly_pct': '93.09', 'paired_properly_qc_failed': 0, 'paired_qc_failed': 0, 'read1': 17688624, 'read1_qc_failed': 0, 'read2': 17688624, 'read2_qc_failed': 0, 'singletons': 0, 'singletons_pct': '0.0', 'singletons_qc_failed': 0, 'total': 39356844, 'total_qc_failed': 0, 'with_itself': 32930940, 'with_itself_qc_failed': 0, '@id': '/samtools-flagstats-quality-metrics/172dd166-db6b-4c07-8a95-27a793cd5aed/', '@type': ['SamtoolsFlagstatsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '172dd166-db6b-4c07-8a95-27a793cd5aed', 'assay_term_id': 'OBI:0001271'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:10:13.497088+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF492MXX/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.82%', '% of reads mapped to too many loci': '0.39%', '% of reads unmapped: other': '4.02%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.86%', 'Average input read length': 92.0, 'Average mapped length': 92.78, 'Deletion average length': 2.08, 'Deletion rate per base': '0.06', 'Insertion average length': 2.1, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 156.36, 'Mismatch rate per base, %': '0.61%', 'Number of chimeric reads': 0.0, 'Number of input reads': 912076.0, 'Number of reads mapped to multiple loci': 116958.0, 'Number of reads mapped to too many loci': 3599.0, 'Number of splices: AT/AC': 25.0, 'Number of splices: Annotated (sjdb)': 52420.0, 'Number of splices: GC/AG': 561.0, 'Number of splices: GT/AG': 52843.0, 'Number of splices: Non-canonical': 29.0, 'Number of splices: Total': 53458.0, 'Uniquely mapped reads %': '65.91', 'Uniquely mapped reads number': 601145.0, '@id': '/star-quality-metrics/b08ac276-f1aa-4987-ac72-7ad6495f981c/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b08ac276-f1aa-4987-ac72-7ad6495f981c', 'read_depth': 718103.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:10:13.530806+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF492MXX/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 2361, 'fraction_of_unique_reads_in_cells': 0.411603, 'mean_genefull_ex50pas_per_cell': 30, 'mean_reads_per_cell': 86, 'mean_UMI_per_cell': 49, 'median_genefull_ex50pas_per_cell': 22, 'median_reads_per_cell': 57, 'median_UMI_per_cell': 33, 'number_of_reads': 912076, 'q30_bases_in_CB_UMI': 0.938519, 'q30_bases_in_rna_read': 0.934347, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.544576, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.571365, 'reads_mapped_to_genome_unique': 0.659095, 'reads_mapped_to_genome_unique_and_multiple': 0.787328, 'reads_with_valid_barcodes': 0.982131, 'sequencing_saturation': 0.392049, 'total_genefull_ex50pas_detected': 9475, 'umis_in_cells': 116601, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 204441, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '143869d534510979d9ed8eada00521b3', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/ad072534-1354-43e4-bd05-8ab917f57c37/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ad072534-1354-43e4-bd05-8ab917f57c37', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:35.458728+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/b29c49c0-9b91-413d-992e-54f46ee4c67e/', 'quality_metric_of': ['/files/ENCFF520CBN/'], 'reproducible_peaks': 161576, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/655867a1-d020-4434-b1df-6420c1034f2e/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '655867a1-d020-4434-b1df-6420c1034f2e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:35.537312+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/b29c49c0-9b91-413d-992e-54f46ee4c67e/', 'quality_metric_of': ['/files/ENCFF520CBN/'], 'frip': 0.34770108154411206, 'min_size': 345, '25_pct': 601, '50_pct': 859, '75_pct': 1355, 'max_size': 20019, 'mean': 1152.4247908105165, '@id': '/chip-peak-enrichment-quality-metrics/b7811cb0-ff09-44a1-a4b9-69d3b6aaa0e9/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b7811cb0-ff09-44a1-a4b9-69d3b6aaa0e9', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:13:29.441915+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF936HWS/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '11.69%', '% of reads mapped to too many loci': '0.43%', '% of reads unmapped: other': '4.48%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '19.22%', 'Average input read length': 90.0, 'Average mapped length': 91.4, 'Deletion average length': 2.03, 'Deletion rate per base': '0.06', 'Insertion average length': 2.13, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 195.97, 'Mismatch rate per base, %': '0.61%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1034279.0, 'Number of reads mapped to multiple loci': 120923.0, 'Number of reads mapped to too many loci': 4412.0, 'Number of splices: AT/AC': 25.0, 'Number of splices: Annotated (sjdb)': 54831.0, 'Number of splices: GC/AG': 521.0, 'Number of splices: GT/AG': 55386.0, 'Number of splices: Non-canonical': 27.0, 'Number of splices: Total': 55959.0, 'Uniquely mapped reads %': '64.18', 'Uniquely mapped reads number': 663831.0, '@id': '/star-quality-metrics/d60efd7e-9871-4185-a4ff-d420745c3f06/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd60efd7e-9871-4185-a4ff-d420745c3f06', 'read_depth': 784754.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:13:29.466415+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF936HWS/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 2012, 'fraction_of_unique_reads_in_cells': 0.385955, 'mean_genefull_ex50pas_per_cell': 35, 'mean_reads_per_cell': 103, 'mean_UMI_per_cell': 56, 'median_genefull_ex50pas_per_cell': 25, 'median_reads_per_cell': 68, 'median_UMI_per_cell': 37, 'number_of_reads': 1034279, 'q30_bases_in_CB_UMI': 0.937666, 'q30_bases_in_rna_read': 0.930696, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.521975, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.548352, 'reads_mapped_to_genome_unique': 0.64183, 'reads_mapped_to_genome_unique_and_multiple': 0.758745, 'reads_with_valid_barcodes': 0.98202, 'sequencing_saturation': 0.411797, 'total_genefull_ex50pas_detected': 9658, 'umis_in_cells': 112984, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 208365, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'a72899b122b10b950da7d29f1bd61bde', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/84624cbc-917a-4086-9707-49b6f9371ba8/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '84624cbc-917a-4086-9707-49b6f9371ba8', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:34.894625+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', 'quality_metric_of': ['/files/ENCFF328OCW/'], 'processing_stage': 'unfiltered', 'total_reads': 117789166, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 116666260, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 99.0, 'paired_reads': 117789166, 'paired_reads_qc_failed': 0, 'read1': 58894583, 'read1_qc_failed': 0, 'read2': 58894583, 'read2_qc_failed': 0, 'properly_paired_reads': 114796840, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.5, 'with_itself': 115872660, 'with_itself_qc_failed': 0, 'singletons': 793600, 'singletons_qc_failed': 0, 'pct_singletons': 0.7000000000000001, 'diff_chroms': 239564, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/588ac303-828a-451d-ae73-3b596e8d16cb/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '588ac303-828a-451d-ae73-3b596e8d16cb', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:35.118126+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', 'quality_metric_of': ['/files/ENCFF557DBY/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 345, 'corr_estimated_fragment_len': 0.11351013332728, 'phantom_peak': 50, 'corr_phantom_peak': 0.1152921, 'argmin_corr': 1500, 'min_corr': 0.109601, 'NSC': 1.035667, 'cross_correlation_plot': {'download': 'ENCFF591OUN.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF591OUN.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '01e0aa780537428925127470c4d13f46'}, 'auc': 0.17847832760930227, 'syn_auc': 0.49541910988990995, 'x_intercept': 0.15291278635035008, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7201213977535817, 'syn_elbow_pt': 0.5084412520619298, 'jsd': 0.1377195269340751, 'syn_jsd': 0.4384593295835177, 'pct_genome_enrich': 15.908663785002819, 'diff_enrich': 29.768946892297386, 'ch_div': 0.2560382668911925, 'jsd_plot': {'download': 'ENCFF591OUN.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF591OUN.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '5bedd3091caaff5c82dda43e9181ab00', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF591OUN.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF591OUN.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '550e2c3b8fe99e53c26d67088d4f6f5f', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/68229321-bd89-4a4c-b027-4febf63e003d/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '68229321-bd89-4a4c-b027-4febf63e003d', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:34.976818+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', 'quality_metric_of': ['/files/ENCFF557DBY/'], 'processing_stage': 'filtered', 'total_reads': 94316264, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 94316264, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 94316264, 'paired_reads_qc_failed': 0, 'read1': 47158132, 'read1_qc_failed': 0, 'read2': 47158132, 'read2_qc_failed': 0, 'properly_paired_reads': 94316264, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 94316264, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/1918313d-d589-44c0-81bf-eab81cfe22cd/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1918313d-d589-44c0-81bf-eab81cfe22cd', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 47158132}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T06:04:35.374161+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', 'quality_metric_of': ['/files/ENCFF557DBY/'], 'unpaired_reads': 0, 'paired_reads': 52474232, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 5316100, 'paired_optical_duplicate_reads': 68572, 'pct_duplicate_reads': 10.1309, 'total_fragments': 52467608, 'distinct_fragments': 47655315, 'positions_with_one_read': 3575471, 'NRF': 0.908281, 'PBC1': 0.914006, 'PBC2': 12.182235, '@id': '/chip-library-quality-metrics/024fac32-3d39-4891-a5f7-cc8fb8236fea/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '024fac32-3d39-4891-a5f7-cc8fb8236fea', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:37.290931+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/52b40ea5-4a37-49cd-a31d-11bda2aaec23/', 'quality_metric_of': ['/files/ENCFF971XDI/'], 'reproducible_peaks': 332075, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/04155e36-5e41-420e-92df-8d160169b300/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '04155e36-5e41-420e-92df-8d160169b300', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:37.374751+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/52b40ea5-4a37-49cd-a31d-11bda2aaec23/', 'quality_metric_of': ['/files/ENCFF971XDI/'], 'frip': 0.34241979541979783, 'min_size': 165, '25_pct': 238, '50_pct': 336, '75_pct': 526, 'max_size': 6615, 'mean': 448.2732063539863, '@id': '/chip-peak-enrichment-quality-metrics/dbf006a9-ec18-489d-a1f7-ffa3350c6225/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'dbf006a9-ec18-489d-a1f7-ffa3350c6225', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:36.665994+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', 'quality_metric_of': ['/files/ENCFF662GGD/'], 'processing_stage': 'unfiltered', 'total_reads': 124231244, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 122154104, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.3, 'paired_reads': 124231244, 'paired_reads_qc_failed': 0, 'read1': 62115622, 'read1_qc_failed': 0, 'read2': 62115622, 'read2_qc_failed': 0, 'properly_paired_reads': 120013208, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.6, 'with_itself': 120786674, 'with_itself_qc_failed': 0, 'singletons': 1367430, 'singletons_qc_failed': 0, 'pct_singletons': 1.0999999999999999, 'diff_chroms': 183237, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/a586c259-17b7-49f9-9245-673ca8a8256c/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a586c259-17b7-49f9-9245-673ca8a8256c', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:37.199582+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', 'quality_metric_of': ['/files/ENCFF233WHR/'], 'unpaired_reads': 0, 'paired_reads': 53968446, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 7262167, 'paired_optical_duplicate_reads': 17145, 'pct_duplicate_reads': 13.456299999999999, 'total_fragments': 53948947, 'distinct_fragments': 47022223, 'positions_with_one_read': 4217669, 'NRF': 0.871606, 'PBC1': 0.887908, 'PBC2': 9.89917, '@id': '/chip-library-quality-metrics/bd99f758-0e76-4fcc-ad1e-12ed74145043/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bd99f758-0e76-4fcc-ad1e-12ed74145043', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:36.749368+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', 'quality_metric_of': ['/files/ENCFF233WHR/'], 'processing_stage': 'filtered', 'total_reads': 93412558, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 93412558, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 93412558, 'paired_reads_qc_failed': 0, 'read1': 46706279, 'read1_qc_failed': 0, 'read2': 46706279, 'read2_qc_failed': 0, 'properly_paired_reads': 93412558, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 93412558, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/9d03dfcb-6951-43a5-9156-371bea33a2f7/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9d03dfcb-6951-43a5-9156-371bea33a2f7', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 46706279}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:51:36.880434+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', 'quality_metric_of': ['/files/ENCFF233WHR/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 165, 'corr_estimated_fragment_len': 0.111653881734379, 'phantom_peak': 40, 'corr_phantom_peak': 0.1150423, 'argmin_corr': 1500, 'min_corr': 0.1072047, 'NSC': 1.041502, 'cross_correlation_plot': {'download': 'ENCFF889EVJ.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF889EVJ.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '91133ed52ec087c5db16ebfa05807469'}, 'auc': 0.1844300962416185, 'syn_auc': 0.49538958297107777, 'x_intercept': 0.1576768131154096, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7055990403358824, 'syn_elbow_pt': 0.49703830392716225, 'jsd': 0.06876950520757998, 'syn_jsd': 0.4250529898688891, 'pct_genome_enrich': 19.243664717348928, 'diff_enrich': 33.32996018234281, 'ch_div': 0.2864076452496714, 'jsd_plot': {'download': 'ENCFF889EVJ.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF889EVJ.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '752dd7fe8f05fc02a6fc9629a2d490de', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF889EVJ.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF889EVJ.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '3851e749e38e4e4d59a66db32bc97a7e', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/39438a73-44c5-4307-bea4-2aca45acfa5b/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '39438a73-44c5-4307-bea4-2aca45acfa5b', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:10:01.598771+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF671RAV/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'e1979a4acb63114c6fdefb62bc853a8e', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'd61af94c7f8a5907cb62fb3a076fe704', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '3e3968f15306b5fbaeac23e9a7b73493', 'width': 1490, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/e7ac0b39-cfd3-48a1-b363-51233dc5f373/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e7ac0b39-cfd3-48a1-b363-51233dc5f373', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:09.677727+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF578OJY/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '795d048f9b215b734c480ee97075999c', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'aab953296b5cd81f196ba14fc8d02973', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '3d48e1c0b20d32df9c27fcd545ebc9a0', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/b40e5692-0aee-4946-b5a2-7764c23c8abd/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b40e5692-0aee-4946-b5a2-7764c23c8abd', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:07.776288+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF475LQE/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c9d8b1fb521a83c059c4485bb4130c05', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '7d0fae1092066a04980e6f2f6a6605bb', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '7375c34b9707fb1e479c4bee40ec1951', 'width': 1490, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/31595b71-51b6-4ba3-b452-f031b287546f/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '31595b71-51b6-4ba3-b452-f031b287546f', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:21.049516+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF066YPM/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '11.81%', '% of reads mapped to too many loci': '0.26%', '% of reads unmapped: other': '2.36%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '17.52%', 'Average input read length': 94.0, 'Average mapped length': 93.95, 'Deletion average length': 2.25, 'Deletion rate per base': '0.07', 'Insertion average length': 1.94, 'Insertion rate per base': '0.10', 'Mapping speed, Million of reads per hour': 245.76, 'Mismatch rate per base, %': '0.56%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1843231.0, 'Number of reads mapped to multiple loci': 217662.0, 'Number of reads mapped to too many loci': 4802.0, 'Number of splices: AT/AC': 25.0, 'Number of splices: Annotated (sjdb)': 112421.0, 'Number of splices: GC/AG': 945.0, 'Number of splices: GT/AG': 112944.0, 'Number of splices: Non-canonical': 68.0, 'Number of splices: Total': 113982.0, 'Uniquely mapped reads %': '68.05', 'Uniquely mapped reads number': 1254314.0, '@id': '/star-quality-metrics/1e5d32ed-2214-47ec-849e-e27e27d51dc4/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1e5d32ed-2214-47ec-849e-e27e27d51dc4', 'read_depth': 1471976.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:21.055252+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF066YPM/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1254, 'fraction_of_unique_reads_in_cells': 0.443258, 'mean_genefull_ex50pas_per_cell': 71, 'mean_reads_per_cell': 351, 'mean_UMI_per_cell': 187, 'median_genefull_ex50pas_per_cell': 51, 'median_reads_per_cell': 234, 'median_UMI_per_cell': 124, 'number_of_reads': 1843231, 'q30_bases_in_CB_UMI': 0.934718, 'q30_bases_in_rna_read': 0.937508, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.539824, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.568676, 'reads_mapped_to_genome_unique': 0.680497, 'reads_mapped_to_genome_unique_and_multiple': 0.798585, 'reads_with_valid_barcodes': 0.972765, 'sequencing_saturation': 0.412093, 'total_genefull_ex50pas_detected': 10424, 'umis_in_cells': 235286, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 441051, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '57b47f5aa296acb784de8409e05d4bd2', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/19b10b64-c566-47c1-b78a-bcc9357f71b2/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '19b10b64-c566-47c1-b78a-bcc9357f71b2', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.228252+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', 'quality_metric_of': ['/files/ENCFF448GYQ/'], 'processing_stage': 'filtered', 'total_reads': 157746794, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 157746794, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 157746794, 'paired_reads_qc_failed': 0, 'read1': 78873397, 'read1_qc_failed': 0, 'read2': 78873397, 'read2_qc_failed': 0, 'properly_paired_reads': 157746794, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 157746794, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/345cfc41-0af6-4431-b42b-7ac9856dc244/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '345cfc41-0af6-4431-b42b-7ac9856dc244', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 78873397}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.593765+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', 'quality_metric_of': ['/files/ENCFF448GYQ/'], 'unpaired_reads': 0, 'paired_reads': 84785177, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 5911780, 'paired_optical_duplicate_reads': 15918, 'pct_duplicate_reads': 6.9727, 'total_fragments': 84782661, 'distinct_fragments': 80586517, 'positions_with_one_read': 3106985, 'NRF': 0.950507, 'PBC1': 0.955935, 'PBC2': 24.79429, '@id': '/chip-library-quality-metrics/62e5efec-8930-481f-b2cb-a915dde167ad/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '62e5efec-8930-481f-b2cb-a915dde167ad', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.348287+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', 'quality_metric_of': ['/files/ENCFF448GYQ/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 445, 'corr_estimated_fragment_len': 0.10535265212252, 'phantom_peak': 45, 'corr_phantom_peak': 0.1046617, 'argmin_corr': 1500, 'min_corr': 0.1050916, 'NSC': 1.002484, 'cross_correlation_plot': {'download': 'ENCFF364PKV.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF364PKV.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': '6b8a2910f515ca1d96db1e402100c3b5'}, 'auc': 0.15580122149411882, 'syn_auc': 0.49626545579755493, 'x_intercept': 0.24758445172601995, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7011177596027165, 'syn_elbow_pt': 0.504973932818066, 'jsd': 0.13093390261177504, 'syn_jsd': 0.44814961297999373, 'pct_genome_enrich': 26.39919871570601, 'diff_enrich': 29.038664407141145, 'ch_div': 0.25942742613778685, 'jsd_plot': {'download': 'ENCFF364PKV.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF364PKV.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': 'cab01599e2fd15c6f8c06793b3399356', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF364PKV.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF364PKV.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'f10242028a38adbbdd344cd8c0adbce8', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/763705ea-cc8f-46e6-af3a-7c9bcf464436/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '763705ea-cc8f-46e6-af3a-7c9bcf464436', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:20.728545+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF619NMF/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.53%', '% of reads mapped to too many loci': '0.23%', '% of reads unmapped: other': '1.86%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '15.52%', 'Average input read length': 96.0, 'Average mapped length': 95.41, 'Deletion average length': 2.19, 'Deletion rate per base': '0.07', 'Insertion average length': 1.99, 'Insertion rate per base': '0.10', 'Mapping speed, Million of reads per hour': 294.7, 'Mismatch rate per base, %': '0.55%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1719096.0, 'Number of reads mapped to multiple loci': 215392.0, 'Number of reads mapped to too many loci': 3917.0, 'Number of splices: AT/AC': 35.0, 'Number of splices: Annotated (sjdb)': 110845.0, 'Number of splices: GC/AG': 882.0, 'Number of splices: GT/AG': 111520.0, 'Number of splices: Non-canonical': 54.0, 'Number of splices: Total': 112491.0, 'Uniquely mapped reads %': '69.87', 'Uniquely mapped reads number': 1201140.0, '@id': '/star-quality-metrics/f98f1286-9714-4d19-8642-8faba1eb1675/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f98f1286-9714-4d19-8642-8faba1eb1675', 'read_depth': 1416532.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:15:21.094376+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF619NMF/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1322, 'fraction_of_unique_reads_in_cells': 0.45542, 'mean_genefull_ex50pas_per_cell': 64, 'mean_reads_per_cell': 327, 'mean_UMI_per_cell': 184, 'median_genefull_ex50pas_per_cell': 50, 'median_reads_per_cell': 223, 'median_UMI_per_cell': 126, 'number_of_reads': 1719096, 'q30_bases_in_CB_UMI': 0.936503, 'q30_bases_in_rna_read': 0.939154, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.553284, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.582489, 'reads_mapped_to_genome_unique': 0.698704, 'reads_mapped_to_genome_unique_and_multiple': 0.823998, 'reads_with_valid_barcodes': 0.973779, 'sequencing_saturation': 0.394098, 'total_genefull_ex50pas_detected': 9840, 'umis_in_cells': 243386, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 433172, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'f59c6bf9b3fe45e6a37bb55eed1baaf5', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/58c1ee90-8c45-44dc-b11a-ddcbb2a797ae/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '58c1ee90-8c45-44dc-b11a-ddcbb2a797ae', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.682623+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a00e3f32-51fe-44de-8a5f-0a2309cf1cc7/', 'quality_metric_of': ['/files/ENCFF451FZM/'], 'reproducible_peaks': 157900, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/e81bb810-7a2c-46b9-abb2-676167586179/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e81bb810-7a2c-46b9-abb2-676167586179', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.765508+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/a00e3f32-51fe-44de-8a5f-0a2309cf1cc7/', 'quality_metric_of': ['/files/ENCFF451FZM/'], 'frip': 0.2982720713804174, 'min_size': 445, '25_pct': 790, '50_pct': 942, '75_pct': 1391, 'max_size': 18757, 'mean': 1212.5168334388854, '@id': '/chip-peak-enrichment-quality-metrics/8cd96c85-b44e-4c5c-a8c6-f44458bf8eca/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8cd96c85-b44e-4c5c-a8c6-f44458bf8eca', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:35:22.147727+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', 'quality_metric_of': ['/files/ENCFF796GUW/'], 'processing_stage': 'unfiltered', 'total_reads': 197743380, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 189154320, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 95.7, 'paired_reads': 197743380, 'paired_reads_qc_failed': 0, 'read1': 98871690, 'read1_qc_failed': 0, 'read2': 98871690, 'read2_qc_failed': 0, 'properly_paired_reads': 184056882, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 93.10000000000001, 'with_itself': 185167478, 'with_itself_qc_failed': 0, 'singletons': 3986842, 'singletons_qc_failed': 0, 'pct_singletons': 2.0, 'diff_chroms': 282267, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/7fd3a432-b63a-481d-a5c6-d0e66cdc752f/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7fd3a432-b63a-481d-a5c6-d0e66cdc752f', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:25.756618+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', 'quality_metric_of': ['/files/ENCFF258JTE/'], 'processing_stage': 'unfiltered', 'total_reads': 122600654, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 113942008, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 92.9, 'paired_reads': 122600654, 'paired_reads_qc_failed': 0, 'read1': 61300327, 'read1_qc_failed': 0, 'read2': 61300327, 'read2_qc_failed': 0, 'properly_paired_reads': 104778280, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 85.5, 'with_itself': 105786026, 'with_itself_qc_failed': 0, 'singletons': 8155982, 'singletons_qc_failed': 0, 'pct_singletons': 6.7, 'diff_chroms': 399611, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/d120b76b-13f9-47ca-ba02-a316765b7771/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd120b76b-13f9-47ca-ba02-a316765b7771', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:26.464837+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/69218a59-53c2-48f6-8283-ea41bed6834f/', 'quality_metric_of': ['/files/ENCFF692CMF/'], 'frip': 0.2934410055141526, 'min_size': 305, '25_pct': 481, '50_pct': 629, '75_pct': 929, 'max_size': 7670, 'mean': 786.614259380389, '@id': '/chip-peak-enrichment-quality-metrics/88bb7d7e-75d6-46c8-8c11-b42d3560480e/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '88bb7d7e-75d6-46c8-8c11-b42d3560480e', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:26.383134+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/69218a59-53c2-48f6-8283-ea41bed6834f/', 'quality_metric_of': ['/files/ENCFF692CMF/'], 'reproducible_peaks': 151353, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/031580a9-29af-4d56-b270-b009c17103a1/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '031580a9-29af-4d56-b270-b009c17103a1', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:25.982041+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', 'quality_metric_of': ['/files/ENCFF084WTX/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 305, 'corr_estimated_fragment_len': 0.112305900622117, 'phantom_peak': 50, 'corr_phantom_peak': 0.1071123, 'argmin_corr': 1500, 'min_corr': 0.1010632, 'NSC': 1.111244, 'cross_correlation_plot': {'download': 'ENCFF558PNA.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF558PNA.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'b7a27d582381b09ccda1b58b0df643d8'}, 'auc': 0.18565903883723886, 'syn_auc': 0.49522186749046093, 'x_intercept': 0.14626973098877372, 'syn_x_intercept': 0.0, 'elbow_pt': 0.7189127587895995, 'syn_elbow_pt': 0.5062321124615019, 'jsd': 0.16682343015253875, 'syn_jsd': 0.4295834282163387, 'pct_genome_enrich': 17.023782151889876, 'diff_enrich': 27.247536978423035, 'ch_div': 0.23340321685959117, 'jsd_plot': {'download': 'ENCFF558PNA.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF558PNA.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '604eba96cb055e932f4f105656601e90', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF558PNA.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF558PNA.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '6f400800dce9f1f91f37a9d5ee94b866', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/f58e111c-6116-41bc-8a45-2f43a7050ce7/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f58e111c-6116-41bc-8a45-2f43a7050ce7', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:25.865083+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', 'quality_metric_of': ['/files/ENCFF084WTX/'], 'processing_stage': 'filtered', 'total_reads': 90345342, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 90345342, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 90345342, 'paired_reads_qc_failed': 0, 'read1': 45172671, 'read1_qc_failed': 0, 'read2': 45172671, 'read2_qc_failed': 0, 'properly_paired_reads': 90345342, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 90345342, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/6df81a53-eec5-450a-abb6-895230f494e2/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6df81a53-eec5-450a-abb6-895230f494e2', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 45172671}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:14:26.292120+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', 'quality_metric_of': ['/files/ENCFF084WTX/'], 'unpaired_reads': 0, 'paired_reads': 49313214, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 4140543, 'paired_optical_duplicate_reads': 15869, 'pct_duplicate_reads': 8.3964, 'total_fragments': 49305497, 'distinct_fragments': 46529448, 'positions_with_one_read': 1856946, 'NRF': 0.943697, 'PBC1': 0.95292, 'PBC2': 23.877282, '@id': '/chip-library-quality-metrics/d7529f9a-751c-439b-9f1c-1b3f82297cf4/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd7529f9a-751c-439b-9f1c-1b3f82297cf4', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T01:13:29.906545+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF578WDG/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '70f8f89279d0679e702787548ab67bba', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '0e3163714b516cbccb0b3d69a1042b89', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '9dae1e5b07e966252b609976bd0f89f2', 'width': 1490, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/7cf8a4f9-145e-42ea-be6c-240a920d9be7/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7cf8a4f9-145e-42ea-be6c-240a920d9be7', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:09.956117+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/278135db-42ed-4a27-a7e4-0ae1dafe0093/', 'quality_metric_of': ['/files/ENCFF573LBE/'], 'reproducible_peaks': 79387, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/435af59e-0cfd-4dd5-814c-771ade2bb7f1/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '435af59e-0cfd-4dd5-814c-771ade2bb7f1', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:10.042255+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/278135db-42ed-4a27-a7e4-0ae1dafe0093/', 'quality_metric_of': ['/files/ENCFF573LBE/'], 'frip': 0.18740336113593753, 'min_size': 525, '25_pct': 941, '50_pct': 1079, '75_pct': 1500, 'max_size': 27197, 'mean': 1315.4195145300869, '@id': '/chip-peak-enrichment-quality-metrics/0a5ba1f2-88ae-4b20-808e-40073f88a1b6/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0a5ba1f2-88ae-4b20-808e-40073f88a1b6', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:09.874129+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', 'quality_metric_of': ['/files/ENCFF647KEC/'], 'unpaired_reads': 0, 'paired_reads': 40528174, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 3144146, 'paired_optical_duplicate_reads': 11862, 'pct_duplicate_reads': 7.757899999999999, 'total_fragments': 40493059, 'distinct_fragments': 37428082, 'positions_with_one_read': 2004359, 'NRF': 0.924309, 'PBC1': 0.938376, 'PBC2': 17.522619, '@id': '/chip-library-quality-metrics/0f786669-b48a-4c45-9131-9986b9e9cea1/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0f786669-b48a-4c45-9131-9986b9e9cea1', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:09.427210+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', 'quality_metric_of': ['/files/ENCFF647KEC/'], 'processing_stage': 'filtered', 'total_reads': 74768056, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 74768056, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 74768056, 'paired_reads_qc_failed': 0, 'read1': 37384028, 'read1_qc_failed': 0, 'read2': 37384028, 'read2_qc_failed': 0, 'properly_paired_reads': 74768056, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 74768056, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/a2e20b0e-6cc6-43c7-8764-8b5091e63e91/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a2e20b0e-6cc6-43c7-8764-8b5091e63e91', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 37384028}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:09.571811+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', 'quality_metric_of': ['/files/ENCFF647KEC/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 525, 'corr_estimated_fragment_len': 0.086317711325885, 'phantom_peak': 50, 'corr_phantom_peak': 0.08357858, 'argmin_corr': 1500, 'min_corr': 0.08595805, 'NSC': 1.004184, 'cross_correlation_plot': {'download': 'ENCFF322FSE.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF322FSE.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'eaad8d6a2fd2db4b98cafd1045d2f919'}, 'auc': 0.15472161030861672, 'syn_auc': 0.49469125576241735, 'x_intercept': 0.22757444528416565, 'syn_x_intercept': 1.007880275663255e-304, 'elbow_pt': 0.7260969358173086, 'syn_elbow_pt': 0.5019043362572931, 'jsd': 0.09425537723487856, 'syn_jsd': 0.4544489969203127, 'pct_genome_enrich': 18.440277827761182, 'diff_enrich': 22.020680354416406, 'ch_div': 0.18994937955585464, 'jsd_plot': {'download': 'ENCFF322FSE.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF322FSE.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '621ec6684065f6b6647d34d48b4bb952', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF322FSE.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF322FSE.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': 'cb31c61e83a5ce006275d7a391eae269', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/8f6f8b18-e4f9-4c01-a37e-24ccede733fd/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '8f6f8b18-e4f9-4c01-a37e-24ccede733fd', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-30T17:45:09.337091+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', 'quality_metric_of': ['/files/ENCFF811UFG/'], 'processing_stage': 'unfiltered', 'total_reads': 104835802, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 103429083, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.7, 'paired_reads': 104835802, 'paired_reads_qc_failed': 0, 'read1': 52417901, 'read1_qc_failed': 0, 'read2': 52417901, 'read2_qc_failed': 0, 'properly_paired_reads': 101473050, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.8, 'with_itself': 102657000, 'with_itself_qc_failed': 0, 'singletons': 772083, 'singletons_qc_failed': 0, 'pct_singletons': 0.7000000000000001, 'diff_chroms': 312479, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/9fcf5407-c8e1-42f3-b573-2c01b36ef72f/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9fcf5407-c8e1-42f3-b573-2c01b36ef72f', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.873484+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/284dc8fa-7727-40f3-afd2-54d5bf83e979/', 'quality_metric_of': ['/files/ENCFF546YHT/'], 'frip': 0.10552585138669393, 'min_size': 115, '25_pct': 149, '50_pct': 199, '75_pct': 275, 'max_size': 2200, 'mean': 229.41484022216278, '@id': '/chip-peak-enrichment-quality-metrics/c9fe2982-bae3-4b94-94f8-4f8df8813ef0/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c9fe2982-bae3-4b94-94f8-4f8df8813ef0', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.787527+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/284dc8fa-7727-40f3-afd2-54d5bf83e979/', 'quality_metric_of': ['/files/ENCFF546YHT/'], 'reproducible_peaks': 190671, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/962964e0-bac6-4a68-977a-06e2f23d2819/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '962964e0-bac6-4a68-977a-06e2f23d2819', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.454233+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', 'quality_metric_of': ['/files/ENCFF715KOC/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 115, 'corr_estimated_fragment_len': 0.124746924387593, 'phantom_peak': 50, 'corr_phantom_peak': 0.1252207, 'argmin_corr': 1500, 'min_corr': 0.1222902, 'NSC': 1.020089, 'cross_correlation_plot': {'download': 'ENCFF076QXB.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF076QXB.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'de4b4dc229140cff6193a5abe0a33339'}, 'auc': 0.24395622816287757, 'syn_auc': 0.49508213859787337, 'x_intercept': 0.12464212597165798, 'syn_x_intercept': 0.0, 'elbow_pt': 0.6238204152138447, 'syn_elbow_pt': 0.5040363255884575, 'jsd': 0.04168424326942246, 'syn_jsd': 0.32864706582301, 'pct_genome_enrich': 27.489723617286714, 'diff_enrich': 19.12916644111554, 'ch_div': 0.163391885172757, 'jsd_plot': {'download': 'ENCFF076QXB.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF076QXB.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': 'c919842d951483fbb51e386dec247777', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF076QXB.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF076QXB.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '907ced28cd7f0570d189cb4b33351895', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/6ef22b8c-aeeb-437f-a011-9f862b572dd0/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6ef22b8c-aeeb-437f-a011-9f862b572dd0', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.707469+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', 'quality_metric_of': ['/files/ENCFF715KOC/'], 'unpaired_reads': 0, 'paired_reads': 47340066, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 4987425, 'paired_optical_duplicate_reads': 18381, 'pct_duplicate_reads': 10.5353, 'total_fragments': 47334948, 'distinct_fragments': 44500744, 'positions_with_one_read': 1964202, 'NRF': 0.940124, 'PBC1': 0.948374, 'PBC2': 21.486252, '@id': '/chip-library-quality-metrics/08c061e7-6480-4b9a-ac85-94748d4fb4c6/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '08c061e7-6480-4b9a-ac85-94748d4fb4c6', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.320095+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', 'quality_metric_of': ['/files/ENCFF715KOC/'], 'processing_stage': 'filtered', 'total_reads': 84705282, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 84705282, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 84705282, 'paired_reads_qc_failed': 0, 'read1': 42352641, 'read1_qc_failed': 0, 'read2': 42352641, 'read2_qc_failed': 0, 'properly_paired_reads': 84705282, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 84705282, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/bf0e50ac-34c3-44b9-89de-15f379a77b91/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bf0e50ac-34c3-44b9-89de-15f379a77b91', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 42352641}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T05:33:58.232552+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', 'quality_metric_of': ['/files/ENCFF071MJY/'], 'processing_stage': 'unfiltered', 'total_reads': 113650626, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 109146157, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 96.0, 'paired_reads': 113650626, 'paired_reads_qc_failed': 0, 'read1': 56825313, 'read1_qc_failed': 0, 'read2': 56825313, 'read2_qc_failed': 0, 'properly_paired_reads': 103871952, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 91.4, 'with_itself': 104882514, 'with_itself_qc_failed': 0, 'singletons': 4263643, 'singletons_qc_failed': 0, 'pct_singletons': 3.8, 'diff_chroms': 390244, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/ab14da37-4281-495e-8c56-2264ddbac1f5/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ab14da37-4281-495e-8c56-2264ddbac1f5', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:56:19.929258+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF696YCS/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.80%', '% of reads mapped to too many loci': '0.16%', '% of reads unmapped: other': '0.69%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.80%', 'Average input read length': 105.0, 'Average mapped length': 103.65, 'Deletion average length': 2.16, 'Deletion rate per base': '0.07', 'Insertion average length': 1.96, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 478.27, 'Mismatch rate per base, %': '0.49%', 'Number of chimeric reads': 0.0, 'Number of input reads': 14215225.0, 'Number of reads mapped to multiple loci': 682813.0, 'Number of reads mapped to too many loci': 23333.0, 'Number of splices: AT/AC': 778.0, 'Number of splices: Annotated (sjdb)': 736178.0, 'Number of splices: GC/AG': 7858.0, 'Number of splices: GT/AG': 741496.0, 'Number of splices: Non-canonical': 697.0, 'Number of splices: Total': 750829.0, 'Uniquely mapped reads %': '88.54', 'Uniquely mapped reads number': 12586828.0, '@id': '/star-quality-metrics/b871f1a1-eeb6-4bd2-b9bf-3fda133d44f4/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b871f1a1-eeb6-4bd2-b9bf-3fda133d44f4', 'read_depth': 13269641.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:56:19.859539+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF696YCS/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 691, 'fraction_of_unique_reads_in_cells': 0.776624, 'mean_genefull_ex50pas_per_cell': 1852, 'mean_reads_per_cell': 12433, 'mean_UMI_per_cell': 4388, 'median_genefull_ex50pas_per_cell': 1576, 'median_reads_per_cell': 8601, 'median_UMI_per_cell': 3007, 'number_of_reads': 14215225, 'q30_bases_in_CB_UMI': 0.93503, 'q30_bases_in_rna_read': 0.935447, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.778242, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.800717, 'reads_mapped_to_genome_unique': 0.885447, 'reads_mapped_to_genome_unique_and_multiple': 0.933481, 'reads_with_valid_barcodes': 0.995869, 'sequencing_saturation': 0.643186, 'total_genefull_ex50pas_detected': 22616, 'umis_in_cells': 3032417, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 8591706, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'f935f7dd6a9aaf94af8a54cdc8f92e7d', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/549e0abf-b5b8-49f0-9567-079ed77eed50/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '549e0abf-b5b8-49f0-9567-079ed77eed50', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:57:31.645966+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF920CLG/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.17%', '% of reads mapped to too many loci': '0.16%', '% of reads unmapped: other': '0.64%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.25%', 'Average input read length': 103.0, 'Average mapped length': 102.41, 'Deletion average length': 2.16, 'Deletion rate per base': '0.07', 'Insertion average length': 1.96, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 481.18, 'Mismatch rate per base, %': '0.48%', 'Number of chimeric reads': 0.0, 'Number of input reads': 14435486.0, 'Number of reads mapped to multiple loci': 745871.0, 'Number of reads mapped to too many loci': 23311.0, 'Number of splices: AT/AC': 845.0, 'Number of splices: Annotated (sjdb)': 741875.0, 'Number of splices: GC/AG': 7849.0, 'Number of splices: GT/AG': 747581.0, 'Number of splices: Non-canonical': 536.0, 'Number of splices: Total': 756811.0, 'Uniquely mapped reads %': '87.78', 'Uniquely mapped reads number': 12672130.0, '@id': '/star-quality-metrics/0c439c30-7d08-4343-b2e8-d51330cd9937/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0c439c30-7d08-4343-b2e8-d51330cd9937', 'read_depth': 13418001.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:57:31.765276+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF920CLG/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 627, 'fraction_of_unique_reads_in_cells': 0.756547, 'mean_genefull_ex50pas_per_cell': 2103, 'mean_reads_per_cell': 13463, 'mean_UMI_per_cell': 5479, 'median_genefull_ex50pas_per_cell': 1782, 'median_reads_per_cell': 8547, 'median_UMI_per_cell': 3542, 'number_of_reads': 14435486, 'q30_bases_in_CB_UMI': 0.935421, 'q30_bases_in_rna_read': 0.935501, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.772981, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.795885, 'reads_mapped_to_genome_unique': 0.877846, 'reads_mapped_to_genome_unique_and_multiple': 0.929515, 'reads_with_valid_barcodes': 0.995046, 'sequencing_saturation': 0.589436, 'total_genefull_ex50pas_detected': 23090, 'umis_in_cells': 3435494, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 8441826, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'da6f02a2eeb96bcd199edac7c6464256', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/233d58b4-ce23-43e9-b5b5-f91adc41e6f2/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '233d58b4-ce23-43e9-b5b5-f91adc41e6f2', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:56:10.164243+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF624KQZ/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '6ba307523c372f120243e38ac7a24845', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c18c5a69123dc5418f9b3bc9b84bf633', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '2f0427325d1a8c231cabfbb9646b4d16', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/7de36fbd-41b7-4ac9-871d-b1c47cf778d0/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7de36fbd-41b7-4ac9-871d-b1c47cf778d0', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:57:27.465595+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF505IKH/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '08e26d5778e4c3073751fd8edc16f089', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '799f5045fe6fc6d9041fe383d14d80e9', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '86f63fa3b826c8cad3cf065578b8968a', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/c9433ae0-97ae-4650-8240-287354165b89/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c9433ae0-97ae-4650-8240-287354165b89', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:06.979290+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF655RIQ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 686, 'fraction_of_unique_reads_in_cells': 0.749156, 'mean_genefull_ex50pas_per_cell': 1904, 'mean_reads_per_cell': 12078, 'mean_UMI_per_cell': 4678, 'median_genefull_ex50pas_per_cell': 1598, 'median_reads_per_cell': 7712, 'median_UMI_per_cell': 3039, 'number_of_reads': 14240759, 'q30_bases_in_CB_UMI': 0.936216, 'q30_bases_in_rna_read': 0.935894, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.776654, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.799385, 'reads_mapped_to_genome_unique': 0.884736, 'reads_mapped_to_genome_unique_and_multiple': 0.935427, 'reads_with_valid_barcodes': 0.996395, 'sequencing_saturation': 0.606928, 'total_genefull_ex50pas_detected': 22950, 'umis_in_cells': 3209666, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 8285768, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'f2aa74c372d8f12e4786c32b2f20e7ab', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/7c033fb9-f2e8-46f7-9042-8167d77bdff3/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7c033fb9-f2e8-46f7-9042-8167d77bdff3', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:07.555491+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF655RIQ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.07%', '% of reads mapped to too many loci': '0.17%', '% of reads unmapped: other': '0.70%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.59%', 'Average input read length': 101.0, 'Average mapped length': 100.03, 'Deletion average length': 2.15, 'Deletion rate per base': '0.06', 'Insertion average length': 1.94, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 563.37, 'Mismatch rate per base, %': '0.48%', 'Number of chimeric reads': 0.0, 'Number of input reads': 14240759.0, 'Number of reads mapped to multiple loci': 721887.0, 'Number of reads mapped to too many loci': 24393.0, 'Number of splices: AT/AC': 815.0, 'Number of splices: Annotated (sjdb)': 740193.0, 'Number of splices: GC/AG': 8132.0, 'Number of splices: GT/AG': 745909.0, 'Number of splices: Non-canonical': 552.0, 'Number of splices: Total': 755408.0, 'Uniquely mapped reads %': '88.47', 'Uniquely mapped reads number': 12599310.0, '@id': '/star-quality-metrics/b28f9efc-0d0f-4ba1-ad09-e1487fff28dd/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b28f9efc-0d0f-4ba1-ad09-e1487fff28dd', 'read_depth': 13321197.0}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:07.780111+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF591BHA/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '78892547be4e022dffcc6bf49f5a95f0', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c50e874e74b68eb64550f5a0dae1fbd7', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'd7b19501ba74c13d7b1377f8705be113', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/3ab93e2c-cd08-45ec-90ea-8f459d618121/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3ab93e2c-cd08-45ec-90ea-8f459d618121', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:47.562378+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF635OWZ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 817, 'fraction_of_unique_reads_in_cells': 0.774567, 'mean_genefull_ex50pas_per_cell': 1650, 'mean_reads_per_cell': 10026, 'mean_UMI_per_cell': 3681, 'median_genefull_ex50pas_per_cell': 1356, 'median_reads_per_cell': 6783, 'median_UMI_per_cell': 2471, 'number_of_reads': 13522225, 'q30_bases_in_CB_UMI': 0.934565, 'q30_bases_in_rna_read': 0.935673, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.782107, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.805538, 'reads_mapped_to_genome_unique': 0.891497, 'reads_mapped_to_genome_unique_and_multiple': 0.938828, 'reads_with_valid_barcodes': 0.995757, 'sequencing_saturation': 0.629088, 'total_genefull_ex50pas_detected': 22725, 'umis_in_cells': 3007850, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 8191682, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'caebf98f1081d44c78713bf32d0a20f6', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/ee11f93f-e602-46b6-8d0a-c13637af1756/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ee11f93f-e602-46b6-8d0a-c13637af1756', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:48.262598+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF635OWZ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.73%', '% of reads mapped to too many loci': '0.17%', '% of reads unmapped: other': '0.70%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.24%', 'Average input read length': 103.0, 'Average mapped length': 101.56, 'Deletion average length': 2.16, 'Deletion rate per base': '0.07', 'Insertion average length': 1.94, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 491.72, 'Mismatch rate per base, %': '0.47%', 'Number of chimeric reads': 0.0, 'Number of input reads': 13522225.0, 'Number of reads mapped to multiple loci': 640012.0, 'Number of reads mapped to too many loci': 23578.0, 'Number of splices: AT/AC': 889.0, 'Number of splices: Annotated (sjdb)': 748055.0, 'Number of splices: GC/AG': 7858.0, 'Number of splices: GT/AG': 753447.0, 'Number of splices: Non-canonical': 568.0, 'Number of splices: Total': 762762.0, 'Uniquely mapped reads %': '89.15', 'Uniquely mapped reads number': 12055025.0, '@id': '/star-quality-metrics/ca057d57-3baa-4fa1-99a9-7a9561080d88/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ca057d57-3baa-4fa1-99a9-7a9561080d88', 'read_depth': 12695037.0}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T04:58:47.510640+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF218GZL/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b650eb1559cbc3e26f809ca73817855e', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'a05abea5f7061df839455863fdcb2959', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'e5000ec47e47171a488c1640a23360f4', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/73c6ee13-d195-4f3a-ac01-fc50eb693801/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '73c6ee13-d195-4f3a-ac01-fc50eb693801', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:00:57.406460+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF397GZZ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.04%', '% of reads mapped to too many loci': '0.16%', '% of reads unmapped: other': '0.67%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.47%', 'Average input read length': 104.0, 'Average mapped length': 103.15, 'Deletion average length': 2.16, 'Deletion rate per base': '0.07', 'Insertion average length': 1.96, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 602.52, 'Mismatch rate per base, %': '0.48%', 'Number of chimeric reads': 0.0, 'Number of input reads': 14226168.0, 'Number of reads mapped to multiple loci': 716426.0, 'Number of reads mapped to too many loci': 23232.0, 'Number of splices: AT/AC': 849.0, 'Number of splices: Annotated (sjdb)': 772646.0, 'Number of splices: GC/AG': 8042.0, 'Number of splices: GT/AG': 777964.0, 'Number of splices: Non-canonical': 633.0, 'Number of splices: Total': 787488.0, 'Uniquely mapped reads %': '88.65', 'Uniquely mapped reads number': 12612048.0, '@id': '/star-quality-metrics/155b4d32-c4b9-44f7-b47d-54e15ff7bb10/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '155b4d32-c4b9-44f7-b47d-54e15ff7bb10', 'read_depth': 13328474.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:00:57.463995+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF397GZZ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 685, 'fraction_of_unique_reads_in_cells': 0.746025, 'mean_genefull_ex50pas_per_cell': 1915, 'mean_reads_per_cell': 12068, 'mean_UMI_per_cell': 4673, 'median_genefull_ex50pas_per_cell': 1607, 'median_reads_per_cell': 8186, 'median_UMI_per_cell': 3134, 'number_of_reads': 14226168, 'q30_bases_in_CB_UMI': 0.938244, 'q30_bases_in_rna_read': 0.936804, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.77895, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.801463, 'reads_mapped_to_genome_unique': 0.886539, 'reads_mapped_to_genome_unique_and_multiple': 0.936898, 'reads_with_valid_barcodes': 0.995943, 'sequencing_saturation': 0.610095, 'total_genefull_ex50pas_detected': 22969, 'umis_in_cells': 3201034, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 8267049, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '2623d6b1ae854eeb200a1489f0beba66', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/0ba63c84-bbf2-47c2-a222-cbc74636350b/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0ba63c84-bbf2-47c2-a222-cbc74636350b', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:00:58.858763+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF160TOS/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '9d93573bf595486b8ce8d61bf8f1dda5', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c92b7fe4725dbeb3c723636ab0e07e2f', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'ba2bb454e6a5e96fd7f19128df752677', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/e480e7cc-675f-471f-8ee4-9a7385c30aff/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e480e7cc-675f-471f-8ee4-9a7385c30aff', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:01:36.662583+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF164YSE/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.68%', '% of reads mapped to too many loci': '0.16%', '% of reads unmapped: other': '0.65%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.80%', 'Average input read length': 104.0, 'Average mapped length': 103.36, 'Deletion average length': 2.16, 'Deletion rate per base': '0.07', 'Insertion average length': 1.97, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 557.35, 'Mismatch rate per base, %': '0.49%', 'Number of chimeric reads': 0.0, 'Number of input reads': 16255947.0, 'Number of reads mapped to multiple loci': 760015.0, 'Number of reads mapped to too many loci': 26088.0, 'Number of splices: AT/AC': 856.0, 'Number of splices: Annotated (sjdb)': 852605.0, 'Number of splices: GC/AG': 9134.0, 'Number of splices: GT/AG': 858761.0, 'Number of splices: Non-canonical': 644.0, 'Number of splices: Total': 869395.0, 'Uniquely mapped reads %': '88.71', 'Uniquely mapped reads number': 14421313.0, '@id': '/star-quality-metrics/eae6fb34-e113-4e95-8246-71e6fd867b1b/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'eae6fb34-e113-4e95-8246-71e6fd867b1b', 'read_depth': 15181328.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:01:36.774900+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF164YSE/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 692, 'fraction_of_unique_reads_in_cells': 0.776313, 'mean_genefull_ex50pas_per_cell': 2138, 'mean_reads_per_cell': 13921, 'mean_UMI_per_cell': 5658, 'median_genefull_ex50pas_per_cell': 1774, 'median_reads_per_cell': 8540, 'median_UMI_per_cell': 3475, 'number_of_reads': 16255947, 'q30_bases_in_CB_UMI': 0.933161, 'q30_bases_in_rna_read': 0.932377, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.763369, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.785645, 'reads_mapped_to_genome_unique': 0.887141, 'reads_mapped_to_genome_unique_and_multiple': 0.933894, 'reads_with_valid_barcodes': 0.972322, 'sequencing_saturation': 0.589998, 'total_genefull_ex50pas_detected': 23320, 'umis_in_cells': 3915907, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 9633485, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '923252ea4eae399585171706df1a2740', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/6e0400e6-1fd2-449b-aa63-1e97688ce1c6/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '6e0400e6-1fd2-449b-aa63-1e97688ce1c6', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:01:42.113330+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF248KCA/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '7ec0abaf6f7be3b0b71937eb591b5b36', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '459a722eb9e031e18715fc540e3c6568', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b0125dc16c0b3d02a546612704682195', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/58e2603b-e900-43ed-83eb-f5a9c29cd770/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '58e2603b-e900-43ed-83eb-f5a9c29cd770', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:06:34.437959+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF238CDU/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.55%', '% of reads mapped to too many loci': '0.19%', '% of reads unmapped: other': '0.80%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.24%', 'Average input read length': 104.0, 'Average mapped length': 102.89, 'Deletion average length': 2.18, 'Deletion rate per base': '0.06', 'Insertion average length': 1.94, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 392.0, 'Mismatch rate per base, %': '0.48%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2068909.0, 'Number of reads mapped to multiple loci': 94039.0, 'Number of reads mapped to too many loci': 3843.0, 'Number of splices: AT/AC': 134.0, 'Number of splices: Annotated (sjdb)': 101041.0, 'Number of splices: GC/AG': 1221.0, 'Number of splices: GT/AG': 101780.0, 'Number of splices: Non-canonical': 71.0, 'Number of splices: Total': 103206.0, 'Uniquely mapped reads %': '89.23', 'Uniquely mapped reads number': 1846024.0, '@id': '/star-quality-metrics/3b52e180-9114-4b27-ba60-9dc90316f05d/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3b52e180-9114-4b27-ba60-9dc90316f05d', 'read_depth': 1940063.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:06:34.508884+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF238CDU/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 255, 'fraction_of_unique_reads_in_cells': 0.706851, 'mean_genefull_ex50pas_per_cell': 823, 'mean_reads_per_cell': 4292, 'mean_UMI_per_cell': 1640, 'median_genefull_ex50pas_per_cell': 533, 'median_reads_per_cell': 2061, 'median_UMI_per_cell': 780, 'number_of_reads': 2068909, 'q30_bases_in_CB_UMI': 0.930261, 'q30_bases_in_rna_read': 0.936386, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.748502, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.770872, 'reads_mapped_to_genome_unique': 0.892269, 'reads_mapped_to_genome_unique_and_multiple': 0.937723, 'reads_with_valid_barcodes': 0.996832, 'sequencing_saturation': 0.611969, 'total_genefull_ex50pas_detected': 15731, 'umis_in_cells': 418454, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1094618, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'acc69f9d0c7bdc40a00763106ef96458', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/54a45761-129e-4f8d-9e14-f4a00a5781aa/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '54a45761-129e-4f8d-9e14-f4a00a5781aa', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:06:26.843924+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF032NUV/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'e785126692d0e8df4d04fcb184a6e36b', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'ffd25ad468b7f9110e79e17b7e7bc181', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '905587f5873253f77ef47b13b5aea746', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/9922d34f-3043-4c6a-aa25-5a7ac598617a/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9922d34f-3043-4c6a-aa25-5a7ac598617a', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:07:52.890386+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF511SEI/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.88%', '% of reads mapped to too many loci': '0.19%', '% of reads unmapped: other': '0.79%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.47%', 'Average input read length': 101.0, 'Average mapped length': 99.55, 'Deletion average length': 2.19, 'Deletion rate per base': '0.06', 'Insertion average length': 1.9, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 360.47, 'Mismatch rate per base, %': '0.47%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2002630.0, 'Number of reads mapped to multiple loci': 97725.0, 'Number of reads mapped to too many loci': 3902.0, 'Number of splices: AT/AC': 111.0, 'Number of splices: Annotated (sjdb)': 96172.0, 'Number of splices: GC/AG': 1085.0, 'Number of splices: GT/AG': 96996.0, 'Number of splices: Non-canonical': 61.0, 'Number of splices: Total': 98253.0, 'Uniquely mapped reads %': '88.66', 'Uniquely mapped reads number': 1775600.0, '@id': '/star-quality-metrics/df4afc96-f1a3-4cb5-9fba-1e5288874785/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'df4afc96-f1a3-4cb5-9fba-1e5288874785', 'read_depth': 1873325.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:07:52.958962+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF511SEI/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 260, 'fraction_of_unique_reads_in_cells': 0.713691, 'mean_genefull_ex50pas_per_cell': 861, 'mean_reads_per_cell': 4097, 'mean_UMI_per_cell': 1588, 'median_genefull_ex50pas_per_cell': 646, 'median_reads_per_cell': 2668, 'median_UMI_per_cell': 1015, 'number_of_reads': 2002630, 'q30_bases_in_CB_UMI': 0.927299, 'q30_bases_in_rna_read': 0.935859, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.745427, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.768917, 'reads_mapped_to_genome_unique': 0.886634, 'reads_mapped_to_genome_unique_and_multiple': 0.935432, 'reads_with_valid_barcodes': 0.997854, 'sequencing_saturation': 0.605104, 'total_genefull_ex50pas_detected': 15694, 'umis_in_cells': 413084, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1065408, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '77e1cc6b7489049bde6bbb0cc6df51fc', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/bbafd5c8-a2c0-4260-80bb-181074b8b1d9/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bbafd5c8-a2c0-4260-80bb-181074b8b1d9', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:07:51.198314+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF946ZTQ/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '1ef967411f12d8ad6b71cc3ab56366c7', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '7da027094a9a65c89e83d8bb1e5e8fee', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '49f46bbceb200cddadf7e314668ad7b1', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/538fe978-38d5-4c6c-b49a-554928f15c0d/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '538fe978-38d5-4c6c-b49a-554928f15c0d', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:09:13.963499+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF367IME/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '891d4c2c2ba07cb0ec8dfa77bff8563b', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '1676db380bb1ae13c6a460f84958dd41', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b6cae187893dc3cd53c05c7958dd2206', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/71468497-af66-446f-aa54-7a44755dfe6c/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '71468497-af66-446f-aa54-7a44755dfe6c', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:09:13.396967+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF403OMT/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 262, 'fraction_of_unique_reads_in_cells': 0.716471, 'mean_genefull_ex50pas_per_cell': 835, 'mean_reads_per_cell': 3848, 'mean_UMI_per_cell': 1555, 'median_genefull_ex50pas_per_cell': 640, 'median_reads_per_cell': 2329, 'median_UMI_per_cell': 945, 'number_of_reads': 1926457, 'q30_bases_in_CB_UMI': 0.923985, 'q30_bases_in_rna_read': 0.931943, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.730465, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.753779, 'reads_mapped_to_genome_unique': 0.881089, 'reads_mapped_to_genome_unique_and_multiple': 0.93023, 'reads_with_valid_barcodes': 0.975113, 'sequencing_saturation': 0.589166, 'total_genefull_ex50pas_detected': 15748, 'umis_in_cells': 407659, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1008225, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'f1d4a215675e5a3f1eefd526f98414ca', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/d2711e4f-1c5a-4306-901e-6203278efb6a/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd2711e4f-1c5a-4306-901e-6203278efb6a', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:09:15.022419+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF403OMT/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.91%', '% of reads mapped to too many loci': '0.18%', '% of reads unmapped: other': '0.71%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.08%', 'Average input read length': 104.0, 'Average mapped length': 102.5, 'Deletion average length': 2.18, 'Deletion rate per base': '0.07', 'Insertion average length': 1.95, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 365.01, 'Mismatch rate per base, %': '0.49%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1926457.0, 'Number of reads mapped to multiple loci': 94669.0, 'Number of reads mapped to too many loci': 3481.0, 'Number of splices: AT/AC': 115.0, 'Number of splices: Annotated (sjdb)': 96789.0, 'Number of splices: GC/AG': 1096.0, 'Number of splices: GT/AG': 97459.0, 'Number of splices: Non-canonical': 76.0, 'Number of splices: Total': 98746.0, 'Uniquely mapped reads %': '88.11', 'Uniquely mapped reads number': 1697380.0, '@id': '/star-quality-metrics/c62765b3-0fbe-4a95-af03-510f8dea71f1/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c62765b3-0fbe-4a95-af03-510f8dea71f1', 'read_depth': 1792049.0}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:10:43.892014+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF958PLV/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c59ae896b91e9d493c3132005b643e1f', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '9a8d68df8ddef844e4c8f632ca9606f8', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c8fce80b59b22894090bf82818800c32', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/a82d18e1-a465-44c9-83a7-9a7be60e1f6d/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'a82d18e1-a465-44c9-83a7-9a7be60e1f6d', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:10:38.763203+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF096MFD/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.76%', '% of reads mapped to too many loci': '0.18%', '% of reads unmapped: other': '0.78%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.96%', 'Average input read length': 104.0, 'Average mapped length': 102.79, 'Deletion average length': 2.18, 'Deletion rate per base': '0.07', 'Insertion average length': 1.96, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 374.76, 'Mismatch rate per base, %': '0.49%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1873808.0, 'Number of reads mapped to multiple loci': 89137.0, 'Number of reads mapped to too many loci': 3427.0, 'Number of splices: AT/AC': 102.0, 'Number of splices: Annotated (sjdb)': 97419.0, 'Number of splices: GC/AG': 1129.0, 'Number of splices: GT/AG': 98119.0, 'Number of splices: Non-canonical': 52.0, 'Number of splices: Total': 99402.0, 'Uniquely mapped reads %': '88.32', 'Uniquely mapped reads number': 1654889.0, '@id': '/star-quality-metrics/b461bf5d-cbb9-4820-bf09-86ba95ec8906/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b461bf5d-cbb9-4820-bf09-86ba95ec8906', 'read_depth': 1744026.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:10:43.174136+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF096MFD/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 214, 'fraction_of_unique_reads_in_cells': 0.691357, 'mean_genefull_ex50pas_per_cell': 834, 'mean_reads_per_cell': 4542, 'mean_UMI_per_cell': 1581, 'median_genefull_ex50pas_per_cell': 599, 'median_reads_per_cell': 2601, 'median_UMI_per_cell': 900, 'number_of_reads': 1873808, 'q30_bases_in_CB_UMI': 0.927782, 'q30_bases_in_rna_read': 0.934981, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.750339, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.773162, 'reads_mapped_to_genome_unique': 0.883169, 'reads_mapped_to_genome_unique_and_multiple': 0.930739, 'reads_with_valid_barcodes': 0.997783, 'sequencing_saturation': 0.644986, 'total_genefull_ex50pas_detected': 14954, 'umis_in_cells': 338370, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 972042, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '53e7d4ec6570288059ec154bf0f2a311', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/5d71974a-662e-4838-ada1-c3e8defbcf10/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5d71974a-662e-4838-ada1-c3e8defbcf10', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:12:15.512861+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF225GYZ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '5.56%', '% of reads mapped to too many loci': '0.19%', '% of reads unmapped: other': '0.72%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '6.14%', 'Average input read length': 103.0, 'Average mapped length': 101.74, 'Deletion average length': 2.22, 'Deletion rate per base': '0.07', 'Insertion average length': 1.91, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 379.78, 'Mismatch rate per base, %': '0.48%', 'Number of chimeric reads': 0.0, 'Number of input reads': 1687924.0, 'Number of reads mapped to multiple loci': 93919.0, 'Number of reads mapped to too many loci': 3200.0, 'Number of splices: AT/AC': 96.0, 'Number of splices: Annotated (sjdb)': 83303.0, 'Number of splices: GC/AG': 1073.0, 'Number of splices: GT/AG': 83838.0, 'Number of splices: Non-canonical': 63.0, 'Number of splices: Total': 85070.0, 'Uniquely mapped reads %': '87.38', 'Uniquely mapped reads number': 1474956.0, '@id': '/star-quality-metrics/76034fc6-ec18-4374-9ee5-efde8bcef1df/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '76034fc6-ec18-4374-9ee5-efde8bcef1df', 'read_depth': 1568875.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:12:15.642547+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF225GYZ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 230, 'fraction_of_unique_reads_in_cells': 0.664742, 'mean_genefull_ex50pas_per_cell': 786, 'mean_reads_per_cell': 3606, 'mean_UMI_per_cell': 1469, 'median_genefull_ex50pas_per_cell': 575, 'median_reads_per_cell': 2329, 'median_UMI_per_cell': 929, 'number_of_reads': 1687924, 'q30_bases_in_CB_UMI': 0.927717, 'q30_bases_in_rna_read': 0.935235, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.739284, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.762239, 'reads_mapped_to_genome_unique': 0.873828, 'reads_mapped_to_genome_unique_and_multiple': 0.92947, 'reads_with_valid_barcodes': 0.99641, 'sequencing_saturation': 0.587353, 'total_genefull_ex50pas_detected': 14977, 'umis_in_cells': 338031, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 829502, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '847d99fde7afdf60a004df404b63fc07', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/d52ba3bc-762e-485f-972b-d007d10a80c1/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd52ba3bc-762e-485f-972b-d007d10a80c1', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:12:12.169983+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF533UXG/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '2ffd0aecb48003bc8c8a3cfa72efe789', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'a11109599c2e89cbc5d567d50ad112bb', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '7e48884ea5e5958e5a330a2b42606cdc', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/3757cf3c-3681-46de-88f9-e33a9689fd72/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3757cf3c-3681-46de-88f9-e33a9689fd72', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:13:40.340183+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF451XCO/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '4.73%', '% of reads mapped to too many loci': '0.19%', '% of reads unmapped: other': '0.80%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '5.23%', 'Average input read length': 102.0, 'Average mapped length': 101.38, 'Deletion average length': 2.2, 'Deletion rate per base': '0.06', 'Insertion average length': 1.93, 'Insertion rate per base': '0.05', 'Mapping speed, Million of reads per hour': 384.33, 'Mismatch rate per base, %': '0.47%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2241909.0, 'Number of reads mapped to multiple loci': 106025.0, 'Number of reads mapped to too many loci': 4202.0, 'Number of splices: AT/AC': 141.0, 'Number of splices: Annotated (sjdb)': 112612.0, 'Number of splices: GC/AG': 1246.0, 'Number of splices: GT/AG': 113473.0, 'Number of splices: Non-canonical': 75.0, 'Number of splices: Total': 114935.0, 'Uniquely mapped reads %': '89.05', 'Uniquely mapped reads number': 1996514.0, '@id': '/star-quality-metrics/ed0bce47-187f-4b6f-9e02-1e8d637a01ca/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ed0bce47-187f-4b6f-9e02-1e8d637a01ca', 'read_depth': 2102539.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:13:40.515640+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF451XCO/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 258, 'fraction_of_unique_reads_in_cells': 0.719833, 'mean_genefull_ex50pas_per_cell': 909, 'mean_reads_per_cell': 4720, 'mean_UMI_per_cell': 1717, 'median_genefull_ex50pas_per_cell': 655, 'median_reads_per_cell': 2832, 'median_UMI_per_cell': 1036, 'number_of_reads': 2241909, 'q30_bases_in_CB_UMI': 0.925, 'q30_bases_in_rna_read': 0.9356, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.754637, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.77788, 'reads_mapped_to_genome_unique': 0.890542, 'reads_mapped_to_genome_unique_and_multiple': 0.937834, 'reads_with_valid_barcodes': 0.997078, 'sequencing_saturation': 0.630837, 'total_genefull_ex50pas_detected': 16072, 'umis_in_cells': 443058, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1217834, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'e3183abf190251e0b29c2c0c71e5e64c', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/c17a5f9a-04ff-48d0-8583-80f7a9e406ff/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c17a5f9a-04ff-48d0-8583-80f7a9e406ff', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:13:40.498544+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF421VUX/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '99f0cdbe223e718d657847cb7271c75e', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'dc672983ed2495ee48a60309e2269c8d', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '86233e40fde682d74608e2069aa6c21e', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/40fdee79-e5b4-47d3-95a5-136fc46148bf/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '40fdee79-e5b4-47d3-95a5-136fc46148bf', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep1_novaseq_alignment_quality_metric'], 'date_created': '2022-02-10T08:16:55.162693+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', 'quality_metric_of': ['/files/ENCFF270IYD/'], 'total_rp': 312775182, 'rp_bl': 287765104, 'frp_bl': 0.92, 'pet': 213313272, 'um_pet': 176009898, 'nr_pet': 19773106, 'pet_red': 0.89, '@id': '/chia-pet-alignment-quality-metrics/d08ade37-3e41-4106-848a-fae66a29377a/', '@type': ['ChiaPetAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'd08ade37-3e41-4106-848a-fae66a29377a'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep1_novaseq_loop_chr_interactions_quality_metric'], 'date_created': '2022-02-10T08:21:48.253026+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', 'quality_metric_of': ['/files/ENCFF571CCD/'], 'intra_inter_pet_ratio': 8.9, 'intra_inter_clust_pet2_ratio': 11.72, 'intra_inter_clust_pet5_ratio': 17.5, 'intra_inter_clust_pet10_ratio': 38.78, '@id': '/chia-pet-chr-interactions-quality-metrics/fab21c1a-a614-4770-af82-7b5fee9b59c7/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'fab21c1a-a614-4770-af82-7b5fee9b59c7'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep2_novaseq_loop_chr_interactions_quality_metric'], 'date_created': '2022-02-10T08:21:48.622103+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', 'quality_metric_of': ['/files/ENCFF721MZJ/'], 'intra_inter_pet_ratio': 9.88, 'intra_inter_clust_pet2_ratio': 13.39, 'intra_inter_clust_pet5_ratio': 22.63, 'intra_inter_clust_pet10_ratio': 66.85, '@id': '/chia-pet-chr-interactions-quality-metrics/b7e2273c-9c45-4063-a16f-19b08a2adc3b/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b7e2273c-9c45-4063-a16f-19b08a2adc3b'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep1_novaseq_peak_enrichment_quality_metric'], 'date_created': '2022-02-10T08:18:49.730672+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', 'quality_metric_of': ['/files/ENCFF061XGQ/'], 'binding_peaks': 88181, '@id': '/chia-pet-peak-enrichment-quality-metrics/ea943d69-310d-49b4-889e-70fc3fca9f2f/', '@type': ['ChiaPetPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ea943d69-310d-49b4-889e-70fc3fca9f2f'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep2_novaseq_alignment_quality_metric'], 'date_created': '2022-02-10T08:16:55.511761+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', 'quality_metric_of': ['/files/ENCFF819XZY/'], 'total_rp': 384483914, 'rp_bl': 341554910, 'frp_bl': 0.89, 'pet': 254070637, 'um_pet': 209747280, 'nr_pet': 29236470, 'pet_red': 0.86, '@id': '/chia-pet-alignment-quality-metrics/18fb0e50-2cc1-44d7-abb0-02a144c65974/', '@type': ['ChiaPetAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '18fb0e50-2cc1-44d7-abb0-02a144c65974'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep2_novaseq_hic_chr_interactions_quality_metric'], 'date_created': '2022-02-10T08:21:47.907795+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', 'quality_metric_of': ['/files/ENCFF838USQ/'], 'intra_inter_pet_ratio': 9.88, 'intra_inter_clust_pet2_ratio': 13.39, 'intra_inter_clust_pet5_ratio': 22.63, 'intra_inter_clust_pet10_ratio': 66.85, '@id': '/chia-pet-chr-interactions-quality-metrics/30c4019f-0b24-4f72-ae99-772836b48e9d/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '30c4019f-0b24-4f72-ae99-772836b48e9d'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep1_novaseq_hic_chr_interactions_quality_metric'], 'date_created': '2022-02-10T08:21:47.559213+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', 'quality_metric_of': ['/files/ENCFF987BJZ/'], 'intra_inter_pet_ratio': 8.9, 'intra_inter_clust_pet2_ratio': 11.72, 'intra_inter_clust_pet5_ratio': 17.5, 'intra_inter_clust_pet10_ratio': 38.78, '@id': '/chia-pet-chr-interactions-quality-metrics/908e8f44-62ab-4fc0-92f2-98688312a1f1/', '@type': ['ChiaPetChrInteractionsQualityMetric', 'QualityMetric', 'Item'], 'uuid': '908e8f44-62ab-4fc0-92f2-98688312a1f1'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/charles-lee/', 'award': '/awards/3UM1HG009409-04S1/', 'aliases': ['charles-lee:HCT116-SMARCA5-Untreated_CTCF_rep2_novaseq_peak_enrichment_quality_metric'], 'date_created': '2022-02-10T08:18:50.081815+00:00', 'submitted_by': '/users/71ab6baf-8f48-4cb8-bb38-70db6114d90d/', 'schema_version': '1', 'step_run': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', 'quality_metric_of': ['/files/ENCFF385INS/'], 'binding_peaks': 96931, '@id': '/chia-pet-peak-enrichment-quality-metrics/b921fb2b-ac29-450b-ae49-9732ccbfa8b4/', '@type': ['ChiaPetPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b921fb2b-ac29-450b-ae49-9732ccbfa8b4'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:57:06.537948+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF713NQQ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 534, 'fraction_of_unique_reads_in_cells': 0.647314, 'mean_genefull_ex50pas_per_cell': 102, 'mean_reads_per_cell': 1853, 'mean_UMI_per_cell': 392, 'median_genefull_ex50pas_per_cell': 81, 'median_reads_per_cell': 1131, 'median_UMI_per_cell': 237, 'number_of_reads': 2696250, 'q30_bases_in_CB_UMI': 0.935249, 'q30_bases_in_rna_read': 0.933574, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.567158, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.613425, 'reads_mapped_to_genome_unique': 0.676939, 'reads_mapped_to_genome_unique_and_multiple': 0.852771, 'reads_with_valid_barcodes': 0.983499, 'sequencing_saturation': 0.777407, 'total_genefull_ex50pas_detected': 9007, 'umis_in_cells': 209598, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 989872, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '098f176142d138bbaf2bc410459a9ac8', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/9f2e87bf-5a32-4ae6-a4ad-511a48402527/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9f2e87bf-5a32-4ae6-a4ad-511a48402527', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:57:06.807421+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF713NQQ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '17.58%', '% of reads mapped to too many loci': '0.12%', '% of reads unmapped: other': '0.81%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '13.80%', 'Average input read length': 97.0, 'Average mapped length': 95.04, 'Deletion average length': 1.95, 'Deletion rate per base': '0.07', 'Insertion average length': 2.16, 'Insertion rate per base': '0.11', 'Mapping speed, Million of reads per hour': 303.33, 'Mismatch rate per base, %': '0.72%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2696250.0, 'Number of reads mapped to multiple loci': 474087.0, 'Number of reads mapped to too many loci': 3142.0, 'Number of splices: AT/AC': 45.0, 'Number of splices: Annotated (sjdb)': 141288.0, 'Number of splices: GC/AG': 1388.0, 'Number of splices: GT/AG': 141906.0, 'Number of splices: Non-canonical': 73.0, 'Number of splices: Total': 143412.0, 'Uniquely mapped reads %': '67.69', 'Uniquely mapped reads number': 1825197.0, '@id': '/star-quality-metrics/2a70b712-9edc-4a23-8bc1-3ea1f3ad59df/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '2a70b712-9edc-4a23-8bc1-3ea1f3ad59df', 'read_depth': 2299284.0}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:57:01.932495+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF981TWA/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b268f12f7c03b83be9201f31343a7396', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '2f5273249eca2cb496ef11129895b005', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '89f424d2c317361cb6d0a21ef3253766', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/35a8cbbe-3dc8-4de8-ac97-dd8858069dcb/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '35a8cbbe-3dc8-4de8-ac97-dd8858069dcb', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:58:16.570674+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF810APW/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '16.43%', '% of reads mapped to too many loci': '0.16%', '% of reads unmapped: other': '1.28%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.18%', 'Average input read length': 98.0, 'Average mapped length': 95.79, 'Deletion average length': 1.95, 'Deletion rate per base': '0.07', 'Insertion average length': 2.15, 'Insertion rate per base': '0.10', 'Mapping speed, Million of reads per hour': 275.42, 'Mismatch rate per base, %': '0.72%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2295150.0, 'Number of reads mapped to multiple loci': 377091.0, 'Number of reads mapped to too many loci': 3604.0, 'Number of splices: AT/AC': 20.0, 'Number of splices: Annotated (sjdb)': 108136.0, 'Number of splices: GC/AG': 1242.0, 'Number of splices: GT/AG': 108676.0, 'Number of splices: Non-canonical': 59.0, 'Number of splices: Total': 109997.0, 'Uniquely mapped reads %': '65.95', 'Uniquely mapped reads number': 1513741.0, '@id': '/star-quality-metrics/91d437c9-bbeb-4ce9-b624-586201f775b7/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '91d437c9-bbeb-4ce9-b624-586201f775b7', 'read_depth': 1890832.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:58:27.022001+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF810APW/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 486, 'fraction_of_unique_reads_in_cells': 0.542354, 'mean_genefull_ex50pas_per_cell': 126, 'mean_reads_per_cell': 1380, 'mean_UMI_per_cell': 532, 'median_genefull_ex50pas_per_cell': 80, 'median_reads_per_cell': 777, 'median_UMI_per_cell': 291, 'number_of_reads': 2295150, 'q30_bases_in_CB_UMI': 0.933505, 'q30_bases_in_rna_read': 0.932989, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.538894, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.577472, 'reads_mapped_to_genome_unique': 0.659539, 'reads_mapped_to_genome_unique_and_multiple': 0.823838, 'reads_with_valid_barcodes': 0.983923, 'sequencing_saturation': 0.582005, 'total_genefull_ex50pas_detected': 9784, 'umis_in_cells': 258766, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 670807, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '757f8bcf1fe1aab1b7be251f500503d6', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/be0d781e-2d02-45db-84ae-dbfc1654c2cd/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'be0d781e-2d02-45db-84ae-dbfc1654c2cd', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:58:16.507946+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF376DKS/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '6d92423eab05ee95872505d32bafe67a', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '88dff138f635a4c17680ebddfc4399e2', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '6f4e4b32d946d46bdacebc70d8fef7e5', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/73f0d1c6-d22b-4682-b248-966a1aa55e08/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '73f0d1c6-d22b-4682-b248-966a1aa55e08', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:59:42.234822+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF529ONA/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '107565908105216db06e766337d96d55', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '9975f85ea7a54ec0f04468786235c0b1', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '2c6fc112aeac0a0767527dcdec5916f1', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/e69a92c8-b181-4086-b558-542863b24936/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e69a92c8-b181-4086-b558-542863b24936', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:01:09.666047+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF899NSP/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c3ff955baaa2f8dd7a4929623925a749', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '4cda068a9166a3acb8bd23df8d182e4a', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '78510c8debfcf0a881aaaf22a4e1a4a0', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/76eea8a3-0e6f-4c1d-8d80-1122f10a2a28/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '76eea8a3-0e6f-4c1d-8d80-1122f10a2a28', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:27.104118+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF615EOG/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 619, 'fraction_of_unique_reads_in_cells': 0.620511, 'mean_genefull_ex50pas_per_cell': 97, 'mean_reads_per_cell': 1300, 'mean_UMI_per_cell': 344, 'median_genefull_ex50pas_per_cell': 81, 'median_reads_per_cell': 800, 'median_UMI_per_cell': 215, 'number_of_reads': 2336536, 'q30_bases_in_CB_UMI': 0.929472, 'q30_bases_in_rna_read': 0.93286, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.555141, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.59832, 'reads_mapped_to_genome_unique': 0.666384, 'reads_mapped_to_genome_unique_and_multiple': 0.836931, 'reads_with_valid_barcodes': 0.982321, 'sequencing_saturation': 0.71746, 'total_genefull_ex50pas_detected': 9315, 'umis_in_cells': 213119, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 804870, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '2a01d881dcbfb4deb20b9cf24e33ddec', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/587abff9-7b7d-43f7-be9e-9b9e00a2e5e8/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '587abff9-7b7d-43f7-be9e-9b9e00a2e5e8', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:27.209198+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF615EOG/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '17.05%', '% of reads mapped to too many loci': '0.13%', '% of reads unmapped: other': '1.05%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '15.12%', 'Average input read length': 97.0, 'Average mapped length': 94.56, 'Deletion average length': 1.93, 'Deletion rate per base': '0.07', 'Insertion average length': 2.2, 'Insertion rate per base': '0.11', 'Mapping speed, Million of reads per hour': 271.34, 'Mismatch rate per base, %': '0.72%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2336536.0, 'Number of reads mapped to multiple loci': 398490.0, 'Number of reads mapped to too many loci': 3101.0, 'Number of splices: AT/AC': 38.0, 'Number of splices: Annotated (sjdb)': 116924.0, 'Number of splices: GC/AG': 1287.0, 'Number of splices: GT/AG': 117356.0, 'Number of splices: Non-canonical': 53.0, 'Number of splices: Total': 118734.0, 'Uniquely mapped reads %': '66.64', 'Uniquely mapped reads number': 1557030.0, '@id': '/star-quality-metrics/7ad8bf55-2375-4017-92d9-6317fee3fb9f/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7ad8bf55-2375-4017-92d9-6317fee3fb9f', 'read_depth': 1955520.0}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:59:41.691933+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF107VVR/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '16.99%', '% of reads mapped to too many loci': '0.15%', '% of reads unmapped: other': '1.12%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.26%', 'Average input read length': 99.0, 'Average mapped length': 96.17, 'Deletion average length': 1.94, 'Deletion rate per base': '0.06', 'Insertion average length': 2.2, 'Insertion rate per base': '0.11', 'Mapping speed, Million of reads per hour': 260.59, 'Mismatch rate per base, %': '0.72%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2171600.0, 'Number of reads mapped to multiple loci': 369023.0, 'Number of reads mapped to too many loci': 3216.0, 'Number of splices: AT/AC': 39.0, 'Number of splices: Annotated (sjdb)': 108493.0, 'Number of splices: GC/AG': 1225.0, 'Number of splices: GT/AG': 108879.0, 'Number of splices: Non-canonical': 40.0, 'Number of splices: Total': 110183.0, 'Uniquely mapped reads %': '65.48', 'Uniquely mapped reads number': 1421915.0, '@id': '/star-quality-metrics/9a0f26ff-f69d-49fc-95b9-82f00301bfd4/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '9a0f26ff-f69d-49fc-95b9-82f00301bfd4', 'read_depth': 1790938.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T05:59:41.730751+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF107VVR/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 684, 'fraction_of_unique_reads_in_cells': 0.591624, 'mean_genefull_ex50pas_per_cell': 98, 'mean_reads_per_cell': 1013, 'mean_UMI_per_cell': 366, 'median_genefull_ex50pas_per_cell': 84, 'median_reads_per_cell': 641, 'median_UMI_per_cell': 226, 'number_of_reads': 2171600, 'q30_bases_in_CB_UMI': 0.933133, 'q30_bases_in_rna_read': 0.933075, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.539501, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.578595, 'reads_mapped_to_genome_unique': 0.654778, 'reads_mapped_to_genome_unique_and_multiple': 0.824709, 'reads_with_valid_barcodes': 0.982673, 'sequencing_saturation': 0.611677, 'total_genefull_ex50pas_detected': 9420, 'umis_in_cells': 250668, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 693136, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '91493f26ffc9e19d355fdb7f18238e2a', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/41458825-5df0-4dec-96c0-88c51d03756c/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '41458825-5df0-4dec-96c0-88c51d03756c', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:01:08.810927+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF595JZA/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 683, 'fraction_of_unique_reads_in_cells': 0.637659, 'mean_genefull_ex50pas_per_cell': 97, 'mean_reads_per_cell': 1179, 'mean_UMI_per_cell': 333, 'median_genefull_ex50pas_per_cell': 75, 'median_reads_per_cell': 702, 'median_UMI_per_cell': 192, 'number_of_reads': 2287729, 'q30_bases_in_CB_UMI': 0.932966, 'q30_bases_in_rna_read': 0.932759, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.55244, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.593041, 'reads_mapped_to_genome_unique': 0.659137, 'reads_mapped_to_genome_unique_and_multiple': 0.830761, 'reads_with_valid_barcodes': 0.982201, 'sequencing_saturation': 0.698149, 'total_genefull_ex50pas_detected': 9679, 'umis_in_cells': 228116, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 805895, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '84e4547ab531b04a6183f40cb282eb7f', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/cc55c968-1dac-4a7f-ba73-c3bc80c844a8/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'cc55c968-1dac-4a7f-ba73-c3bc80c844a8', 'assay_term_id': 'OBI:0002631'}, {'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:01:08.987060+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF595JZA/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '17.16%', '% of reads mapped to too many loci': '0.14%', '% of reads unmapped: other': '1.18%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '15.60%', 'Average input read length': 98.0, 'Average mapped length': 95.77, 'Deletion average length': 1.94, 'Deletion rate per base': '0.07', 'Insertion average length': 2.17, 'Insertion rate per base': '0.10', 'Mapping speed, Million of reads per hour': 274.53, 'Mismatch rate per base, %': '0.72%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2287729.0, 'Number of reads mapped to multiple loci': 392630.0, 'Number of reads mapped to too many loci': 3148.0, 'Number of splices: AT/AC': 43.0, 'Number of splices: Annotated (sjdb)': 117324.0, 'Number of splices: GC/AG': 1280.0, 'Number of splices: GT/AG': 117959.0, 'Number of splices: Non-canonical': 45.0, 'Number of splices: Total': 119327.0, 'Uniquely mapped reads %': '65.91', 'Uniquely mapped reads number': 1507926.0, '@id': '/star-quality-metrics/e0e4a535-b256-45cf-9e60-a6e97bbe4baf/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'e0e4a535-b256-45cf-9e60-a6e97bbe4baf', 'read_depth': 1900556.0}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:21.953312+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF999GQZ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '11.59%', '% of reads mapped to too many loci': '0.24%', '% of reads unmapped: other': '2.54%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.98%', 'Average input read length': 96.0, 'Average mapped length': 94.71, 'Deletion average length': 1.9, 'Deletion rate per base': '0.07', 'Insertion average length': 2.21, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 314.32, 'Mismatch rate per base, %': '0.68%', 'Number of chimeric reads': 0.0, 'Number of input reads': 3230466.0, 'Number of reads mapped to multiple loci': 374435.0, 'Number of reads mapped to too many loci': 7719.0, 'Number of splices: AT/AC': 66.0, 'Number of splices: Annotated (sjdb)': 197096.0, 'Number of splices: GC/AG': 2214.0, 'Number of splices: GT/AG': 198033.0, 'Number of splices: Non-canonical': 63.0, 'Number of splices: Total': 200376.0, 'Uniquely mapped reads %': '68.65', 'Uniquely mapped reads number': 2217675.0, '@id': '/star-quality-metrics/83d2ebb4-25b1-48c3-a0d0-5f6c94faf6ee/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '83d2ebb4-25b1-48c3-a0d0-5f6c94faf6ee', 'read_depth': 2592110.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:21.996674+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF999GQZ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 976, 'fraction_of_unique_reads_in_cells': 0.52137, 'mean_genefull_ex50pas_per_cell': 134, 'mean_reads_per_cell': 1019, 'mean_UMI_per_cell': 272, 'median_genefull_ex50pas_per_cell': 88, 'median_reads_per_cell': 640, 'median_UMI_per_cell': 173, 'number_of_reads': 3230466, 'q30_bases_in_CB_UMI': 0.93138, 'q30_bases_in_rna_read': 0.931655, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.591, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.615453, 'reads_mapped_to_genome_unique': 0.686488, 'reads_mapped_to_genome_unique_and_multiple': 0.802395, 'reads_with_valid_barcodes': 0.984117, 'sequencing_saturation': 0.706271, 'total_genefull_ex50pas_detected': 12940, 'umis_in_cells': 266211, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 995402, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'a1925fee18bfaa01006b87add060a0c6', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/ee45d706-c25a-4c6c-aed4-f37066e5151f/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ee45d706-c25a-4c6c-aed4-f37066e5151f', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:19.289453+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF706SDC/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'f0d0465f4fab6dd3eb0db21ab94cd2c4', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '95e84290b0e65436c1d9631536c5c2c0', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b5106e0a162e98d9fabae44b7299ae7b', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/63ba1a3c-1ac8-4e46-9363-cedd9edd0086/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '63ba1a3c-1ac8-4e46-9363-cedd9edd0086', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:47.993464+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF960QUW/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '54fb5b33ab85b63cb85e20bbb562ca91', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'ca947d699990c37c9651ae4f388cc10c', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '43a7bf0204f4cdc7009152e48916e252', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/f6577233-8e1a-4739-8c40-f2ee6e1517ec/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'f6577233-8e1a-4739-8c40-f2ee6e1517ec', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-26T06:03:21.441507+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF594HKT/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'b0f5b4c6399e37b129b6ad500effc033', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'ee926fde0c6ce64ce09e807ed2639932', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'bdc62bdce6c925b92ecfdc87f0b01b8b', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/c6b545fa-2813-4ef2-89bb-72147f21bcf5/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c6b545fa-2813-4ef2-89bb-72147f21bcf5', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:48.269026+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF513TDF/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '11.54%', '% of reads mapped to too many loci': '0.23%', '% of reads unmapped: other': '2.42%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '16.33%', 'Average input read length': 95.0, 'Average mapped length': 93.68, 'Deletion average length': 1.9, 'Deletion rate per base': '0.07', 'Insertion average length': 2.16, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 291.92, 'Mismatch rate per base, %': '0.66%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2919170.0, 'Number of reads mapped to multiple loci': 336919.0, 'Number of reads mapped to too many loci': 6633.0, 'Number of splices: AT/AC': 74.0, 'Number of splices: Annotated (sjdb)': 176305.0, 'Number of splices: GC/AG': 2381.0, 'Number of splices: GT/AG': 176665.0, 'Number of splices: Non-canonical': 63.0, 'Number of splices: Total': 179183.0, 'Uniquely mapped reads %': '69.48', 'Uniquely mapped reads number': 2028169.0, '@id': '/star-quality-metrics/bbbc1238-2003-42d9-98be-95b14f298329/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'bbbc1238-2003-42d9-98be-95b14f298329', 'read_depth': 2365088.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:48.290066+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF513TDF/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1122, 'fraction_of_unique_reads_in_cells': 0.5364, 'mean_genefull_ex50pas_per_cell': 105, 'mean_reads_per_cell': 831, 'mean_UMI_per_cell': 209, 'median_genefull_ex50pas_per_cell': 70, 'median_reads_per_cell': 522, 'median_UMI_per_cell': 134, 'number_of_reads': 2919170, 'q30_bases_in_CB_UMI': 0.928947, 'q30_bases_in_rna_read': 0.932019, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.595479, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.620875, 'reads_mapped_to_genome_unique': 0.694776, 'reads_mapped_to_genome_unique_and_multiple': 0.810192, 'reads_with_valid_barcodes': 0.984537, 'sequencing_saturation': 0.723386, 'total_genefull_ex50pas_detected': 12137, 'umis_in_cells': 235057, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 932427, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '59129b71dfac104e811427f59bca66d1', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/140d11b7-1502-4275-abbc-15748f4328ef/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '140d11b7-1502-4275-abbc-15748f4328ef', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:48.237469+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF606EZP/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '12.44%', '% of reads mapped to too many loci': '0.20%', '% of reads unmapped: other': '2.02%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '14.51%', 'Average input read length': 95.0, 'Average mapped length': 93.91, 'Deletion average length': 1.88, 'Deletion rate per base': '0.06', 'Insertion average length': 2.23, 'Insertion rate per base': '0.08', 'Mapping speed, Million of reads per hour': 288.09, 'Mismatch rate per base, %': '0.68%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2880860.0, 'Number of reads mapped to multiple loci': 358472.0, 'Number of reads mapped to too many loci': 5777.0, 'Number of splices: AT/AC': 77.0, 'Number of splices: Annotated (sjdb)': 188642.0, 'Number of splices: GC/AG': 2172.0, 'Number of splices: GT/AG': 189564.0, 'Number of splices: Non-canonical': 30.0, 'Number of splices: Total': 191843.0, 'Uniquely mapped reads %': '70.83', 'Uniquely mapped reads number': 2040499.0, '@id': '/star-quality-metrics/61df94ad-0df7-413c-8a13-934399ef9ca3/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '61df94ad-0df7-413c-8a13-934399ef9ca3', 'read_depth': 2398971.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:48.286645+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF606EZP/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1400, 'fraction_of_unique_reads_in_cells': 0.584531, 'mean_genefull_ex50pas_per_cell': 75, 'mean_reads_per_cell': 733, 'mean_UMI_per_cell': 145, 'median_genefull_ex50pas_per_cell': 48, 'median_reads_per_cell': 474, 'median_UMI_per_cell': 97, 'number_of_reads': 2880860, 'q30_bases_in_CB_UMI': 0.93164, 'q30_bases_in_rna_read': 0.933337, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.609712, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.63666, 'reads_mapped_to_genome_unique': 0.708295, 'reads_mapped_to_genome_unique_and_multiple': 0.832727, 'reads_with_valid_barcodes': 0.985355, 'sequencing_saturation': 0.786101, 'total_genefull_ex50pas_detected': 11423, 'umis_in_cells': 204025, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 1026725, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': 'ee8cb416e2d7a45a29f398cc4406e005', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/05d81066-daa7-4eeb-a748-310f7b9eb049/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '05d81066-daa7-4eeb-a748-310f7b9eb049', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:58:47.780930+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF853KPG/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'c0f41abc64eab63a85200e15a0219059', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '265e9542a3cad114c6a3836edb68ae96', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'bbbdde00073f27968219d37c2661ef74', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/5974c545-85f5-45cd-a8f8-d5856097c112/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5974c545-85f5-45cd-a8f8-d5856097c112', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:59:43.149878+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF378HAO/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '3fca72fb879d561b7d327c7a7d513fca', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'cc5e386edfeb8256dfa6585a6ae18123', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': 'ae797e32616fe672967cffeb200005b3', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/0f1b77f2-8e14-4b99-a72e-2f815d37ecac/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '0f1b77f2-8e14-4b99-a72e-2f815d37ecac', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:59:45.333088+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF806RAC/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '11.43%', '% of reads mapped to too many loci': '0.24%', '% of reads unmapped: other': '2.36%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '17.70%', 'Average input read length': 96.0, 'Average mapped length': 95.15, 'Deletion average length': 1.91, 'Deletion rate per base': '0.06', 'Insertion average length': 2.19, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 373.12, 'Mismatch rate per base, %': '0.68%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2694776.0, 'Number of reads mapped to multiple loci': 307964.0, 'Number of reads mapped to too many loci': 6387.0, 'Number of splices: AT/AC': 50.0, 'Number of splices: Annotated (sjdb)': 168937.0, 'Number of splices: GC/AG': 2001.0, 'Number of splices: GT/AG': 169412.0, 'Number of splices: Non-canonical': 90.0, 'Number of splices: Total': 171553.0, 'Uniquely mapped reads %': '68.27', 'Uniquely mapped reads number': 1839759.0, '@id': '/star-quality-metrics/20475032-21d4-4435-92d9-5e494553bc4c/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '20475032-21d4-4435-92d9-5e494553bc4c', 'read_depth': 2147723.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T21:59:45.353305+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF806RAC/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1063, 'fraction_of_unique_reads_in_cells': 0.478799, 'mean_genefull_ex50pas_per_cell': 113, 'mean_reads_per_cell': 697, 'mean_UMI_per_cell': 238, 'median_genefull_ex50pas_per_cell': 81, 'median_reads_per_cell': 436, 'median_UMI_per_cell': 153, 'number_of_reads': 2694776, 'q30_bases_in_CB_UMI': 0.930586, 'q30_bases_in_rna_read': 0.93171, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.574258, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.598363, 'reads_mapped_to_genome_unique': 0.682713, 'reads_mapped_to_genome_unique_and_multiple': 0.796995, 'reads_with_valid_barcodes': 0.986147, 'sequencing_saturation': 0.618449, 'total_genefull_ex50pas_detected': 12292, 'umis_in_cells': 253993, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 740939, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '1ec7b67749c68fc648817018a60050c7', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/b894beee-f8f0-494e-8987-e7774dfec8c8/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'b894beee-f8f0-494e-8987-e7774dfec8c8', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " \"[{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T22:00:18.851759+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '8', 'step_run': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', 'quality_metric_of': ['/files/ENCFF526VEJ/'], '% of chimeric reads': '0.00%', '% of reads mapped to multiple loci': '10.89%', '% of reads mapped to too many loci': '0.27%', '% of reads unmapped: other': '2.70%', '% of reads unmapped: too many mismatches': '0.00%', '% of reads unmapped: too short': '17.21%', 'Average input read length': 95.0, 'Average mapped length': 94.6, 'Deletion average length': 1.93, 'Deletion rate per base': '0.06', 'Insertion average length': 2.18, 'Insertion rate per base': '0.07', 'Mapping speed, Million of reads per hour': 265.03, 'Mismatch rate per base, %': '0.68%', 'Number of chimeric reads': 0.0, 'Number of input reads': 2871158.0, 'Number of reads mapped to multiple loci': 312586.0, 'Number of reads mapped to too many loci': 7771.0, 'Number of splices: AT/AC': 65.0, 'Number of splices: Annotated (sjdb)': 171344.0, 'Number of splices: GC/AG': 2469.0, 'Number of splices: GT/AG': 171584.0, 'Number of splices: Non-canonical': 69.0, 'Number of splices: Total': 174187.0, 'Uniquely mapped reads %': '68.93', 'Uniquely mapped reads number': 1979205.0, '@id': '/star-quality-metrics/7213f699-eec2-4e5a-ac35-e7b06fdb4374/', '@type': ['StarQualityMetric', 'QualityMetric', 'Item'], 'uuid': '7213f699-eec2-4e5a-ac35-e7b06fdb4374', 'read_depth': 2291791.0}, {'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T22:00:19.202132+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF526VEJ/'], 'mode': 'GeneFull_Ex50pAS', 'estimated_number_of_cells': 1338, 'fraction_of_unique_reads_in_cells': 0.485824, 'mean_genefull_ex50pas_per_cell': 109, 'mean_reads_per_cell': 595, 'mean_UMI_per_cell': 219, 'median_genefull_ex50pas_per_cell': 72, 'median_reads_per_cell': 354, 'median_UMI_per_cell': 135, 'number_of_reads': 2871158, 'q30_bases_in_CB_UMI': 0.931486, 'q30_bases_in_rna_read': 0.931545, 'reads_mapped_to_genefull_ex50pas_unique_genefull_ex50pas': 0.57093, 'reads_mapped_to_genefull_ex50pas_unique_and_multiple_gene_ex50pas': 0.594822, 'reads_mapped_to_genome_unique': 0.68934, 'reads_mapped_to_genome_unique_and_multiple': 0.798211, 'reads_with_valid_barcodes': 0.987189, 'sequencing_saturation': 0.587361, 'total_genefull_ex50pas_detected': 13479, 'umis_in_cells': 294130, 'unique_reads_in_cells_mapped_to_genefull_ex50pas': 796378, 'barcode_rank_plot': {'download': 'umi_per_cell.png', 'href': '@@download/barcode_rank_plot/umi_per_cell.png', 'type': 'image/png', 'md5sum': '575bd90d9170cad90d360e6e51d81034', 'width': 640, 'height': 480}, '@id': '/star-solo-quality-metric/800b878c-4420-459e-99a6-4e7859fcf0a8/', '@type': ['StarSoloQualityMetric', 'QualityMetric', 'Item'], 'uuid': '800b878c-4420-459e-99a6-4e7859fcf0a8', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'single-cell RNA sequencing assay', 'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': [], 'date_created': '2023-05-31T22:00:19.140830+00:00', 'submitted_by': '/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/', 'schema_version': '1', 'step_run': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', 'quality_metric_of': ['/files/ENCFF411IYO/'], 'total_counts_vs_pct_mitochondria': {'download': 'pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_pct_mitochondria/pct_count_mt.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '81cc35a4db7557b59c0a4d437976bfc3', 'width': 640, 'height': 480}, 'total_counts_vs_genes_by_count': {'download': 'n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/total_counts_vs_genes_by_count/n_genes_by_counts.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '09ce6e9fd308e4c8d83b991326aeea0f', 'width': 640, 'height': 480}, 'counts_violin_plot': {'download': 'qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'href': '@@download/counts_violin_plot/qc_metric_violin.GeneFull_Ex50pAS_EM_raw.png', 'type': 'image/png', 'md5sum': '10ca84781f3c608cff7e4b89d8167220', 'width': 1489, 'height': 490}, '@id': '/scrna-seq-counts-summary-quality-metric/5b9659fd-10d8-436b-bc26-2e64a771f8ba/', '@type': ['ScrnaSeqCountsSummaryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '5b9659fd-10d8-436b-bc26-2e64a771f8ba', 'assay_term_id': 'OBI:0002631'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:39.125140+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/f07a5df3-2ee2-4fc8-8a49-4acda3ae757f/', 'quality_metric_of': ['/files/ENCFF430JPF/'], 'frip': 0.3368675003486986, 'min_size': 360, '25_pct': 560, '50_pct': 709, '75_pct': 1076, 'max_size': 22409, 'mean': 954.3851444555138, '@id': '/chip-peak-enrichment-quality-metrics/3fdc48ac-3893-40ab-b6b3-547335302ab1/', '@type': ['ChipPeakEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '3fdc48ac-3893-40ab-b6b3-547335302ab1', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:39.037814+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '2', 'step_run': '/analysis-step-runs/f07a5df3-2ee2-4fc8-8a49-4acda3ae757f/', 'quality_metric_of': ['/files/ENCFF430JPF/'], 'reproducible_peaks': 208784, 'rescue_ratio': 0.0, 'self_consistency_ratio': 1.0, 'reproducibility': 'pass', '@id': '/chip-replication-quality-metrics/1acc456e-caf6-48b7-9131-65cecfdc4e66/', '@type': ['ChipReplicationQualityMetric', 'QualityMetric', 'Item'], 'uuid': '1acc456e-caf6-48b7-9131-65cecfdc4e66', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:38.451355+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', 'quality_metric_of': ['/files/ENCFF776MAT/'], 'processing_stage': 'unfiltered', 'total_reads': 182290418, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 180670867, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 99.1, 'paired_reads': 182290418, 'paired_reads_qc_failed': 0, 'read1': 91145209, 'read1_qc_failed': 0, 'read2': 91145209, 'read2_qc_failed': 0, 'properly_paired_reads': 178125616, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 97.7, 'with_itself': 179646044, 'with_itself_qc_failed': 0, 'singletons': 1024823, 'singletons_qc_failed': 0, 'pct_singletons': 0.6, 'diff_chroms': 288460, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/ac136465-f5a7-467f-8451-707fa2fbc98f/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'ac136465-f5a7-467f-8451-707fa2fbc98f', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:38.952421+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', 'quality_metric_of': ['/files/ENCFF885JGR/'], 'unpaired_reads': 0, 'paired_reads': 82576468, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 10601705, 'paired_optical_duplicate_reads': 243155, 'pct_duplicate_reads': 12.8387, 'total_fragments': 82569477, 'distinct_fragments': 72858773, 'positions_with_one_read': 6565698, 'NRF': 0.882394, 'PBC1': 0.892043, 'PBC2': 9.89889, '@id': '/chip-library-quality-metrics/624bbed6-6361-4ea0-a098-f27f39e66101/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '624bbed6-6361-4ea0-a098-f27f39e66101', 'assay_term_id': 'OBI:0002160'}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:38.537154+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', 'quality_metric_of': ['/files/ENCFF885JGR/'], 'processing_stage': 'filtered', 'total_reads': 143949526, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 143949526, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 143949526, 'paired_reads_qc_failed': 0, 'read1': 71974763, 'read1_qc_failed': 0, 'read2': 71974763, 'read2_qc_failed': 0, 'properly_paired_reads': 143949526, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 143949526, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/c8806544-59dc-47b5-84e0-5a471ce239da/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': 'c8806544-59dc-47b5-84e0-5a471ce239da', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 71974763}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-13T04:38:38.658358+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', 'quality_metric_of': ['/files/ENCFF885JGR/'], 'subsampled_reads': 15000000, 'estimated_fragment_len': 360, 'corr_estimated_fragment_len': 0.113066442068299, 'phantom_peak': 45, 'corr_phantom_peak': 0.1148406, 'argmin_corr': 1500, 'min_corr': 0.1119376, 'NSC': 1.010085, 'cross_correlation_plot': {'download': 'ENCFF897JUL.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'href': '@@download/cross_correlation_plot/ENCFF897JUL.merged.trim_50bp.srt.filt.no_chrM.15M.cc.plot.pdf', 'type': 'application/pdf', 'md5sum': 'e6665ef14e2180fb818e8c73527230aa'}, 'auc': 0.15844152600436104, 'syn_auc': 0.4962857255079507, 'x_intercept': 0.2429052102096999, 'syn_x_intercept': 0.0, 'elbow_pt': 0.6948405709019154, 'syn_elbow_pt': 0.4993477220712155, 'jsd': 0.12785274864656948, 'syn_jsd': 0.4451447949370881, 'pct_genome_enrich': 26.360130430175673, 'diff_enrich': 30.840132337092896, 'ch_div': 0.2736793866975493, 'jsd_plot': {'download': 'ENCFF897JUL.merged.nodup.jsd_plot.png', 'href': '@@download/jsd_plot/ENCFF897JUL.merged.nodup.jsd_plot.png', 'type': 'image/png', 'md5sum': '928488e761a4c4d93cb7fc4d94d22558', 'width': 640, 'height': 480}, 'gc_bias_plot': {'download': 'ENCFF897JUL.merged.nodup.gc_plot.png', 'href': '@@download/gc_bias_plot/ENCFF897JUL.merged.nodup.gc_plot.png', 'type': 'image/png', 'md5sum': '9d49718f86384080a3c1a4d2035ef2b3', 'width': 640, 'height': 480}, '@id': '/chip-alignment-enrichment-quality-metrics/340a8dbc-ab77-45e4-954d-c12193033219/', '@type': ['ChipAlignmentEnrichmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '340a8dbc-ab77-45e4-954d-c12193033219', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-11T15:49:15.251343+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4b5c096f-226f-46d7-bbb5-3fdae64e36c2/', 'quality_metric_of': ['/files/ENCFF132WWU/'], 'processing_stage': 'unfiltered', 'total_reads': 282417638, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 277025909, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 98.1, 'paired_reads': 282417638, 'paired_reads_qc_failed': 0, 'read1': 141208819, 'read1_qc_failed': 0, 'read2': 141208819, 'read2_qc_failed': 0, 'properly_paired_reads': 273490234, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 96.8, 'with_itself': 275501496, 'with_itself_qc_failed': 0, 'singletons': 1524413, 'singletons_qc_failed': 0, 'pct_singletons': 0.5, 'diff_chroms': 407951, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/64cf1c84-0a42-4c08-8127-5a3e11fcba9d/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '64cf1c84-0a42-4c08-8127-5a3e11fcba9d', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " \"[{'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-11T15:49:15.347250+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4b5c096f-226f-46d7-bbb5-3fdae64e36c2/', 'quality_metric_of': ['/files/ENCFF720TEW/'], 'processing_stage': 'filtered', 'total_reads': 220348908, 'total_reads_qc_failed': 0, 'duplicate_reads': 0, 'duplicate_reads_qc_failed': 0, 'mapped_reads': 220348908, 'mapped_reads_qc_failed': 0, 'pct_mapped_reads': 100.0, 'paired_reads': 220348908, 'paired_reads_qc_failed': 0, 'read1': 110174454, 'read1_qc_failed': 0, 'read2': 110174454, 'read2_qc_failed': 0, 'properly_paired_reads': 220348908, 'properly_paired_reads_qc_failed': 0, 'pct_properly_paired_reads': 100.0, 'with_itself': 220348908, 'with_itself_qc_failed': 0, 'singletons': 0, 'singletons_qc_failed': 0, 'pct_singletons': 0.0, 'diff_chroms': 0, 'diff_chroms_qc_failed': 0, '@id': '/chip-alignment-quality-metrics/111f78c0-84d9-4e48-93d5-40820faf3d81/', '@type': ['ChipAlignmentQualityMetric', 'QualityMetric', 'Item'], 'uuid': '111f78c0-84d9-4e48-93d5-40820faf3d81', 'assay_term_id': 'OBI:0002160', 'usable_fragments': 110174454}, {'assay_term_name': 'Mint-ChIP-seq', 'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'date_created': '2022-03-11T15:49:15.439417+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'schema_version': '1', 'step_run': '/analysis-step-runs/4b5c096f-226f-46d7-bbb5-3fdae64e36c2/', 'quality_metric_of': ['/files/ENCFF720TEW/'], 'unpaired_reads': 0, 'paired_reads': 121041566, 'unmapped_reads': 0, 'unpaired_duplicate_reads': 0, 'paired_duplicate_reads': 10867112, 'paired_optical_duplicate_reads': 192814, 'pct_duplicate_reads': 8.978, 'total_fragments': 120280756, 'distinct_fragments': 110672165, 'positions_with_one_read': 6358701, 'NRF': 0.920115, 'PBC1': 0.932001, 'PBC2': 16.221331, '@id': '/chip-library-quality-metrics/001139c9-26bb-4b1e-badb-29e48c9a2395/', '@type': ['ChipLibraryQualityMetric', 'QualityMetric', 'Item'], 'uuid': '001139c9-26bb-4b1e-badb-29e48c9a2395', 'assay_term_id': 'OBI:0002160'}]\",\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " '[]',\n",
       " ...]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "[\"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:05.489151+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-alignment-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@id': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0ef6bdca-0a26-4f44-a59c-7d6edddf871e'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:11.089688+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-star-signal-generation-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/457fb955-4881-4d4b-b09a-7659885311ef/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '457fb955-4881-4d4b-b09a-7659885311ef'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:05:00.996184+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/8dc8a0c8-2b51-4c5b-b192-0181c2342dcc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8dc8a0c8-2b51-4c5b-b192-0181c2342dcc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:43.896008+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/f8e03920-450f-4e6d-aad7-886aa81234c8/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'f8e03920-450f-4e6d-aad7-886aa81234c8'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:50.930221+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'fa14e0d6-274f-41aa-8bf2-27334cadc5f2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:45.805546+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/aa7272e8-e887-4362-bbd9-b5215f46db9e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'aa7272e8-e887-4362-bbd9-b5215f46db9e'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:01.497461+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-kallisto-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', '@id': '/analysis-step-runs/498dcd9b-bd2d-4248-9afa-09e46162954d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '498dcd9b-bd2d-4248-9afa-09e46162954d'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:56.843199+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-tads-calling-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', '@id': '/analysis-step-runs/6418042c-cef6-49f5-b83a-f66ea71d03f1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6418042c-cef6-49f5-b83a-f66ea71d03f1'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:05.489151+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-alignment-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@id': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0ef6bdca-0a26-4f44-a59c-7d6edddf871e'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:50.930221+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/fa14e0d6-274f-41aa-8bf2-27334cadc5f2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'fa14e0d6-274f-41aa-8bf2-27334cadc5f2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:43.896008+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/f8e03920-450f-4e6d-aad7-886aa81234c8/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'f8e03920-450f-4e6d-aad7-886aa81234c8'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:05:05.396746+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/7630ea30-bf88-49be-8cf7-234a2780e7b7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7630ea30-bf88-49be-8cf7-234a2780e7b7'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:40.566485+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:05:03.307773+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/1e93b456-4047-4a11-9c1b-5407955b6328/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '1e93b456-4047-4a11-9c1b-5407955b6328'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:37.665068+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a98e7af4-c173-4aa6-a789-391b3e535fd0'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:35.973877+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-pooled-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/6b935fdd-3b21-4b41-9a1a-2752eb50e93e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6b935fdd-3b21-4b41-9a1a-2752eb50e93e'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:33.364013+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a89b4c03-9b77-4fe8-9a2e-438e0c92b02c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a89b4c03-9b77-4fe8-9a2e-438e0c92b02c'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T20:14:39.661347+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@id': '/analysis-step-runs/c8adc0d0-b24c-491b-a685-4670c5e3331b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8adc0d0-b24c-491b-a685-4670c5e3331b'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:42.155226+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-replicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/36e582a0-9b8e-4a3d-a109-799d59a5d72a/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '36e582a0-9b8e-4a3d-a109-799d59a5d72a'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.681045+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@id': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6d381d81-b697-4e47-a33b-8d24ab583cdb'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:37.665068+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a98e7af4-c173-4aa6-a789-391b3e535fd0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.273439+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@id': '/analysis-step-runs/d5d38709-71e3-43db-b71d-166e6efde2c7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd5d38709-71e3-43db-b71d-166e6efde2c7'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:42.155226+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-replicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/36e582a0-9b8e-4a3d-a109-799d59a5d72a/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '36e582a0-9b8e-4a3d-a109-799d59a5d72a'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.105693+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@id': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '55f9b653-e7ad-4f12-acad-4acffe9fd89f'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:40.566485+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8d9d3bdf-5a6e-4964-b5d3-92f019aaed5b'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.822625+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@id': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2b5dd93c-914f-4b6c-8748-2542431cd396'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:35.973877+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-pooled-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/6b935fdd-3b21-4b41-9a1a-2752eb50e93e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6b935fdd-3b21-4b41-9a1a-2752eb50e93e'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-08-12T16:25:42.250311+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-bc27eb44-3f5b-4abc-8f3a-ed0e448c77d1-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '@id': '/analysis-step-runs/285487eb-3941-40ea-b52c-38e0087a09b7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '285487eb-3941-40ea-b52c-38e0087a09b7'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.952678+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@id': '/analysis-step-runs/aecc7ec8-5872-4e0f-a0dc-75cf76506a83/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'aecc7ec8-5872-4e0f-a0dc-75cf76506a83'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:33.364013+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a89b4c03-9b77-4fe8-9a2e-438e0c92b02c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a89b4c03-9b77-4fe8-9a2e-438e0c92b02c'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-08-12T16:25:42.250311+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-bc27eb44-3f5b-4abc-8f3a-ed0e448c77d1-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '@id': '/analysis-step-runs/285487eb-3941-40ea-b52c-38e0087a09b7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '285487eb-3941-40ea-b52c-38e0087a09b7'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.627557+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@id': '/analysis-step-runs/8bb52c5a-fe4a-476d-9552-81f60b3f3e0e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8bb52c5a-fe4a-476d-9552-81f60b3f3e0e'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:04.729233+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@id': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '39a7813e-0a93-42fd-bec8-8d49591b7520'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:54.158939+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-merging-quantification-interaction-calling-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@id': '/analysis-step-runs/15607b8b-0c8b-4a62-bd7f-cb14add2813b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '15607b8b-0c8b-4a62-bd7f-cb14add2813b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:04.729233+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@id': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '39a7813e-0a93-42fd-bec8-8d49591b7520'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:04.729233+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@id': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '39a7813e-0a93-42fd-bec8-8d49591b7520'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:22.921604+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/e2ea0042-83a3-4215-8c54-b13e67f25cf6/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'e2ea0042-83a3-4215-8c54-b13e67f25cf6'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.952678+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@id': '/analysis-step-runs/aecc7ec8-5872-4e0f-a0dc-75cf76506a83/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'aecc7ec8-5872-4e0f-a0dc-75cf76506a83'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:04.729233+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@id': '/analysis-step-runs/39a7813e-0a93-42fd-bec8-8d49591b7520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '39a7813e-0a93-42fd-bec8-8d49591b7520'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:20.548466+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c8b19118-ceed-44bf-a40e-0b6703867e7b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8b19118-ceed-44bf-a40e-0b6703867e7b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2023-01-31T00:14:46.505386+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-2457f17a-4d31-4b20-94ac-5cb0f3eb1ec2-53ed72a10bc65af5a9333e83f047307aa7a149db5bf330f7efe8195366efb8b6'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/343f6cf3-327c-415c-a5b1-93989dcfd520/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '343f6cf3-327c-415c-a5b1-93989dcfd520'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T20:14:39.661347+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@id': '/analysis-step-runs/c8adc0d0-b24c-491b-a685-4670c5e3331b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8adc0d0-b24c-491b-a685-4670c5e3331b'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:37.665068+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a98e7af4-c173-4aa6-a789-391b3e535fd0'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:20.548466+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c8b19118-ceed-44bf-a40e-0b6703867e7b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8b19118-ceed-44bf-a40e-0b6703867e7b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:35.423810+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-dhs-accessibility-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', '@id': '/analysis-step-runs/cc74b096-9c68-48b2-abcc-066a6d1e338f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'cc74b096-9c68-48b2-abcc-066a6d1e338f'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.627557+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@id': '/analysis-step-runs/8bb52c5a-fe4a-476d-9552-81f60b3f3e0e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8bb52c5a-fe4a-476d-9552-81f60b3f3e0e'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:33.364013+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a89b4c03-9b77-4fe8-9a2e-438e0c92b02c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a89b4c03-9b77-4fe8-9a2e-438e0c92b02c'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:05.891577+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '492fbb1c-310c-4a55-ade5-a27ba508be96'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:10.401396+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-tads-calling-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', '@id': '/analysis-step-runs/b73c8e25-27f6-4e54-b131-de7b02f0ad32/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b73c8e25-27f6-4e54-b131-de7b02f0ad32'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.681045+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@id': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6d381d81-b697-4e47-a33b-8d24ab583cdb'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:37.665068+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-pseudoreplicated-idr-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@id': '/analysis-step-runs/a98e7af4-c173-4aa6-a789-391b3e535fd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a98e7af4-c173-4aa6-a789-391b3e535fd0'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:25.247517+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/f37c1620-007d-4e95-868d-0e94a6158b31/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'f37c1620-007d-4e95-868d-0e94a6158b31'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:06.863791+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-merging-quantification-interaction-calling-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@id': '/analysis-step-runs/29f56206-d24c-4f6c-b0d0-d30fa6def06f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '29f56206-d24c-4f6c-b0d0-d30fa6def06f'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.273439+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@id': '/analysis-step-runs/d5d38709-71e3-43db-b71d-166e6efde2c7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd5d38709-71e3-43db-b71d-166e6efde2c7'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:33.364013+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-signal-generation-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a89b4c03-9b77-4fe8-9a2e-438e0c92b02c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a89b4c03-9b77-4fe8-9a2e-438e0c92b02c'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:29:05.891577+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5d122b3b-d3d4-4461-9dd2-c6fe28ddb6db-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/492fbb1c-310c-4a55-ade5-a27ba508be96/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '492fbb1c-310c-4a55-ade5-a27ba508be96'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:04.268953+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-pairs-generation-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', '@id': '/analysis-step-runs/32637be8-3d89-45b2-bc31-9f436e95bcf1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '32637be8-3d89-45b2-bc31-9f436e95bcf1'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.105693+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@id': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '55f9b653-e7ad-4f12-acad-4acffe9fd89f'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:15:58.989038+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-alignment-merging-filtering-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', '@id': '/analysis-step-runs/4712d703-abf1-4e9d-b339-137fc09bbc45/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '4712d703-abf1-4e9d-b339-137fc09bbc45'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.822625+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@id': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2b5dd93c-914f-4b6c-8748-2542431cd396'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:13.130667+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-loop-calling-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', '@id': '/analysis-step-runs/85f70870-7d94-4399-ab16-50d7db96895b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '85f70870-7d94-4399-ab16-50d7db96895b'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-08T00:16:06.863791+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-merging-quantification-interaction-calling-step-v-1-6884e206-12cb-44fe-90ae-cb96946717aa-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@id': '/analysis-step-runs/29f56206-d24c-4f6c-b0d0-d30fa6def06f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '29f56206-d24c-4f6c-b0d0-d30fa6def06f'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-09-26T20:45:14.793278+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-a8832f89-ff5d-4f46-8673-d43198daa959-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/86cfd783-e120-4e9d-9004-9c5205b7de25/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '86cfd783-e120-4e9d-9004-9c5205b7de25'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-09-26T20:45:14.793278+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:chip-seq-alignment-step-v-2-a8832f89-ff5d-4f46-8673-d43198daa959-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/86cfd783-e120-4e9d-9004-9c5205b7de25/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '86cfd783-e120-4e9d-9004-9c5205b7de25'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:54.158939+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-merging-quantification-interaction-calling-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@id': '/analysis-step-runs/15607b8b-0c8b-4a62-bd7f-cb14add2813b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '15607b8b-0c8b-4a62-bd7f-cb14add2813b'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:46.603172+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/e1e44b4a-7ff7-46a3-8d1c-116f1e4fe38e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'e1e44b4a-7ff7-46a3-8d1c-116f1e4fe38e'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:00:22.676004+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/d39d5d5e-981a-4b0c-9a5c-be4edeedd849/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd39d5d5e-981a-4b0c-9a5c-be4edeedd849'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:54.824839+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/95674aac-9277-4b18-84d7-3b5aa227512c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '95674aac-9277-4b18-84d7-3b5aa227512c'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:00:31.048824+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/8dc44c46-1971-446d-8939-035443e82eab/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8dc44c46-1971-446d-8939-035443e82eab'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:46.603172+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/e1e44b4a-7ff7-46a3-8d1c-116f1e4fe38e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'e1e44b4a-7ff7-46a3-8d1c-116f1e4fe38e'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:00:22.676004+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/d39d5d5e-981a-4b0c-9a5c-be4edeedd849/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd39d5d5e-981a-4b0c-9a5c-be4edeedd849'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:25.817804+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '10bd3351-3cf8-43ee-a592-1c02f2a12426'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:00:36.269333+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/c9c90fbb-5698-4afd-a7d4-2a3761585343/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c9c90fbb-5698-4afd-a7d4-2a3761585343'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:25.817804+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/10bd3351-3cf8-43ee-a592-1c02f2a12426/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '10bd3351-3cf8-43ee-a592-1c02f2a12426'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:59:58.022264+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:37:59.966850+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-9dd41367-213d-4272-9737-861b896687b4-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/9414cf1f-ddaf-43bb-809f-5f32657136c6/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9414cf1f-ddaf-43bb-809f-5f32657136c6'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:59:58.022264+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-83e7169f-eb21-4231-b7cb-fb228e8f17da-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a40da3c3-b0eb-4056-8885-d1c2b2ddd1a2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:52.497956+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-pairs-generation-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', '@id': '/analysis-step-runs/e06a506e-adcd-4cfc-9932-08ca9b56b20c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'e06a506e-adcd-4cfc-9932-08ca9b56b20c'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:33:00.255489+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/350c1f09-d182-43f1-b33f-18b63609a0b1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '350c1f09-d182-43f1-b33f-18b63609a0b1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:32:47.288376+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/5cd4d22e-b30c-4263-be5c-592f888d8050/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5cd4d22e-b30c-4263-be5c-592f888d8050'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:32:26.314972+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '4c1f7294-47ca-4290-88bf-d35edfd3da2b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:54.238892+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/7ace5429-50ef-40ed-90a5-0e0882e32ac8/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7ace5429-50ef-40ed-90a5-0e0882e32ac8'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:32:54.850540+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/da82ff3b-4f7e-434b-81b5-01f1e69e6186/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'da82ff3b-4f7e-434b-81b5-01f1e69e6186'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:49.861439+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/ba09bf7d-80a1-4f7a-9727-f2847982b2ca/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ba09bf7d-80a1-4f7a-9727-f2847982b2ca'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:32:47.288376+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/5cd4d22e-b30c-4263-be5c-592f888d8050/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5cd4d22e-b30c-4263-be5c-592f888d8050'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:40.105165+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7328450c-83c1-4e82-bc5b-c7fb6d14a633'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:05:00.996184+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-30db70f4-2f52-42c1-a824-0fc3e6961dc5-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/8dc8a0c8-2b51-4c5b-b192-0181c2342dcc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8dc8a0c8-2b51-4c5b-b192-0181c2342dcc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:52.014876+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/e46c8ef3-665a-4d9e-8a14-1238f10c2ae8/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'e46c8ef3-665a-4d9e-8a14-1238f10c2ae8'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:32:26.314972+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-7667404f-21b1-44f3-aee2-3cd9cfc739c9-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/4c1f7294-47ca-4290-88bf-d35edfd3da2b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '4c1f7294-47ca-4290-88bf-d35edfd3da2b'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:48.954021+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-alignment-merging-filtering-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', '@id': '/analysis-step-runs/0b515aa6-3db5-451b-870f-11b2388a464d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0b515aa6-3db5-451b-870f-11b2388a464d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:40.105165+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/7328450c-83c1-4e82-bc5b-c7fb6d14a633/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7328450c-83c1-4e82-bc5b-c7fb6d14a633'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:47.624399+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/51ab203c-c726-4d52-8113-1e96582aaa65/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '51ab203c-c726-4d52-8113-1e96582aaa65'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:11:49.861439+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-931aa156-3407-4c06-b816-3d5bd61c0580-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/ba09bf7d-80a1-4f7a-9727-f2847982b2ca/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ba09bf7d-80a1-4f7a-9727-f2847982b2ca'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:14.117036+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2e3090b7-799c-477a-a821-8064d62f4265'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:14.117036+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/2e3090b7-799c-477a-a821-8064d62f4265/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2e3090b7-799c-477a-a821-8064d62f4265'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T18:00:13.498512+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-dhs-accessibility-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-1.14.0'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', '@id': '/analysis-step-runs/8e1bca2b-d5bf-4c22-9329-eeee6afd2216/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8e1bca2b-d5bf-4c22-9329-eeee6afd2216'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:42.893397+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/519ff374-a2f7-4232-be9f-a8a404783d0d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '519ff374-a2f7-4232-be9f-a8a404783d0d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:15.260407+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-rsem-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@id': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9ea46028-1556-469d-a67b-0f4745fe5d4b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:35.045833+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/bf46e96f-a3c9-4d42-9f28-5812ce7be550/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'bf46e96f-a3c9-4d42-9f28-5812ce7be550'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:15.260407+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-rsem-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@id': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9ea46028-1556-469d-a67b-0f4745fe5d4b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:05:35.045833+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-b4fe5f8b-e016-403f-80e1-b1ad2e220f02-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/bf46e96f-a3c9-4d42-9f28-5812ce7be550/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'bf46e96f-a3c9-4d42-9f28-5812ce7be550'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-11-01T04:55:50.959844+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_hic-diploidification-step-v-1_-6ab7ee15-4453-4c01-99f5-407daf8a0353-e857fdb9a2b8561ea1780266edb71009db1ed2c81ca66b428097f7e8390dce44'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@id': '/analysis-step-runs/517ecec3-da52-4553-9198-6456e657e599/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '517ecec3-da52-4553-9198-6456e657e599'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:36:23.978567+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/25591894-5558-4371-a0b9-849ebb752090/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '25591894-5558-4371-a0b9-849ebb752090'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:35:57.842713+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9e3f0c14-f240-4478-9604-870f8b00c709'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:11.089688+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-star-signal-generation-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/457fb955-4881-4d4b-b09a-7659885311ef/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '457fb955-4881-4d4b-b09a-7659885311ef'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:36:20.829532+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/31fe7a00-7e41-47e6-bbfd-06ec91848ebf/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '31fe7a00-7e41-47e6-bbfd-06ec91848ebf'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:05.489151+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-alignment-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@id': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0ef6bdca-0a26-4f44-a59c-7d6edddf871e'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:36:17.778113+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c1744ac8-2dd3-41d0-a2df-23767320b704/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c1744ac8-2dd3-41d0-a2df-23767320b704'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/robert-waterston/', 'award': '/awards/U41HG007355/', 'date_created': '2022-06-02T13:42:47.038529+00:00', 'submitted_by': '/users/8cab8958-22c9-4b27-ba37-7696e8b604c4/', 'aliases': ['robert-waterston:tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-92293586-817f-47c8-8389-f8ab087b6d50-'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/3e3bd7c8-c13e-42a2-977d-5905a2323661/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '3e3bd7c8-c13e-42a2-977d-5905a2323661'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:11.089688+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-star-signal-generation-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/457fb955-4881-4d4b-b09a-7659885311ef/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '457fb955-4881-4d4b-b09a-7659885311ef'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:36:17.778113+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c1744ac8-2dd3-41d0-a2df-23767320b704/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c1744ac8-2dd3-41d0-a2df-23767320b704'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:15.260407+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-rsem-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@id': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9ea46028-1556-469d-a67b-0f4745fe5d4b'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:35:57.842713+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-a4ef3737-072e-42aa-bc4f-9045002a22a3-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/9e3f0c14-f240-4478-9604-870f8b00c709/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9e3f0c14-f240-4478-9604-870f8b00c709'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:11.089688+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-star-signal-generation-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/457fb955-4881-4d4b-b09a-7659885311ef/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '457fb955-4881-4d4b-b09a-7659885311ef'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:05.489151+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-alignment-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@id': '/analysis-step-runs/0ef6bdca-0a26-4f44-a59c-7d6edddf871e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0ef6bdca-0a26-4f44-a59c-7d6edddf871e'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:15.260407+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-rsem-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@id': '/analysis-step-runs/9ea46028-1556-469d-a67b-0f4745fe5d4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '9ea46028-1556-469d-a67b-0f4745fe5d4b'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-04-12T17:59:58.764756+00:00', 'submitted_by': '/users/1e75d989-a438-4d77-a451-8a297fd3636e/', 'aliases': ['encode-processing-pipeline:hic-loop-calling-step-v-1-cda79be7-9995-4060-99ce-ee05dfb8a387-480cf0b87bb19569a9e62ea22f090f8729375352954e158579a30a21b7039c02'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', '@id': '/analysis-step-runs/0795b70e-7da4-460b-85ab-28fcce9dbfbb/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '0795b70e-7da4-460b-85ab-28fcce9dbfbb'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2021-12-31T08:51:01.497461+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:_analysis-steps_bulk-rna-seq-kallisto-quantification-step-v-1_-a7457a0d-a627-4df1-80ba-1c8de2a41eba-bdcac4249199303e3742c3635e3874df40dc11943f1809ec23f17f07a0b81ce9'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', '@id': '/analysis-step-runs/498dcd9b-bd2d-4248-9afa-09e46162954d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '498dcd9b-bd2d-4248-9afa-09e46162954d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:33.303237+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/39e74081-38a5-4f49-b4f1-8842b3ec48c4/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '39e74081-38a5-4f49-b4f1-8842b3ec48c4'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:31.172910+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/b29c49c0-9b91-413d-992e-54f46ee4c67e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b29c49c0-9b91-413d-992e-54f46ee4c67e'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:28.603130+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/360bed05-b4c0-4585-a282-2f3a1721a9d1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '360bed05-b4c0-4585-a282-2f3a1721a9d1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:28.603130+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/360bed05-b4c0-4585-a282-2f3a1721a9d1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '360bed05-b4c0-4585-a282-2f3a1721a9d1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:12.575812+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5e6e9930-b3f0-49f5-a4e2-aaa97e27335a'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T06:04:12.575812+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-dff1c5b9-1aea-4967-8e35-63d7bc3635f7-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/5e6e9930-b3f0-49f5-a4e2-aaa97e27335a/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5e6e9930-b3f0-49f5-a4e2-aaa97e27335a'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:35.242664+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/10ff7af4-6619-4b1d-b993-b528b336f0fc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '10ff7af4-6619-4b1d-b993-b528b336f0fc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:30.787625+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a791151d-85b3-419b-9c6d-321d2f64ff94/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a791151d-85b3-419b-9c6d-321d2f64ff94'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:33.055165+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/52b40ea5-4a37-49cd-a31d-11bda2aaec23/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '52b40ea5-4a37-49cd-a31d-11bda2aaec23'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:20.907601+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:30.787625+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/a791151d-85b3-419b-9c6d-321d2f64ff94/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a791151d-85b3-419b-9c6d-321d2f64ff94'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:51:20.907601+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-82f2548a-1089-4132-938a-f00965b7a5ad-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '80fc4bde-6a76-4b53-b28c-b49c8ac8ae4b'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:20.638250+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/f87deffb-e666-455f-8213-bbeadf65f77e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'f87deffb-e666-455f-8213-bbeadf65f77e'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:15.904498+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/ac3a7452-7abc-4c8a-a4fe-012c65194ef7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ac3a7452-7abc-4c8a-a4fe-012c65194ef7'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:15.904498+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/ac3a7452-7abc-4c8a-a4fe-012c65194ef7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ac3a7452-7abc-4c8a-a4fe-012c65194ef7'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:00.689289+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b184f90d-8604-4fc9-877a-fcd69eac16f3'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:18.389631+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/a00e3f32-51fe-44de-8a5f-0a2309cf1cc7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'a00e3f32-51fe-44de-8a5f-0a2309cf1cc7'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:35:00.689289+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5e8666f6-27e1-42ca-baf0-43da7edb2c1d-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/b184f90d-8604-4fc9-877a-fcd69eac16f3/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b184f90d-8604-4fc9-877a-fcd69eac16f3'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:13:58.319294+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5e8b97ae-7711-4b0f-be3a-bae20e1668d3'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:14:21.315343+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/69218a59-53c2-48f6-8283-ea41bed6834f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '69218a59-53c2-48f6-8283-ea41bed6834f'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:14:18.279974+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/b236c0bf-f741-4edd-ac7b-3b63f5c61f31/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b236c0bf-f741-4edd-ac7b-3b63f5c61f31'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:14:18.279974+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/b236c0bf-f741-4edd-ac7b-3b63f5c61f31/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b236c0bf-f741-4edd-ac7b-3b63f5c61f31'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:14:23.948764+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/3c0b2b59-c8ac-4292-94d6-e7da248a01ac/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '3c0b2b59-c8ac-4292-94d6-e7da248a01ac'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:13:58.319294+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-e0f66ac9-c48d-4fe0-bf7a-cf6f8cddc985-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/5e8b97ae-7711-4b0f-be3a-bae20e1668d3/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '5e8b97ae-7711-4b0f-be3a-bae20e1668d3'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:44:59.409506+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/293b2e56-4188-4e9e-aad9-48834d3257d4/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '293b2e56-4188-4e9e-aad9-48834d3257d4'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:44:59.409506+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/293b2e56-4188-4e9e-aad9-48834d3257d4/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '293b2e56-4188-4e9e-aad9-48834d3257d4'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:45:02.697505+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/278135db-42ed-4a27-a7e4-0ae1dafe0093/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '278135db-42ed-4a27-a7e4-0ae1dafe0093'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:44:38.398338+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b0ce6877-6480-4dc2-a682-d6fe2671af27'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:45:06.843419+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/caf5417d-545c-407a-aca5-18e0b48b9563/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'caf5417d-545c-407a-aca5-18e0b48b9563'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:44:38.398338+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-7f4bce76-bc62-4f7c-aa73-992e9fd9d86b-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/b0ce6877-6480-4dc2-a682-d6fe2671af27/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'b0ce6877-6480-4dc2-a682-d6fe2671af27'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:54.030741+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/284dc8fa-7727-40f3-afd2-54d5bf83e979/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '284dc8fa-7727-40f3-afd2-54d5bf83e979'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:51.530883+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c348b726-2635-4e6f-a90a-8e4b08ad2996/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c348b726-2635-4e6f-a90a-8e4b08ad2996'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:51.530883+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/c348b726-2635-4e6f-a90a-8e4b08ad2996/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c348b726-2635-4e6f-a90a-8e4b08ad2996'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:37.066848+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '25fc156e-3459-40c2-a35a-2dfe9b64bda2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:56.531072+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/89219414-0400-4d10-8e61-6e12c6bd641c/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '89219414-0400-4d10-8e61-6e12c6bd641c'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T05:33:37.066848+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-357fc3f2-49b3-4e47-991d-47e8517a0ea2-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/25fc156e-3459-40c2-a35a-2dfe9b64bda2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '25fc156e-3459-40c2-a35a-2dfe9b64bda2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.822625+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@id': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2b5dd93c-914f-4b6c-8748-2542431cd396'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.273439+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@id': '/analysis-step-runs/d5d38709-71e3-43db-b71d-166e6efde2c7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd5d38709-71e3-43db-b71d-166e6efde2c7'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.822625+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@id': '/analysis-step-runs/2b5dd93c-914f-4b6c-8748-2542431cd396/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '2b5dd93c-914f-4b6c-8748-2542431cd396'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T20:14:39.661347+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@id': '/analysis-step-runs/c8adc0d0-b24c-491b-a685-4670c5e3331b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8adc0d0-b24c-491b-a685-4670c5e3331b'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.952678+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@id': '/analysis-step-runs/aecc7ec8-5872-4e0f-a0dc-75cf76506a83/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'aecc7ec8-5872-4e0f-a0dc-75cf76506a83'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.273439+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@id': '/analysis-step-runs/d5d38709-71e3-43db-b71d-166e6efde2c7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd5d38709-71e3-43db-b71d-166e6efde2c7'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.105693+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@id': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '55f9b653-e7ad-4f12-acad-4acffe9fd89f'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.528805+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@id': '/analysis-step-runs/dbbe742c-d17b-4ad2-8efc-2e1a1152bb84/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'dbbe742c-d17b-4ad2-8efc-2e1a1152bb84'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.681045+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@id': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6d381d81-b697-4e47-a33b-8d24ab583cdb'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.627557+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@id': '/analysis-step-runs/8bb52c5a-fe4a-476d-9552-81f60b3f3e0e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8bb52c5a-fe4a-476d-9552-81f60b3f3e0e'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T20:14:39.661347+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@id': '/analysis-step-runs/c8adc0d0-b24c-491b-a685-4670c5e3331b/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'c8adc0d0-b24c-491b-a685-4670c5e3331b'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:21.681045+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@id': '/analysis-step-runs/6d381d81-b697-4e47-a33b-8d24ab583cdb/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '6d381d81-b697-4e47-a33b-8d24ab583cdb'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.627557+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@id': '/analysis-step-runs/8bb52c5a-fe4a-476d-9552-81f60b3f3e0e/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '8bb52c5a-fe4a-476d-9552-81f60b3f3e0e'}\",\n",
       " \"{'status': 'released', 'date_created': '2020-07-24T17:33:22.105693+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@id': '/analysis-step-runs/55f9b653-e7ad-4f12-acad-4acffe9fd89f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '55f9b653-e7ad-4f12-acad-4acffe9fd89f'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2020-09-18T19:54:40.952678+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_step_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@id': '/analysis-step-runs/aecc7ec8-5872-4e0f-a0dc-75cf76506a83/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'aecc7ec8-5872-4e0f-a0dc-75cf76506a83'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.660549+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-alignment-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@id': '/analysis-step-runs/d9674070-79dc-4a3c-a0d7-3143caeddc1d/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd9674070-79dc-4a3c-a0d7-3143caeddc1d'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'date_created': '2021-12-14T22:16:46.875646+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'aliases': ['barbara-wold:starsolo-split-quantification-step-run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@id': '/analysis-step-runs/acba5c57-3740-4bbf-87ff-8afcdff736bc/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'acba5c57-3740-4bbf-87ff-8afcdff736bc'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'date_created': '2023-05-17T21:28:31.825876+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': ['barbara-wold:starsolo_merging_step_v0_run'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@id': '/analysis-step-runs/7df1790e-dcd9-46b8-bd0f-76d3b19cfee2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7df1790e-dcd9-46b8-bd0f-76d3b19cfee2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:31.518996+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/7987bb81-22ef-42cf-9875-4b9ca75bfbd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7987bb81-22ef-42cf-9875-4b9ca75bfbd0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:31.518996+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/7987bb81-22ef-42cf-9875-4b9ca75bfbd0/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '7987bb81-22ef-42cf-9875-4b9ca75bfbd0'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:34.532549+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-step-v-1-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@id': '/analysis-step-runs/f07a5df3-2ee2-4fc8-8a49-4acda3ae757f/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'f07a5df3-2ee2-4fc8-8a49-4acda3ae757f'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:36.818897+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/1ea7a6e7-9933-4b21-9e2c-2eba3d53c033/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '1ea7a6e7-9933-4b21-9e2c-2eba3d53c033'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:11.232650+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ea5e139b-2ecd-4fc9-89bc-8b2c45410739'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-13T04:38:11.232650+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-5966fc22-18a0-4b02-ba27-6fc4815c925f-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/ea5e139b-2ecd-4fc9-89bc-8b2c45410739/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'ea5e139b-2ecd-4fc9-89bc-8b2c45410739'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-11T15:48:54.129983+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-fee73b46-14e2-4b50-ad11-768314a8e805-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/4b5c096f-226f-46d7-bbb5-3fdae64e36c2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '4b5c096f-226f-46d7-bbb5-3fdae64e36c2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-11T15:48:54.129983+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:chip-seq-alignment-step-v-2-fee73b46-14e2-4b50-ad11-768314a8e805-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@id': '/analysis-step-runs/4b5c096f-226f-46d7-bbb5-3fdae64e36c2/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '4b5c096f-226f-46d7-bbb5-3fdae64e36c2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:48:04.798781+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-b9315fb6-ea6b-453d-b82a-241b44c00f23-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@id': '/analysis-step-runs/995b2608-6a1f-4616-aea6-12c8b58c15c7/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': '995b2608-6a1f-4616-aea6-12c8b58c15c7'}\",\n",
       " \"{'status': 'released', 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'date_created': '2022-03-30T17:47:59.586782+00:00', 'submitted_by': '/users/6bae687f-b77a-46b9-af0e-a02c135cf42e/', 'aliases': ['encode-processing-pipeline:histone-chip-signal-generation-step-v-1-b9315fb6-ea6b-453d-b82a-241b44c00f23-593cc2f296c86fb4ba30a4b15e646d3c57f64f97dd2b60bc5e8ecd05df02a47d'], 'schema_version': '5', 'analysis_step_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@id': '/analysis-step-runs/d5c10423-231c-4efa-aafc-8b63e8a6ddd1/', '@type': ['AnalysisStepRun', 'Item'], 'uuid': 'd5c10423-231c-4efa-aafc-8b63e8a6ddd1'}\",\n",
       " ...]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "[\"{'software_versions': [{'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'aliases': ['dnanexus:star_v_2_5_1_b'], 'version': '2.5.1b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/9d43c048-c70c-42ab-895d-77a69fceb42b/'], 'date_created': '2014-08-13T01:30:14.893029+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'star', 'title': 'STAR', 'source_url': 'https://github.com/alexdobin/STAR/releases', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'STAR (Spliced Transcript Alignment to a Reference) aligns short and bulk RNA-seq reads to a reference genome using uncompressed suffix arrays.', '@id': '/software/star/', '@type': ['Software', 'Item'], 'uuid': 'adb49a18-c66d-4bbe-9cab-bea6fd12f8a3', 'versions': ['/software-versions/834e3145-b752-4486-a613-c0c625262051/', '/software-versions/195baf07-de0a-460d-9686-59d82bd75e50/', '/software-versions/e038206f-d791-4744-9ee5-7d22bd1f39ee/', '/software-versions/0bfb7a0a-afc7-4453-b735-ee878f23e7a9/', '/software-versions/19f75e90-e991-4dc8-837d-8cd97ab5fb91/', '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '/software-versions/f533aae8-827c-4dfe-bfe3-81143728bad8/', '/software-versions/de76db97-c034-4113-9053-74be4af14b1e/', '/software-versions/e7a3ccb9-f3e3-405e-9ee5-7ae53ece93e8/', '/software-versions/dd2d092b-61f7-491e-a127-5099a6a32d9b/', '/software-versions/ef94cd69-7316-480d-b785-e1cb36d1ea97/', '/software-versions/055f6c18-7b20-4499-9f5b-ac4bec338c47/', '/software-versions/475e856f-4d0e-459d-8417-23abd75613e3/', '/software-versions/cab07b5a-9dd9-4417-947f-2771d21db315/', '/software-versions/53519944-8257-4702-8167-0e32140ae5d6/', '/software-versions/d6aa7af7-54ef-4759-b589-91deafbde9ae/']}, 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'schema_version': '4', 'notes': 'Using bin/Linux_x86_64_static/STAR executable', 'download_checksum': '4bff1086765377f4b1706c31769367b7', 'date_created': '2016-01-20T20:45:16.541995+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/alexdobin/STAR/archive/2.5.1b.tar.gz', '@id': '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '70a085ad-194e-4de0-838c-7af257680f54'}], 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:07:57.849387+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-align-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-alignment-step', 'title': 'bulk RNA-seq alignment step', 'major_version': 1, 'analysis_step_types': ['alignment', 'QA calculation'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments', 'transcriptome alignments'], '@id': '/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ee4d1b35-e531-4900-8217-cd7719a96216', 'name': 'bulk-rna-seq-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/']}, 'schema_version': '4', 'date_created': '2020-01-08T21:18:33.262328+00:00', 'aliases': ['encode:bulk-rna-seq-star-alignment-step-version-1'], 'status': 'released', '@id': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5047d5aa-bb3b-441b-9dce-4a94e9d064dd', 'name': 'bulk-rna-seq-alignment-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:09:03.779901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-signal-generation-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-star-signal-generation-step', 'title': 'bulk RNA-seq star signal generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal of all reads', 'signal of unique reads', 'plus strand signal of all reads', 'plus strand signal of unique reads', 'minus strand signal of all reads', 'minus strand signal of unique reads'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '790beb4b-3b09-449c-a1fa-7dca6c802b74', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/']}, 'status': 'released', 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'software_versions': [{'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', 'schema_version': '4', 'status': 'released', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': [], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'download_checksum': '8d0c29618778013088f5fe7bd1ddb238', 'date_created': '2020-01-09T20:07:05.507787+00:00', 'version': '385', '@id': '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5d8ce593-b423-402e-93d1-f2eeca0a2f90'}, {'aliases': [], 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2020-01-08T22:50:34.724469+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/', 'status': 'released', 'aliases': ['encode:bedSort'], 'schema_version': '10', 'software_type': ['other'], 'name': 'bedsort', 'title': 'bedSort', 'source_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedSort', 'purpose': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'UCSC Genome Browser tool for sorting .bed files by chrom,chromStart.', '@id': '/software/bedsort/', '@type': ['Software', 'Item'], 'uuid': '81cba982-508f-4fd4-ab1a-5d27d6c5897c', 'versions': ['/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/']}, 'date_created': '2020-01-08T22:51:18.955130+00:00', 'download_checksum': '11abd4745b970cae6d5aa9c44429f3a2', 'status': 'released', 'version': '385', 'schema_version': '4', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', '@id': '/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8141bb60-2775-483b-8ecc-85956209ba4a'}], 'date_created': '2020-01-08T21:21:55.421044+00:00', 'aliases': ['encode:bulk-rnaseq-signal-generation-step-version-1'], 'schema_version': '4', '@id': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '53e87326-7e11-4230-8d39-66a8b8471a9c', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'minor_version': 0, 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:10.006985+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step', 'title': 'TF ChIP seq pseudoreplicated idr thresholed peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['IDR thresholded peaks'], 'output_file_types': ['IDR thresholded peaks'], 'parents': ['/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '492a7752-97ed-4c0d-82d9-87d0007d31ab', 'name': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/']}, 'date_created': '2019-11-15T05:51:07.285379+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'], '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1b02e66b-ed49-4dc6-b24a-1b229c076a5e', 'name': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'minor_version': 0, 'date_created': '2019-11-15T05:51:10.764375+00:00', 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:11.407842+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step', 'title': 'TF ChIP seq pooled pseudoreplicated idr thresholed peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['IDR thresholded peaks'], 'output_file_types': ['IDR thresholded peaks'], 'parents': ['/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/'], '@id': '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'b936f3c4-72ee-4735-9f7b-e1779ea1a232', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'], 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '071417b5-cd2b-48b9-a225-fb75ead07fb6', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'aliases': ['encode:bulk-rnaseq-kallisto-step-version-1'], 'status': 'released', 'schema_version': '4', 'date_created': '2020-01-08T21:19:38.395952+00:00', 'minor_version': 0, 'software_versions': [{'schema_version': '4', 'date_created': '2019-02-12T23:18:11.475019+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'downloaded_url': 'https://github.com/pachterlab/kallisto/releases/download/v0.44.0/kallisto_linux-v0.44.0.tar.gz', 'aliases': ['encode:kallisto_v0.44.0'], 'version': 'v0.44.0', 'download_checksum': 'af1e787706ba16cb49c171b59eec6063', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-02-12T23:15:57.870930+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/pachterlab/kallisto', 'status': 'released', 'aliases': ['encode:kallisto'], 'schema_version': '10', 'name': 'kallisto', 'title': 'kallisto', 'used_by': ['community', 'uniform processing pipelines'], 'description': 'kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment. On benchmarks with standard RNA-Seq data, kallisto can quantify 30 million human reads in less than 3 minutes on a Mac desktop computer using only the read sequences and a transcriptome index that itself takes less than 10 minutes to build. Pseudoalignment of reads preserves the key information needed for quantification, and kallisto is therefore not only fast, but also as accurate than existing quantification tools. In fact, because the pseudoalignment procedure is robust to errors in the reads, in many benchmarks kallisto significantly outperforms existing tools.', '@id': '/software/kallisto/', '@type': ['Software', 'Item'], 'uuid': 'c0d93c45-b348-438b-abde-80a868a5a625', 'versions': ['/software-versions/bd220526-b2aa-4fe4-bf36-56e5da908589/']}, '@id': '/software-versions/bd220526-b2aa-4fe4-bf36-56e5da908589/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bd220526-b2aa-4fe4-bf36-56e5da908589'}], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:12:19.473065+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-kallisto-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-kallisto-quantification-step', 'title': 'bulk RNA-seq kallisto quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome index', 'reads'], 'output_file_types': ['transcript quantifications'], '@id': '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '93fe5f94-18a6-4fc4-a7a6-3c05da84a4b5', 'name': 'bulk-rna-seq-kallisto-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/']}, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', '@id': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bd3ed9ca-922b-4b2b-a0a1-e103ec36379d', 'name': 'bulk-rna-seq-kallisto-quantification-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-07T20:00:21.296053+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.692240+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-tads-calling-step'], 'schema_version': '17', 'step_label': 'hic-tads-calling-step', 'title': 'Hi-C TADs calling step', 'major_version': 1, 'analysis_step_types': ['contact domain identification'], 'input_file_types': ['contact matrix'], 'output_file_types': ['contact domains'], 'parents': ['/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/'], '@id': '/analysis-steps/hic-tads-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '60232aac-6b7d-4510-b36a-5c28c90eff6e', 'name': 'hic-tads-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-tads-calling-step-v-1-0/', '/analysis-step-versions/hic-tads-calling-step-v-1-1/']}, 'software_versions': [{'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28a5945c-d1ba-4fb6-99ce-76ae5b292025', 'name': 'hic-tads-calling-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'software_versions': [{'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'aliases': ['dnanexus:star_v_2_5_1_b'], 'version': '2.5.1b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/9d43c048-c70c-42ab-895d-77a69fceb42b/'], 'date_created': '2014-08-13T01:30:14.893029+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'star', 'title': 'STAR', 'source_url': 'https://github.com/alexdobin/STAR/releases', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'STAR (Spliced Transcript Alignment to a Reference) aligns short and bulk RNA-seq reads to a reference genome using uncompressed suffix arrays.', '@id': '/software/star/', '@type': ['Software', 'Item'], 'uuid': 'adb49a18-c66d-4bbe-9cab-bea6fd12f8a3', 'versions': ['/software-versions/834e3145-b752-4486-a613-c0c625262051/', '/software-versions/195baf07-de0a-460d-9686-59d82bd75e50/', '/software-versions/e038206f-d791-4744-9ee5-7d22bd1f39ee/', '/software-versions/0bfb7a0a-afc7-4453-b735-ee878f23e7a9/', '/software-versions/19f75e90-e991-4dc8-837d-8cd97ab5fb91/', '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '/software-versions/f533aae8-827c-4dfe-bfe3-81143728bad8/', '/software-versions/de76db97-c034-4113-9053-74be4af14b1e/', '/software-versions/e7a3ccb9-f3e3-405e-9ee5-7ae53ece93e8/', '/software-versions/dd2d092b-61f7-491e-a127-5099a6a32d9b/', '/software-versions/ef94cd69-7316-480d-b785-e1cb36d1ea97/', '/software-versions/055f6c18-7b20-4499-9f5b-ac4bec338c47/', '/software-versions/475e856f-4d0e-459d-8417-23abd75613e3/', '/software-versions/cab07b5a-9dd9-4417-947f-2771d21db315/', '/software-versions/53519944-8257-4702-8167-0e32140ae5d6/', '/software-versions/d6aa7af7-54ef-4759-b589-91deafbde9ae/']}, 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'schema_version': '4', 'notes': 'Using bin/Linux_x86_64_static/STAR executable', 'download_checksum': '4bff1086765377f4b1706c31769367b7', 'date_created': '2016-01-20T20:45:16.541995+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/alexdobin/STAR/archive/2.5.1b.tar.gz', '@id': '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '70a085ad-194e-4de0-838c-7af257680f54'}], 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:07:57.849387+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-align-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-alignment-step', 'title': 'bulk RNA-seq alignment step', 'major_version': 1, 'analysis_step_types': ['alignment', 'QA calculation'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments', 'transcriptome alignments'], '@id': '/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ee4d1b35-e531-4900-8217-cd7719a96216', 'name': 'bulk-rna-seq-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/']}, 'schema_version': '4', 'date_created': '2020-01-08T21:18:33.262328+00:00', 'aliases': ['encode:bulk-rna-seq-star-alignment-step-version-1'], 'status': 'released', '@id': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5047d5aa-bb3b-441b-9dce-4a94e9d064dd', 'name': 'bulk-rna-seq-alignment-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'status': 'released', 'minor_version': 0, 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:10.006985+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step', 'title': 'TF ChIP seq pseudoreplicated idr thresholed peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['IDR thresholded peaks'], 'output_file_types': ['IDR thresholded peaks'], 'parents': ['/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '492a7752-97ed-4c0d-82d9-87d0007d31ab', 'name': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/']}, 'date_created': '2019-11-15T05:51:07.285379+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'], '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1b02e66b-ed49-4dc6-b24a-1b229c076a5e', 'name': 'tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'minor_version': 0, 'date_created': '2019-11-15T05:51:10.402582+00:00', 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:11.083132+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pooled-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pooled pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['pooling', 'peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e4b31518-205e-43b1-afb8-522e1cc64da6', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0'], 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4a8ba9ff-b074-43f9-9a33-bb89aa1d492c', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:06.966916+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.643967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e99869e1-3599-4292-a6be-af013173a646', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4e54cd81-c5ba-4866-b997-06c5cc1fffed', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:14.088095+00:00', 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'aliases': ['encode:tf-chip-seq-pooled-signal-generation-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:11.568267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pooled-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-pooled-signal-generation-step', 'title': 'TF ChIP seq pooled signal generation step', 'major_version': 1, 'analysis_step_types': ['pooling', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '417c237e-a277-45b6-b128-ab7e12edc120', 'name': 'tf-chip-pooled-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '3c9027e6-37ac-46be-9cdf-e860b14c55bb', 'name': 'tf-chip-pooled-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.463433+00:00', 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v-1-0'], 'status': 'released', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.360683+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-signal-generation-step', 'title': 'TF ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '997f4a70-ad7f-4d41-b988-60ab9d817c4d', 'name': 'tf-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/']}, '@id': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8036faee-410a-41be-a2b8-d310f23ad475', 'name': 'tf-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2020-09-18T20:14:09.493363+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T20:13:32.587576+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed'], 'schema_version': '17', 'step_label': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step', 'title': 'Ruan Lab ChIA-PIPE spp bed to bigBed bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['peaks'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dc647c58-f1c2-46bb-87dc-f6adf4c77fc5', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ec36803a-f075-4d03-8b48-3e911cbfb55d', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:09.718485+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-replicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:10.495959+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-replicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-replicated-idr-step', 'title': 'TF ChIP seq replicated idr step', 'major_version': 1, 'analysis_step_types': ['pooling', 'peak calling', 'IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'conservative IDR thresholded peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '80126972-1f5a-4530-bbb7-2fc62d1c61ed', 'name': 'tf-chip-seq-replicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bfb08d3-9f8f-431f-a884-e2e689c64d07', 'name': 'tf-chip-seq-replicated-idr-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.383176+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.823285+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step', 'title': 'Ruan Lab ChIA-PIPE bam to hic bam2pairs, Juicer tools step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'c82edbd4-cdfb-4c5d-a77f-52b5d92611db', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-16T22:02:16.098864+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs_af9c8fc'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-16T21:52:38.400497+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs'], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bam2pairs', 'title': 'bam2pairs', 'source_url': 'http://github.com/4dn-dcic/pairix/tree/master/util/bam2pairs', 'used_by': ['ENCODE'], 'description': 'This script converts a paired-end bam file to a pairs file.', '@id': '/software/bam2pairs/', '@type': ['Software', 'Item'], 'uuid': 'fd4e6d1c-ff0d-42b5-88f1-cda9f330c291', 'versions': ['/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/']}, 'version': 'commit af9c8fc', 'downloaded_url': 'http://github.com/4dn-dcic/pairix/commit/af9c8fc60f201778df67581aba56a7ae23745ef6', '@id': '/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '867f4bda-802f-4e88-a35e-78886c2759ce'}, {'date_created': '2020-07-10T22:23:58.840954+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['erez-aiden:juicertools_1.7.5'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '1.7.5', 'download_checksum': 'b0dc301fed2ce85a2e6baa34bfc1cdff', 'downloaded_url': 'http://hicfiles.tc4ga.com.s3.amazonaws.com/public/juicer/juicer_tools.1.7.5_linux_x64_jcuda.0.8.jar', '@id': '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '7a54892a-47d1-464f-9407-2b1870cc9039'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '9635b2de-3625-45b0-bbee-429388d0561a', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.966916+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.643967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e99869e1-3599-4292-a6be-af013173a646', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4e54cd81-c5ba-4866-b997-06c5cc1fffed', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:51.001604+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.462793+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedgraph bedtools step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['unfiltered alignments', 'exclusion list regions'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '62e7591a-f6eb-4298-b744-0f353ff089ee', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/']}, 'software_versions': [{'date_created': '2017-06-27T03:42:14.714951+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'status': 'released', 'aliases': ['encode:bedtools 2.26.0'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'version': '2.26.0', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.26.0', '@id': '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '13e61050-fe4f-4002-9ae0-8d1adcf81a04'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '82f524da-1b07-4272-ab8e-f0c901aaf279', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:09.718485+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-replicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:10.495959+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-replicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-replicated-idr-step', 'title': 'TF ChIP seq replicated idr step', 'major_version': 1, 'analysis_step_types': ['pooling', 'peak calling', 'IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'conservative IDR thresholded peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '80126972-1f5a-4530-bbb7-2fc62d1c61ed', 'name': 'tf-chip-seq-replicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-replicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bfb08d3-9f8f-431f-a884-e2e689c64d07', 'name': 'tf-chip-seq-replicated-idr-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.849098+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.303421+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_control_bam_to_bed'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bed-peak-calling-spp-step', 'title': 'Ruan Lab ChIA-PIPE bam to bed SPP step', 'major_version': 1, 'analysis_step_types': ['peak calling'], 'input_file_types': ['unfiltered alignments'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0846ae82-e09a-4b92-a0a9-f67627fb461d', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:spp 1.13'], 'version': '1.13', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/8a6e0ea0-6b0e-4fdc-885e-297c19cd75aa/'], 'date_created': '2014-08-13T01:30:13.985607+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'spp', 'title': 'SPP', 'source_url': 'http://compbio.med.harvard.edu/Supplements/ChIP-seq/', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A ChIP-seq peak calling algorithm, implemented as an R package, that accounts for the offset in forward-strand and reverse-strand reads to improve resolution, compares enrichment in signal to background or control experiments, and can also estimate whether the available number of reads is sufficient to achieve saturation, meaning that additional reads would not allow identification of additional peaks. SPP will be used in the ENCODE 3 uniform peak calling pipeline.', '@id': '/software/spp/', '@type': ['Software', 'Item'], 'uuid': 'e0c8602f-bd34-49cf-ae0a-e85f13e82470', 'versions': ['/software-versions/beb915d5-157c-48d2-9867-7d2bbf8917fe/', '/software-versions/021b7765-014b-48b5-8258-436c8f378e3c/', '/software-versions/e1ed8453-f332-49c8-b8c0-bb33c1494da4/', '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/']}, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'download_checksum': 'bdea5fbf5fbd09243f7ddd6cf8a062c8', 'date_created': '2016-06-09T15:50:40.358685+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/hms-dbmi/spp/archive/1.13.tar.gz', '@id': '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ffb5a817-e963-4694-9aae-cd89a832e1a1'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bbcef8b-821d-4abe-b571-7e255b9865ac', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'minor_version': 0, 'date_created': '2019-11-15T05:51:10.402582+00:00', 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:11.083132+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pooled-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pooled pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['pooling', 'peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e4b31518-205e-43b1-afb8-522e1cc64da6', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0'], 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4a8ba9ff-b074-43f9-9a33-bb89aa1d492c', 'name': 'tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.541370+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.993488+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedpe CPU step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['alignments'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'b9a0e911-05bb-4c58-8c57-07d65db1e268', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '2d9ae5c9-4942-4407-a321-0c31831a4173', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:14.088095+00:00', 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'aliases': ['encode:tf-chip-seq-pooled-signal-generation-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:11.568267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pooled-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-pooled-signal-generation-step', 'title': 'TF ChIP seq pooled signal generation step', 'major_version': 1, 'analysis_step_types': ['pooling', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '417c237e-a277-45b6-b128-ab7e12edc120', 'name': 'tf-chip-pooled-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-pooled-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '3c9027e6-37ac-46be-9cdf-e860b14c55bb', 'name': 'tf-chip-pooled-signal-generation-step-v-1-0'}\",\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'schema_version': '4', 'aliases': ['encode:chip-seq-alignment-step-v-2-0'], 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], 'status': 'released', 'date_created': '2019-11-15T05:51:06.302710+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'minor_version': 0, '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b7563f06-06bc-4fb5-8f2f-9c49cc7abba9', 'name': 'chip-seq-alignment-step-v-2-0'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.847238+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.428239+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig'], 'schema_version': '17', 'step_label': 'ruan-lab-bedgraph-conversion-to-bigwig-step', 'title': 'Ruan Lab ChIA-PIPE bedgraph to bigWig bedGraphToBigWig step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['signal of unique reads'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '12c47b68-0fe9-4e69-8b54-7b509b376a37', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:bedgraphtobigwig 4'], 'version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'schema_version': '4', 'download_checksum': 'c20443ee1d095089e79facdfbc214a89', 'date_created': '2015-02-13T16:12:16.673800+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentUtils/blob/v302.1.0/bin/linux.x86_64/bedGraphToBigWig', '@id': '/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fa03d911-5b7c-40b6-9649-9086aa4b1597'}], '@id': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'edc72129-d859-4047-861b-10135504cc18', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.463433+00:00', 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v-1-0'], 'status': 'released', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.360683+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-signal-generation-step', 'title': 'TF ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '997f4a70-ad7f-4d41-b988-60ab9d817c4d', 'name': 'tf-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/']}, '@id': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8036faee-410a-41be-a2b8-d310f23ad475', 'name': 'tf-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'schema_version': '4', 'aliases': ['encode:chip-seq-alignment-step-v-2-0'], 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], 'status': 'released', 'date_created': '2019-11-15T05:51:06.302710+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'minor_version': 0, '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b7563f06-06bc-4fb5-8f2f-9c49cc7abba9', 'name': 'chip-seq-alignment-step-v-2-0'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.499328+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.076302+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract'], 'schema_version': '17', 'step_label': 'ruan-lab-biginteract-conversion-step', 'title': 'Ruan Lab ChIA-PIPE bedpe to bigInteract bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['loops'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a8e0199f-9e9d-4dca-b5b6-95163a6eae5c', 'name': 'ruan-lab-biginteract-conversion-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bbd376ff-f3c2-444b-b09b-94bacbed91e8', 'name': 'ruan-lab-biginteract-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-05-24T18:07:44.465565+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'deleted', 'aliases': ['encode:chip-seq-alignment-step-v-2-3'], 'schema_version': '4', 'minor_version': 3, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28bb298d-25d8-4599-9c26-f12dedd2a62f', 'name': 'chip-seq-alignment-step-v-2-3'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T19:59:21.389725+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.072273+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-merging-quantification-interaction-calling-step'], 'schema_version': '17', 'step_label': 'hic-merging-quantification-interaction-calling-step', 'title': 'Hi-C merging, quantification, and interaction calling step', 'major_version': 1, 'analysis_step_types': ['filtering', 'quantification', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'd6bcf0ba-1bc0-4174-85d7-443d57b42eba', 'name': 'hic-merging-quantification-interaction-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-0/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '00e63923-0fc6-42d1-a196-f880f01f96e8', 'name': 'hic-merging-quantification-interaction-calling-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-05-24T18:07:44.465565+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'deleted', 'aliases': ['encode:chip-seq-alignment-step-v-2-3'], 'schema_version': '4', 'minor_version': 3, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28bb298d-25d8-4599-9c26-f12dedd2a62f', 'name': 'chip-seq-alignment-step-v-2-3'}\",\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-05-24T18:07:44.465565+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'deleted', 'aliases': ['encode:chip-seq-alignment-step-v-2-3'], 'schema_version': '4', 'minor_version': 3, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28bb298d-25d8-4599-9c26-f12dedd2a62f', 'name': 'chip-seq-alignment-step-v-2-3'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.847238+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.428239+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig'], 'schema_version': '17', 'step_label': 'ruan-lab-bedgraph-conversion-to-bigwig-step', 'title': 'Ruan Lab ChIA-PIPE bedgraph to bigWig bedGraphToBigWig step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['signal of unique reads'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '12c47b68-0fe9-4e69-8b54-7b509b376a37', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:bedgraphtobigwig 4'], 'version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'schema_version': '4', 'download_checksum': 'c20443ee1d095089e79facdfbc214a89', 'date_created': '2015-02-13T16:12:16.673800+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentUtils/blob/v302.1.0/bin/linux.x86_64/bedGraphToBigWig', '@id': '/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fa03d911-5b7c-40b6-9649-9086aa4b1597'}], '@id': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'edc72129-d859-4047-861b-10135504cc18', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-05-24T18:07:44.465565+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'deleted', 'aliases': ['encode:chip-seq-alignment-step-v-2-3'], 'schema_version': '4', 'minor_version': 3, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'date_created': '2019-10-18T16:26:04.956018+00:00', 'version': '2.3.4.3', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/5b64d8ad-0b79-4694-a7d2-dc13033f9504/'], 'date_created': '2016-12-07T21:59:25.746320+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'url': 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'bowtie2', 'title': 'Bowtie 2', 'purpose': ['ChIP-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.', '@id': '/software/bowtie2/', '@type': ['Software', 'Item'], 'uuid': '9a935746-3c3f-46d7-8b0a-415c2b0cecd0', 'versions': ['/software-versions/da7c531b-26f1-40a5-aa1a-5ed4f99e4d90/', '/software-versions/9c51ea8f-ac73-4a25-8783-d31bce2d5aac/', '/software-versions/b38b963c-dbe8-4de7-a2dc-ee2c593a67c6/', '/software-versions/e7dc5254-5485-44ca-afc0-0677413cde47/', '/software-versions/12c34690-5edd-4da8-a3bc-0790b0d38e14/', '/software-versions/6584bfb8-8dcd-4f46-b098-55eebc85a369/', '/software-versions/a8abe783-46a9-46cd-b5fb-23ebf39b6f71/', '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '/software-versions/45f61442-c961-435d-a919-5dd88243a4fc/', '/software-versions/75d9e6e5-1f0e-423f-9512-01b4295292e9/', '/software-versions/22cbb2b4-194f-409d-ac6e-7483118c63b4/', '/software-versions/505c6bc6-bda6-4cea-90cd-24d20ef3cd11/']}, 'status': 'released', 'downloaded_url': 'https://github.com/BenLangmead/bowtie2/archive/v2.3.4.3.tar.gz', 'schema_version': '4', 'aliases': ['encode:bowtie2.3.4.3'], 'download_checksum': '9f8bb660362813277d12580da551b0c7', '@id': '/software-versions/681b30ef-91b7-463f-919d-e7718c8efaf5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '681b30ef-91b7-463f-919d-e7718c8efaf5'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-3/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28bb298d-25d8-4599-9c26-f12dedd2a62f', 'name': 'chip-seq-alignment-step-v-2-3'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-09-18T20:14:09.493363+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T20:13:32.587576+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed'], 'schema_version': '17', 'step_label': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step', 'title': 'Ruan Lab ChIA-PIPE spp bed to bigBed bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['peaks'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dc647c58-f1c2-46bb-87dc-f6adf4c77fc5', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ec36803a-f075-4d03-8b48-3e911cbfb55d', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.966916+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.643967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e99869e1-3599-4292-a6be-af013173a646', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4e54cd81-c5ba-4866-b997-06c5cc1fffed', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-02-01T00:54:20.055540+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-dhs-accessibility-step-v-1-0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2022-02-01T00:50:45.846012+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-dhs-accessibility-step'], 'schema_version': '17', 'step_label': 'hic-dhs-accessibility-step', 'title': 'HiC DHS accessibility step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments', 'chromosome sizes'], 'output_file_types': ['nuclease cleavage frequency'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-dhs-accessibility-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '1911dc2b-5c4d-48c4-a2f9-cf8465f7d4b4', 'name': 'hic-dhs-accessibility-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', 'versions': ['/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/']}, 'software_versions': [{'date_created': '2022-01-21T18:39:05.524946+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:juicer_tools_v2.14.00'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.14.00', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v.2.14.00', '@id': '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '80a415ad-b0ed-4e38-bc11-d361318ace00'}, {'date_created': '2022-02-01T00:54:04.308073+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:bedgraphtobigwig_v381'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'version': '381', '@id': '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a746cac0-0797-4874-817e-88f688c33a58'}], '@id': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ceeb2768-81a1-4050-bc1c-0443a4ed35fb', 'name': 'hic-dhs-accessibility-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.499328+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.076302+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract'], 'schema_version': '17', 'step_label': 'ruan-lab-biginteract-conversion-step', 'title': 'Ruan Lab ChIA-PIPE bedpe to bigInteract bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['loops'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a8e0199f-9e9d-4dca-b5b6-95163a6eae5c', 'name': 'ruan-lab-biginteract-conversion-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bbd376ff-f3c2-444b-b09b-94bacbed91e8', 'name': 'ruan-lab-biginteract-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.463433+00:00', 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v-1-0'], 'status': 'released', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.360683+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-signal-generation-step', 'title': 'TF ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '997f4a70-ad7f-4d41-b988-60ab9d817c4d', 'name': 'tf-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/']}, '@id': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8036faee-410a-41be-a2b8-d310f23ad475', 'name': 'tf-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T20:00:21.296053+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.692240+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-tads-calling-step'], 'schema_version': '17', 'step_label': 'hic-tads-calling-step', 'title': 'Hi-C TADs calling step', 'major_version': 1, 'analysis_step_types': ['contact domain identification'], 'input_file_types': ['contact matrix'], 'output_file_types': ['contact domains'], 'parents': ['/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/'], '@id': '/analysis-steps/hic-tads-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '60232aac-6b7d-4510-b36a-5c28c90eff6e', 'name': 'hic-tads-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-tads-calling-step-v-1-0/', '/analysis-step-versions/hic-tads-calling-step-v-1-1/']}, 'software_versions': [{'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-tads-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '28a5945c-d1ba-4fb6-99ce-76ae5b292025', 'name': 'hic-tads-calling-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.383176+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.823285+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step', 'title': 'Ruan Lab ChIA-PIPE bam to hic bam2pairs, Juicer tools step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'c82edbd4-cdfb-4c5d-a77f-52b5d92611db', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-16T22:02:16.098864+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs_af9c8fc'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-16T21:52:38.400497+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs'], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bam2pairs', 'title': 'bam2pairs', 'source_url': 'http://github.com/4dn-dcic/pairix/tree/master/util/bam2pairs', 'used_by': ['ENCODE'], 'description': 'This script converts a paired-end bam file to a pairs file.', '@id': '/software/bam2pairs/', '@type': ['Software', 'Item'], 'uuid': 'fd4e6d1c-ff0d-42b5-88f1-cda9f330c291', 'versions': ['/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/']}, 'version': 'commit af9c8fc', 'downloaded_url': 'http://github.com/4dn-dcic/pairix/commit/af9c8fc60f201778df67581aba56a7ae23745ef6', '@id': '/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '867f4bda-802f-4e88-a35e-78886c2759ce'}, {'date_created': '2020-07-10T22:23:58.840954+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['erez-aiden:juicertools_1.7.5'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '1.7.5', 'download_checksum': 'b0dc301fed2ce85a2e6baa34bfc1cdff', 'downloaded_url': 'http://hicfiles.tc4ga.com.s3.amazonaws.com/public/juicer/juicer_tools.1.7.5_linux_x64_jcuda.0.8.jar', '@id': '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '7a54892a-47d1-464f-9407-2b1870cc9039'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '9635b2de-3625-45b0-bbee-429388d0561a', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.966916+00:00', 'schema_version': '4', 'software_versions': [{'schema_version': '4', 'downloaded_url': 'https://github.com/kundajelab/idr/releases/tag/2.0.4.2', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/d4d9c83f-d3ee-4955-bf72-88eb47512159/', '/publications/8312fc0c-b241-4cb2-9b01-1438910550ad/'], 'date_created': '2014-08-13T01:30:16.750466+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'idr', 'title': 'Irreproducible Discovery Rate (IDR)', 'source_url': 'https://sites.google.com/site/anshulkundaje/projects/idr', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Measures consistency between replicates in high-throughput experiments.  Also uses reproducibility in score rankings between peaks in each replicate to determine an optimal cutoff for significance.  The core IDR R package can be downloaded from the IDR download page: http://cran.r-project.org/web/packages/idr/index.html', '@id': '/software/idr/', '@type': ['Software', 'Item'], 'uuid': 'bd493748-5623-41f7-8867-3e090a0e8c58', 'versions': ['/software-versions/3a4bfb4d-b655-430d-a27e-011c1cdbd174/', '/software-versions/7fe855cf-255b-4950-aef9-10b066254240/', '/software-versions/b4682da5-8c2f-44d0-b931-0590a0bf3737/', '/software-versions/14b46941-7c68-48e3-bce7-6bbda5a57621/', '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/']}, 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'date_created': '2019-03-07T18:42:55.955925+00:00', 'status': 'released', 'download_checksum': '3e827d17b8aedcf9dfcd3039eec688dd', 'version': '2.0.4.2', 'aliases': ['encode:idr_2.0.4.2'], '@id': '/software-versions/fabc37f2-612c-4fb1-b975-63fd2545cb9a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fabc37f2-612c-4fb1-b975-63fd2545cb9a'}, {'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}, {'date_created': '2020-02-07T18:11:49.341859+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'downloaded_url': 'https://github.com/kundajelab/phantompeakqualtools/releases/tag/1.2.1', 'schema_version': '4', 'download_checksum': 'abed85f87a33090afd6ee2feb4fee44b', 'version': '1.2.1', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/6d54a249-990c-48c1-82a8-685bb3fd3b2b/', '/publications/f291aded-5b34-4238-9611-eedc4e1ef6f5/'], 'date_created': '2014-08-13T01:30:18.335561+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['quality metric', 'filtering'], 'name': 'phantompeakqualtools', 'title': 'Phantompeakqualtools', 'source_url': 'http://code.google.com/p/phantompeakqualtools/', 'purpose': ['data QC', 'ChIP-seq', 'DNase-seq', 'RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Used to generate these quality metrics: NSC and RSC. The NSC (Normalized strand cross-correlation) and RSC (relative strand cross-correlation) metrics use cross-correlation of stranded read density profiles to measure enrichment independently of peak calling.', '@id': '/software/phantompeakqualtools/', '@type': ['Software', 'Item'], 'uuid': 'dcf9781d-1038-47c8-9943-e285c7a6c785', 'versions': ['/software-versions/88756cca-1e79-4086-92ad-5f15ebb1fd5a/', '/software-versions/22379c5f-b5a2-4bd9-ab3e-3b76843ea814/', '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/']}, 'aliases': ['encode:phantompeaksqualtools_v1.2.1'], '@id': '/software-versions/89bc4295-df3d-465e-9a1a-2472d06d3d04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '89bc4295-df3d-465e-9a1a-2472d06d3d04'}], 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.643967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-pseudoreplicated-idr-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-pseudoreplicated-idr-step', 'title': 'TF ChIP seq pseudoreplicated idr step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'pseudoreplicated IDR'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['IDR ranked peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e99869e1-3599-4292-a6be-af013173a646', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/tf-chip-seq-pseudoreplicated-idr-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '4e54cd81-c5ba-4866-b997-06c5cc1fffed', 'name': 'tf-chip-seq-pseudoreplicated-idr-step-v-1-0'}\",\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-07T19:59:21.389725+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.072273+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-merging-quantification-interaction-calling-step'], 'schema_version': '17', 'step_label': 'hic-merging-quantification-interaction-calling-step', 'title': 'Hi-C merging, quantification, and interaction calling step', 'major_version': 1, 'analysis_step_types': ['filtering', 'quantification', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'd6bcf0ba-1bc0-4174-85d7-443d57b42eba', 'name': 'hic-merging-quantification-interaction-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-0/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '00e63923-0fc6-42d1-a196-f880f01f96e8', 'name': 'hic-merging-quantification-interaction-calling-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:51.001604+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.462793+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedgraph bedtools step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['unfiltered alignments', 'exclusion list regions'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '62e7591a-f6eb-4298-b744-0f353ff089ee', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/']}, 'software_versions': [{'date_created': '2017-06-27T03:42:14.714951+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'status': 'released', 'aliases': ['encode:bedtools 2.26.0'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'version': '2.26.0', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.26.0', '@id': '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '13e61050-fe4f-4002-9ae0-8d1adcf81a04'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '82f524da-1b07-4272-ab8e-f0c901aaf279', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:06.463433+00:00', 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v-1-0'], 'status': 'released', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:09.360683+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-signal-generation-step', 'title': 'TF ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/tf-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '997f4a70-ad7f-4d41-b988-60ab9d817c4d', 'name': 'tf-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/']}, '@id': '/analysis-step-versions/tf-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8036faee-410a-41be-a2b8-d310f23ad475', 'name': 'tf-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'date_created': '2021-06-24T00:05:34.748921+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-pairs-generation-step-1.0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.283583+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-pairs-generation-step'], 'schema_version': '17', 'step_label': 'hic-pairs-generation-step', 'title': 'Hi-C pairs generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['alignments'], 'output_file_types': ['pairs'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-pairs-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '2b2e735f-9e63-4193-9492-181a93f9aff6', 'name': 'hic-pairs-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/hic-pairs-generation-step-v-1-0/']}, 'software_versions': [{'date_created': '2021-06-24T00:30:25.776231+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-06-24T00:29:17.445634+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'pairix', 'title': 'Pairix', 'source_url': 'https://github.com/4dn-dcic/pairix', 'purpose': ['HiC'], 'used_by': ['ENCODE'], 'description': 'Pairix is a tool for indexing and querying on a block-compressed text file containing pairs of genomic coordinates.', '@id': '/software/pairix/', '@type': ['Software', 'Item'], 'uuid': 'c3e435be-b7c2-4755-b867-e3cea3d60afd', 'versions': ['/software-versions/924886cc-214c-440b-a9d0-b045fbc02f47/']}, 'version': '0.3.6', 'download_checksum': '72a9208377443c7abd4ea3c2418be890', 'downloaded_url': 'https://github.com/4dn-dcic/pairix/releases/tag/0.3.6', '@id': '/software-versions/924886cc-214c-440b-a9d0-b045fbc02f47/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '924886cc-214c-440b-a9d0-b045fbc02f47'}], '@id': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '38f5e3ed-3bcc-4e45-a445-673994e52056', 'name': 'hic-pairs-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.849098+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.303421+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_control_bam_to_bed'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bed-peak-calling-spp-step', 'title': 'Ruan Lab ChIA-PIPE bam to bed SPP step', 'major_version': 1, 'analysis_step_types': ['peak calling'], 'input_file_types': ['unfiltered alignments'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0846ae82-e09a-4b92-a0a9-f67627fb461d', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:spp 1.13'], 'version': '1.13', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/8a6e0ea0-6b0e-4fdc-885e-297c19cd75aa/'], 'date_created': '2014-08-13T01:30:13.985607+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'spp', 'title': 'SPP', 'source_url': 'http://compbio.med.harvard.edu/Supplements/ChIP-seq/', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A ChIP-seq peak calling algorithm, implemented as an R package, that accounts for the offset in forward-strand and reverse-strand reads to improve resolution, compares enrichment in signal to background or control experiments, and can also estimate whether the available number of reads is sufficient to achieve saturation, meaning that additional reads would not allow identification of additional peaks. SPP will be used in the ENCODE 3 uniform peak calling pipeline.', '@id': '/software/spp/', '@type': ['Software', 'Item'], 'uuid': 'e0c8602f-bd34-49cf-ae0a-e85f13e82470', 'versions': ['/software-versions/beb915d5-157c-48d2-9867-7d2bbf8917fe/', '/software-versions/021b7765-014b-48b5-8258-436c8f378e3c/', '/software-versions/e1ed8453-f332-49c8-b8c0-bb33c1494da4/', '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/']}, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'download_checksum': 'bdea5fbf5fbd09243f7ddd6cf8a062c8', 'date_created': '2016-06-09T15:50:40.358685+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/hms-dbmi/spp/archive/1.13.tar.gz', '@id': '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ffb5a817-e963-4694-9aae-cd89a832e1a1'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bbcef8b-821d-4abe-b571-7e255b9865ac', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T19:58:06.031901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:46.890251+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-alignment-merging-filtering-step'], 'schema_version': '17', 'step_label': 'hic-alignment-merging-filtering-step', 'title': 'Hi-C alignment, merging, and filtering step', 'major_version': 1, 'analysis_step_types': ['alignment', 'filtering'], 'input_file_types': ['reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '942efe88-2dd8-46a0-a890-3bd738ab6417', 'name': 'hic-alignment-merging-filtering-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-0/', '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1e3ed49f-bfbb-4718-ae33-c2b19440a250', 'name': 'hic-alignment-merging-filtering-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.541370+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.993488+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedpe CPU step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['alignments'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'b9a0e911-05bb-4c58-8c57-07d65db1e268', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '2d9ae5c9-4942-4407-a321-0c31831a4173', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-01-21T18:40:57.697876+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.481486+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-loop-calling-step'], 'schema_version': '17', 'step_label': 'hic-loop-calling-step', 'title': 'Hi-C loop calling step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['contact matrix'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/'], '@id': '/analysis-steps/hic-loop-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1013ae0-19ac-4f74-b9c6-87cf0369c3e5', 'name': 'hic-loop-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-loop-calling-step-v-1-0/', '/analysis-step-versions/hic-loop-calling-step-v-1-1/', '/analysis-step-versions/hic-loop-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2022-01-21T18:39:05.524946+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:juicer_tools_v2.14.00'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.14.00', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v.2.14.00', '@id': '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '80a415ad-b0ed-4e38-bc11-d361318ace00'}], '@id': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '67b98dd1-cbd5-40c8-942c-b64be7e49654', 'name': 'hic-loop-calling-step-v-1-2'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T19:59:21.389725+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.072273+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-merging-quantification-interaction-calling-step'], 'schema_version': '17', 'step_label': 'hic-merging-quantification-interaction-calling-step', 'title': 'Hi-C merging, quantification, and interaction calling step', 'major_version': 1, 'analysis_step_types': ['filtering', 'quantification', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'd6bcf0ba-1bc0-4174-85d7-443d57b42eba', 'name': 'hic-merging-quantification-interaction-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-0/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '00e63923-0fc6-42d1-a196-f880f01f96e8', 'name': 'hic-merging-quantification-interaction-calling-step-v-1-1'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T19:59:21.389725+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.072273+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-merging-quantification-interaction-calling-step'], 'schema_version': '17', 'step_label': 'hic-merging-quantification-interaction-calling-step', 'title': 'Hi-C merging, quantification, and interaction calling step', 'major_version': 1, 'analysis_step_types': ['filtering', 'quantification', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'd6bcf0ba-1bc0-4174-85d7-443d57b42eba', 'name': 'hic-merging-quantification-interaction-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-0/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-merging-quantification-interaction-calling-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '00e63923-0fc6-42d1-a196-f880f01f96e8', 'name': 'hic-merging-quantification-interaction-calling-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-06-24T00:05:34.748921+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-pairs-generation-step-1.0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.283583+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-pairs-generation-step'], 'schema_version': '17', 'step_label': 'hic-pairs-generation-step', 'title': 'Hi-C pairs generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['alignments'], 'output_file_types': ['pairs'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-pairs-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '2b2e735f-9e63-4193-9492-181a93f9aff6', 'name': 'hic-pairs-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/hic-pairs-generation-step-v-1-0/']}, 'software_versions': [{'date_created': '2021-06-24T00:30:25.776231+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-06-24T00:29:17.445634+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'pairix', 'title': 'Pairix', 'source_url': 'https://github.com/4dn-dcic/pairix', 'purpose': ['HiC'], 'used_by': ['ENCODE'], 'description': 'Pairix is a tool for indexing and querying on a block-compressed text file containing pairs of genomic coordinates.', '@id': '/software/pairix/', '@type': ['Software', 'Item'], 'uuid': 'c3e435be-b7c2-4755-b867-e3cea3d60afd', 'versions': ['/software-versions/924886cc-214c-440b-a9d0-b045fbc02f47/']}, 'version': '0.3.6', 'download_checksum': '72a9208377443c7abd4ea3c2418be890', 'downloaded_url': 'https://github.com/4dn-dcic/pairix/releases/tag/0.3.6', '@id': '/software-versions/924886cc-214c-440b-a9d0-b045fbc02f47/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '924886cc-214c-440b-a9d0-b045fbc02f47'}], '@id': '/analysis-step-versions/hic-pairs-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '38f5e3ed-3bcc-4e45-a445-673994e52056', 'name': 'hic-pairs-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-07T19:58:06.031901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:46.890251+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-alignment-merging-filtering-step'], 'schema_version': '17', 'step_label': 'hic-alignment-merging-filtering-step', 'title': 'Hi-C alignment, merging, and filtering step', 'major_version': 1, 'analysis_step_types': ['alignment', 'filtering'], 'input_file_types': ['reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '942efe88-2dd8-46a0-a890-3bd738ab6417', 'name': 'hic-alignment-merging-filtering-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-0/', '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'date_created': '2021-02-13T00:43:41.411160+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.6', 'downloaded_url': 'http://www.htslib.org/doc/1.6/samtools.html', '@id': '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc'}, {'date_created': '2022-01-27T17:48:14.805523+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': 'encode_tag_disk1', 'downloaded_url': 'https://github.com/aidenlab/juicer/releases/tag/encode_tag_disk1', '@id': '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bcbce2dc-35bf-4f21-8fa6-b41037f42fce'}, {'date_created': '2021-12-07T19:55:51.417861+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.13.06', 'download_checksum': 'b59392771c21d8a62de1904a6c94c14f', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v2.13.06', '@id': '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '38f71a81-a929-4117-bbfe-ea7e46aad7a4'}], '@id': '/analysis-step-versions/hic-alignment-merging-filtering-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1e3ed49f-bfbb-4718-ae33-c2b19440a250', 'name': 'hic-alignment-merging-filtering-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-02-01T00:54:20.055540+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-dhs-accessibility-step-v-1-0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2022-02-01T00:50:45.846012+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-dhs-accessibility-step'], 'schema_version': '17', 'step_label': 'hic-dhs-accessibility-step', 'title': 'HiC DHS accessibility step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments', 'chromosome sizes'], 'output_file_types': ['nuclease cleavage frequency'], 'parents': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/'], '@id': '/analysis-steps/hic-dhs-accessibility-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '1911dc2b-5c4d-48c4-a2f9-cf8465f7d4b4', 'name': 'hic-dhs-accessibility-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', 'versions': ['/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/']}, 'software_versions': [{'date_created': '2022-01-21T18:39:05.524946+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:juicer_tools_v2.14.00'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.14.00', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v.2.14.00', '@id': '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '80a415ad-b0ed-4e38-bc11-d361318ace00'}, {'date_created': '2022-02-01T00:54:04.308073+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:bedgraphtobigwig_v381'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'version': '381', '@id': '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a746cac0-0797-4874-817e-88f688c33a58'}], '@id': '/analysis-step-versions/hic-dhs-accessibility-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ceeb2768-81a1-4050-bc1c-0443a4ed35fb', 'name': 'hic-dhs-accessibility-step-v-1-0'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step-version-1'], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:13:35.445702+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-rsem-quantification-step', 'title': 'bulk RNA-seq rsem quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome alignments', 'genome index'], 'output_file_types': ['gene quantifications'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e33f9c72-3c30-4167-944c-7b9fffe20d89', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'download_checksum': '2e20daf7e8f96b43762f4c6f056dce8b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/7e5ae696-ee4c-49d0-9aca-11c132138f76/'], 'date_created': '2015-02-12T21:48:58.503051+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['transcript identification'], 'name': 'rsem', 'title': 'RSEM', 'source_url': 'http://deweylab.biostat.wisc.edu/rsem/', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A software package for estimating gene and isoform expression levels from RNA-Seq data.  RSEM has enabled valuable guidance for cost-efficient design of quantification experiments with RNA-Seq, which is currently relatively expensive.', '@id': '/software/rsem/', '@type': ['Software', 'Item'], 'uuid': '151bcaa2-8ffc-4429-b013-ab96a7d359e9', 'versions': ['/software-versions/08a31dbd-e957-4004-bf0c-8506388d9fed/', '/software-versions/893ce943-f35b-4e1b-bf03-cf843ba84c4b/', '/software-versions/efaf0ed5-0232-4b11-be33-902e9904858f/', '/software-versions/1be77f15-196e-4301-b27a-56053e398cfe/', '/software-versions/53932186-69c6-4338-82c1-74653bee6007/', '/software-versions/d371a8e0-ed98-4af1-98b8-6e49268d6c56/', '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '/software-versions/f8de23fe-2bcc-4253-9489-199edff353b1/']}, 'schema_version': '4', 'status': 'released', 'aliases': [], 'date_created': '2019-05-30T22:44:42.827280+00:00', 'downloaded_url': 'https://github.com/deweylab/RSEM/archive/v1.2.31.zip', 'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.2.31', '@id': '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef826860-2063-4bab-9e0c-4ad3c5ae6941'}], 'minor_version': 0, 'date_created': '2020-01-08T21:20:55.889957+00:00', '@id': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1c431e16-146c-4554-ac4e-3d5fa29116c8', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step-version-1'], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:13:35.445702+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-rsem-quantification-step', 'title': 'bulk RNA-seq rsem quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome alignments', 'genome index'], 'output_file_types': ['gene quantifications'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e33f9c72-3c30-4167-944c-7b9fffe20d89', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'download_checksum': '2e20daf7e8f96b43762f4c6f056dce8b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/7e5ae696-ee4c-49d0-9aca-11c132138f76/'], 'date_created': '2015-02-12T21:48:58.503051+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['transcript identification'], 'name': 'rsem', 'title': 'RSEM', 'source_url': 'http://deweylab.biostat.wisc.edu/rsem/', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A software package for estimating gene and isoform expression levels from RNA-Seq data.  RSEM has enabled valuable guidance for cost-efficient design of quantification experiments with RNA-Seq, which is currently relatively expensive.', '@id': '/software/rsem/', '@type': ['Software', 'Item'], 'uuid': '151bcaa2-8ffc-4429-b013-ab96a7d359e9', 'versions': ['/software-versions/08a31dbd-e957-4004-bf0c-8506388d9fed/', '/software-versions/893ce943-f35b-4e1b-bf03-cf843ba84c4b/', '/software-versions/efaf0ed5-0232-4b11-be33-902e9904858f/', '/software-versions/1be77f15-196e-4301-b27a-56053e398cfe/', '/software-versions/53932186-69c6-4338-82c1-74653bee6007/', '/software-versions/d371a8e0-ed98-4af1-98b8-6e49268d6c56/', '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '/software-versions/f8de23fe-2bcc-4253-9489-199edff353b1/']}, 'schema_version': '4', 'status': 'released', 'aliases': [], 'date_created': '2019-05-30T22:44:42.827280+00:00', 'downloaded_url': 'https://github.com/deweylab/RSEM/archive/v1.2.31.zip', 'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.2.31', '@id': '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef826860-2063-4bab-9e0c-4ad3c5ae6941'}], 'minor_version': 0, 'date_created': '2020-01-08T21:20:55.889957+00:00', '@id': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1c431e16-146c-4554-ac4e-3d5fa29116c8', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-03-16T21:27:45.038904+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step-versions'], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2022-03-16T21:19:02.589268+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:hic-diploidification-step'], 'schema_version': '17', 'step_label': 'hic-diploidification-step', 'title': 'HiC diploidification step', 'major_version': 1, 'analysis_step_types': ['signal generation', 'interaction calling'], 'input_file_types': ['phased variant calls', 'alignments'], 'output_file_types': ['haplotype-specific contact matrix', 'haplotype-specific nuclease cleavage frequency', 'haplotype-specific nuclease cleavage corrected frequency'], '@id': '/analysis-steps/hic-diploidification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dfe1f8f0-5767-4a2c-9589-90d192860d53', 'name': 'hic-diploidification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2022-03-16T20:44:16.079055+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/UM1HG009375/', 'aliases': ['encode:hic_diploidification'], 'accession': 'ENCPL478DPO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C diploidification pipeline', 'analysis_steps': ['/analysis-steps/hic-diploidification-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'pipeline_version': 1, '@id': '/pipelines/ENCPL478DPO/', '@type': ['Pipeline', 'Item'], 'uuid': '6feecc72-2038-4d3c-a6b0-64350e4f69bb'}], 'current_version': '/analysis-step-versions/hic-diploidification-step-v-1-1/', 'versions': ['/analysis-step-versions/hic-diploidification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-03-16T21:00:49.888416+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:samtools-1.14'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'version': '1.14', 'downloaded_url': 'https://sourceforge.net/projects/samtools/files/samtools/1.14/', '@id': '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '371fa24d-e400-4aac-a26a-f91e26b18ff5'}, {'date_created': '2022-03-16T21:03:17.757736+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:3d-dna-63029'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2022-03-16T20:55:51.504347+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': '3d-dna', 'title': '3d-dna', 'source_url': 'https://github.com/aidenlab/3d-dna', 'description': 'We begin with a series of iterative steps whose goal is to eliminate misjoins in the input scaffolds. Each step begins with a scaffold pool (initially, this pool is the set of input scaffolds themselves). The scaffolding algorithm is used to order and orient these scaffolds. Next, the misjoin correction algorithm is applied to detect errors in the scaffold pool, thus creating an edited scaffold pool. Finally, the edited scaffold pool is used as an input for the next iteration of the misjoin correction algorithm. The ultimate effect of these iterations is to reliably detect misjoins in the input scaffolds without removing correctly assembled sequence. After this process is complete, the scaffolding algorithm is applied to the revised input scaffolds, and the output – a single “megascaffold” which concatenates all the chromosomes – is retained for post-processing.', '@id': '/software/3d-dna/', '@type': ['Software', 'Item'], 'uuid': 'd2529df0-78cc-46a2-b01f-c9b48cc842b2', 'versions': ['/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/']}, 'version': '63029aa3bc5ba9bbdad9dd9771ace583cc95e273', 'downloaded_url': 'https://github.com/aidenlab/3d-dna/commit/63029aa3bc5ba9bbdad9dd9771ace583cc95e273', '@id': '/software-versions/ef0bec8e-2d22-457f-a3a6-ad03df6b1acc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef0bec8e-2d22-457f-a3a6-ad03df6b1acc'}, {'date_created': '2022-03-16T21:24:39.432362+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-02-23T00:07:48.376455+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/theaidenlab/juicer', 'status': 'released', 'aliases': ['encode:juicer'], 'schema_version': '10', 'name': 'juicer', 'title': 'Juicer', 'purpose': ['HiC'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'Juicer is a platform for analyzing high-resolution Hi-C data.', '@id': '/software/juicer/', '@type': ['Software', 'Item'], 'uuid': 'f828efd1-773a-449c-ba40-0e0144713fcb', 'versions': ['/software-versions/fc2bf4d3-271e-4086-a24f-d250d5d6f1cf/', '/software-versions/cb060af3-ecc0-4876-82c6-4c26649d8cd4/', '/software-versions/2dc6e2e4-950a-4eb1-9c47-32a042c63079/', '/software-versions/709cf9ca-fee5-49a0-9f61-b5718159d447/', '/software-versions/c43a5c4f-486d-4561-aa55-4d512db5b3de/', '/software-versions/de62a1ad-ca0e-413c-aa4e-0e24a2278e60/', '/software-versions/f263b742-cfb0-4710-bbc8-fa1c2c8d1a67/', '/software-versions/b78e3e43-fdd2-4f6f-a442-b322d7251a72/', '/software-versions/b2ab19ae-3223-41ae-a328-2dedcde4add4/', '/software-versions/64c1f49b-cdcd-4758-aa56-5e7bc19b1368/', '/software-versions/5b87cad9-a611-4e6c-a375-2c7672765cb9/', '/software-versions/75fc6160-ee1d-4e5b-8235-27809de94538/', '/software-versions/4942d964-75f3-481f-bc66-43fd6021dab3/', '/software-versions/bcbce2dc-35bf-4f21-8fa6-b41037f42fce/', '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '/software-versions/b0798510-0c4d-4632-925c-deaaafa6f7c1/']}, 'version': '50d557f1d4725a475071fce5975839602bd311e5', 'downloaded_url': 'https://github.com/aidenlab/juicer/commit/50d557f1d4725a475071fce5975839602bd311e5', '@id': '/software-versions/57ca1252-879d-4f2a-bfa1-8e45ce391059/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '57ca1252-879d-4f2a-bfa1-8e45ce391059'}], '@id': '/analysis-step-versions/hic-diploidification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'aaf260d0-aa38-47bf-b1b1-dbe7bad27449', 'name': 'hic-diploidification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:09:03.779901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-signal-generation-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-star-signal-generation-step', 'title': 'bulk RNA-seq star signal generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal of all reads', 'signal of unique reads', 'plus strand signal of all reads', 'plus strand signal of unique reads', 'minus strand signal of all reads', 'minus strand signal of unique reads'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '790beb4b-3b09-449c-a1fa-7dca6c802b74', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/']}, 'status': 'released', 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'software_versions': [{'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', 'schema_version': '4', 'status': 'released', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': [], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'download_checksum': '8d0c29618778013088f5fe7bd1ddb238', 'date_created': '2020-01-09T20:07:05.507787+00:00', 'version': '385', '@id': '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5d8ce593-b423-402e-93d1-f2eeca0a2f90'}, {'aliases': [], 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2020-01-08T22:50:34.724469+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/', 'status': 'released', 'aliases': ['encode:bedSort'], 'schema_version': '10', 'software_type': ['other'], 'name': 'bedsort', 'title': 'bedSort', 'source_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedSort', 'purpose': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'UCSC Genome Browser tool for sorting .bed files by chrom,chromStart.', '@id': '/software/bedsort/', '@type': ['Software', 'Item'], 'uuid': '81cba982-508f-4fd4-ab1a-5d27d6c5897c', 'versions': ['/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/']}, 'date_created': '2020-01-08T22:51:18.955130+00:00', 'download_checksum': '11abd4745b970cae6d5aa9c44429f3a2', 'status': 'released', 'version': '385', 'schema_version': '4', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', '@id': '/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8141bb60-2775-483b-8ecc-85956209ba4a'}], 'date_created': '2020-01-08T21:21:55.421044+00:00', 'aliases': ['encode:bulk-rnaseq-signal-generation-step-version-1'], 'schema_version': '4', '@id': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '53e87326-7e11-4230-8d39-66a8b8471a9c', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'software_versions': [{'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'aliases': ['dnanexus:star_v_2_5_1_b'], 'version': '2.5.1b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/9d43c048-c70c-42ab-895d-77a69fceb42b/'], 'date_created': '2014-08-13T01:30:14.893029+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'star', 'title': 'STAR', 'source_url': 'https://github.com/alexdobin/STAR/releases', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'STAR (Spliced Transcript Alignment to a Reference) aligns short and bulk RNA-seq reads to a reference genome using uncompressed suffix arrays.', '@id': '/software/star/', '@type': ['Software', 'Item'], 'uuid': 'adb49a18-c66d-4bbe-9cab-bea6fd12f8a3', 'versions': ['/software-versions/834e3145-b752-4486-a613-c0c625262051/', '/software-versions/195baf07-de0a-460d-9686-59d82bd75e50/', '/software-versions/e038206f-d791-4744-9ee5-7d22bd1f39ee/', '/software-versions/0bfb7a0a-afc7-4453-b735-ee878f23e7a9/', '/software-versions/19f75e90-e991-4dc8-837d-8cd97ab5fb91/', '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '/software-versions/f533aae8-827c-4dfe-bfe3-81143728bad8/', '/software-versions/de76db97-c034-4113-9053-74be4af14b1e/', '/software-versions/e7a3ccb9-f3e3-405e-9ee5-7ae53ece93e8/', '/software-versions/dd2d092b-61f7-491e-a127-5099a6a32d9b/', '/software-versions/ef94cd69-7316-480d-b785-e1cb36d1ea97/', '/software-versions/055f6c18-7b20-4499-9f5b-ac4bec338c47/', '/software-versions/475e856f-4d0e-459d-8417-23abd75613e3/', '/software-versions/cab07b5a-9dd9-4417-947f-2771d21db315/', '/software-versions/53519944-8257-4702-8167-0e32140ae5d6/', '/software-versions/d6aa7af7-54ef-4759-b589-91deafbde9ae/']}, 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'schema_version': '4', 'notes': 'Using bin/Linux_x86_64_static/STAR executable', 'download_checksum': '4bff1086765377f4b1706c31769367b7', 'date_created': '2016-01-20T20:45:16.541995+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/alexdobin/STAR/archive/2.5.1b.tar.gz', '@id': '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '70a085ad-194e-4de0-838c-7af257680f54'}], 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:07:57.849387+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-align-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-alignment-step', 'title': 'bulk RNA-seq alignment step', 'major_version': 1, 'analysis_step_types': ['alignment', 'QA calculation'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments', 'transcriptome alignments'], '@id': '/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ee4d1b35-e531-4900-8217-cd7719a96216', 'name': 'bulk-rna-seq-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/']}, 'schema_version': '4', 'date_created': '2020-01-08T21:18:33.262328+00:00', 'aliases': ['encode:bulk-rna-seq-star-alignment-step-version-1'], 'status': 'released', '@id': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5047d5aa-bb3b-441b-9dce-4a94e9d064dd', 'name': 'bulk-rna-seq-alignment-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'aliases': ['encode:tf-chip-seq-replicated-conservative-idr-thresholded-peaks-file-format-conversion-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:10.885105+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step', 'title': 'TF ChIP seq replicated idr thresholed peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['conservative IDR thresholded peaks', 'IDR thresholded peaks'], 'output_file_types': ['conservative IDR thresholded peaks', 'IDR thresholded peaks'], 'parents': ['/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/'], '@id': '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '5581ccf8-28ea-47b6-9c4b-c22d796cd749', 'name': 'tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}], 'current_version': '/analysis-step-versions/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/']}, 'minor_version': 0, 'schema_version': '4', 'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'date_created': '2019-11-15T05:51:10.205834+00:00', 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], '@id': '/analysis-step-versions/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'afa7862d-d1d2-4adf-98dd-c73d95e2766f', 'name': 'tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:09:03.779901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-signal-generation-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-star-signal-generation-step', 'title': 'bulk RNA-seq star signal generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal of all reads', 'signal of unique reads', 'plus strand signal of all reads', 'plus strand signal of unique reads', 'minus strand signal of all reads', 'minus strand signal of unique reads'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '790beb4b-3b09-449c-a1fa-7dca6c802b74', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/']}, 'status': 'released', 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'software_versions': [{'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', 'schema_version': '4', 'status': 'released', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': [], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'download_checksum': '8d0c29618778013088f5fe7bd1ddb238', 'date_created': '2020-01-09T20:07:05.507787+00:00', 'version': '385', '@id': '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5d8ce593-b423-402e-93d1-f2eeca0a2f90'}, {'aliases': [], 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2020-01-08T22:50:34.724469+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/', 'status': 'released', 'aliases': ['encode:bedSort'], 'schema_version': '10', 'software_type': ['other'], 'name': 'bedsort', 'title': 'bedSort', 'source_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedSort', 'purpose': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'UCSC Genome Browser tool for sorting .bed files by chrom,chromStart.', '@id': '/software/bedsort/', '@type': ['Software', 'Item'], 'uuid': '81cba982-508f-4fd4-ab1a-5d27d6c5897c', 'versions': ['/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/']}, 'date_created': '2020-01-08T22:51:18.955130+00:00', 'download_checksum': '11abd4745b970cae6d5aa9c44429f3a2', 'status': 'released', 'version': '385', 'schema_version': '4', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', '@id': '/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8141bb60-2775-483b-8ecc-85956209ba4a'}], 'date_created': '2020-01-08T21:21:55.421044+00:00', 'aliases': ['encode:bulk-rnaseq-signal-generation-step-version-1'], 'schema_version': '4', '@id': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '53e87326-7e11-4230-8d39-66a8b8471a9c', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step-version-1'], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:13:35.445702+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-rsem-quantification-step', 'title': 'bulk RNA-seq rsem quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome alignments', 'genome index'], 'output_file_types': ['gene quantifications'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e33f9c72-3c30-4167-944c-7b9fffe20d89', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'download_checksum': '2e20daf7e8f96b43762f4c6f056dce8b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/7e5ae696-ee4c-49d0-9aca-11c132138f76/'], 'date_created': '2015-02-12T21:48:58.503051+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['transcript identification'], 'name': 'rsem', 'title': 'RSEM', 'source_url': 'http://deweylab.biostat.wisc.edu/rsem/', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A software package for estimating gene and isoform expression levels from RNA-Seq data.  RSEM has enabled valuable guidance for cost-efficient design of quantification experiments with RNA-Seq, which is currently relatively expensive.', '@id': '/software/rsem/', '@type': ['Software', 'Item'], 'uuid': '151bcaa2-8ffc-4429-b013-ab96a7d359e9', 'versions': ['/software-versions/08a31dbd-e957-4004-bf0c-8506388d9fed/', '/software-versions/893ce943-f35b-4e1b-bf03-cf843ba84c4b/', '/software-versions/efaf0ed5-0232-4b11-be33-902e9904858f/', '/software-versions/1be77f15-196e-4301-b27a-56053e398cfe/', '/software-versions/53932186-69c6-4338-82c1-74653bee6007/', '/software-versions/d371a8e0-ed98-4af1-98b8-6e49268d6c56/', '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '/software-versions/f8de23fe-2bcc-4253-9489-199edff353b1/']}, 'schema_version': '4', 'status': 'released', 'aliases': [], 'date_created': '2019-05-30T22:44:42.827280+00:00', 'downloaded_url': 'https://github.com/deweylab/RSEM/archive/v1.2.31.zip', 'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.2.31', '@id': '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef826860-2063-4bab-9e0c-4ad3c5ae6941'}], 'minor_version': 0, 'date_created': '2020-01-08T21:20:55.889957+00:00', '@id': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1c431e16-146c-4554-ac4e-3d5fa29116c8', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:09:03.779901+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-signal-generation-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-star-signal-generation-step', 'title': 'bulk RNA-seq star signal generation step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal of all reads', 'signal of unique reads', 'plus strand signal of all reads', 'plus strand signal of unique reads', 'minus strand signal of all reads', 'minus strand signal of unique reads'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '790beb4b-3b09-449c-a1fa-7dca6c802b74', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/']}, 'status': 'released', 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'software_versions': [{'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', 'schema_version': '4', 'status': 'released', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'aliases': [], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'download_checksum': '8d0c29618778013088f5fe7bd1ddb238', 'date_created': '2020-01-09T20:07:05.507787+00:00', 'version': '385', '@id': '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5d8ce593-b423-402e-93d1-f2eeca0a2f90'}, {'aliases': [], 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2020-01-08T22:50:34.724469+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/', 'status': 'released', 'aliases': ['encode:bedSort'], 'schema_version': '10', 'software_type': ['other'], 'name': 'bedsort', 'title': 'bedSort', 'source_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/bedSort', 'purpose': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq'], 'used_by': ['uniform processing pipelines', 'ENCODE'], 'description': 'UCSC Genome Browser tool for sorting .bed files by chrom,chromStart.', '@id': '/software/bedsort/', '@type': ['Software', 'Item'], 'uuid': '81cba982-508f-4fd4-ab1a-5d27d6c5897c', 'versions': ['/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/']}, 'date_created': '2020-01-08T22:51:18.955130+00:00', 'download_checksum': '11abd4745b970cae6d5aa9c44429f3a2', 'status': 'released', 'version': '385', 'schema_version': '4', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentutils_v385_bin_bulkrna', '@id': '/software-versions/8141bb60-2775-483b-8ecc-85956209ba4a/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '8141bb60-2775-483b-8ecc-85956209ba4a'}], 'date_created': '2020-01-08T21:21:55.421044+00:00', 'aliases': ['encode:bulk-rnaseq-signal-generation-step-version-1'], 'schema_version': '4', '@id': '/analysis-step-versions/bulk-rna-seq-star-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '53e87326-7e11-4230-8d39-66a8b8471a9c', 'name': 'bulk-rna-seq-star-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'software_versions': [{'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'aliases': ['dnanexus:star_v_2_5_1_b'], 'version': '2.5.1b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/9d43c048-c70c-42ab-895d-77a69fceb42b/'], 'date_created': '2014-08-13T01:30:14.893029+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'star', 'title': 'STAR', 'source_url': 'https://github.com/alexdobin/STAR/releases', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'STAR (Spliced Transcript Alignment to a Reference) aligns short and bulk RNA-seq reads to a reference genome using uncompressed suffix arrays.', '@id': '/software/star/', '@type': ['Software', 'Item'], 'uuid': 'adb49a18-c66d-4bbe-9cab-bea6fd12f8a3', 'versions': ['/software-versions/834e3145-b752-4486-a613-c0c625262051/', '/software-versions/195baf07-de0a-460d-9686-59d82bd75e50/', '/software-versions/e038206f-d791-4744-9ee5-7d22bd1f39ee/', '/software-versions/0bfb7a0a-afc7-4453-b735-ee878f23e7a9/', '/software-versions/19f75e90-e991-4dc8-837d-8cd97ab5fb91/', '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '/software-versions/f533aae8-827c-4dfe-bfe3-81143728bad8/', '/software-versions/de76db97-c034-4113-9053-74be4af14b1e/', '/software-versions/e7a3ccb9-f3e3-405e-9ee5-7ae53ece93e8/', '/software-versions/dd2d092b-61f7-491e-a127-5099a6a32d9b/', '/software-versions/ef94cd69-7316-480d-b785-e1cb36d1ea97/', '/software-versions/055f6c18-7b20-4499-9f5b-ac4bec338c47/', '/software-versions/475e856f-4d0e-459d-8417-23abd75613e3/', '/software-versions/cab07b5a-9dd9-4417-947f-2771d21db315/', '/software-versions/53519944-8257-4702-8167-0e32140ae5d6/', '/software-versions/d6aa7af7-54ef-4759-b589-91deafbde9ae/']}, 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'schema_version': '4', 'notes': 'Using bin/Linux_x86_64_static/STAR executable', 'download_checksum': '4bff1086765377f4b1706c31769367b7', 'date_created': '2016-01-20T20:45:16.541995+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/alexdobin/STAR/archive/2.5.1b.tar.gz', '@id': '/software-versions/70a085ad-194e-4de0-838c-7af257680f54/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '70a085ad-194e-4de0-838c-7af257680f54'}], 'minor_version': 0, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:07:57.849387+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-align-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-alignment-step', 'title': 'bulk RNA-seq alignment step', 'major_version': 1, 'analysis_step_types': ['alignment', 'QA calculation'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments', 'transcriptome alignments'], '@id': '/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ee4d1b35-e531-4900-8217-cd7719a96216', 'name': 'bulk-rna-seq-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-1/']}, 'schema_version': '4', 'date_created': '2020-01-08T21:18:33.262328+00:00', 'aliases': ['encode:bulk-rna-seq-star-alignment-step-version-1'], 'status': 'released', '@id': '/analysis-step-versions/bulk-rna-seq-alignment-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5047d5aa-bb3b-441b-9dce-4a94e9d064dd', 'name': 'bulk-rna-seq-alignment-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step-version-1'], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:13:35.445702+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-rsem-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-rsem-quantification-step', 'title': 'bulk RNA-seq rsem quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome alignments', 'genome index'], 'output_file_types': ['gene quantifications'], 'parents': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/'], '@id': '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'e33f9c72-3c30-4167-944c-7b9fffe20d89', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'download_checksum': '2e20daf7e8f96b43762f4c6f056dce8b', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/7e5ae696-ee4c-49d0-9aca-11c132138f76/'], 'date_created': '2015-02-12T21:48:58.503051+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['transcript identification'], 'name': 'rsem', 'title': 'RSEM', 'source_url': 'http://deweylab.biostat.wisc.edu/rsem/', 'purpose': ['RNA-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A software package for estimating gene and isoform expression levels from RNA-Seq data.  RSEM has enabled valuable guidance for cost-efficient design of quantification experiments with RNA-Seq, which is currently relatively expensive.', '@id': '/software/rsem/', '@type': ['Software', 'Item'], 'uuid': '151bcaa2-8ffc-4429-b013-ab96a7d359e9', 'versions': ['/software-versions/08a31dbd-e957-4004-bf0c-8506388d9fed/', '/software-versions/893ce943-f35b-4e1b-bf03-cf843ba84c4b/', '/software-versions/efaf0ed5-0232-4b11-be33-902e9904858f/', '/software-versions/1be77f15-196e-4301-b27a-56053e398cfe/', '/software-versions/53932186-69c6-4338-82c1-74653bee6007/', '/software-versions/d371a8e0-ed98-4af1-98b8-6e49268d6c56/', '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '/software-versions/f8de23fe-2bcc-4253-9489-199edff353b1/']}, 'schema_version': '4', 'status': 'released', 'aliases': [], 'date_created': '2019-05-30T22:44:42.827280+00:00', 'downloaded_url': 'https://github.com/deweylab/RSEM/archive/v1.2.31.zip', 'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.2.31', '@id': '/software-versions/ef826860-2063-4bab-9e0c-4ad3c5ae6941/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef826860-2063-4bab-9e0c-4ad3c5ae6941'}], 'minor_version': 0, 'date_created': '2020-01-08T21:20:55.889957+00:00', '@id': '/analysis-step-versions/bulk-rna-seq-rsem-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '1c431e16-146c-4554-ac4e-3d5fa29116c8', 'name': 'bulk-rna-seq-rsem-quantification-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-01-21T18:40:57.697876+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2021-06-24T00:01:47.481486+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:hic-loop-calling-step'], 'schema_version': '17', 'step_label': 'hic-loop-calling-step', 'title': 'Hi-C loop calling step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['contact matrix'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/'], '@id': '/analysis-steps/hic-loop-calling-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1013ae0-19ac-4f74-b9c6-87cf0369c3e5', 'name': 'hic-loop-calling-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-06-24T00:06:04.145855+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL839OAB', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['HiC'], 'status': 'released', 'title': 'Hi-C pipeline', 'analysis_steps': ['/analysis-steps/hic-alignment-merging-filtering-step-v-1/', '/analysis-steps/hic-merging-quantification-interaction-calling-step-v-1/', '/analysis-steps/hic-pairs-generation-step-v-1/', '/analysis-steps/hic-loop-calling-step-v-1/', '/analysis-steps/hic-tads-calling-step-v-1/', '/analysis-steps/hic-eigenvector-creation-step-v-1/', '/analysis-steps/hic-genome-compartments-identification-step-v-1/', '/analysis-steps/hic-delta-step-v-1/', '/analysis-steps/hic-dhs-accessibility-step-v-1/', '/analysis-steps/hic-dhs-merging-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/hic-pipeline', 'standards_page': '/hic/', '@id': '/pipelines/ENCPL839OAB/', '@type': ['Pipeline', 'Item'], 'uuid': '676e390f-f058-4b78-8b90-b308cd3f78fc'}], 'current_version': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', 'versions': ['/analysis-step-versions/hic-loop-calling-step-v-1-0/', '/analysis-step-versions/hic-loop-calling-step-v-1-1/', '/analysis-step-versions/hic-loop-calling-step-v-1-2/']}, 'software_versions': [{'date_created': '2022-01-21T18:39:05.524946+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': ['encode:juicer_tools_v2.14.00'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '2.14.00', 'downloaded_url': 'https://github.com/aidenlab/Juicebox/releases/tag/v.2.14.00', '@id': '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '80a415ad-b0ed-4e38-bc11-d361318ace00'}], '@id': '/analysis-step-versions/hic-loop-calling-step-v-1-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '67b98dd1-cbd5-40c8-942c-b64be7e49654', 'name': 'hic-loop-calling-step-v-1-2'}\",\n",
       " None,\n",
       " \"{'aliases': ['encode:bulk-rnaseq-kallisto-step-version-1'], 'status': 'released', 'schema_version': '4', 'date_created': '2020-01-08T21:19:38.395952+00:00', 'minor_version': 0, 'software_versions': [{'schema_version': '4', 'date_created': '2019-02-12T23:18:11.475019+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'downloaded_url': 'https://github.com/pachterlab/kallisto/releases/download/v0.44.0/kallisto_linux-v0.44.0.tar.gz', 'aliases': ['encode:kallisto_v0.44.0'], 'version': 'v0.44.0', 'download_checksum': 'af1e787706ba16cb49c171b59eec6063', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-02-12T23:15:57.870930+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'url': 'https://github.com/pachterlab/kallisto', 'status': 'released', 'aliases': ['encode:kallisto'], 'schema_version': '10', 'name': 'kallisto', 'title': 'kallisto', 'used_by': ['community', 'uniform processing pipelines'], 'description': 'kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment. On benchmarks with standard RNA-Seq data, kallisto can quantify 30 million human reads in less than 3 minutes on a Mac desktop computer using only the read sequences and a transcriptome index that itself takes less than 10 minutes to build. Pseudoalignment of reads preserves the key information needed for quantification, and kallisto is therefore not only fast, but also as accurate than existing quantification tools. In fact, because the pseudoalignment procedure is robust to errors in the reads, in many benchmarks kallisto significantly outperforms existing tools.', '@id': '/software/kallisto/', '@type': ['Software', 'Item'], 'uuid': 'c0d93c45-b348-438b-abde-80a868a5a625', 'versions': ['/software-versions/bd220526-b2aa-4fe4-bf36-56e5da908589/']}, '@id': '/software-versions/bd220526-b2aa-4fe4-bf36-56e5da908589/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bd220526-b2aa-4fe4-bf36-56e5da908589'}], 'analysis_step': {'documents': [], 'date_created': '2020-01-08T21:12:19.473065+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['encode:bulk-rnaseq-kallisto-step'], 'schema_version': '17', 'step_label': 'bulk-rna-seq-kallisto-quantification-step', 'title': 'bulk RNA-seq kallisto quantification step', 'major_version': 1, 'analysis_step_types': ['quantification'], 'input_file_types': ['transcriptome index', 'reads'], 'output_file_types': ['transcript quantifications'], '@id': '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '93fe5f94-18a6-4fc4-a7a6-3c05da84a4b5', 'name': 'bulk-rna-seq-kallisto-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2020-01-08T21:25:16.758323+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL862USL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['RNA-seq', 'CRISPR genome editing followed by RNA-seq', 'shRNA knockdown followed by RNA-seq', 'CRISPRi followed by RNA-seq', 'siRNA knockdown followed by RNA-seq'], 'status': 'released', 'title': 'Bulk RNA-seq', 'analysis_steps': ['/analysis-steps/bulk-rna-seq-alignment-step-v-1/', '/analysis-steps/bulk-rna-seq-rsem-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-kallisto-quantification-step-v-1/', '/analysis-steps/bulk-rna-seq-star-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/rna-seq-pipeline/releases/tag/1.2.4', 'reference_filesets': ['/references/ENCSR151GDH/', '/references/ENCSR496QMW/'], '@id': '/pipelines/ENCPL862USL/', '@type': ['Pipeline', 'Item'], 'uuid': 'cb9f028e-fdb3-4c32-8c3e-a361b72d2bf6'}], 'current_version': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', 'versions': ['/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/']}, 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', '@id': '/analysis-step-versions/bulk-rna-seq-kallisto-quantification-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bd3ed9ca-922b-4b2b-a0a1-e103ec36379d', 'name': 'bulk-rna-seq-kallisto-quantification-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.541370+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.993488+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedpe CPU step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['alignments'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'b9a0e911-05bb-4c58-8c57-07d65db1e268', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '2d9ae5c9-4942-4407-a321-0c31831a4173', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:51.001604+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.462793+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedgraph bedtools step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['unfiltered alignments', 'exclusion list regions'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '62e7591a-f6eb-4298-b744-0f353ff089ee', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/']}, 'software_versions': [{'date_created': '2017-06-27T03:42:14.714951+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'status': 'released', 'aliases': ['encode:bedtools 2.26.0'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'version': '2.26.0', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.26.0', '@id': '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '13e61050-fe4f-4002-9ae0-8d1adcf81a04'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '82f524da-1b07-4272-ab8e-f0c901aaf279', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.541370+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.993488+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_loop'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedpe CPU step', 'major_version': 1, 'analysis_step_types': ['chromatin loop identification'], 'input_file_types': ['alignments'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'b9a0e911-05bb-4c58-8c57-07d65db1e268', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '2d9ae5c9-4942-4407-a321-0c31831a4173', 'name': 'ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T20:14:09.493363+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T20:13:32.587576+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed'], 'schema_version': '17', 'step_label': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step', 'title': 'Ruan Lab ChIA-PIPE spp bed to bigBed bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['peaks'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dc647c58-f1c2-46bb-87dc-f6adf4c77fc5', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ec36803a-f075-4d03-8b48-3e911cbfb55d', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.847238+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.428239+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig'], 'schema_version': '17', 'step_label': 'ruan-lab-bedgraph-conversion-to-bigwig-step', 'title': 'Ruan Lab ChIA-PIPE bedgraph to bigWig bedGraphToBigWig step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['signal of unique reads'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '12c47b68-0fe9-4e69-8b54-7b509b376a37', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:bedgraphtobigwig 4'], 'version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'schema_version': '4', 'download_checksum': 'c20443ee1d095089e79facdfbc214a89', 'date_created': '2015-02-13T16:12:16.673800+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentUtils/blob/v302.1.0/bin/linux.x86_64/bedGraphToBigWig', '@id': '/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fa03d911-5b7c-40b6-9649-9086aa4b1597'}], '@id': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'edc72129-d859-4047-861b-10135504cc18', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:51.001604+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph_bedtools_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.462793+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bedgraph'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step', 'title': 'Ruan Lab ChIA-PIPE bam to bedgraph bedtools step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['unfiltered alignments', 'exclusion list regions'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '62e7591a-f6eb-4298-b744-0f353ff089ee', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/']}, 'software_versions': [{'date_created': '2017-06-27T03:42:14.714951+00:00', 'submitted_by': '/users/332d0e03-a907-4f53-8358-bb00118277c8/', 'status': 'released', 'aliases': ['encode:bedtools 2.26.0'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'version': '2.26.0', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.26.0', '@id': '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '13e61050-fe4f-4002-9ae0-8d1adcf81a04'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '82f524da-1b07-4272-ab8e-f0c901aaf279', 'name': 'ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.849098+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.303421+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_control_bam_to_bed'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bed-peak-calling-spp-step', 'title': 'Ruan Lab ChIA-PIPE bam to bed SPP step', 'major_version': 1, 'analysis_step_types': ['peak calling'], 'input_file_types': ['unfiltered alignments'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0846ae82-e09a-4b92-a0a9-f67627fb461d', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:spp 1.13'], 'version': '1.13', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/8a6e0ea0-6b0e-4fdc-885e-297c19cd75aa/'], 'date_created': '2014-08-13T01:30:13.985607+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'spp', 'title': 'SPP', 'source_url': 'http://compbio.med.harvard.edu/Supplements/ChIP-seq/', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A ChIP-seq peak calling algorithm, implemented as an R package, that accounts for the offset in forward-strand and reverse-strand reads to improve resolution, compares enrichment in signal to background or control experiments, and can also estimate whether the available number of reads is sufficient to achieve saturation, meaning that additional reads would not allow identification of additional peaks. SPP will be used in the ENCODE 3 uniform peak calling pipeline.', '@id': '/software/spp/', '@type': ['Software', 'Item'], 'uuid': 'e0c8602f-bd34-49cf-ae0a-e85f13e82470', 'versions': ['/software-versions/beb915d5-157c-48d2-9867-7d2bbf8917fe/', '/software-versions/021b7765-014b-48b5-8258-436c8f378e3c/', '/software-versions/e1ed8453-f332-49c8-b8c0-bb33c1494da4/', '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/']}, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'download_checksum': 'bdea5fbf5fbd09243f7ddd6cf8a062c8', 'date_created': '2016-06-09T15:50:40.358685+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/hms-dbmi/spp/archive/1.13.tar.gz', '@id': '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ffb5a817-e963-4694-9aae-cd89a832e1a1'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bbcef8b-821d-4abe-b571-7e255b9865ac', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.232991+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.661531+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_fastq_to_bam'], 'schema_version': '17', 'step_label': 'ruan-lab-fastq-to-bam-alignment-cpu-step', 'title': 'Ruan Lab ChIA-PIPE fastq to bam CPU step', 'major_version': 1, 'analysis_step_types': ['filtering', 'classification', 'alignment'], 'input_file_types': ['reads', 'genome reference'], 'output_file_types': ['unfiltered alignments', 'alignments'], '@id': '/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'ac1b0720-644f-4f8d-bfeb-366b5c80feb1', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-10T22:23:58.663516+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU_0.0.1a-r2'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-10T20:29:34.214359+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:CPU'], 'schema_version': '10', 'name': 'cpu', 'title': 'CPU', 'source_url': 'http://github.com/cheehongsg/CPU', 'purpose': ['ChIA-PET'], 'used_by': ['ENCODE'], 'description': 'ChIA-PET Utilities is a collection of efficient specialized programs for processing ChIA-PET data from raw reads to interactions.', '@id': '/software/cpu/', '@type': ['Software', 'Item'], 'uuid': '76e32489-fb6f-4978-b99b-6dc7a0159d3f', 'versions': ['/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/']}, 'version': '0.0.1a Release 2', 'downloaded_url': 'http://github.com/cheehongsg/CPU/releases/tag/v0.0.1a-r2', '@id': '/software-versions/0fbcdcdc-3683-4f6b-9227-dcf6d287f09f/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '0fbcdcdc-3683-4f6b-9227-dcf6d287f09f'}], '@id': '/analysis-step-versions/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '24c53733-ece3-4e7b-8ed5-8d850e66e2e5', 'name': 'ruan-lab-fastq-to-bam-alignment-cpu-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.383176+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.823285+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step', 'title': 'Ruan Lab ChIA-PIPE bam to hic bam2pairs, Juicer tools step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'c82edbd4-cdfb-4c5d-a77f-52b5d92611db', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-16T22:02:16.098864+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs_af9c8fc'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-16T21:52:38.400497+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs'], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bam2pairs', 'title': 'bam2pairs', 'source_url': 'http://github.com/4dn-dcic/pairix/tree/master/util/bam2pairs', 'used_by': ['ENCODE'], 'description': 'This script converts a paired-end bam file to a pairs file.', '@id': '/software/bam2pairs/', '@type': ['Software', 'Item'], 'uuid': 'fd4e6d1c-ff0d-42b5-88f1-cda9f330c291', 'versions': ['/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/']}, 'version': 'commit af9c8fc', 'downloaded_url': 'http://github.com/4dn-dcic/pairix/commit/af9c8fc60f201778df67581aba56a7ae23745ef6', '@id': '/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '867f4bda-802f-4e88-a35e-78886c2759ce'}, {'date_created': '2020-07-10T22:23:58.840954+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['erez-aiden:juicertools_1.7.5'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '1.7.5', 'download_checksum': 'b0dc301fed2ce85a2e6baa34bfc1cdff', 'downloaded_url': 'http://hicfiles.tc4ga.com.s3.amazonaws.com/public/juicer/juicer_tools.1.7.5_linux_x64_jcuda.0.8.jar', '@id': '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '7a54892a-47d1-464f-9407-2b1870cc9039'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '9635b2de-3625-45b0-bbee-429388d0561a', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.499328+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.076302+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract'], 'schema_version': '17', 'step_label': 'ruan-lab-biginteract-conversion-step', 'title': 'Ruan Lab ChIA-PIPE bedpe to bigInteract bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['loops'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a8e0199f-9e9d-4dca-b5b6-95163a6eae5c', 'name': 'ruan-lab-biginteract-conversion-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bbd376ff-f3c2-444b-b09b-94bacbed91e8', 'name': 'ruan-lab-biginteract-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T20:14:09.493363+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T20:13:32.587576+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_spp_bed_to_bigbed'], 'schema_version': '17', 'step_label': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step', 'title': 'Ruan Lab ChIA-PIPE spp bed to bigBed bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['peaks'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'dc647c58-f1c2-46bb-87dc-f6adf4c77fc5', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'ec36803a-f075-4d03-8b48-3e911cbfb55d', 'name': 'ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2020-07-24T17:32:50.383176+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:23.823285+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_hic'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step', 'title': 'Ruan Lab ChIA-PIPE bam to hic bam2pairs, Juicer tools step', 'major_version': 1, 'analysis_step_types': ['file format conversion', 'interaction calling'], 'input_file_types': ['alignments'], 'output_file_types': ['contact matrix'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'c82edbd4-cdfb-4c5d-a77f-52b5d92611db', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/']}, 'software_versions': [{'date_created': '2020-07-16T22:02:16.098864+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs_af9c8fc'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2020-07-16T21:52:38.400497+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['encode:bam2pairs'], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bam2pairs', 'title': 'bam2pairs', 'source_url': 'http://github.com/4dn-dcic/pairix/tree/master/util/bam2pairs', 'used_by': ['ENCODE'], 'description': 'This script converts a paired-end bam file to a pairs file.', '@id': '/software/bam2pairs/', '@type': ['Software', 'Item'], 'uuid': 'fd4e6d1c-ff0d-42b5-88f1-cda9f330c291', 'versions': ['/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/']}, 'version': 'commit af9c8fc', 'downloaded_url': 'http://github.com/4dn-dcic/pairix/commit/af9c8fc60f201778df67581aba56a7ae23745ef6', '@id': '/software-versions/867f4bda-802f-4e88-a35e-78886c2759ce/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '867f4bda-802f-4e88-a35e-78886c2759ce'}, {'date_created': '2020-07-10T22:23:58.840954+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['erez-aiden:juicertools_1.7.5'], 'schema_version': '4', 'software': {'lab': '/labs/erez-aiden/', 'award': '/awards/UM1HG009375/', 'references': [], 'date_created': '2018-06-08T17:17:08.677188+00:00', 'submitted_by': '/users/a3f5ce81-93e3-4769-8552-7f8024ee29f9/', 'status': 'released', 'aliases': ['erez-lieberman:juicertools'], 'schema_version': '10', 'name': 'juicertools', 'title': 'juicertools', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Software package for analysis of Hi-C data.', '@id': '/software/juicertools/', '@type': ['Software', 'Item'], 'uuid': 'bb23eed2-d4f8-4ab7-8685-91592d097017', 'versions': ['/software-versions/b9c1599f-23ae-4d75-bc24-815b55d317ce/', '/software-versions/5fec2dd1-c4a4-4ebf-a728-ccbc9d4ae318/', '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '/software-versions/baf6c06f-352d-4f70-ac93-b52131e66a24/', '/software-versions/63b38d7b-3c68-4529-867c-4c9fa435e38c/', '/software-versions/048625bd-b0b4-443a-b08f-e66431be8a18/', '/software-versions/38f71a81-a929-4117-bbfe-ea7e46aad7a4/', '/software-versions/80a415ad-b0ed-4e38-bc11-d361318ace00/', '/software-versions/9188181b-b38b-4d99-90ae-f9dc685ac2ef/']}, 'version': '1.7.5', 'download_checksum': 'b0dc301fed2ce85a2e6baa34bfc1cdff', 'downloaded_url': 'http://hicfiles.tc4ga.com.s3.amazonaws.com/public/juicer/juicer_tools.1.7.5_linux_x64_jcuda.0.8.jar', '@id': '/software-versions/7a54892a-47d1-464f-9407-2b1870cc9039/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '7a54892a-47d1-464f-9407-2b1870cc9039'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '9635b2de-3625-45b0-bbee-429388d0561a', 'name': 'ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-09-18T19:51:19.499328+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.076302+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedpe_to_biginteract'], 'schema_version': '17', 'step_label': 'ruan-lab-biginteract-conversion-step', 'title': 'Ruan Lab ChIA-PIPE bedpe to bigInteract bedToBigBed step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['loops'], 'output_file_types': ['loops'], 'parents': ['/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a8e0199f-9e9d-4dca-b5b6-95163a6eae5c', 'name': 'ruan-lab-biginteract-conversion-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/']}, 'software_versions': [{'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'aliases': ['encode:bedtobigbed v369'], 'downloaded_url': 'http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/', 'schema_version': '4', 'version': '369', 'status': 'released', 'submitted_by': '/users/76091563-a959-4a9c-929c-9acfa1a0a078/', 'date_created': '2019-05-21T22:22:11.679587+00:00', '@id': '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '55771f3c-384b-4cd0-aa5b-7420d3f72676'}], '@id': '/analysis-step-versions/ruan-lab-biginteract-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'bbd376ff-f3c2-444b-b09b-94bacbed91e8', 'name': 'ruan-lab-biginteract-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2020-07-24T17:32:50.849098+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bam_to_bed_spp_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-07-24T17:32:24.303421+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_control_bam_to_bed'], 'schema_version': '17', 'step_label': 'ruan-lab-bam-to-bed-peak-calling-spp-step', 'title': 'Ruan Lab ChIA-PIPE bam to bed SPP step', 'major_version': 1, 'analysis_step_types': ['peak calling'], 'input_file_types': ['unfiltered alignments'], 'output_file_types': ['peaks'], 'parents': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0846ae82-e09a-4b92-a0a9-f67627fb461d', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:spp 1.13'], 'version': '1.13', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/8a6e0ea0-6b0e-4fdc-885e-297c19cd75aa/'], 'date_created': '2014-08-13T01:30:13.985607+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'spp', 'title': 'SPP', 'source_url': 'http://compbio.med.harvard.edu/Supplements/ChIP-seq/', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A ChIP-seq peak calling algorithm, implemented as an R package, that accounts for the offset in forward-strand and reverse-strand reads to improve resolution, compares enrichment in signal to background or control experiments, and can also estimate whether the available number of reads is sufficient to achieve saturation, meaning that additional reads would not allow identification of additional peaks. SPP will be used in the ENCODE 3 uniform peak calling pipeline.', '@id': '/software/spp/', '@type': ['Software', 'Item'], 'uuid': 'e0c8602f-bd34-49cf-ae0a-e85f13e82470', 'versions': ['/software-versions/beb915d5-157c-48d2-9867-7d2bbf8917fe/', '/software-versions/021b7765-014b-48b5-8258-436c8f378e3c/', '/software-versions/e1ed8453-f332-49c8-b8c0-bb33c1494da4/', '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '/software-versions/d7608aac-8fc1-49ef-ba54-72c79aa57061/']}, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'schema_version': '4', 'download_checksum': 'bdea5fbf5fbd09243f7ddd6cf8a062c8', 'date_created': '2016-06-09T15:50:40.358685+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/hms-dbmi/spp/archive/1.13.tar.gz', '@id': '/software-versions/ffb5a817-e963-4694-9aae-cd89a832e1a1/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ffb5a817-e963-4694-9aae-cd89a832e1a1'}], '@id': '/analysis-step-versions/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5bbcef8b-821d-4abe-b571-7e255b9865ac', 'name': 'ruan-lab-bam-to-bed-peak-calling-spp-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'date_created': '2020-09-18T19:51:19.847238+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig_version'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2020-09-18T18:41:44.428239+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'status': 'released', 'aliases': ['yijun-ruan:chiapipe_bedgraph_to_bigwig'], 'schema_version': '17', 'step_label': 'ruan-lab-bedgraph-conversion-to-bigwig-step', 'title': 'Ruan Lab ChIA-PIPE bedgraph to bigWig bedGraphToBigWig step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['signal of unique reads'], 'output_file_types': ['signal of unique reads'], 'parents': ['/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/'], '@id': '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '12c47b68-0fe9-4e69-8b54-7b509b376a37', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1', 'pipelines': [{'documents': ['/documents/47cf3c63-12c4-4b32-afb5-0e8141918ffa/'], 'date_created': '2020-07-24T17:35:04.186569+00:00', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'references': [], 'lab': '/labs/yijun-ruan/', 'award': '/awards/UM1HG009409/', 'aliases': ['yijun-ruan:chia-pipe-pipeline'], 'accession': 'ENCPL169TBL', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIA-PET'], 'status': 'released', 'title': 'Ruan Lab ChIA-PIPE Pipeline', 'description': 'ChIA-PIPE is a fully automated pipeline for ChIA-PET data processing, quality assessment, visualization, and analysis. ChIA-PIPE performs linker filtering, read mapping, peak calling, and loop calling and automates quality control assessment for each dataset. To enable visualization, ChIA-PIPE generates input files for two-dimensional contact map viewing with Juicebox and HiGlass and provides a new dockerized visualization tool for high-resolution, browser-based exploration of peaks and loops. To enable structural interpretation, ChIA-PIPE calls chromatin contact domains, resolves allele-specific peaks and loops, and annotates enhancer-promoter loops. ChIA-PIPE also supports the analysis of other related chromatin-mapping data types.', 'analysis_steps': ['/analysis-steps/ruan-lab-fastq-to-bam-alignment-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-hic-interaction-calling-conversion-bams2pairs-juicertools-step-v-1/', '/analysis-steps/ruan-lab-bam-to-loop-chromatin-loop-identification-cpu-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-macs2-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bed-peak-calling-spp-step-v-1/', '/analysis-steps/ruan-lab-bam-to-bedgraph-genome-cov-bedtools-step-v-1/', '/analysis-steps/ruan-lab-biginteract-conversion-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bigbed-conversion-spp-bedtobigbed-step-v-1/', '/analysis-steps/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1/'], 'source_url': 'http://github.com/TheJacksonLaboratory/ChIA-PIPE', '@id': '/pipelines/ENCPL169TBL/', '@type': ['Pipeline', 'Item'], 'uuid': 'b885c8d7-061c-4abc-8669-6d7639d67b01'}], 'current_version': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', 'versions': ['/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/']}, 'software_versions': [{'aliases': ['encode:bedgraphtobigwig 4'], 'version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-09T16:26:04.154964+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedgraphtobigwig', 'title': 'bedGraphToBigWig', 'source_url': 'https://github.com/ENCODE-DCC/kentUtils', 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'Convert bedGraph to bigWig file.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891', '@id': '/software/bedgraphtobigwig/', '@type': ['Software', 'Item'], 'uuid': '904cb1b7-b4bd-4eac-a339-50b7b9e7414b', 'versions': ['/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '/software-versions/32963773-e32a-48b6-8a78-59733d2f2333/', '/software-versions/5d8ce593-b423-402e-93d1-f2eeca0a2f90/', '/software-versions/9520d8c7-a37f-4e01-9c84-ada587a495c6/', '/software-versions/9aa0a086-a170-4a25-bc90-69feeb713c38/', '/software-versions/633add07-9f9e-4521-8181-371c33e38826/', '/software-versions/a746cac0-0797-4874-817e-88f688c33a58/']}, 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'schema_version': '4', 'download_checksum': 'c20443ee1d095089e79facdfbc214a89', 'date_created': '2015-02-13T16:12:16.673800+00:00', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/kentUtils/blob/v302.1.0/bin/linux.x86_64/bedGraphToBigWig', '@id': '/software-versions/fa03d911-5b7c-40b6-9649-9086aa4b1597/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'fa03d911-5b7c-40b6-9649-9086aa4b1597'}], '@id': '/analysis-step-versions/ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'edc72129-d859-4047-861b-10135504cc18', 'name': 'ruan-lab-bedgraph-conversion-to-bigwig-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " None,\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.607170+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.003191+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-alignment-step', 'title': 'STARsolo alignment step', 'major_version': 1, 'analysis_step_types': ['alignment'], 'input_file_types': ['genome index', 'reads'], 'output_file_types': ['alignments'], '@id': '/analysis-steps/starsolo-alignment-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '9567f797-9412-4d8c-9f10-6e18bd35c787', 'name': 'starsolo-alignment-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-alignment-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-alignment-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '134cc1e8-0725-42f4-8e5a-ff8fe178a23c', 'name': 'starsolo-alignment-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2021-12-14T22:15:36.817179+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 1, 'analysis_step': {'documents': [], 'date_created': '2021-12-14T22:15:07.208387+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '17', 'step_label': 'starsolo-quantification-step', 'title': 'STARsolo quantification step', 'major_version': 1, 'analysis_step_types': ['quantification', 'filtering'], 'input_file_types': ['alignments', 'inclusion list'], 'output_file_types': ['sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of all reads', 'sparse gene count matrix of unique reads', 'unfiltered sparse gene count matrix of unique reads', 'unfiltered sparse splice junction count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-alignment-step-v-1/'], '@id': '/analysis-steps/starsolo-quantification-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '015dbf5c-b722-4f3e-abf4-d376b95096f1', 'name': 'starsolo-quantification-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', 'versions': ['/analysis-step-versions/starsolo-quantification-step-v-1-1/']}, 'software_versions': [{'date_created': '2022-02-10T21:41:47.876419+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': ['barbara-wold:starsolo 2.7.10a'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2021-12-14T22:14:10.766199+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'name': 'starsolo', 'title': 'STARsolo', 'source_url': 'https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md', 'description': 'STARsolo is a tool for mapping, demultiplexing, and quantification for single cell RNA-seq.', '@id': '/software/starsolo/', '@type': ['Software', 'Item'], 'uuid': '80ba5bac-afa7-46bd-b536-ba6e873f70e4', 'versions': ['/software-versions/e874ad65-ad64-4fe9-bf46-833ab89befca/', '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/']}, 'version': '2.7.10a', 'downloaded_url': 'https://github.com/alexdobin/STAR/releases/tag/2.7.10a', '@id': '/software-versions/ce211075-8eaa-4952-bda1-275e597e980b/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ce211075-8eaa-4952-bda1-275e597e980b'}], '@id': '/analysis-step-versions/starsolo-quantification-step-v-1-1/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b6611d99-10dd-47fe-9f31-ff0ed63ae0a0', 'name': 'starsolo-quantification-step-v-1-1'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " \"{'date_created': '2023-05-17T21:07:26.509984+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo_merging_step_v0'], 'schema_version': '4', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2023-05-17T20:56:32.688857+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:starsolo-merging-step'], 'schema_version': '17', 'step_label': 'starsolo-merging-step', 'title': 'STARsolo merging step', 'major_version': 1, 'analysis_step_types': ['merging'], 'input_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'output_file_types': ['unfiltered sparse gene count matrix of all reads', 'unfiltered sparse gene count matrix of unique reads'], 'parents': ['/analysis-steps/starsolo-quantification-step-v-1/'], '@id': '/analysis-steps/starsolo-merging-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '4fa66cb8-7f34-4e9b-89d8-a9395e025dfc', 'name': 'starsolo-merging-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2021-12-14T22:16:14.887553+00:00', 'submitted_by': '/users/85c75ed3-269b-4273-b4ba-e269ee350dd6/', 'references': [], 'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'aliases': ['barbara-wold:starsolo_pipeline'], 'accession': 'ENCPL257SYI', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['single-cell RNA sequencing assay'], 'status': 'released', 'title': 'Single-cell RNA-seq STARsolo Pipeline', 'description': 'Pipeline for processing 10X-style short read, single-cell RNA-seq data.', 'analysis_steps': ['/analysis-steps/starsolo-alignment-step-v-1/', '/analysis-steps/starsolo-quantification-step-v-1/', '/analysis-steps/starsolo-merging-step-v-1/'], 'source_url': 'https://github.com/detrout/woldlab-rna-seq/releases/tag/1.2.5', '@id': '/pipelines/ENCPL257SYI/', '@type': ['Pipeline', 'Item'], 'uuid': '752652b3-68fb-4fda-a42e-6362628e3e5e'}], 'current_version': '/analysis-step-versions/starsolo-merging-step-v-1-0/', 'versions': ['/analysis-step-versions/starsolo-merging-step-v-1-0/']}, 'software_versions': [{'date_created': '2023-05-17T20:53:41.352354+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': ['barbara-wold:mex_gene_archive_0.2.3'], 'schema_version': '4', 'software': {'lab': '/labs/barbara-wold/', 'award': '/awards/UM1HG009443/', 'references': [], 'date_created': '2023-05-17T20:51:29.176732+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'mex_gene_archive', 'title': 'mex_gene_archive', 'source_url': 'https://github.com/detrout/mex_gene_archive', 'used_by': ['ENCODE'], 'description': 'mex_gene_archive is a minimal file format designed to meet the needs of archiving sparse gene matrices in a format compatible with the ENCODE 4 Data Coordination Center.', '@id': '/software/mex_gene_archive/', '@type': ['Software', 'Item'], 'uuid': 'ff98ad0b-010d-490c-a1c6-0963c8427162', 'versions': ['/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/']}, 'version': '0.2.3', 'downloaded_url': 'https://github.com/detrout/mex_gene_archive/releases/tag/0.2.3', '@id': '/software-versions/ad339383-de8c-4071-9d41-df910cc3d5bc/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ad339383-de8c-4071-9d41-df910cc3d5bc'}], '@id': '/analysis-step-versions/starsolo-merging-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '8f0f1213-8c4f-4e4c-8a50-660cdd0ac028', 'name': 'starsolo-merging-step-v-1-0'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " \"{'date_created': '2019-11-15T05:51:11.103370+00:00', 'schema_version': '4', 'software_versions': [{'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-06-23T22:20:17.989102+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'status': 'released', 'aliases': ['encode:overlap_peaks'], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'overlap_peaks', 'title': 'overlap_peaks.py', 'source_url': 'https://github.com/ENCODE-DCC/tf_chipseq/tree/master/dnanexus/overlap_peaks', 'used_by': ['uniform processing pipelines'], 'description': 'overlap_peaks.py is a method to select overlapping peaks among replicates and psuedo-replicates', '@id': '/software/overlap_peaks/', '@type': ['Software', 'Item'], 'uuid': '620dfb1a-149f-46d9-b9e1-15bd5ed6f8d9', 'versions': ['/software-versions/64b2d05e-3347-4edd-b0a9-efbf1a259fbd/', '/software-versions/7e3ee587-f88d-404f-aabe-d44d6e6c99a4/', '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': 'ATAC v1.6.1', 'schema_version': '4', 'status': 'released', 'downloaded_url': 'https://github.com/ENCODE-DCC/atac-seq-pipeline/releases/tag/v1.6.1', 'download_checksum': '735bf408ed2a443969b5b722d33d923d', 'date_created': '2020-02-07T18:08:04.707992+00:00', 'aliases': ['encode:encode_task_overlap_ATAC_v1.6.1'], '@id': '/software-versions/a0ece1ec-a61e-49c3-ab39-b3cb64425659/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'a0ece1ec-a61e-49c3-ab39-b3cb64425659'}], 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'], 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.192652+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-step', 'title': 'histone ChIP seq pseudoreplicated partition concordance step', 'major_version': 1, 'analysis_step_types': ['peak calling', 'partition concordance'], 'input_file_types': ['alignments', 'exclusion list regions'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '741334f2-db96-4f1a-82ee-55dfaebced33', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/']}, 'minor_version': 0, 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'b279b2ae-8da9-408f-bd7d-74dba20ec407', 'name': 'histone-chip-seq-pseudoreplicated-overlap-step-v-1-0'}\",\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'date_created': '2022-06-20T16:35:32.398515+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'minor_version': 2, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:05.864615+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:chip-seq-alignment-step-v2'], 'schema_version': '17', 'step_label': 'chip-seq-alignment-step', 'title': 'ChIP seq alignment step', 'major_version': 2, 'analysis_step_types': ['read trimming', 'fastq concatenation', 'alignment', 'filtering'], 'input_file_types': ['reads', 'genome index'], 'output_file_types': ['alignments', 'unfiltered alignments'], '@id': '/analysis-steps/chip-seq-alignment-step-v-2/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'a1566d04-868e-4149-b1d9-5164999d563b', 'name': 'chip-seq-alignment-step-v-2', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T05:56:02.884967+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL367MAS', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-replicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pooled-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/', '/analysis-steps/tf-chip-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL367MAS/', '@type': ['Pipeline', 'Item'], 'uuid': '7160c087-629e-40fc-bcda-c43104fa34d8'}, {'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}, {'documents': [], 'date_created': '2019-11-15T05:53:48.357267+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL481MLO', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq'], 'status': 'released', 'title': 'Transcription factor ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/tf-chip-signal-generation-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-step-v-1/', '/analysis-steps/tf-chip-seq-pseudoreplicated-idr-thresholded-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL481MLO/', '@type': ['Pipeline', 'Item'], 'uuid': '460c58d6-2da3-4570-b38d-a9d905150b05'}], 'current_version': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', 'versions': ['/analysis-step-versions/chip-seq-alignment-step-v-2-0/', '/analysis-step-versions/chip-seq-alignment-step-v-2-1/', '/analysis-step-versions/chip-seq-alignment-step-v-2-2/']}, 'software_versions': [{'status': 'released', 'downloaded_url': 'http://sourceforge.net/projects/bio-bwa/files/bwa-0.7.17.tar.bz2/download', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/e20c4884-62b4-4e55-a853-b8d3ff8740b0/'], 'date_created': '2015-02-12T21:35:56.411340+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://bio-bwa.sourceforge.net/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['aligner'], 'name': 'bwa', 'title': 'BWA', 'source_url': 'http://bio-bwa.sourceforge.net/', 'purpose': ['DNase-seq', 'ChIP-seq'], 'used_by': ['uniform processing pipelines'], 'description': 'BWA is a software package for mapping low-divergent sequences based on a Burrows-Wheeler index against a large reference genome, such as the human genome.  Publications for the short read alignment component is found at PMID: 19451168, while PMID: 20080505 outlines the algorithm to align sequences >200bp up to 1Mb.', '@id': '/software/bwa/', '@type': ['Software', 'Item'], 'uuid': '37a517fc-1f47-4fd4-864b-a5f37f6347d8', 'versions': ['/software-versions/de3c16f7-be94-4b2f-a7b4-c44ba67735d7/', '/software-versions/c707737d-71cf-488e-8430-03621e7de06c/', '/software-versions/bbaad0ce-90db-4a19-937a-7c171b05ed4b/', '/software-versions/c5c9e884-6d42-4ec2-bdb9-d73870ceddee/', '/software-versions/ba333483-df65-4f48-b210-6fdc9572c1de/', '/software-versions/85010760-a946-4f22-a6aa-6f94cc32e528/', '/software-versions/b85286f2-34ea-4c92-ac14-02eaac5da565/', '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '/software-versions/cbb9c467-eaed-4e63-b657-56d3f4091dce/']}, 'version': '0.7.17', 'submitted_by': '/users/38dc09c2-4c05-49d2-b73d-4b75261fcbf1/', 'schema_version': '4', 'date_created': '2019-11-27T22:44:33.534035+00:00', 'aliases': ['encode:bwa_0.7.17'], '@id': '/software-versions/c15354da-2e6d-40d0-8877-e0cb601e76cd/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'c15354da-2e6d-40d0-8877-e0cb601e76cd'}, {'submitted_by': '/users/7e944358-625c-426c-975c-7d875e22f753/', 'version': '1.9', 'download_checksum': 'e2f2e401af7e8309075ba4545118819d', 'aliases': ['encode:samtools 1.9'], 'status': 'released', 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:54:17.918472+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://www.htslib.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'samtools', 'title': 'Samtools', 'source_url': 'http://sourceforge.net/projects/samtools/files/samtools/', 'used_by': ['community', 'ENCODE'], 'description': 'Samtools is a suite of programs for interacting with high-throughput sequencing data.  SAMtools implements various utilities for post-processing alignments in the SAM format, such as indexing, variant caller and alignment viewer, and thus provides universal tools for processing read alignments (PMID:19505943).', '@id': '/software/samtools/', '@type': ['Software', 'Item'], 'uuid': 'ce18defa-8989-4067-9ac7-2f13734223da', 'versions': ['/software-versions/d44db1ed-3af1-4e3b-ae7a-b0fb1085877f/', '/software-versions/f21b7bbf-6178-4f1a-8c58-caeb6488bfc3/', '/software-versions/2f4d48db-d4ec-4e9e-8c22-9fc5ee6c49ac/', '/software-versions/ba604a77-9683-4f88-b733-6a14d17e5c1d/', '/software-versions/4c911ae3-9007-4777-9680-2c526ab8fdbb/', '/software-versions/db773727-4185-4f43-8ec9-1671fc9c5d02/', '/software-versions/bebd8903-70cc-44a8-bd5e-42414431e685/', '/software-versions/6b5dd7cf-aa67-457c-9a79-64818e3cc5ff/', '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '/software-versions/aba3a24e-9348-4134-afe8-f19cdf22ee11/', '/software-versions/ff27407c-47d2-4d4a-85fe-f64e25a43ce2/', '/software-versions/9fa6c427-6912-4529-8664-d7aaf54e9b5f/', '/software-versions/8e3f7c06-d296-4cba-9958-1ccfa9a2b9bc/', '/software-versions/4cee829a-2538-4b19-9adc-6eb981f8d972/', '/software-versions/51c1e7da-b774-4f8a-acf8-d325678b2d3b/', '/software-versions/9b66aecd-fa8b-4bd6-84db-5b0ca2e87a9e/', '/software-versions/371fa24d-e400-4aac-a26a-f91e26b18ff5/']}, 'downloaded_url': 'https://github.com/samtools/samtools/releases/tag/1.9', 'date_created': '2019-06-06T20:57:11.142628+00:00', '@id': '/software-versions/ef27c5aa-7ffc-4ba2-836f-6b857918f72d/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'ef27c5aa-7ffc-4ba2-836f-6b857918f72d'}, {'version': '0.6.6', 'status': 'released', 'downloaded_url': 'https://github.com/biod/sambamba/releases/tag/v0.6.6', 'software': {'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'references': [], 'date_created': '2019-03-07T17:23:28.697654+00:00', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', 'url': 'http://lomereiter.github.io/sambamba/', 'status': 'released', 'aliases': ['encode:sambamba'], 'schema_version': '10', 'name': 'sambamba', 'title': 'Sambamba', 'source_url': 'https://github.com/biod/sambamba', 'used_by': ['uniform processing pipelines'], 'description': 'Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files. Because of its efficiency is an important work horse running in many sequencing centres around the world today.', '@id': '/software/sambamba/', '@type': ['Software', 'Item'], 'uuid': '071a8bd2-dc64-4fd7-aa20-c9ec83990284', 'versions': ['/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '/software-versions/a9d543a4-de63-4a27-9e21-adeb8abdcf8f/']}, 'aliases': ['encode:sambamba_0.6.6'], 'date_created': '2019-03-07T18:33:58.365633+00:00', 'schema_version': '4', 'download_checksum': '9d0e5401e3ba7f19af240f9d64795bbb', 'submitted_by': '/users/6667a92a-d202-493a-8c7d-7a56d1380356/', '@id': '/software-versions/bf964430-60ec-4703-92b7-e51eed8e4029/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'bf964430-60ec-4703-92b7-e51eed8e4029'}, {'date_created': '2020-02-07T17:56:54.938070+00:00', 'schema_version': '4', 'aliases': ['encode:picard_v2.20.7'], 'download_checksum': 'f48a12017ed75f4b3fe8cb338b99f51f', 'version': '2.20.7', 'downloaded_url': 'https://github.com/broadinstitute/picard/releases/tag/2.20.7', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-12T21:52:56.611384+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://broadinstitute.github.io/picard/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['filtering', 'other'], 'name': 'picard', 'title': 'Picard', 'source_url': 'https://github.com/broadinstitute/picard', 'used_by': ['uniform processing pipelines'], 'description': 'A set of tools (in Java) for working with next generation high-throughput sequencing (HTS) data in the BAM format.  Picard is implemented using the HTSJDK Java library HTSJDK, supporting accessing of common file formats, such as SAM and VCF, used for high-throughput sequencing data.  Currenty no published paper for Picard software.', '@id': '/software/picard/', '@type': ['Software', 'Item'], 'uuid': 'baf75b1a-6b0b-4c84-b2a1-d52b11119ddd', 'versions': ['/software-versions/c11ff7d0-30b5-4946-9fb5-8a29783bdc0f/', '/software-versions/37969002-2985-4d2c-9d1d-2f7160d80019/', '/software-versions/3ad2356f-dc51-4b5e-9498-ab3382f018f4/', '/software-versions/797e6012-5a6c-4dcc-ae9d-df45f0a19686/', '/software-versions/8fdae858-164e-4408-82ab-c6e8d2fdeba4/', '/software-versions/34e7993a-0dff-49dc-8203-ad314cb3f692/', '/software-versions/7df968e0-9d30-47c0-9ec9-0e1a7f64566d/', '/software-versions/62620171-90e2-471c-88cf-21732179a1ee/', '/software-versions/0fbc9f1d-70dd-45ab-963d-e659218fcf25/', '/software-versions/906aecb3-3faa-4059-85e0-df8ebe498434/', '/software-versions/2b4fc768-9192-4858-b004-f9d0c3b5e534/', '/software-versions/c2d8f4fb-b4a0-4b6d-96d7-acd92f6a0d04/', '/software-versions/5f2c9789-d184-4a10-a572-6cb2ff3a5ed0/', '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '/software-versions/a391f978-2e3a-487c-b759-9d33a2bf389f/', '/software-versions/98f2318f-83c0-4f6b-80b2-2627de15432f/', '/software-versions/2022ed71-90e6-4520-bfbd-b40ade09fb5d/', '/software-versions/51062a12-733d-4028-99f3-1db06595b494/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/f16e96b6-7903-46f3-b6c7-d888278e43a3/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'f16e96b6-7903-46f3-b6c7-d888278e43a3'}, {'date_created': '2020-02-07T17:59:02.763886+00:00', 'schema_version': '4', 'aliases': ['encode:cutadapt_v.2.5'], 'download_checksum': '65428b9295e77c674e31b22408f16f79', 'version': '2.5', 'downloaded_url': 'https://github.com/marcelm/cutadapt/releases/tag/v2.5', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2016-01-08T20:56:40.441207+00:00', 'submitted_by': '/users/a00d89e0-a488-4adf-9738-25149afc4087/', 'url': 'https://cutadapt.readthedocs.org/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'cutadapt', 'title': 'cutadapt', 'bug_tracker_url': 'https://github.com/marcelm/cutadapt/issues', 'source_url': 'https://pypi.python.org/pypi/cutadapt', 'used_by': ['community', 'uniform processing pipelines', 'ENCODE'], 'description': 'Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.', '@id': '/software/cutadapt/', '@type': ['Software', 'Item'], 'uuid': '1520d573-61a9-4867-ace9-bf0139d6ea14', 'versions': ['/software-versions/06dc46e3-5089-4f78-a331-fdebd95f83dd/', '/software-versions/bb5928d1-8153-4fe4-99e5-e8389dfa5e02/', '/software-versions/f0e164d8-b55b-4015-98a3-b8a48c11d115/', '/software-versions/697f881b-c16a-40fc-93a9-60fdf0cbef2b/', '/software-versions/d45d0827-cc45-4c6a-ba88-9e1ca8fc8195/', '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '/software-versions/16e1fd37-325b-4d29-990e-baaee6162e9d/', '/software-versions/cccef9c9-9049-4088-9488-2011c4160bc5/', '/software-versions/28cdfc04-76cb-468b-8e0c-26bf0fd7c76c/', '/software-versions/68f9e3e0-25a9-4b16-a0c2-6fcd65b19109/', '/software-versions/15da865f-7355-4cdb-a088-1a28f8257bfa/']}, 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', '@id': '/software-versions/b0dd1949-b903-4439-a707-1013e445825e/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'b0dd1949-b903-4439-a707-1013e445825e'}, {'date_created': '2021-08-10T16:38:46.301171+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '4', 'software': {'lab': '/labs/mark-gerstein/', 'award': '/awards/U24HG009446/', 'references': [], 'date_created': '2021-08-10T16:37:24.936629+00:00', 'submitted_by': '/users/43f2f757-5cbf-490a-9787-a1ee85a4cdcd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['other'], 'name': 'ptools_bin', 'title': 'ptools_bin', 'source_url': 'https://github.com/ENCODE-DCC/ptools_bin', 'used_by': ['uniform processing pipelines'], 'description': 'A data-sanitization software allowing raw functional genomics reads to be shared while minimizing privacy leakage, enabling principled privacy-utility trade-offs.', '@id': '/software/ptools_bin/', '@type': ['Software', 'Item'], 'uuid': 'b54331ae-1b58-4018-a61d-b066042d9e7d', 'versions': ['/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '/software-versions/2a1fd295-dd86-4769-b4ef-98afe12168e1/']}, 'version': '0.0.4', 'download_checksum': '0b1c731ac141309feb20287446ad69f9', 'downloaded_url': 'https://github.com/ENCODE-DCC/ptools_bin/releases/tag/0.0.4', '@id': '/software-versions/5b116086-fc3f-44aa-9186-4855342aa8c5/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '5b116086-fc3f-44aa-9186-4855342aa8c5'}], '@id': '/analysis-step-versions/chip-seq-alignment-step-v-2-2/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd9a1da27-79c8-4e51-b2d6-3aaccd9fa948', 'name': 'chip-seq-alignment-step-v-2-2'}\",\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " None,\n",
       " \"{'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'], 'software_versions': [{'date_created': '2020-02-07T18:10:25.805733+00:00', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'status': 'released', 'aliases': ['encode:bedToBigBed_v377'], 'schema_version': '4', 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': [], 'date_created': '2015-02-18T01:10:01.762666+00:00', 'submitted_by': '/users/81a6cc12-2847-4e2e-8f2c-f566699eb29e/', 'url': 'http://genome.ucsc.edu/goldenpath/help/bigBed.html', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedToBigBed', 'title': 'bedToBigBed', 'source_url': 'http://hgdownload.cse.ucsc.edu/admin/exe/', 'used_by': ['ENCODE', 'uniform processing pipelines', 'DAC'], 'description': 'bedToBigBed takes a standard bed file or a non-standard bed file with associated .as file to create a compressed bigBed version.  Description of Big Binary Indexed (BBI) files and visualization of next-generation sequencing experiment results explained by W.J. Kent, PMCID: PMC2922891.', '@id': '/software/bedToBigBed/', '@type': ['Software', 'Item'], 'uuid': '1431a1b1-a52e-4170-92b7-dfbd4bb5b8fe', 'versions': ['/software-versions/143e5445-7f10-48fe-9455-5f1a777773ca/', '/software-versions/8605897a-3732-42a1-9ad9-bb916f383fcb/', '/software-versions/55771f3c-384b-4cd0-aa5b-7420d3f72676/', '/software-versions/c2e7bed4-7017-44c8-875f-283ec968f937/', '/software-versions/616bc458-2260-462e-8065-54240204da57/', '/software-versions/37d9036c-f115-4226-9c0f-b7c1a24974b5/', '/software-versions/4d04578e-b1c4-49a8-b920-9de0793c0fc1/', '/software-versions/9d4d2689-e337-4609-859a-a365c635fead/', '/software-versions/55d79a5d-2203-440e-9145-d5a3cfe70e2a/', '/software-versions/318b9616-dce4-4631-9d4e-e214cb9693e9/']}, 'version': '377', 'download_checksum': '4cb015c529baacbd3484e89762115342', 'downloaded_url': 'http://github.com/ucscGenomeBrowser/kent/releases/tag/v377_base', '@id': '/software-versions/616bc458-2260-462e-8065-54240204da57/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '616bc458-2260-462e-8065-54240204da57'}], 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.411007+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step', 'title': 'histone ChIP seq pseudoreplicated overlap stable peaks file format conversion step', 'major_version': 1, 'analysis_step_types': ['file format conversion'], 'input_file_types': ['pseudoreplicated peaks'], 'output_file_types': ['pseudoreplicated peaks'], 'parents': ['/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/'], '@id': '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': '0bbf3525-a5d5-414b-a6f8-5b19b3221d1f', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/']}, 'status': 'released', 'date_created': '2019-11-15T05:51:11.272362+00:00', 'schema_version': '4', '@id': '/analysis-step-versions/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': 'd57c37fa-5643-4406-a55a-ac997a0dc21c', 'name': 'histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1-0'}\",\n",
       " \"{'status': 'released', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v-1-0'], 'date_created': '2019-11-15T05:51:10.930259+00:00', 'minor_version': 0, 'analysis_step': {'documents': [], 'date_created': '2019-11-15T05:48:06.019936+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'status': 'released', 'aliases': ['encode:histone-chip-seq-signal-generation-step-v1'], 'schema_version': '17', 'step_label': 'histone-chip-signal-generation-step', 'title': 'histone ChIP signal generation step', 'major_version': 1, 'analysis_step_types': ['signal generation'], 'input_file_types': ['alignments'], 'output_file_types': ['signal p-value', 'fold change over control'], 'parents': ['/analysis-steps/chip-seq-alignment-step-v-2/'], '@id': '/analysis-steps/histone-chip-signal-generation-step-v-1/', '@type': ['AnalysisStep', 'Item'], 'uuid': 'f835db81-76b8-46c7-8c1d-c022785e36c3', 'name': 'histone-chip-signal-generation-step-v-1', 'pipelines': [{'documents': [], 'date_created': '2019-11-15T06:00:09.014370+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL809GEM', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2 (unreplicated)', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL809GEM/', '@type': ['Pipeline', 'Item'], 'uuid': '6f145ae9-e452-4301-9f4d-4db3cb736d00'}, {'documents': [], 'date_created': '2019-11-15T05:59:19.099757+00:00', 'submitted_by': '/users/98fb23d3-0d79-4c3d-981d-01539e6589f1/', 'references': [], 'lab': '/labs/encode-processing-pipeline/', 'award': '/awards/U24HG009397/', 'aliases': [], 'accession': 'ENCPL612HIG', 'schema_version': '14', 'alternate_accessions': [], 'assay_term_names': ['ChIP-seq', 'Mint-ChIP-seq'], 'status': 'released', 'title': 'Histone ChIP-seq 2', 'analysis_steps': ['/analysis-steps/chip-seq-alignment-step-v-2/', '/analysis-steps/histone-chip-signal-generation-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-replicated-overlap-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-pseudoreplicated-overlap-stable-peaks-file-format-conversion-step-v-1/', '/analysis-steps/histone-chip-seq-pooled-signal-generation-step-v-1/'], 'source_url': 'https://github.com/ENCODE-DCC/chip-seq-pipeline2/releases/tag/v1.3.5.1', 'reference_filesets': ['/references/ENCSR174MOP/', '/references/ENCSR487FYI/', '/references/ENCSR928IAF/', '/references/ENCSR879BNV/'], '@id': '/pipelines/ENCPL612HIG/', '@type': ['Pipeline', 'Item'], 'uuid': '01634d2e-6e8c-40ac-bb92-75bdef070b0b'}], 'current_version': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', 'versions': ['/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/']}, 'schema_version': '4', 'software_versions': [{'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'version': '2.2.4', 'aliases': ['encode:macs_v2.2.4'], 'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/4c104c82-6e8a-4683-b654-5f1576743c6e/', '/publications/30bd11d9-f05d-4bee-9512-c8199c8794e2/'], 'date_created': '2014-08-13T01:30:14.690368+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['peak caller'], 'name': 'macs', 'title': 'MACS', 'bug_tracker_url': 'http://groups.google.com/group/macs-announcement', 'source_url': 'http://liulab.dfci.harvard.edu/MACS/', 'purpose': ['ChIP-seq'], 'used_by': ['ENCODE', 'uniform processing pipelines'], 'description': 'A widely-used, fast, robust ChIP-seq peak-finding algorithm that accounts for the offset in forward-strand and reverse-strand reads to improve resolution and uses a dynamic Poisson distribution to effectively capture local biases in the genome.  MACS 1.4 was used in the ENCODE 2 uniform peak calling pipeline.', '@id': '/software/macs/', '@type': ['Software', 'Item'], 'uuid': 'ab3eee7d-a025-4c3a-873b-b2c5562ebdc7', 'versions': ['/software-versions/12342c09-ec5b-458b-8b6b-13ce9f11f1e7/', '/software-versions/4e64fac3-776f-402c-a470-744f8514f39f/', '/software-versions/f06f8ba6-cf38-4da7-8906-80358e67c1e1/', '/software-versions/5900819c-32f0-4f8d-80d6-976dd224723e/', '/software-versions/1c1265da-7599-4208-aec6-807d7b9b9b07/', '/software-versions/700bf164-a6de-4fc9-8d69-a2b812385afb/', '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '/software-versions/dde87254-07b6-46a8-9c1b-00838f01d14a/']}, 'date_created': '2020-02-07T18:06:18.418775+00:00', 'schema_version': '4', 'downloaded_url': 'https://pypi.org/project/MACS2/2.2.4/#files', 'status': 'released', 'download_checksum': 'e61cc35ad0fafc7e1beac3639815dc24', '@id': '/software-versions/41f43709-2239-4388-8121-0ab8e3eae8d7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': '41f43709-2239-4388-8121-0ab8e3eae8d7'}, {'software': {'lab': '/labs/encode-consortium/', 'award': '/awards/ENCODE/', 'references': ['/publications/bdd1b360-c377-450d-969a-efc1a7e8611f/'], 'date_created': '2014-08-13T01:30:20.545705+00:00', 'submitted_by': '/users/ce2bde01-07ec-4b8a-b179-554ef95b71dd/', 'notes': 'Previously hosted at https://code.google.com/p/bedtools/', 'status': 'released', 'aliases': [], 'schema_version': '10', 'software_type': ['file format conversion'], 'name': 'bedtools', 'title': 'BEDTools', 'source_url': 'https://github.com/arq5x/bedtools', 'purpose': ['ChIP-seq', 'DNase-seq'], 'used_by': ['DAC', 'uniform processing pipelines', 'ENCODE'], 'description': 'Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetics: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, and VCF.', '@id': '/software/bedtools/', '@type': ['Software', 'Item'], 'uuid': 'e43bdb79-c746-4c40-815a-6cc27f3688a8', 'versions': ['/software-versions/2cc07124-16be-4333-8943-4bce96059af3/', '/software-versions/97914185-dc92-436a-a729-e9b82610d92e/', '/software-versions/0062f99c-54c1-49be-bc45-b37271616422/', '/software-versions/6404f5d1-636c-407f-8742-55d89a990993/', '/software-versions/13e61050-fe4f-4002-9ae0-8d1adcf81a04/', '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '/software-versions/c12e0857-dee3-41fd-8247-00a5848f00ff/', '/software-versions/be697d3b-cf28-46b2-9a99-5219b9ac1bad/', '/software-versions/d2b3e267-77eb-4eb6-bb58-9167b7e95d91/', '/software-versions/ce0d87d2-8ecf-4374-bcaf-1654654fb18a/', '/software-versions/b64f37e9-9ee7-48fc-8085-90c74d75df2b/']}, 'download_checksum': '4baadff63ca829ce8e4a13e0e2885dd0', 'schema_version': '4', 'aliases': ['encode:bedtools_v2.29.0'], 'date_created': '2020-02-07T18:34:13.262448+00:00', 'version': '2.29.0', 'status': 'released', 'submitted_by': '/users/64974e05-12df-4733-ae90-b402298b142e/', 'downloaded_url': 'https://github.com/arq5x/bedtools2/releases/tag/v2.29.0', '@id': '/software-versions/d22a5931-5436-4958-bb35-f70a93af2ed7/', '@type': ['SoftwareVersion', 'Item'], 'uuid': 'd22a5931-5436-4958-bb35-f70a93af2ed7'}], '@id': '/analysis-step-versions/histone-chip-signal-generation-step-v-1-0/', '@type': ['AnalysisStepVersion', 'Item'], 'uuid': '5e3bf2f9-9982-4076-9ecc-8829fb676856', 'name': 'histone-chip-signal-generation-step-v-1-0'}\",\n",
       " ...]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Inspect data\n",
    "\n",
    "for column in encode_df[\"quality_metrics\", \"step_run\", \"analysis_step_version\"].iter_columns():\n",
    "    display(column.to_list())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n",
      "sys:1: MapWithoutReturnDtypeWarning: Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. Specify `return_dtype` to silence this warning.\n"
     ]
    }
   ],
   "source": [
    "# Convert select columns to valid JSON\n",
    "\n",
    "encode_df = encode_df.select(pl.col(\"quality_metrics\", \"step_run\", \"analysis_step_version\").map_elements(lambda x: ast.literal_eval(x)))\n",
    "display(encode_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Done\n",
    "\n",
    "clean_encode = encode_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "display(clean_encode)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
